---
title: Fecal microbiota signatures are predicitve of response to therapy among Ustekinumab-treated
  Crohnâ€™s Disease patients.
csl: mbio.csl
fontsize: 11pt
output:
  pdf_document:
    keep_tex: yes
    latex_engine: xelatex
  md_document: default
  html_document: default
header-includes:
- \usepackage{setspace}
- \doublespacing
- \usepackage{lineno}
- \linenumbers
- \renewcommand{\familydefault}{\sfdefault}
- \usepackage{graphicx}
geometry: margin=1.0in
bibliography: My_Library_copy.bib
---

```{r knitr_settings, cache=FALSE, include=FALSE}
source('../code/knitr_settings.R')
knitr::opts_chunk$set("cache" = T)
source('../code/R_packages_setup.R')
```


```{r readdata, include=F}
#Read in all data
source('../code/APmd.setup.R')




screen_RESPwk22<-APmd_RSPwk22[APmd_RSPwk22$visit=="Screening",] #removes any group with NA in RESPONSEwk22

screen_data<-screen_RESPwk22				#na.omit(screen_RESPwk22)
summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))


screen_data<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Not",]
screen_data<-screen_data[screen_data$TRTGRINDMAN!="Treated_Not",]

summary(screen_data[,c("TRTGR", "TRTGRINDMAN", "visit", "RESPONSEwk6", "RelRSPwk6", "REMISSwk6")])


screen.trtd<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Placebo",]
screen.trtd<-screen.trtd[screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(screen.trtd[,1:6])
screen.plac<-screen_data[screen_data$TRTGRINDMAN!="Treated_Placebo",]
screen.plac<-screen.plac[screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(screen.plac[,1:6])

nao.screen_data<-na.omit(screen_data)

nao.screen.trtd<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Placebo_Placebo",]
nao.screen.trtd<-nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(nao.screen.trtd[,1:6])
nao.screen.plac<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Treated_Placebo",]
nao.screen.plac<-nao.screen.plac[nao.screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(nao.screen.plac[,1:6])


tax<- read.table('../data/Jan400.simple.taxonomy.txt', header=T, sep="\t")
nao.screen.all.distmat<-read.table("../data/Jan400.screening.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.all.dist<-as.dist(nao.screen.all.distmat)
nao.screen.trtd.distmat<-read.table("../data/Jan400.screening.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.trtd.dist<-as.dist(nao.screen.trtd.distmat)
nao.screen.plac.distmat<-read.table("../data/Jan400.screening.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.plac.dist<-as.dist(nao.screen.plac.distmat)

screen.all.distmat<-read.table("../data/Jan400.screen.all.thetayc.0.03.square.ave.dist")
screen.all.dist<-as.dist(screen.all.distmat)
screen.trtd.distmat<-read.table("../data/Jan400.screen.ust.thetayc.0.03.square.ave.dist")
screen.trtd.dist<-as.dist(screen.trtd.distmat)
screen.plac.distmat<-read.table("../data/Jan400.screen.plac.thetayc.0.03.square.ave.dist")
screen.plac.dist<-as.dist(screen.plac.distmat)

trtd_trtd.data.distmat<-read.table("../data/Jan400.trtd_trtd.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
trtd_trtd.data.dist<-as.dist(trtd_trtd.data.distmat)
trtd_plac.data.distmat<-read.table("../data/Jan400.trtd_plac.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
trtd_plac.data.dist<-as.dist(trtd_plac.data.distmat)
plac_plac.data.distmat<-read.table("../data/Jan400.plac_plac.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
plac_plac.data.dist<-as.dist(plac_plac.data.distmat)
plac_trtd.data.distmat<-read.table("../data/Jan400.plac_trtd.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
plac_trtd.data.dist<-as.dist(plac_trtd.data.distmat)
```	

```{r sum, include=F}
summary(nao.screen_data[,c("TRTGR")])
summary(screen_data[,c("TRTGR")])
```

```{r phylum_otu, include=FALSE}

#screen_data <- screen_data[screen_data$TRTGR=="Treated",]
source("../code/SF1_week6phyla.R")

pdf("figures/SF1_wk6phyla.pdf", height = 10, width = 12)
plot_grid(phyRabRelRSPwk6_figure2, phyRabREMISSwk6_figure2, labels = c("A", "B"), align = "v", nrow=2)
dev.off()



#trtd.screen_data <- screen_data
#screen_data <- trtd.screen_data[trtd.screen_data$TRTGR=="Treated",]
source("../code/Figure5_REMISSwk6.sig.otu.figure.R")

screen_data.trtd <- screen_data[screen_data$TRTGR=="Treated",]
screen_data.trtd_groups <- screen_data.trtd[, c("group", "TRTGR")]
screen_data.trtd_Otus <- screen_data.trtd[, c("group", "TRTGR", "REMISSwk6")]
screen.trtd_otu_rel_abund <- merge(screen_data.trtd_groups, otu_rel_abund, by.x = "group", by.y = "row.names")
rownames(screen.trtd_otu_rel_abund) <- screen.trtd_otu_rel_abund$group
screen.trtd_otu_rel_abund <- screen.trtd_otu_rel_abund[,-c(1:2)]

screen.trtd_rel_abund_Otus <- merge(screen_data.trtd_Otus, otu_rel_abund, by.x = "group", by.y = "row.names")
rownames(screen.trtd_rel_abund_Otus) <- screen.trtd_rel_abund_Otus$group
screen.trtd_rel_abund_Otus <- screen.trtd_rel_abund_Otus[,-c(1:2)]
screen.trtd_rel_abund_Otus <- subset(screen.trtd_rel_abund_Otus, select = c("REMISSwk6", "Otu00001", "Otu00007"))
trtd.REMwk6.YES.relabund <- screen.trtd_rel_abund_Otus[screen.trtd_rel_abund_Otus$REMISSwk6=="Yes",]
trtd.REMwk6.NO.relabund <- screen.trtd_rel_abund_Otus[screen.trtd_rel_abund_Otus$REMISSwk6=="No",]

median(trtd.REMwk6.YES.relabund$Otu00001)
quantile(trtd.REMwk6.YES.relabund$Otu00001, type = 2)
quantile(trtd.REMwk6.NO.relabund$Otu00001, type = 2)
quantile(trtd.REMwk6.YES.relabund$Otu00007, type = 2)
quantile(trtd.REMwk6.NO.relabund$Otu00007, type = 2)
#screen_data <- trtd.screen_data 
```

```{r cdai.cor.test,  include=F}
set.seed(32016)
cdai.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, cdail_wk0)))
n.cdai.cortest<-nrow(cdai.cor.test.data)
n.cdai.cortest
cor.cdai<-cor.test(cdai.cor.test.data$cdail_wk0, cdai.cor.test.data$invsimpson, method = "spearman")
cor.cdai
cor.cdai.pval<-signif(cor.cdai$p.value, digits = 1)
cor.cdai.pval
cor.cdai.rho<-signif(cor.cdai$estimate, digits = 1)
cor.cdai.rho


```

```{r cdaiwk0betadiv, include=F}
set.seed(32016)
cdaiwk0.adonis<-adonis(nao.screen.all.dist ~ cdail_wk0, nao.screen_data)
cdaiwk0.adonis
cdaiwk0.adonis.pval<-signif(as.numeric(na.omit(cdaiwk0.adonis$aov.tab$`Pr(>F)`)), digits = 1)

CDAI_min<-toString(c('Min=', min(nao.screen_data$cdail_wk0)))
CDAI_min<-gsub(",", "", CDAI_min)
CDAI_min<-gsub(" ", "", CDAI_min)
CDAI_med<-toString(c("Median=", median(nao.screen_data$cdail_wk0)))
CDAI_med<-gsub(",", "", CDAI_med)
CDAI_med<-gsub(" ", "", CDAI_med)
CDAI_max<-toString(c("Max=", max(nao.screen_data$cdail_wk0)))
CDAI_max<-gsub(",", "", CDAI_max)
CDAI_max<-gsub(" ", "", CDAI_max)
CDAI_sum<-toString(c(CDAI_min, CDAI_med, CDAI_max))
CDAI_n<-n.cdai.cortest
CDAI_Spear_P<- toString(c('(', cor.cdai.pval, ')'))
CDAI_Spear_P<-gsub(",", "", CDAI_Spear_P)
CDAI_Spear_P<-gsub(" ", "", CDAI_Spear_P)
CDAI_Spear_rho<-toString(c('rho =', cor.cdai.rho))
CDAI_Spear_rho<-gsub(",", "", CDAI_Spear_rho)
#CDAI_Spear_rho<-gsub(" ", "", CDAI_Spear_rho)
CDAI_Spear_Prho<-c(CDAI_Spear_rho, CDAI_Spear_P)
CDAI_Spear<-toString(c(CDAI_Spear_Prho))
CDAI_Spear<-gsub(",", "", CDAI_Spear)
#CDAI_AMOVA_p<-toString(c(cdaiwk0.adonis.pval))
#CDAI_AMOVA_p<-gsub(",", "", CDAI_AMOVA_p)
#CDAI_AMOVA_p<-gsub(" ", "", CDAI_AMOVA_p)
#CDAI_AMOVA<-toString(c(CDAI_AMOVA_p))
#CDAI_AMOVA<-gsub(",", "", CDAI_AMOVA)
CDAI<- matrix(c('CDAI', CDAI_Spear_rho, cor.cdai$p.value, as.numeric(na.omit(cdaiwk0.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r stoolfreq adiv, include=F}
set.seed(32016)
stool.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, CDSTOOLM_Screening)))
n.stool.cortest<-nrow(stool.cor.test.data)
n.stool.cortest
cor.stool<-cor.test(stool.cor.test.data$CDSTOOLM_Screening, stool.cor.test.data$invsimpson, method = "spearman")
cor.stool
cor.stool.pval<-signif(cor.stool$p.value, digits = 1)
cor.stool.pval
cor.stool.rho<-signif(cor.stool$estimate, digits = 1)
cor.stool.rho

```

```{r stoolfreq bdiv, include=F}
set.seed(32016)
stool.adonis<-adonis(nao.screen.all.dist ~ CDSTOOLM_Screening, nao.screen_data)
stool.adonis
stool.adonis.pval<-signif(as.numeric(na.omit(stool.adonis$aov.tab$`Pr(>F)`)), digits = 1)

Stool_min<-toString(c('Min=', min(nao.screen_data$CDSTOOLM_Screening)))
Stool_min<-gsub(",", "", Stool_min)
Stool_min<-gsub(" ", "", Stool_min)
Stool_med<-toString(c("Median=", median(nao.screen_data$CDSTOOLM_Screening)))
Stool_med<-gsub(",", "", Stool_med)
Stool_med<-gsub(" ", "", Stool_med)
Stool_max<-toString(c("Max=", max(nao.screen_data$CDSTOOLM_Screening)))
Stool_max<-gsub(",", "", Stool_max)
Stool_max<-gsub(" ", "", Stool_max)
Stool_sum<-toString(c(Stool_min, Stool_med, Stool_max))
Stool_n<-n.stool.cortest
Stool_Spear_P<- toString(c("(", cor.stool.pval, ")"))
Stool_Spear_P<-gsub(",", "", Stool_Spear_P)
Stool_Spear_P<-gsub(" ", "", Stool_Spear_P)
Stool_Spear_rho<-toString(c('rho =', cor.stool.rho))
Stool_Spear_rho<-gsub(",", "", Stool_Spear_rho)
#Stool_Spear_rho<-gsub(" ", "", Stool_Spear_rho)
Stool_Spear_Prho<-c(Stool_Spear_rho, Stool_Spear_P)
Stool_Spear<-toString(c(Stool_Spear_Prho))
Stool_Spear<-gsub(",", "", Stool_Spear)

Stool<- matrix(c('Loose Stool Frequency (per week)', Stool_Spear_rho, cor.stool$p.value, as.numeric(na.omit(stool.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```


```{r CRP.cor.test, include=F}
set.seed(32016)
CRP.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BL_CRP)))
n.CRP.cortest<-nrow(CRP.cor.test.data)
n.CRP.cortest
cor.CRP<-cor.test(CRP.cor.test.data$BL_CRP, CRP.cor.test.data$invsimpson, method = "spearman")
cor.CRP
cor.CRP.pval<-signif(cor.CRP$p.value, digits = 1)
cor.CRP.pval
cor.CRP.rho<-signif(cor.CRP$estimate, digits = 1)
cor.CRP.rho

```

```{r BL_CRP.adonis,  include=F}
set.seed(32016)
BL_CRP.adonis<-adonis(nao.screen.all.dist ~ BL_CRP, nao.screen_data)
BL_CRP.adonis
BL_CRP.adonis.pval<-signif(as.numeric(na.omit(BL_CRP.adonis$aov.tab$`Pr(>F)`)), digits = 1)

CRP_min<-toString(c('Min=', min(nao.screen_data$BL_CRP)))
CRP_min<-gsub(",", "", CRP_min)
CRP_min<-gsub(" ", "", CRP_min)
CRP_med<-toString(c("Median=", median(nao.screen_data$BL_CRP)))
CRP_med<-gsub(",", "", CRP_med)
CRP_med<-gsub(" ", "", CRP_med)
CRP_max<-toString(c("Max=", max(nao.screen_data$BL_CRP)))
CRP_max<-gsub(",", "", CRP_max)
CRP_max<-gsub(" ", "", CRP_max)
CRP_sum<-toString(c(CRP_min, CRP_med, CRP_max))
CRP_n<-n.CRP.cortest
CRP_Spear_P<- toString(c('(', cor.CRP.pval, ")"))
CRP_Spear_P<-gsub(",", "", CRP_Spear_P)
CRP_Spear_P<-gsub(" ", "", CRP_Spear_P)
CRP_Spear_rho<-toString(c('rho =', cor.CRP.rho))
CRP_Spear_rho<-gsub(",", "", CRP_Spear_rho)
#CRP_Spear_rho<-gsub(" ", "", CRP_Spear_rho)
CRP_Spear_Prho<-c(CRP_Spear_rho, CRP_Spear_P)
CRP_Spear<-toString(c(CRP_Spear_Prho))
CRP_Spear<-gsub(",", "", CRP_Spear)

CRP<- matrix(c('C-Reactive Protein (mg/L serum)', CRP_Spear_rho, cor.CRP$p.value, as.numeric(na.omit(BL_CRP.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r FCALP.cor.test, include=F}
set.seed(32016)
FCALP.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BL_FCALP)))
n.FCALP.cortest<-nrow(FCALP.cor.test.data)
n.FCALP.cortest
cor.FCALP<-cor.test(FCALP.cor.test.data$BL_FCALP, FCALP.cor.test.data$invsimpson, method = "spearman")
cor.FCALP
cor.FCALP.pval<-signif(cor.FCALP$p.value, digits = 1)
cor.FCALP.pval
cor.FCALP.rho<-signif(cor.FCALP$estimate, digits = 1)
cor.FCALP.rho

```

```{r BL_FCALP.adonis,  include=F}
set.seed(32016)
BL_FCALP.adonis<-adonis(nao.screen.all.dist ~ BL_FCALP, nao.screen_data)
BL_FCALP.adonis
BL_FCALP.adonis.pval<-signif(as.numeric(na.omit(BL_FCALP.adonis$aov.tab$`Pr(>F)`)), digits = 1)

FCALP_min<-toString(c('Min=', min(nao.screen_data$BL_FCALP)))
FCALP_min<-gsub(",", "", FCALP_min)
FCALP_min<-gsub(" ", "", FCALP_min)
FCALP_med<-toString(c("Median=", median(nao.screen_data$BL_FCALP)))
FCALP_med<-gsub(",", "", FCALP_med)
FCALP_med<-gsub(" ", "", FCALP_med)
FCALP_max<-toString(c("Max=", max(nao.screen_data$BL_FCALP)))
FCALP_max<-gsub(",", "", FCALP_max)
FCALP_max<-gsub(" ", "", FCALP_max)
FCALP_sum<-toString(c(FCALP_min, FCALP_med, FCALP_max))
FCALP_n<-n.FCALP.cortest
FCALP_Spear_P<- toString(c('(', cor.FCALP.pval,")"))
FCALP_Spear_P<-gsub(",", "", FCALP_Spear_P)
FCALP_Spear_P<-gsub(" ", "", FCALP_Spear_P)
FCALP_Spear_rho<-toString(c('rho =', cor.FCALP.rho))
FCALP_Spear_rho<-gsub(",", "", FCALP_Spear_rho)
#FCALP_Spear_rho<-gsub(" ", "", FCALP_Spear_rho)
FCALP_Spear_Prho<-c(FCALP_Spear_rho, FCALP_Spear_P)
FCALP_Spear<-toString(c(FCALP_Spear_Prho))
FCALP_Spear<-gsub(",", "", FCALP_Spear)
FCALP_AMOVA_p<-toString(c(BL_FCALP.adonis.pval))
FCALP_AMOVA_p<-gsub(",", "", FCALP_AMOVA_p)
FCALP_AMOVA_p<-gsub(" ", "", FCALP_AMOVA_p)
FCALP_AMOVA<-toString(c(FCALP_AMOVA_p))
FCALP_AMOVA<-gsub(",", "", FCALP_AMOVA)
FCALP<- matrix(c('Fecal Calprotectin (Âµg/g)', FCALP_Spear_rho, cor.FCALP$p.value, as.numeric(na.omit(BL_FCALP.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r FLACT.cor.test, include=F}
set.seed(32016)
FLACT.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BL_FLACT)))
n.FLACT.cortest<-nrow(FLACT.cor.test.data)
n.FLACT.cortest
cor.FLACT<-cor.test(FLACT.cor.test.data$BL_FLACT, FLACT.cor.test.data$invsimpson, method = "spearman")
cor.FLACT
cor.FLACT.pval<-signif(cor.FLACT$p.value, digits = 1)
cor.FLACT.pval
cor.FLACT.rho<-signif(cor.FLACT$estimate, digits = 1)
cor.FLACT.rho

```

```{r BL_FLACT.adonis,  include=F}
set.seed(32016)
BL_FLACT.adonis<-adonis(nao.screen.all.dist ~ BL_FLACT, nao.screen_data)
BL_FLACT.adonis
BL_FLACT.adonis.pval<-signif(as.numeric(na.omit(BL_FLACT.adonis$aov.tab$`Pr(>F)`)), digits = 1)

FLACT_min<-toString(c('Min=', min(nao.screen_data$BL_FLACT)))
FLACT_min<-gsub(",", "", FLACT_min)
FLACT_min<-gsub(" ", "", FLACT_min)
FLACT_med<-toString(c("Median=", median(nao.screen_data$BL_FLACT)))
FLACT_med<-gsub(",", "", FLACT_med)
FLACT_med<-gsub(" ", "", FLACT_med)
FLACT_max<-toString(c("Max=", max(nao.screen_data$BL_FLACT)))
FLACT_max<-gsub(",", "", FLACT_max)
FLACT_max<-gsub(" ", "", FLACT_max)
FLACT_sum<-toString(c(FLACT_min, FLACT_med, FLACT_max))
FLACT_n<-n.FLACT.cortest
FLACT_Spear_P<- toString(c('(', cor.FLACT.pval,")"))
FLACT_Spear_P<-gsub(",", "", FLACT_Spear_P)
FLACT_Spear_P<-gsub(" ", "", FLACT_Spear_P)
FLACT_Spear_rho<-toString(c('rho =', cor.FLACT.rho))
FLACT_Spear_rho<-gsub(",", "", FLACT_Spear_rho)
#FLACT_Spear_rho<-gsub(" ", "", FLACT_Spear_rho)
FLACT_Spear_Prho<-c(FLACT_Spear_rho, FLACT_Spear_P)
FLACT_Spear<-toString(c(FLACT_Spear_Prho))
FLACT_Spear<-gsub(",", "", FLACT_Spear)
FLACT_AMOVA_p<-toString(c(BL_FLACT.adonis.pval))
FLACT_AMOVA_p<-gsub(",", "", FLACT_AMOVA_p)
FLACT_AMOVA_p<-gsub(" ", "", FLACT_AMOVA_p)
FLACT_AMOVA<-toString(c(FLACT_AMOVA_p))
FLACT_AMOVA<-gsub(",", "", FLACT_AMOVA)
FLACT<- matrix(c('Fecal Lactoferrin (Âµg/g)', FLACT_Spear_rho, cor.FLACT$p.value, as.numeric(na.omit(BL_FLACT.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r BMI.cortest, include=F}
set.seed(32016)
BMI.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, invsimpson, BL_BMI)))
n.BMI.cortest<-nrow(BMI.cor.test.data)
n.BMI.cortest
cor.BMI<-cor.test(BMI.cor.test.data$BL_BMI, BMI.cor.test.data$invsimpson, method = "spearman")
cor.BMI
cor.BMI.pval<-signif(cor.BMI$p.value, digits = 1)
cor.BMI.pval
cor.BMI.rho<-signif(cor.BMI$estimate, digits = 1)
cor.BMI.rho

```

```{r BL_BMI.adonis,  include=F}
set.seed(32016)
BL_BMI.adonis<-adonis(nao.screen.all.dist ~ BL_BMI, nao.screen_data)
BL_BMI.adonis
BL_BMI.adonis.pval<-signif(as.numeric(na.omit(BL_BMI.adonis$aov.tab$`Pr(>F)`)), digits = 1)

BMI_min<-toString(c('Min=', min(nao.screen_data$BL_BMI)))
BMI_min<-gsub(",", "", BMI_min)
BMI_min<-gsub(" ", "", BMI_min)
BMI_med<-toString(c("Median=", median(nao.screen_data$BL_BMI)))
BMI_med<-gsub(",", "", BMI_med)
BMI_med<-gsub(" ", "", BMI_med)
BMI_max<-toString(c("Max=", max(nao.screen_data$BL_BMI)))
BMI_max<-gsub(",", "", BMI_max)
BMI_max<-gsub(" ", "", BMI_max)
BMI_sum<-toString(c(BMI_min, BMI_med, BMI_max))
BMI_n<-n.BMI.cortest
BMI_Spear_P<- toString(c('(', cor.BMI.pval, ")"))
BMI_Spear_P<-gsub(",", "", BMI_Spear_P)
BMI_Spear_P<-gsub(" ", "", BMI_Spear_P)
BMI_Spear_rho<-toString(c('rho =', cor.BMI.rho))
BMI_Spear_rho<-gsub(",", "", BMI_Spear_rho)
#BMI_Spear_rho<-gsub(" ", "", BMI_Spear_rho)
BMI_Spear_Prho<-c(BMI_Spear_rho, BMI_Spear_P)
BMI_Spear<-toString(c(BMI_Spear_Prho))
BMI_Spear<-gsub(",", "", BMI_Spear)
BMI_AMOVA_p<-toString(c(BL_BMI.adonis.pval))
BMI_AMOVA_p<-gsub(",", "", BMI_AMOVA_p)
BMI_AMOVA_p<-gsub(" ", "", BMI_AMOVA_p)
BMI_AMOVA<-toString(c(BMI_AMOVA_p))
BMI_AMOVA<-gsub(",", "", BMI_AMOVA)
BMI<- matrix(c('BMI', BMI_Spear_rho, cor.BMI$p.value, as.numeric(na.omit(BL_BMI.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r weightcor, include=F}
set.seed(32016)
weight.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, CDWEIGHM_Screening)))
n.weight.cortest<-nrow(weight.cor.test.data)
n.weight.cortest
cor.weight<-cor.test(weight.cor.test.data$CDWEIGHM_Screening, weight.cor.test.data$invsimpson, method = "spearman")
cor.weight
cor.weight.pval<-signif(cor.weight$p.value, digits = 1)
cor.weight.pval
cor.weight.rho<-signif(cor.weight$estimate, digits = 1)
cor.weight.rho

```

```{r CDWEIGHM_Screening.adonis,  include=F}
set.seed(32016)
CDWEIGHM_Screening.adonis<-adonis(nao.screen.all.dist ~ CDWEIGHM_Screening, nao.screen_data)
CDWEIGHM_Screening.adonis
CDWEIGHM_Screening.adonis.pval<-signif(as.numeric(na.omit(CDWEIGHM_Screening.adonis$aov.tab$`Pr(>F)`)), digits = 1)

weight_min<-toString(c('Min=', min(nao.screen_data$CDWEIGHM_Screening)))
weight_min<-gsub(",", "", weight_min)
weight_min<-gsub(" ", "", weight_min)
weight_med<-toString(c("Median=", median(nao.screen_data$CDWEIGHM_Screening)))
weight_med<-gsub(",", "", weight_med)
weight_med<-gsub(" ", "", weight_med)
weight_max<-toString(c("Max=", max(nao.screen_data$CDWEIGHM_Screening)))
weight_max<-gsub(",", "", weight_max)
weight_max<-gsub(" ", "", weight_max)
weight_sum<-toString(c(weight_min, weight_med, weight_max))
weight_n<-n.weight.cortest
weight_Spear_P<- toString(c('(', cor.weight.pval, ")"))
weight_Spear_P<-gsub(",", "", weight_Spear_P)
weight_Spear_P<-gsub(" ", "", weight_Spear_P)
weight_Spear_rho<-toString(c('rho =', cor.weight.rho))
weight_Spear_rho<-gsub(",", "", weight_Spear_rho)
#weight_Spear_rho<-gsub(" ", "", weight_Spear_rho)
weight_Spear_Prho<-c(weight_Spear_rho, weight_Spear_P)
weight_Spear<-toString(c(weight_Spear_Prho))
weight_Spear<-gsub(",", "", weight_Spear)
weight_AMOVA_p<-toString(c(CDWEIGHM_Screening.adonis.pval))
weight_AMOVA_p<-gsub(",", "", weight_AMOVA_p)
weight_AMOVA_p<-gsub(" ", "", weight_AMOVA_p)
weight_AMOVA<-toString(c(weight_AMOVA_p))
weight_AMOVA<-gsub(",", "", weight_AMOVA)
weight<- matrix(c('Weight (kg)', weight_Spear_rho, cor.weight$p.value, as.numeric(na.omit(CDWEIGHM_Screening.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r AGE.cor.test, include=F}
set.seed(32016)
AGE.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, invsimpson, AGE)))
n.AGE.cortest<-nrow(AGE.cor.test.data)
n.AGE.cortest
cor.AGE<-cor.test(AGE.cor.test.data$AGE, AGE.cor.test.data$invsimpson, method = "spearman")
cor.AGE
cor.AGE.pval<-signif(cor.AGE$p.value, digits = 1)
cor.AGE.pval
cor.AGE.rho<-signif(cor.AGE$estimate, digits = 1)
cor.AGE.rho


AGE.total.mean<-mean(AGE.cor.test.data$AGE)
AGE.total.sd<-sd(AGE.cor.test.data$AGE)

AGE.trtd<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Treated",]
AGE.trtd.mean<-signif(mean(AGE.cor.test.data$AGE), digits = 2)
AGE.trtd.sd<-sd(AGE.cor.test.data$AGE)

AGE.plac<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Placebo",]
AGE.plac.mean<-signif(mean(AGE.cor.test.data$AGE), digits=2)
AGE.plac.sd<-sd(AGE.cor.test.data$AGE)



```

```{r AGE.adonis,  include=F}
set.seed(32016)
AGE.adonis<-adonis(nao.screen.all.dist ~ AGE, nao.screen_data)
AGE.adonis
AGE.adonis.pval<-signif(as.numeric(na.omit(AGE.adonis$aov.tab$`Pr(>F)`)), digits = 1)

AGE_min<-toString(c('Min=', min(nao.screen_data$AGE)))
AGE_min<-gsub(",", "", AGE_min)
AGE_min<-gsub(" ", "", AGE_min)
AGE_med<-toString(c("Median=", median(nao.screen_data$AGE)))
AGE_med<-gsub(",", "", AGE_med)
AGE_med<-gsub(" ", "", AGE_med)
AGE_max<-toString(c("Max=", max(nao.screen_data$AGE)))
AGE_max<-gsub(",", "", AGE_max)
AGE_max<-gsub(" ", "", AGE_max)
AGE_sum<-toString(c(AGE_min, AGE_med, AGE_max))
AGE_n<-n.AGE.cortest

AGE_Spear_P<- toString(c('(', cor.AGE.pval, ")"))
AGE_Spear_P<-gsub(",", "", AGE_Spear_P)
AGE_Spear_P<-gsub(" ", "", AGE_Spear_P)
AGE_Spear_rho<-toString(c('rho =', cor.AGE.rho))
AGE_Spear_rho<-gsub(",", "", AGE_Spear_rho)
#AGE_Spear_rho<-gsub("rho", expression("\U03C1"), AGE_Spear_rho)
AGE_Spear_Prho<-c(AGE_Spear_rho, AGE_Spear_P)
AGE_Spear<-toString(c(AGE_Spear_Prho))
AGE_Spear<-gsub(",", "", AGE_Spear)
AGE_AMOVA_p<-toString(c(AGE.adonis.pval))
AGE_AMOVA_p<-gsub(",", "", AGE_AMOVA_p)
AGE_AMOVA_p<-gsub(" ", "", AGE_AMOVA_p)
AGE_AMOVA<-toString(c(AGE_AMOVA_p))
AGE_AMOVA<-gsub(",", "", AGE_AMOVA)
AGE<- matrix(c('Age (years)', AGE_Spear_rho, cor.AGE$p.value, as.numeric(na.omit(AGE.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r SEX-wilcox, include=F}
set.seed(32016)
SEX.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, SEX)))
n.MSEX.wilcoxtest<-nrow(SEX.wilcox.test.data[SEX.wilcox.test.data$SEX=="M",])
n.MSEX.wilcoxtest
n.FSEX.wilcoxtest<-nrow(SEX.wilcox.test.data[SEX.wilcox.test.data$SEX=="F",])
n.FSEX.wilcoxtest
wilcox.SEX<-wilcox.test(invsimpson ~ SEX, SEX.wilcox.test.data)
summary(wilcox.SEX)
wilcox.SEX.pval<-signif(wilcox.SEX$p.value, digits = 1)
wilcox.SEX.pval
wilcox.SEX.method<-wilcox.SEX$method
subT<-wilcox.SEX.method

```

```{r SEX.adonis,  include=F}
set.seed(32016)
SEX.adonis<-adonis(nao.screen.all.dist ~ SEX, nao.screen_data)
SEX.adonis
SEX.adonis.pval<-signif(as.numeric(na.omit(SEX.adonis$aov.tab$`Pr(>F)`)), digits = 1)
MSEX<-toString(c(n.MSEX.wilcoxtest))
FSEX<-toString(c(n.FSEX.wilcoxtest))


SEX_sum<-toString(c(FSEX, "/", MSEX))
SEX_sum<-gsub(",", "", SEX_sum)
SEX_sum<-gsub(" ", "", SEX_sum)

SEX<- matrix(c('Sex (F/M)', "-", wilcox.SEX$p.value, as.numeric(na.omit(SEX.adonis$aov.tab$`Pr(>F)`))), ncol = 4)

```

```{r BCORT-wilcox, include=F}
set.seed(32016)
BCORT.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BCORT)))
YesBCORT.wilcoxtest<-BCORT.wilcox.test.data[BCORT.wilcox.test.data$BCORT=="Yes",]
n.YesBCORT.wilcoxtest<-nrow(YesBCORT.wilcoxtest)
NoBCORT.wilcoxtest<-BCORT.wilcox.test.data[BCORT.wilcox.test.data$BCORT=="No",]
n.NoBCORT.wilcoxtest<-nrow(NoBCORT.wilcoxtest)
wilcox.BCORT<-wilcox.test(invsimpson ~ BCORT, BCORT.wilcox.test.data)
summary(wilcox.BCORT)
wilcox.BCORT.pval<-signif(wilcox.BCORT$p.value, digits = 1)
wilcox.BCORT.pval
wilcox.BCORT.method<-wilcox.BCORT$method
subT<-wilcox.BCORT.method
nYesString<-toString(c("Yes=", n.YesBCORT.wilcoxtest))
nYesString<-gsub(",", "", nYesString)
nNoString<-toString(c("No=", n.NoBCORT.wilcoxtest))
nNoString<-gsub(",", "", nNoString)

```

```{r BCORT.adonis,  include=F}
set.seed(32016)
BCORT.adonis<-adonis(nao.screen.all.dist ~ BCORT, nao.screen_data)
BCORT.adonis
BCORT.adonis.pval<-signif(as.numeric(na.omit(BCORT.adonis$aov.tab$`Pr(>F)`)), digits = 1)

YesBCORT<-toString(c(n.YesBCORT.wilcoxtest))

NoBCORT<-toString(c(n.NoBCORT.wilcoxtest))


BCORT_sum<-toString(c(YesBCORT, "/", NoBCORT))
BCORT_sum<-gsub(",", "", BCORT_sum)
BCORT_sum<-gsub(" ", "", BCORT_sum)

BCORT<- matrix(c('Corticosteroid Use (Y/N)', "-", wilcox.BCORT$p.value, as.numeric(na.omit(BCORT.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r DDUR.cortest, include=F}
set.seed(32016)
cor.test(nao.screen_data$invsimpson, nao.screen_data$DDUR, method = "spearman")


DDUR.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, DDUR)))
n.DDUR.cortest<-nrow(DDUR.cor.test.data)
n.DDUR.cortest
cor.DDUR<-cor.test(DDUR.cor.test.data$DDUR, DDUR.cor.test.data$invsimpson, method = "spearman")
cor.DDUR
cor.DDUR.pval<-signif(cor.DDUR$p.value, digits = 1)
cor.DDUR.pval
cor.DDUR.rho<-signif(cor.DDUR$estimate, digits = 1)
cor.DDUR.rho


```

```{r DDUR.adonis,  include=F}
set.seed(32016)
DDUR.adonis<-adonis(nao.screen.all.dist ~ DDUR, nao.screen_data)
DDUR.adonis
DDUR.adonis.pval<-signif(as.numeric(na.omit(DDUR.adonis$aov.tab$`Pr(>F)`)), digits = 1)

DDUR_min<-toString(c('Min=', min(nao.screen_data$DDUR)))
DDUR_min<-gsub(",", "", DDUR_min)
DDUR_min<-gsub(" ", "", DDUR_min)
DDUR_med<-toString(c("Median=", median(nao.screen_data$DDUR)))
DDUR_med<-gsub(",", "", DDUR_med)
DDUR_med<-gsub(" ", "", DDUR_med)
DDUR_max<-toString(c("Max=", max(nao.screen_data$DDUR)))
DDUR_max<-gsub(",", "", DDUR_max)
DDUR_max<-gsub(" ", "", DDUR_max)
DDUR_sum<-toString(c(DDUR_min, DDUR_med, DDUR_max))
DDUR_n<-n.DDUR.cortest
DDUR_Spear_P<- toString(c("(", cor.DDUR.pval, ")"))
DDUR_Spear_P<-gsub(",", "", DDUR_Spear_P)
DDUR_Spear_P<-gsub(" ", "", DDUR_Spear_P)
DDUR_Spear_rho<-toString(c('rho =', cor.DDUR.rho))
DDUR_Spear_rho<-gsub(",", "", DDUR_Spear_rho)
#DDUR_Spear_rho<-gsub(" ", "", DDUR_Spear_rho)
DDUR_Spear_Prho<-c(DDUR_Spear_rho, DDUR_Spear_P)
DDUR_Spear_Prho<-toString(c(DDUR_Spear_Prho))
DDUR_Spear_Prho<-gsub(",", "", DDUR_Spear_Prho)

DDUR<- matrix(c('Disease Duration (years)', DDUR_Spear_rho, cor.DDUR$p.value, as.numeric(na.omit(DDUR.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r kwtissinvol, include=F}
set.seed(32016)
summary(nao.screen_data$tissinvol)

tissinvol.kw.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, tissinvol)))
levels(tissinvol.kw.data$tissinvol)[2]<-"Colon-Ileum"
tissinvol.kw<-kruskal.test(invsimpson ~ tissinvol, tissinvol.kw.data)
tissinvol.kw
#plot(invsimpson ~ tissinvol, tissinvol.kw.data)
Colon.kw<-tissinvol.kw.data[tissinvol.kw.data$tissinvol=="Colon",]
n.Colon.kw<-nrow(Colon.kw)
Colon_Ileum.kw<-tissinvol.kw.data[tissinvol.kw.data$tissinvol=="Colon-Ileum",]
n.Colon_Ileum.kw<-nrow(Colon_Ileum.kw)
Ileum.kw<-tissinvol.kw.data[tissinvol.kw.data$tissinvol=="Ileum",]
n.Ileum.kw<-nrow(Ileum.kw)
kmc<-kruskalmc(invsimpson ~ tissinvol, tissinvol.kw.data)
kw.tissinvol<-kruskal.test(invsimpson ~ tissinvol, tissinvol.kw.data)
summary(kw.tissinvol)
kw.tissinvol.pval<-signif(kw.tissinvol$p.value, digits = 1)
kw.tissinvol.pval
kw.tissinvol.method<-kw.tissinvol$method
subT<-kw.tissinvol.method
nColonString<-toString(c("Colon =", n.Colon.kw))
nColonString<-gsub(",", "", nColonString)
nColon_IleumString<-toString(c("Colon-Ileum =", n.Colon_Ileum.kw))
nColon_IleumString<-gsub(",", "", nColon_IleumString)
nIleumString<-toString(c("Ileum =", n.Ileum.kw))
nIleumString<-gsub(",", "", nIleumString)

kruskalmc(invsimpson ~ tissinvol, tissinvol.kw.data)
```

```{r tissinvol.adonis,  include=F}
set.seed(32016)
tissinvol.adonis<-adonis(nao.screen.all.dist ~ tissinvol, nao.screen_data)
tissinvol.adonis
tissinvol.adonis.pval<-signif(as.numeric(na.omit(tissinvol.adonis$aov.tab$`Pr(>F)`)), digits = 1)

Colon<-toString(c('Colon=', n.Colon.kw))
Colon<-gsub(",", "", Colon)
Colon<-gsub(" ", "", Colon)
ColIle<-toString(c('Colon-Ileum=', n.Colon_Ileum.kw))
ColIle<-gsub(",", "", ColIle)
ColIle<-gsub(" ", "", ColIle)
Ileum<-toString(c('Ileum=', n.Ileum.kw))
Ileum<-gsub(",", "", Ileum)
Ileum<-gsub(" ", "", Ileum)

tissinvol_sum<-toString(c(Colon, ColIle, Ileum))

tissinvol<- matrix(c('Tissue Involvement', "-", kw.tissinvol$p.value, as.numeric(na.omit(tissinvol.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r BOWSTRICT-wilcox, include=F}
set.seed(32016)

BOWSTRICT.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BOWSTRICT)))
n.YesBOWSTRICT.wilcoxtest<-nrow(BOWSTRICT.wilcox.test.data[BOWSTRICT.wilcox.test.data$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.wilcoxtest
n.NoBOWSTRICT.wilcoxtest<-nrow(BOWSTRICT.wilcox.test.data[BOWSTRICT.wilcox.test.data$BOWSTRICT=="Other",])
n.NoBOWSTRICT.wilcoxtest
wilcox.BOWSTRICT<-wilcox.test(invsimpson ~ BOWSTRICT, BOWSTRICT.wilcox.test.data)
summary(wilcox.BOWSTRICT)
wilcox.BOWSTRICT.pval<-signif(wilcox.BOWSTRICT$p.value, digits = 1)
wilcox.BOWSTRICT.pval

#plot(invsimpson~BOWSTRICT, nao.screen_data)
```

```{r BOWSTRICT.adonis,  include=F}
set.seed(32016)
BOWSTRICT.adonis<-adonis(nao.screen.all.dist ~ BOWSTRICT, nao.screen_data)
BOWSTRICT.adonis
BOWSTRICT.adonis.pval<-signif(as.numeric(na.omit(BOWSTRICT.adonis$aov.tab$`Pr(>F)`)), digits = 1)

YesBOWSTRICT<-toString(c('Yes=', n.YesBOWSTRICT.wilcoxtest))
YesBOWSTRICT<-gsub(",", "", YesBOWSTRICT)
YesBOWSTRICT<-gsub(" ", "", YesBOWSTRICT)
NoBOWSTRICT<-toString(c("No=", n.NoBOWSTRICT.wilcoxtest))
NoBOWSTRICT<-gsub(",", "", NoBOWSTRICT)
NoBOWSTRICT<-gsub(" ", "", NoBOWSTRICT)

BOWSTRICT_sum<-toString(c(NoBOWSTRICT, YesBOWSTRICT))

	
BOWSTRICT<-matrix(c('Bowel Stricture', "-", wilcox.BOWSTRICT$p.value, as.numeric(na.omit(BOWSTRICT.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r cohort description, include=F, cache=F}
N.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR)))
N_trtd <- nrow(N.cohort.data[N.cohort.data$TRTGR=="Treated",])
N_plac <- nrow(N.cohort.data[N.cohort.data$TRTGR=="Placebo",])
N_total <- nrow(N.cohort.data)
N.cohort<- matrix(c('', paste("n =", N_trtd), paste("n =", N_plac), paste("n =",N_total)), ncol = 4)

set.seed(32016)
AGE.total.mean<-signif(mean(AGE.cor.test.data$AGE), digits = 2)
AGE.total.sd<-signif(sd(AGE.cor.test.data$AGE), digits = 2)
AGE_total<-toString(c(AGE.total.mean, "Â±", AGE.total.sd))
AGE_total<-gsub(",", "", AGE_total)
#AGE_total<-gsub(" ", "", AGE_total)

AGE.trtd<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Treated",]
AGE.trtd.mean<-signif(mean(AGE.trtd$AGE), digits = 2)
AGE.trtd.sd<-signif(sd(AGE.trtd$AGE), digits = 2)
AGE_trtd<-toString(c(AGE.trtd.mean, " Â± ", AGE.trtd.sd))
AGE_trtd<-gsub(",", "", AGE_trtd)
#AGE_trtd<-gsub(" ", "", AGE_trtd)

AGE.plac<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Placebo",]
AGE.plac.mean<-signif(mean(AGE.plac$AGE), digits = 2)
AGE.plac.sd<-signif(sd(AGE.plac$AGE), digits = 2)
AGE_plac<-toString(c(AGE.plac.mean, " Â± ", AGE.plac.sd))
AGE_plac<-gsub(",", "", AGE_plac)
#AGE_plac<-gsub(" ", "", AGE_plac)

AGE.cohort<- matrix(c('Age (years)', AGE_trtd, AGE_plac, AGE_total), ncol = 4)

SEX.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, SEX)))
n.MSEX.total<-nrow(SEX.cohort.data[SEX.cohort.data$SEX=="M",])
n.MSEX.total
n.FSEX.total<-nrow(SEX.cohort.data[SEX.cohort.data$SEX=="F",])
n.FSEX.total

SEX_total<-toString(c(signif(n.MSEX.total/(n.FSEX.total+n.MSEX.total)*100, 3)))
SEX_total<-gsub(",", "", SEX_total)
#SEX_total<-gsub(" ", "", SEX_total)

SEX.trtd<-SEX.cohort.data[SEX.cohort.data$TRTGR=="Treated",]
n.MSEX.trtd<-nrow(SEX.trtd[SEX.trtd$SEX=="M",])
n.MSEX.trtd
n.FSEX.trtd<-nrow(SEX.trtd[SEX.trtd$SEX=="F",])
n.FSEX.trtd
SEX_trtd<-toString(c(signif(n.MSEX.trtd/(n.FSEX.trtd + n.MSEX.trtd)*100, 3)))
SEX_trtd<-gsub(",", "", SEX_trtd)
#SEX_trtd<-gsub(" ", "", SEX_trtd)


SEX.plac<-SEX.cohort.data[SEX.cohort.data$TRTGR=="Placebo",]
n.MSEX.plac<-nrow(SEX.plac[SEX.plac$SEX=="M",])
n.MSEX.plac
n.FSEX.plac<-nrow(SEX.plac[SEX.plac$SEX=="F",])
n.FSEX.plac
SEX_plac<-toString(c(signif(n.MSEX.plac/(n.FSEX.plac +n.MSEX.plac)*100, 3)))
SEX_plac<-gsub(",", "", SEX_plac)
#SEX_plac<-gsub(" ", "", SEX_plac)

SEX.cohort<- matrix(c('Sex (% Male)', SEX_trtd, SEX_plac, SEX_total), ncol = 4)

RACE.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, RACE)))
n.CRACE.total<-nrow(RACE.cohort.data[RACE.cohort.data$RACE=="Caucasian",])
n.CRACE.total
n.ORACE.total<-nrow(RACE.cohort.data[RACE.cohort.data$RACE!="Caucasian",])
n.ORACE.total

RACE_total<-toString(c(signif(n.CRACE.total/(n.CRACE.total+n.ORACE.total)*100, 3)))
RACE_total<-gsub(",", "", RACE_total)
#RACE_total<-gsub(" ", "", RACE_total)

RACE.trtd<-RACE.cohort.data[RACE.cohort.data$TRTGR=="Treated",]
n.CRACE.trtd<-nrow(RACE.trtd[RACE.trtd$RACE=="Caucasian",])
n.CRACE.trtd
n.ORACE.trtd<-nrow(RACE.trtd[RACE.trtd$RACE!="Caucasian",])
n.ORACE.trtd
RACE_trtd<-toString(c(signif(n.CRACE.trtd/(n.CRACE.trtd+n.ORACE.trtd)*100, 3)))
RACE_trtd<-gsub(",", "", RACE_trtd)
#RACE_trtd<-gsub(" ", "", RACE_trtd)


RACE.plac<-RACE.cohort.data[RACE.cohort.data$TRTGR=="Placebo",]
n.CRACE.plac<-nrow(RACE.plac[RACE.plac$RACE=="Caucasian",])
n.CRACE.plac
n.ORACE.plac<-nrow(RACE.plac[RACE.plac$RACE!="Caucasian",])
n.ORACE.plac
RACE_plac<-toString(c(signif(n.CRACE.plac/(n.CRACE.plac+n.ORACE.plac)*100, 3)))
RACE_plac<-gsub(",", "", RACE_plac)
#RACE_plac<-gsub(" ", "", RACE_plac)

RACE.cohort<- matrix(c('Race (% Caucasian)', RACE_trtd, RACE_plac, RACE_total), ncol = 4)

BCORT.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, BCORT)))
n.YesBCORT.total<-nrow(BCORT.cohort.data[BCORT.cohort.data$BCORT=="Yes",])
n.YesBCORT.total
n.NoBCORT.total<-nrow(BCORT.cohort.data[BCORT.cohort.data$BCORT!="Yes",])
n.NoBCORT.total

BCORT_total<-toString(c(signif(n.YesBCORT.total/(n.YesBCORT.total+n.NoBCORT.total)*100, 3)))
BCORT_total<-gsub(",", "", BCORT_total)
BCORT_total<-gsub(" ", "", BCORT_total)

BCORT.trtd<-BCORT.cohort.data[BCORT.cohort.data$TRTGR=="Treated",]
n.YesBCORT.trtd<-nrow(BCORT.trtd[BCORT.trtd$BCORT=="Yes",])
n.YesBCORT.trtd
n.NoBCORT.trtd<-nrow(BCORT.trtd[BCORT.trtd$BCORT!="Yes",])
n.NoBCORT.trtd
BCORT_trtd<-toString(c(signif(n.YesBCORT.trtd/(n.YesBCORT.trtd+n.NoBCORT.trtd)*100, 3)))
BCORT_trtd<-gsub(",", "", BCORT_trtd)
BCORT_trtd<-gsub(" ", "", BCORT_trtd)


BCORT.plac<-BCORT.cohort.data[BCORT.cohort.data$TRTGR=="Placebo",]
n.YesBCORT.plac<-nrow(BCORT.plac[BCORT.plac$BCORT=="Yes",])
n.YesBCORT.plac
n.NoBCORT.plac<-nrow(BCORT.plac[BCORT.plac$BCORT!="Yes",])
n.NoBCORT.plac
BCORT_plac<-toString(c(signif(n.YesBCORT.plac/(n.YesBCORT.plac+n.NoBCORT.plac)*100, 3)))
BCORT_plac<-gsub(",", "", BCORT_plac)
BCORT_plac<-gsub(" ", "", BCORT_plac)

BCORT.cohort<- matrix(c('Corticosteroid Use (%)', BCORT_trtd, BCORT_plac, BCORT_total), ncol = 4)

BL_BMI.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, BL_BMI)))
BL_BMI.total.mean<-signif(mean(BL_BMI.cohort.data$BL_BMI), digits = 2)
BL_BMI.total.sd<-signif(sd(BL_BMI.cohort.data$BL_BMI), digits = 2)
BL_BMI_total<-toString(c(BL_BMI.total.mean, " Â± ", BL_BMI.total.sd))
BL_BMI_total<-gsub(",", "", BL_BMI_total)
#BL_BMI_total<-gsub(" ", "", BL_BMI_total)

BL_BMI.trtd<-BL_BMI.cohort.data[BL_BMI.cohort.data$TRTGR=="Treated",]
BL_BMI.trtd.mean<-signif(mean(BL_BMI.trtd$BL_BMI), digits = 2)
BL_BMI.trtd.sd<-signif(sd(BL_BMI.trtd$BL_BMI), digits = 2)
BL_BMI_trtd<-toString(c(BL_BMI.trtd.mean, " Â± ", BL_BMI.trtd.sd))
BL_BMI_trtd<-gsub(",", "", BL_BMI_trtd)
#BL_BMI_trtd<-gsub(" ", "", BL_BMI_trtd)

BL_BMI.plac<-BL_BMI.cohort.data[BL_BMI.cohort.data$TRTGR=="Placebo",]
BL_BMI.plac.mean<-signif(mean(BL_BMI.plac$BL_BMI), digits = 2)
BL_BMI.plac.sd<-signif(sd(BL_BMI.plac$BL_BMI), digits = 2)
BL_BMI_plac<-toString(c(BL_BMI.plac.mean, " Â± ", BL_BMI.plac.sd))
BL_BMI_plac<-gsub(",", "", BL_BMI_plac)
#BL_BMI_plac<-gsub(" ", "", BL_BMI_plac)

BL_BMI.cohort<- matrix(c('BMI (kg/m^2)', BL_BMI_trtd, BL_BMI_plac, BL_BMI_total), ncol = 4)


DDUR.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, DDUR)))
DDUR.total.mean<-signif(mean(DDUR.cohort.data$DDUR), digits = 2)
DDUR.total.sd<-signif(sd(DDUR.cohort.data$DDUR), digits = 2)
DDUR_total<-toString(c(DDUR.total.mean, " Â± ", DDUR.total.sd))
DDUR_total<-gsub(",", "", DDUR_total)
#DDUR_total<-gsub(" ", "", DDUR_total)

DDUR.trtd<-DDUR.cohort.data[DDUR.cohort.data$TRTGR=="Treated",]
DDUR.trtd.mean<-signif(mean(DDUR.trtd$DDUR), digits = 2)
DDUR.trtd.sd<-signif(sd(DDUR.trtd$DDUR), digits = 2)
DDUR_trtd<-toString(c(DDUR.trtd.mean, " Â± ", DDUR.trtd.sd))
DDUR_trtd<-gsub(",", "", DDUR_trtd)
#DDUR_trtd<-gsub(" ", "", DDUR_trtd)

DDUR.plac<-DDUR.cohort.data[DDUR.cohort.data$TRTGR=="Placebo",]
DDUR.plac.mean<-signif(mean(DDUR.plac$DDUR), digits = 2)
DDUR.plac.sd<-signif(sd(DDUR.plac$DDUR), digits = 2)
DDUR_plac<-toString(c(DDUR.plac.mean, " Â± ", DDUR.plac.sd))
DDUR_plac<-gsub(",", "", DDUR_plac)
#DDUR_plac<-gsub(" ", "", DDUR_plac)

DDUR.cohort<- matrix(c('Disease Duration (years)', DDUR_trtd, DDUR_plac, DDUR_total), ncol = 4)

cdail_wk0.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, cdail_wk0)))
cdail_wk0.total.mean<-signif(mean(cdail_wk0.cohort.data$cdail_wk0), digits = 2)
cdail_wk0.total.sd<-signif(sd(cdail_wk0.cohort.data$cdail_wk0), digits = 2)
cdail_wk0_total<-toString(c(cdail_wk0.total.mean, " Â± ", cdail_wk0.total.sd))
cdail_wk0_total<-gsub(",", "", cdail_wk0_total)
#cdail_wk0_total<-gsub(" ", "", cdail_wk0_total)

cdail_wk0.trtd<-cdail_wk0.cohort.data[cdail_wk0.cohort.data$TRTGR=="Treated",]
cdail_wk0.trtd.mean<-signif(mean(cdail_wk0.trtd$cdail_wk0), digits = 2)
cdail_wk0.trtd.sd<-signif(sd(cdail_wk0.trtd$cdail_wk0), digits = 2)
cdail_wk0_trtd<-toString(c(cdail_wk0.trtd.mean, " Â± ", cdail_wk0.trtd.sd))
cdail_wk0_trtd<-gsub(",", "", cdail_wk0_trtd)
#cdail_wk0_trtd<-gsub(" ", "", cdail_wk0_trtd)

cdail_wk0.plac<-cdail_wk0.cohort.data[cdail_wk0.cohort.data$TRTGR=="Placebo",]
cdail_wk0.plac.mean<-signif(mean(cdail_wk0.plac$cdail_wk0), digits = 2)
cdail_wk0.plac.sd<-signif(sd(cdail_wk0.plac$cdail_wk0), digits = 2)
cdail_wk0_plac<-toString(c(cdail_wk0.plac.mean, " Â± ", cdail_wk0.plac.sd))
cdail_wk0_plac<-gsub(",", "", cdail_wk0_plac)
#cdail_wk0_plac<-gsub(" ", "", cdail_wk0_plac)

cdail_wk0.cohort<- matrix(c('CDAI', cdail_wk0_trtd, cdail_wk0_plac, cdail_wk0_total), ncol = 4)

BOWSTRICT.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, BOWSTRICT)))
n.YesBOWSTRICT.total<-nrow(BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.total
n.NoBOWSTRICT.total<-nrow(BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$BOWSTRICT=="Other",])
n.NoBOWSTRICT.total

BOWSTRICT_total<-toString(c(signif(n.YesBOWSTRICT.total/(n.YesBOWSTRICT.total+n.NoBOWSTRICT.total)*100, 3)))
BOWSTRICT_total<-gsub(",", "", BOWSTRICT_total)
BOWSTRICT_total<-gsub(" ", "", BOWSTRICT_total)

BOWSTRICT.trtd<-BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$TRTGR=="Treated",]
n.YesBOWSTRICT.trtd<-nrow(BOWSTRICT.trtd[BOWSTRICT.trtd$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.trtd
n.NoBOWSTRICT.trtd<-nrow(BOWSTRICT.trtd[BOWSTRICT.trtd$BOWSTRICT=="Other",])
n.NoBOWSTRICT.trtd
BOWSTRICT_trtd<-toString(c(signif(n.YesBOWSTRICT.trtd/(n.YesBOWSTRICT.trtd+n.NoBOWSTRICT.trtd)*100, 3)))
BOWSTRICT_trtd<-gsub(",", "", BOWSTRICT_trtd)
BOWSTRICT_trtd<-gsub(" ", "", BOWSTRICT_trtd)


BOWSTRICT.plac<-BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$TRTGR=="Placebo",]
n.YesBOWSTRICT.plac<-nrow(BOWSTRICT.plac[BOWSTRICT.plac$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.plac
n.NoBOWSTRICT.plac<-nrow(BOWSTRICT.plac[BOWSTRICT.plac$BOWSTRICT=="Other",])
n.NoBOWSTRICT.plac
BOWSTRICT_plac<-toString(c(signif(n.YesBOWSTRICT.plac/(n.YesBOWSTRICT.plac+n.NoBOWSTRICT.plac)*100, 3)))
BOWSTRICT_plac<-gsub(",", "", BOWSTRICT_plac)
BOWSTRICT_plac<-gsub(" ", "", BOWSTRICT_plac)

BOWSTRICT.cohort<- matrix(c('Bowel Stricture (%)', BOWSTRICT_trtd, BOWSTRICT_plac, BOWSTRICT_total), ncol = 4)

tissinvol.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, tissinvol)))
n.Colon.total<-nrow(tissinvol.cohort.data[tissinvol.cohort.data$tissinvol=="Colon",])
n.Colon.total <- signif(n.Colon.total/N_total*100, 3)
n.Colon_Ileum.total<-nrow(tissinvol.cohort.data[tissinvol.cohort.data$tissinvol=="Colon_Ileum",])
n.Colon_Ileum.total <- signif(n.Colon_Ileum.total/N_total*100, 3)
n.Ileum.total<-nrow(tissinvol.cohort.data[tissinvol.cohort.data$tissinvol=="Ileum",])
n.Ileum.total <- signif(n.Ileum.total/N_total*100,3)


tissinvol_total<-toString(c(n.Colon.total, "/", n.Colon_Ileum.total, "/", n.Ileum.total))
tissinvol_total<-gsub(",", "", tissinvol_total)
tissinvol_total<-gsub(" ", "", tissinvol_total)

tissinvol.trtd<-tissinvol.cohort.data[tissinvol.cohort.data$TRTGR=="Treated",]
n.Colon.trtd<-nrow(tissinvol.trtd[tissinvol.trtd$tissinvol=="Colon",])
n.Colon.trtd <- signif(n.Colon.trtd/N_trtd*100, 3)
n.Colon_Ileum.trtd<-nrow(tissinvol.trtd[tissinvol.trtd$tissinvol=="Colon_Ileum",])
n.Colon_Ileum.trtd <- signif(n.Colon_Ileum.trtd/N_trtd*100, 3)
n.Ileum.trtd<-nrow(tissinvol.trtd[tissinvol.trtd$tissinvol=="Ileum",])
n.Ileum.trtd <- signif(n.Ileum.trtd/N_trtd*100, 3)

tissinvol_trtd<-toString(c(n.Colon.trtd, "/", n.Colon_Ileum.trtd, "/", n.Ileum.trtd))
tissinvol_trtd<-gsub(",", "", tissinvol_trtd)
tissinvol_trtd<-gsub(" ", "", tissinvol_trtd)

tissinvol.plac<-tissinvol.cohort.data[tissinvol.cohort.data$TRTGR=="Placebo",]
n.Colon.plac<-nrow(tissinvol.plac[tissinvol.plac$tissinvol=="Colon",])
n.Colon.plac <- signif(n.Colon.plac/N_plac*100, 3)
n.Colon_Ileum.plac<-nrow(tissinvol.plac[tissinvol.plac$tissinvol=="Colon_Ileum",])
n.Colon_Ileum.plac <- signif(n.Colon_Ileum.plac/N_plac*100, 3)
n.Ileum.plac<-nrow(tissinvol.plac[tissinvol.plac$tissinvol=="Ileum",])
n.Ileum.plac <- signif(n.Ileum.plac/N_plac*100, 3)


tissinvol_plac<-toString(c(n.Colon.plac, "/", n.Colon_Ileum.plac, "/", n.Ileum.plac))
tissinvol_plac<-gsub(",", "", tissinvol_plac)
tissinvol_plac<-gsub(" ", "", tissinvol_plac)

tissinvol.cohort<- matrix(c('Tissue Involvement (%) \nColon/Ileocolic/Ileal', tissinvol_trtd, tissinvol_plac, tissinvol_total), ncol = 4)
```


```{r wk6-diversity, include=F}
set.seed(32016)
all.RelRSPwk6.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk6)))

YesRelRSPwk6.wilcoxtest <- all.RelRSPwk6.wilcox.test.data[all.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",]
Yes.inv.25quantile <- signif(quantile(YesRelRSPwk6.wilcoxtest$invsimpson, .25, type = 2), digits=3)
Yes.inv.75quantile <- signif(quantile(YesRelRSPwk6.wilcoxtest$invsimpson, .75, type = 2), digits=4)

NoRelRSPwk6.wilcoxtest <- all.RelRSPwk6.wilcox.test.data[all.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",]
No.inv.25quantile <- signif(quantile(NoRelRSPwk6.wilcoxtest$invsimpson, .25, type = 2), digits=3)
No.inv.75quantile <- signif(quantile(NoRelRSPwk6.wilcoxtest$invsimpson, .75, type = 2), digits=4)

all.n.YesRelRSPwk6.wilcoxtest<-nrow(all.RelRSPwk6.wilcox.test.data[all.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",])
all.n.YesRelRSPwk6.wilcoxtest
all.n.NoRelRSPwk6.wilcoxtest<-nrow(all.RelRSPwk6.wilcox.test.data[all.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",])
all.n.NoRelRSPwk6.wilcoxtest

all.RelRSPwk6.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk6, TRTGR)))
all.RelRSPwk6.kw.test.data$rRSPwk6TRTGR <- as.factor(paste(all.RelRSPwk6.kw.test.data$RelRSPwk6, all.RelRSPwk6.kw.test.data$TRTGR, sep = "_"))
all.RelRSPwk6.kw.test <- kruskal.test(all.RelRSPwk6.kw.test.data$invsimpson ~ all.RelRSPwk6.kw.test.data$rRSPwk6TRTGR, all.RelRSPwk6.kw.test.data)
all.RelRSPwk6.kw.test.pval <- all.RelRSPwk6.kw.test$p.value

set.seed(32016)
all.RelRSPwk6.adonis<-adonis(nao.screen.all.dist ~ RelRSPwk6, nao.screen_data, strata=nao.screen_data$TRTGR)
all.RelRSPwk6.adonis
all.RelRSPwk6.adonis.pval<-signif(as.numeric(na.omit(all.RelRSPwk6.adonis$aov.tab$`Pr(>F)`)), digits = 2)

all.YesRelRSPwk6<-toString(c(all.n.YesRelRSPwk6.wilcoxtest))
all.YesRelRSPwk6<-gsub(",", "", all.YesRelRSPwk6)
all.YesRelRSPwk6<-gsub(" ", "", all.YesRelRSPwk6)
all.NoRelRSPwk6<-toString(c(all.n.NoRelRSPwk6.wilcoxtest))
all.NoRelRSPwk6<-gsub(",", "", all.NoRelRSPwk6)
all.NoRelRSPwk6<-gsub(" ", "", all.NoRelRSPwk6)

all.RelRSPwk6_sum<-toString(c(all.NoRelRSPwk6, all.YesRelRSPwk6))

trtd.RelRSPwk6.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, RelRSPwk6)))
trtd.n.YesRelRSPwk6.wilcoxtest<-nrow(trtd.RelRSPwk6.wilcox.test.data[trtd.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",])
trtd.n.YesRelRSPwk6.wilcoxtest
trtd.n.NoRelRSPwk6.wilcoxtest<-nrow(trtd.RelRSPwk6.wilcox.test.data[trtd.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",])
trtd.n.NoRelRSPwk6.wilcoxtest

trtd.YesRelRSPwk6<-toString(c(trtd.n.YesRelRSPwk6.wilcoxtest))
trtd.YesRelRSPwk6<-gsub(",", "", trtd.YesRelRSPwk6)
trtd.YesRelRSPwk6<-gsub(" ", "", trtd.YesRelRSPwk6)
trtd.NoRelRSPwk6<-toString(c(trtd.n.NoRelRSPwk6.wilcoxtest))
trtd.NoRelRSPwk6<-gsub(",", "", trtd.NoRelRSPwk6)
trtd.NoRelRSPwk6<-gsub(" ", "", trtd.NoRelRSPwk6)

trtd.RelRSPwk6_sum<-toString(c(trtd.NoRelRSPwk6, trtd.YesRelRSPwk6))

plac.RelRSPwk6.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, RelRSPwk6)))
plac.n.YesRelRSPwk6.wilcoxtest<-nrow(plac.RelRSPwk6.wilcox.test.data[plac.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",])
plac.n.YesRelRSPwk6.wilcoxtest
plac.n.NoRelRSPwk6.wilcoxtest<-nrow(plac.RelRSPwk6.wilcox.test.data[plac.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",])
plac.n.NoRelRSPwk6.wilcoxtest

plac.YesRelRSPwk6<-toString(c(plac.n.YesRelRSPwk6.wilcoxtest))
plac.YesRelRSPwk6<-gsub(",", "", plac.YesRelRSPwk6)
plac.YesRelRSPwk6<-gsub(" ", "", plac.YesRelRSPwk6)
plac.NoRelRSPwk6<-toString(c(plac.n.NoRelRSPwk6.wilcoxtest))
plac.NoRelRSPwk6<-gsub(",", "", plac.NoRelRSPwk6)
plac.NoRelRSPwk6<-gsub(" ", "", plac.NoRelRSPwk6)

plac.RelRSPwk6_sum<-toString(c(plac.NoRelRSPwk6, plac.YesRelRSPwk6))

all.RelRSPwk6_AMOVA_p<-toString(c(all.RelRSPwk6.adonis.pval))
all.RelRSPwk6_AMOVA_p<-gsub(",", "", all.RelRSPwk6_AMOVA_p)
all.RelRSPwk6_AMOVA_p<-gsub(" ", "", all.RelRSPwk6_AMOVA_p)
all.RelRSPwk6_AMOVA<-toString(c(all.RelRSPwk6_AMOVA_p))
all.RelRSPwk6_AMOVA<-gsub(",", "", all.RelRSPwk6_AMOVA)
all.RelRSPwk6<-matrix(c('Week 6 Response (No, Yes)', trtd.RelRSPwk6_sum, plac.RelRSPwk6_sum, all.RelRSPwk6.kw.test.pval, all.RelRSPwk6_AMOVA), ncol = 5)

set.seed(32016)
all.REMISSwk6.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk6)))

all.REMISSwk6.invsimp <- all.REMISSwk6.wilcox.test.data[,-3]
row.names(all.REMISSwk6.invsimp) <- all.REMISSwk6.invsimp$group
all.REMISSwk6.invsimp <- all.REMISSwk6.invsimp[,-1]
REMwk6.invsimp.matrix <- aggregate(all.REMISSwk6.invsimp, by=list(all.REMISSwk6.wilcox.test.data$REMISSwk6), median)

row.names(REMwk6.invsimp.matrix) <- REMwk6.invsimp.matrix$Group.1
median.REMwk6.invsimp.matrix <- subset(REMwk6.invsimp.matrix, select=-c(Group.1))
median.REMwk6.invsimp.matrix <- as.matrix(t(subset(median.REMwk6.invsimp.matrix)))
colnames(median.REMwk6.invsimp.matrix) <- REMwk6.invsimp.matrix$Group.1
median.REMwk6.invsimp <- data.frame(median.REMwk6.invsimp.matrix)

all.REMISSwk6.Yes <- all.REMISSwk6.wilcox.test.data[all.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",]
quantile(all.REMISSwk6.Yes$invsimpson, type = 2)

REMwk6.YES.invsimp <- signif(median.REMwk6.invsimp$Yes, 4)

all.REMISSwk6.No <- all.REMISSwk6.wilcox.test.data[all.REMISSwk6.wilcox.test.data$REMISSwk6=="No",]
quantile(all.REMISSwk6.No$invsimpson, type = 2)

REMwk6.NO.invsimp <- signif(median.REMwk6.invsimp$No, 3)


all.REMISSwk6.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk6, TRTGR)))
all.REMISSwk6.kw.test.data$REMwk6TRTGR <- as.factor(paste(all.REMISSwk6.kw.test.data$REMISSwk6, all.REMISSwk6.kw.test.data$TRTGR, sep = "_"))
all.REMISSwk6.kw.test <- kruskal.test(all.REMISSwk6.kw.test.data$invsimpson ~ all.REMISSwk6.kw.test.data$REMwk6TRTGR, all.REMISSwk6.kw.test.data)
all.REMISSwk6.kw.test
all.REMISSwk6.kw.test.pval <- all.REMISSwk6.kw.test$p.value
all.REMISSwk6.kwmc.test <- kruskalmc(all.REMISSwk6.kw.test.data$invsimpson ~ all.REMISSwk6.kw.test.data$REMwk6TRTGR, all.REMISSwk6.kw.test.data)
all.REMISSwk6.kwmc.test

all.n.YesREMISSwk6.wilcoxtest<-nrow(all.REMISSwk6.wilcox.test.data[all.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",])
all.n.YesREMISSwk6.wilcoxtest
all.n.NoREMISSwk6.wilcoxtest<-nrow(all.REMISSwk6.wilcox.test.data[all.REMISSwk6.wilcox.test.data$REMISSwk6=="No",])
all.n.NoREMISSwk6.wilcoxtest
all.wilcox.REMISSwk6<-wilcox.test(invsimpson ~ REMISSwk6, all.REMISSwk6.wilcox.test.data)
all.wilcox.REMISSwk6
all.wilcox.REMISSwk6.pval<-signif(all.wilcox.REMISSwk6$p.value, digits = 2)
all.wilcox.REMISSwk6.pval

set.seed(32016)
all.REMISSwk6.adonis<-adonis(nao.screen.all.dist ~ REMISSwk6, nao.screen_data, strata=nao.screen_data$TRTGR)
all.REMISSwk6.adonis
all.REMISSwk6.adonis.pval<-signif(as.numeric(na.omit(all.REMISSwk6.adonis$aov.tab$`Pr(>F)`)), digits = 2)

all.YesREMISSwk6<-toString(c(all.n.YesREMISSwk6.wilcoxtest))
all.YesREMISSwk6<-gsub(",", "", all.YesREMISSwk6)
all.YesREMISSwk6<-gsub(" ", "", all.YesREMISSwk6)
all.NoREMISSwk6<-toString(c(all.n.NoREMISSwk6.wilcoxtest))
all.NoREMISSwk6<-gsub(",", "", all.NoREMISSwk6)
all.NoREMISSwk6<-gsub(" ", "", all.NoREMISSwk6)

all.REMISSwk6_sum<-toString(c(all.NoREMISSwk6, all.YesREMISSwk6))

trtd.REMISSwk6.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, REMISSwk6)))
trtd.n.YesREMISSwk6.wilcoxtest<-nrow(trtd.REMISSwk6.wilcox.test.data[trtd.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",])
trtd.n.YesREMISSwk6.wilcoxtest
trtd.n.NoREMISSwk6.wilcoxtest<-nrow(trtd.REMISSwk6.wilcox.test.data[trtd.REMISSwk6.wilcox.test.data$REMISSwk6=="No",])
trtd.n.NoREMISSwk6.wilcoxtest
trtd.YesREMISSwk6<-toString(c(trtd.n.YesREMISSwk6.wilcoxtest))
trtd.YesREMISSwk6<-gsub(",", "", trtd.YesREMISSwk6)
trtd.YesREMISSwk6<-gsub(" ", "", trtd.YesREMISSwk6)
trtd.NoREMISSwk6<-toString(c(trtd.n.NoREMISSwk6.wilcoxtest))
trtd.NoREMISSwk6<-gsub(",", "", trtd.NoREMISSwk6)
trtd.NoREMISSwk6<-gsub(" ", "", trtd.NoREMISSwk6)

trtd.REMISSwk6_sum<-toString(c(trtd.NoREMISSwk6, trtd.YesREMISSwk6))

plac.REMISSwk6.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, REMISSwk6)))
plac.n.YesREMISSwk6.wilcoxtest<-nrow(plac.REMISSwk6.wilcox.test.data[plac.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",])
plac.n.YesREMISSwk6.wilcoxtest
plac.n.NoREMISSwk6.wilcoxtest<-nrow(plac.REMISSwk6.wilcox.test.data[plac.REMISSwk6.wilcox.test.data$REMISSwk6=="No",])
plac.n.NoREMISSwk6.wilcoxtest
plac.YesREMISSwk6<-toString(c(plac.n.YesREMISSwk6.wilcoxtest))
plac.YesREMISSwk6<-gsub(",", "", plac.YesREMISSwk6)
plac.YesREMISSwk6<-gsub(" ", "", plac.YesREMISSwk6)
plac.NoREMISSwk6<-toString(c(plac.n.NoREMISSwk6.wilcoxtest))
plac.NoREMISSwk6<-gsub(",", "", plac.NoREMISSwk6)
plac.NoREMISSwk6<-gsub(" ", "", plac.NoREMISSwk6)

plac.REMISSwk6_sum<-toString(c(plac.NoREMISSwk6, plac.YesREMISSwk6))




all.REMISSwk6_AMOVA_p<-toString(c(all.REMISSwk6.adonis.pval))
all.REMISSwk6_AMOVA_p<-gsub(",", "", all.REMISSwk6_AMOVA_p)
all.REMISSwk6_AMOVA_p<-gsub(" ", "", all.REMISSwk6_AMOVA_p)
all.REMISSwk6_AMOVA<-toString(c(all.REMISSwk6_AMOVA_p))
all.REMISSwk6_AMOVA<-gsub(",", "", all.REMISSwk6_AMOVA)
all.REMISSwk6<-matrix(c('Week 6 Remission (No, Yes)', trtd.REMISSwk6_sum, plac.REMISSwk6_sum, all.REMISSwk6.kw.test.pval, all.REMISSwk6_AMOVA), ncol = 5)
```

```{r wk22-diversity, include=F}
set.seed(32016)
all.RelRSPwk22.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk22)))

all.RelRSPwk22.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk22, TRTGRINDMAN)))
all.RelRSPwk22.kw.test.data$rRSPwk22TRTGRINDMAN <- as.factor(paste(all.RelRSPwk22.kw.test.data$RelRSPwk22, all.RelRSPwk22.kw.test.data$TRTGRINDMAN, sep = "_"))
all.RelRSPwk22.kw.test <- kruskal.test(all.RelRSPwk22.kw.test.data$invsimpson ~ all.RelRSPwk22.kw.test.data$rRSPwk22TRTGRINDMAN, all.RelRSPwk22.kw.test.data)
all.RelRSPwk22.kw.test.pval <- all.RelRSPwk22.kw.test$p.value


all.n.YesRelRSPwk22.wilcoxtest<-nrow(all.RelRSPwk22.wilcox.test.data[all.RelRSPwk22.wilcox.test.data$RelRSPwk22=="Yes",])
all.n.YesRelRSPwk22.wilcoxtest
all.n.NoRelRSPwk22.wilcoxtest<-nrow(all.RelRSPwk22.wilcox.test.data[all.RelRSPwk22.wilcox.test.data$RelRSPwk22=="No",])
all.n.NoRelRSPwk22.wilcoxtest
all.wilcox.RelRSPwk22<-wilcox.test(invsimpson ~ RelRSPwk22, all.RelRSPwk22.wilcox.test.data)
all.wilcox.RelRSPwk22
all.wilcox.RelRSPwk22.pval<-signif(all.wilcox.RelRSPwk22$p.value, digits = 2)
all.wilcox.RelRSPwk22.pval

set.seed(32016)
kruskalmc(nao.screen_data$invsimpson, nao.screen_data$RelRSPwk22, data=nao.screen_data)

set.seed(32016)
all.RelRSPwk22.adonis<-adonis(nao.screen.all.dist ~ RelRSPwk22, nao.screen_data, strata=nao.screen_data$TRTGRINDMAN)
all.RelRSPwk22.adonis
all.RelRSPwk22.adonis.pval<-signif(as.numeric(na.omit(all.RelRSPwk22.adonis$aov.tab$`Pr(>F)`)), digits = 2)

all.YesRelRSPwk22<-toString(c(all.n.YesRelRSPwk22.wilcoxtest))
all.YesRelRSPwk22<-gsub(",", "", all.YesRelRSPwk22)
all.YesRelRSPwk22<-gsub(" ", "", all.YesRelRSPwk22)
all.NoRelRSPwk22<-toString(c(all.n.NoRelRSPwk22.wilcoxtest))
all.NoRelRSPwk22<-gsub(",", "", all.NoRelRSPwk22)
all.NoRelRSPwk22<-gsub(" ", "", all.NoRelRSPwk22)

all.RelRSPwk22_sum<-toString(c(all.NoRelRSPwk22, all.YesRelRSPwk22))

trtd.RelRSPwk22.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, RelRSPwk22, TRTGRINDMAN)))
trtd.RelRSPwk22.wilcox.test.data<-trtd.RelRSPwk22.wilcox.test.data[trtd.RelRSPwk22.wilcox.test.data$TRTGRINDMAN=="Treated_Treated",]
n.trtd.RelRSPwk22.wilcox.test.data <- nrow(trtd.RelRSPwk22.wilcox.test.data)
trtd.n.YesRelRSPwk22.wilcoxtest<-nrow(trtd.RelRSPwk22.wilcox.test.data[trtd.RelRSPwk22.wilcox.test.data$RelRSPwk22=="Yes",])
trtd.n.YesRelRSPwk22.wilcoxtest
trtd.n.NoRelRSPwk22.wilcoxtest<-nrow(trtd.RelRSPwk22.wilcox.test.data[trtd.RelRSPwk22.wilcox.test.data$RelRSPwk22=="No",])
trtd.n.NoRelRSPwk22.wilcoxtest
trtd.YesRelRSPwk22<-toString(c(trtd.n.YesRelRSPwk22.wilcoxtest))
trtd.YesRelRSPwk22<-gsub(",", "", trtd.YesRelRSPwk22)
trtd.YesRelRSPwk22<-gsub(" ", "", trtd.YesRelRSPwk22)
trtd.NoRelRSPwk22<-toString(c(trtd.n.NoRelRSPwk22.wilcoxtest))
trtd.NoRelRSPwk22<-gsub(",", "", trtd.NoRelRSPwk22)
trtd.NoRelRSPwk22<-gsub(" ", "", trtd.NoRelRSPwk22)

trtd.RelRSPwk22_sum<-toString(c(trtd.NoRelRSPwk22, trtd.YesRelRSPwk22))

plac.RelRSPwk22.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, RelRSPwk22, TRTGRINDMAN)))
plac.RelRSPwk22.wilcox.test.data<-plac.RelRSPwk22.wilcox.test.data[plac.RelRSPwk22.wilcox.test.data$TRTGRINDMAN=="Placebo_Placebo",]
n.plac.RelRSPwk22.wilcox.test.data <- nrow(plac.RelRSPwk22.wilcox.test.data)
plac.n.YesRelRSPwk22.wilcoxtest<-nrow(plac.RelRSPwk22.wilcox.test.data[plac.RelRSPwk22.wilcox.test.data$RelRSPwk22=="Yes",])
plac.n.YesRelRSPwk22.wilcoxtest
plac.n.NoRelRSPwk22.wilcoxtest<-nrow(plac.RelRSPwk22.wilcox.test.data[plac.RelRSPwk22.wilcox.test.data$RelRSPwk22=="No",])
plac.n.NoRelRSPwk22.wilcoxtest
plac.YesRelRSPwk22<-toString(c(plac.n.YesRelRSPwk22.wilcoxtest))
plac.YesRelRSPwk22<-gsub(",", "", plac.YesRelRSPwk22)
plac.YesRelRSPwk22<-gsub(" ", "", plac.YesRelRSPwk22)
plac.NoRelRSPwk22<-toString(c(plac.n.NoRelRSPwk22.wilcoxtest))
plac.NoRelRSPwk22<-gsub(",", "", plac.NoRelRSPwk22)
plac.NoRelRSPwk22<-gsub(" ", "", plac.NoRelRSPwk22)

plac.RelRSPwk22_sum<-toString(c(plac.NoRelRSPwk22, plac.YesRelRSPwk22))


all.RelRSPwk22_AMOVA_p<-toString(c(all.RelRSPwk22.adonis.pval))
all.RelRSPwk22_AMOVA_p<-gsub(",", "", all.RelRSPwk22_AMOVA_p)
all.RelRSPwk22_AMOVA_p<-gsub(" ", "", all.RelRSPwk22_AMOVA_p)
all.RelRSPwk22_AMOVA<-toString(c(all.RelRSPwk22_AMOVA_p))
all.RelRSPwk22_AMOVA<-gsub(",", "", all.RelRSPwk22_AMOVA)
all.RelRSPwk22<-matrix(c('Week 22 Response (No, Yes)', trtd.RelRSPwk22_sum, plac.RelRSPwk22_sum, all.RelRSPwk22.kw.test.pval, all.RelRSPwk22_AMOVA), ncol = 5)

set.seed(32016)
all.REMISSwk22.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk22)))
all.REMISSwk22.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk22, TRTGRINDMAN)))

all.REMISSwk22.kw.test.data$REMwk6TRTGRINDMAN <-
	as.factor(paste(all.REMISSwk22.kw.test.data$REMISSwk22, all.REMISSwk22.kw.test.data$TRTGRINDMAN, sep = "_"))
all.REMISSwk22.kw.test <- kruskal.test(all.REMISSwk22.kw.test.data$invsimpson ~ all.REMISSwk22.kw.test.data$REMwk6TRTGRINDMAN, all.REMISSwk22.kw.test.data)
all.REMISSwk22.kw.test.pval <- all.REMISSwk22.kw.test$p.value

all.n.YesREMISSwk22.wilcoxtest<-nrow(all.REMISSwk22.wilcox.test.data[all.REMISSwk22.wilcox.test.data$REMISSwk22=="Yes",])
all.n.YesREMISSwk22.wilcoxtest
all.n.NoREMISSwk22.wilcoxtest<-nrow(all.REMISSwk22.wilcox.test.data[all.REMISSwk22.wilcox.test.data$REMISSwk22=="No",])
all.n.NoREMISSwk22.wilcoxtest
all.wilcox.REMISSwk22<-wilcox.test(invsimpson ~ REMISSwk22, all.REMISSwk22.wilcox.test.data)
all.wilcox.REMISSwk22
all.wilcox.REMISSwk22.pval<-signif(all.wilcox.REMISSwk22$p.value, digits = 2)
all.wilcox.REMISSwk22.pval

set.seed(32016)
all.REMISSwk22.adonis<-adonis(nao.screen.all.dist ~ REMISSwk22, nao.screen_data, strata=nao.screen_data$TRTGRINDMAN)
all.REMISSwk22.adonis
all.REMISSwk22.adonis.pval<-signif(as.numeric(na.omit(all.REMISSwk22.adonis$aov.tab$`Pr(>F)`)), digits = 2)

kruskalmc(nao.screen_data$invsimpson, nao.screen_data$REMISSwk22, data=nao.screen.trtd)

all.YesREMISSwk22<-toString(c(all.n.YesREMISSwk22.wilcoxtest))
all.YesREMISSwk22<-gsub(",", "", all.YesREMISSwk22)
all.YesREMISSwk22<-gsub(" ", "", all.YesREMISSwk22)
all.NoREMISSwk22<-toString(c(all.n.NoREMISSwk22.wilcoxtest))
all.NoREMISSwk22<-gsub(",", "", all.NoREMISSwk22)
all.NoREMISSwk22<-gsub(" ", "", all.NoREMISSwk22)

all.REMISSwk22_sum<-toString(c(all.NoREMISSwk22, all.YesREMISSwk22))

trtd.REMISSwk22.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, REMISSwk22, TRTGRINDMAN)))
trtd.REMISSwk22.wilcox.test.data<-trtd.REMISSwk22.wilcox.test.data[trtd.REMISSwk22.wilcox.test.data$TRTGRINDMAN=="Treated_Treated",]
trtd.n.YesREMISSwk22.wilcoxtest<-nrow(trtd.REMISSwk22.wilcox.test.data[trtd.REMISSwk22.wilcox.test.data$REMISSwk22=="Yes",])
trtd.n.YesREMISSwk22.wilcoxtest
trtd.n.NoREMISSwk22.wilcoxtest<-nrow(trtd.REMISSwk22.wilcox.test.data[trtd.REMISSwk22.wilcox.test.data$REMISSwk22=="No",])
trtd.n.NoREMISSwk22.wilcoxtest
trtd.YesREMISSwk22<-toString(c(trtd.n.YesREMISSwk22.wilcoxtest))
trtd.YesREMISSwk22<-gsub(",", "", trtd.YesREMISSwk22)
trtd.YesREMISSwk22<-gsub(" ", "", trtd.YesREMISSwk22)
trtd.NoREMISSwk22<-toString(c(trtd.n.NoREMISSwk22.wilcoxtest))
trtd.NoREMISSwk22<-gsub(",", "", trtd.NoREMISSwk22)
trtd.NoREMISSwk22<-gsub(" ", "", trtd.NoREMISSwk22)

trtd.REMISSwk22_sum<-toString(c(trtd.NoREMISSwk22, trtd.YesREMISSwk22))

plac.REMISSwk22.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, REMISSwk22, TRTGRINDMAN)))
plac.REMISSwk22.wilcox.test.data<-plac.REMISSwk22.wilcox.test.data[plac.REMISSwk22.wilcox.test.data$TRTGRINDMAN=="Placebo_Placebo",]
plac.n.YesREMISSwk22.wilcoxtest<-nrow(plac.REMISSwk22.wilcox.test.data[plac.REMISSwk22.wilcox.test.data$REMISSwk22=="Yes",])
plac.n.YesREMISSwk22.wilcoxtest
plac.n.NoREMISSwk22.wilcoxtest<-nrow(plac.REMISSwk22.wilcox.test.data[plac.REMISSwk22.wilcox.test.data$REMISSwk22=="No",])
plac.n.NoREMISSwk22.wilcoxtest
plac.YesREMISSwk22<-toString(c(plac.n.YesREMISSwk22.wilcoxtest))
plac.YesREMISSwk22<-gsub(",", "", plac.YesREMISSwk22)
plac.YesREMISSwk22<-gsub(" ", "", plac.YesREMISSwk22)
plac.NoREMISSwk22<-toString(c(plac.n.NoREMISSwk22.wilcoxtest))
plac.NoREMISSwk22<-gsub(",", "", plac.NoREMISSwk22)
plac.NoREMISSwk22<-gsub(" ", "", plac.NoREMISSwk22)

plac.REMISSwk22_sum<-toString(c(plac.NoREMISSwk22, plac.YesREMISSwk22))


all.REMISSwk22_AMOVA_p<-toString(c(all.REMISSwk22.adonis.pval))
all.REMISSwk22_AMOVA_p<-gsub(",", "", all.REMISSwk22_AMOVA_p)
all.REMISSwk22_AMOVA_p<-gsub(" ", "", all.REMISSwk22_AMOVA_p)
all.REMISSwk22_AMOVA<-toString(c(all.REMISSwk22_AMOVA_p))
all.REMISSwk22_AMOVA<-gsub(",", "", all.REMISSwk22_AMOVA)
all.REMISSwk22<-matrix(c('Week 22 Remission (No, Yes)', trtd.REMISSwk22_sum, plac.REMISSwk22_sum, all.REMISSwk22.kw.test.pval, all.REMISSwk22_AMOVA), ncol = 5)

```

```{r tbl1padj, include=F}
colnames(CDAI) <- c('Clinical Variable', 'Summary', 'Species Richness (Alpha-diversity)', 'Community Structure (beta-diversity)')
tbl1<-rbind(CDAI, Stool)

cohortdiv<-c("CRP", "FCALP", "FLACT", "BMI", "weight", "AGE", "SEX", "BCORT", "DDUR", "tissinvol")

for (i in cohortdiv){
	tbl1<-rbind(tbl1, get(i))
}

tbl1 <- data.frame(tbl1)
adiv_p <- as.numeric(as.character(tbl1$Species.Richness..Alpha.diversity.))

tbl1_adiv_adj_p <- p.adjust(adiv_p, method="BH")
tbl1$Species.Richness..Alpha.diversity.<-as.factor(round(tbl1_adiv_adj_p, 3))

bdiv_p <- as.numeric(as.character(tbl1$Community.Structure..beta.diversity.))

tbl1_bdiv_adj_p <- p.adjust(bdiv_p, method="BH")
tbl1$Community.Structure..beta.diversity.<-as.factor(round(tbl1_bdiv_adj_p, 3))

tbl1

```

```{r SUpptable1diversity, fig.height=5, fig.width=10, include=F}

source('../code/table2diversity.R')

# pdf(file='tables/table2diversity.pdf', height=4, width=6)
# grid.table(table2div)
# dev.off()
```

```{r timereaddata, include=FALSE}
#Read in all data
#metadata
source('../code/APmd.setup.R')

screen_RESPwk22<-APmd_RSPwk22 #[APmd_RSPwk22$visit=="Screening",] #removes any group with NA in RESPONSEwk22

screen_data<-screen_RESPwk22				#na.omit(screen_RESPwk22)

#screen_data$CDSTOOLM_Screening[screen_data$CDSTOOLM_Screening=="."]<-"NA"
#screen_data$CDSTOOLM_Screening <-as.integer(screen_data$CDSTOOLM_Screening)
#screen_data$CDPAINM_Screening[screen_data$CDPAINM_Screening=="."]<-"NA"
#screen_data$CDPAINM_Screening <-as.integer(screen_data$CDPAINM_Screening)
#screen_data$CDWELBEM_Screening[screen_data$CDWELBEM_Screening=="."]<-"NA"
#screen_data$CDWELBEM_Screening <-as.integer(screen_data$CDWELBEM_Screening)
#screen_data$CDHCTM_Screening[screen_data$CDHCTM_Screening=="."]<-"NA"
#screen_data$CDHCTM_Screening <-as.integer(screen_data$CDHCTM_Screening)
#screen_data$CDAMASSM_Screening<-as.factor(screen_data$CDAMASSM_Screening)

#axes <- read.table('../data/Jan400.screen_ust.3107.subsample.thetayc.0.03.lt.ave.nmds.by3.axes', header=T)
tax<- read.table('../data/Jan400.simple.taxonomy.txt', header=T, sep="\t")
#summary(APmd_RSPwk22[,c("TRTGR", "visit")])
#summary(screen_RESPwk22[,1:49])
summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

levs<-screen_data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs
subidsover4<-levs[levs$no_rows>=4,]
summary(subidsover4)
fourtimesubs<-as.character(subidsover4$USUBJID)

fourtime_data <- screen_data[screen_data$USUBJID %in% fourtimesubs,]
#summary(subset(fourtime_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

#x5tp<-fourtime_data[fourtime_data$USUBJID=="C0743T26045400006",]
#subset(x5tp, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson))

#Remove "duplicate" sample with worse coverage and nseqs
alltime_data<-fourtime_data
alltime_data<-alltime_data[alltime_data$TRTGRINDMAN!="Placebo_Treated", ]
alltime_data<-alltime_data[alltime_data$TRTGRINDMAN!="Treated_Placebo", ] #remove TRTGRINDMAN=Placebo_Treated for easier analysisof all time point data placebo vs treated

summary(subset(alltime_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
alltp.ust<-alltime_data[alltime_data$TRTGRINDMAN=="Treated_Treated",]
#alltp.ust<-alltp.ust[alltp.ust$TRTGRINDMAN!="Placebo_Treated",]
summary(alltp.ust[,1:6])
alltp.placplac<-alltime_data[alltime_data$TRTGRINDMAN=="Placebo_Placebo",]
#alltp.placplac<-alltp.plac[alltp.plac$TRTGRINDMAN="Treated_Treated",]
summary(alltp.placplac[,1:6])

alltp.ust.RESPyes<-alltp.ust[alltp.ust$RelRSPwk22=="Yes",]
summary(alltp.ust.RESPyes[,1:6])
alltp.ust.RESPno<-alltp.ust[alltp.ust$RelRSPwk22=="No",]
summary(alltp.ust.RESPno[,1:6])

alltp.placplac.RESPyes<-alltp.placplac[alltp.placplac$RelRSPwk22=="Yes",]
summary(alltp.placplac.RESPyes[,1:6])
alltp.placplac.RESPno<-alltp.placplac[alltp.placplac$RelRSPwk22=="No",]
summary(alltp.placplac.RESPno[,1:6])

alltp.all.respY<-alltime_data[alltime_data$RelRSPwk22=="Yes",]
alltp.all.respN<-alltime_data[alltime_data$RelRSPwk22=="No",]
summary(alltp.all.respY[,1:6])

no22data<-screen_data[screen_data$visit!="Week 22",]
no22data<-no22data[no22data$TRTGRINDMAN!="Placebo_Not",]
no22data<-no22data[no22data$TRTGRINDMAN!="Treated_Not",]
summary(no22data$visit)
summary(subset(no22data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
levs<-no22data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs
subidsover3<-levs[levs$no_rows>=3,]
summary(subidsover3)
subidis4<-levs[levs$no_rows==4,]
no22subs3times<-as.character(subidsover3$USUBJID)
no22subs3times

no22_3x_data <- no22data[no22data$USUBJID %in% no22subs3times,]
summary(subset(no22_3x_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

x4tp<-no22_3x_data[no22_3x_data$USUBJID=="C0743T26045400006",]
subset(x4tp, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson))

#Remove "duplicate" sample with worse coverage and nseqs
wk046_data<-no22_3x_data[no22_3x_data$group!='E4859941-5',]

summary(subset(wk046_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

wk046_data<-wk046_data[wk046_data$USUBJID!='C0743T26030100002',]

summary(subset(wk046_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8, INDTRTGR)))

wk046wk22remitters<-wk046_data[wk046_data$REMISSwk22=="Yes",]
summary(subset(wk046wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

trtd.wk046wk22remitters<-wk046wk22remitters[wk046wk22remitters$TRTGRINDMAN!="Placebo_Placebo",]
trtd.wk046wk22remitters<-trtd.wk046wk22remitters[trtd.wk046wk22remitters$TRTGRINDMAN!="Placebo_Treated",]
summary(subset(trtd.wk046wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

plac.wk046wk22remitters<-wk046wk22remitters[wk046wk22remitters$TRTGRINDMAN!="Treated_Placebo",]
plac.wk046wk22remitters<-plac.wk046wk22remitters[plac.wk046wk22remitters$TRTGRINDMAN!="Treated_Treated",]
summary(subset(plac.wk046wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))


all.distmat<-read.table("../data/Jan400.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
all.dist<-as.dist(all.distmat)
ustdistmat<-read.table("../data/Jan400.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
ustdist<-as.dist(ustdistmat)
pladistmat<-read.table("../data/Jan400.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
pladist<-as.dist(pladistmat)
alltp_distmat<-read.table("../data/Jan400.alltp.noPU.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_dist<-as.dist(alltp_distmat)
alltp_ustdistmat<-read.table("../data/Jan400.all.na.omit.cmd.ust.alltp.thetayc.square.ave.dist")
alltp_ustdist<-as.dist(alltp_ustdistmat)
alltp_placplacdistmat<-read.table("../data/Jan400.alltp.plac.noPU.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_placplacdist<-as.dist(alltp_placplacdistmat)

wk046_distmat<-read.table("../data/Jan400.wk046.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk046_dist<-as.dist(wk046_distmat)
wk046_ustdistmat<-read.table("../data/Jan400.wk046.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk046_ustdist<-as.dist(wk046_ustdistmat)
wk046_placdistmat<-read.table("../data/Jan400.wk046.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk046_placdist<-as.dist(wk046_placdistmat)

wk0.ustdistmat<-read.table("../data/Jan400.alltp.ust.wk0.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk0.ustdist<-as.dist(wk0.ustdistmat)
wk4.ustdistmat<-read.table("../data/Jan400.alltp.ust.wk4.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk4.ustdist<-as.dist(wk4.ustdistmat)
wk6.ustdistmat<-read.table("../data/Jan400.alltp.ust.wk6.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk6.ustdist<-as.dist(wk6.ustdistmat)

wk0.placdistmat<-read.table("../data/Jan400.alltp.plac.wk0.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk0.placdist<-as.dist(wk0.placdistmat)
wk4.placdistmat<-read.table("../data/Jan400.alltp.plac.wk4.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk4.placdist<-as.dist(wk4.placdistmat)
wk6.placdistmat<-read.table("../data/Jan400.alltp.plac.wk6.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk6.placdist<-as.dist(wk6.placdistmat)


alltp_ustdistmat.respY<-read.table("../data/Jan400.alltp.noPU.trtd.relRSP.Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_ustdist.respY<-as.dist(alltp_ustdistmat.respY)
alltp_placdistmat.respY<-read.table("../data/Jan400.alltp.noPU.plac.relRSP.Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_placdist.respY<-as.dist(alltp_placdistmat.respY)
alltp_distmat.respY<-read.table("../data/Jan400.alltp.noPU.relRSP.Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_dist.respY<-as.dist(alltp_distmat.respY)
alltp_distmat.respN<-read.table("../data/Jan400.alltp.noPU.relRSP.N.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_dist.respN<-as.dist(alltp_distmat.respN)

wk046.ust<-wk046_data[wk046_data$TRTGRINDMAN!="Placebo_Placebo",]
wk046.ust<-wk046.ust[wk046.ust$TRTGRINDMAN!="Placebo_Treated",]
summary(wk046.ust[,1:6])
wk046.plac<-wk046_data[wk046_data$TRTGRINDMAN!="Treated_Placebo",]
wk046.plac<-wk046.plac[wk046.plac$TRTGRINDMAN!="Treated_Treated",]
summary(wk046.plac[,1:6])

wk046.ust.relRSP22Y<-wk046.ust[wk046.ust$RelRSPwk22=="Yes",]
wk046.ust.relRSP22Y.distmat<-read.table("../data/Jan400.wk046.ust.relRSP22Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046.ust.relRSP22Y.dist<-as.dist(wk046.ust.relRSP22Y.distmat)

wk046.plac.relRSP22Y<-wk046.plac[wk046.plac$RelRSPwk22 =="Yes",]
wk046.plac.relRSP22Y.distmat<-read.table("../data/Jan400.wk046.plac.relRSP22Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046.plac.relRSP22Y.dist<-as.dist(wk046.plac.relRSP22Y.distmat)

wk046_data.relRSP22Y<-wk046_data[wk046_data$RelRSPwk22=="Yes",]
wk046_data.relRSP22Y.distmat<-read.table("../data/Jan400.wk046_data.relRSP22Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046_data.relRSP22Y.dist<-as.dist(wk046_data.relRSP22Y.distmat)

wk22remitters<-alltime_data[alltime_data$REMISSwk22=="Yes",]
summary(subset(wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

trtd.wk22remitters<-wk22remitters[wk22remitters$TRTGRINDMAN!="Placebo_Placebo",]
trtd.wk22remitters<-trtd.wk22remitters[trtd.wk22remitters$TRTGRINDMAN!="Placebo_Treated",]
summary(subset(trtd.wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

plac.wk22remitters<-wk22remitters[wk22remitters$TRTGRINDMAN!="Treated_Placebo",]
plac.wk22remitters<-plac.wk22remitters[plac.wk22remitters$TRTGRINDMAN!="Treated_Treated",]
summary(subset(plac.wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

wk22remitters.distmat<-read.table("../data/Jan400.alltp.wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk22remitters.dist<-as.dist(wk22remitters.distmat)
trtd.wk22remitters.distmat<-read.table("../data/Jan400.alltp.trtd.wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
trtd.wk22remitters.dist<-as.dist(trtd.wk22remitters.distmat)
plac.wk22remitters.distmat<-read.table("../data/Jan400.alltp.plac.wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
plac.wk22remitters.dist<-as.dist(plac.wk22remitters.distmat)

wk046wk22remitters.distmat<-read.table("../data/Jan400.wk046wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046wk22remitters.dist<-as.dist(wk046wk22remitters.distmat)
trtd.wk046wk22remitters.distmat<-read.table("../data/Jan400.trtd.wk046wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
trtd.wk046wk22remitters.dist<-as.dist(trtd.wk046wk22remitters.distmat)
plac.wk046wk22remitters.distmat<-read.table("../data/Jan400.plac.wk046wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
plac.wk046wk22remitters.dist<-as.dist(plac.wk046wk22remitters.distmat)
```

```{r vars, include=F}
alltp.all.respY<-alltime_data[alltime_data$RelRSPwk22=="Yes",]
alltp.ust.RESPy<-alltp.ust[alltp.ust$RelRSPwk22=="Yes",]
alltp.placplac.RESPy<-alltp.placplac[alltp.placplac$RelRSPwk22=="Yes",]

#set.seed(32016)
#adonis(alltp_dist ~ TRTGR*RelRSPwk22, alltime_data, strata = alltime_data$visit)
```

```{r alltp.adiv, include=F}
alltime_data <- alltime_data[alltime_data$TRTGRINDMAN != "Treated_Placebo",]

summary(alltime_data[, c("USUBJID", "visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")])

#change in adiv over time?
wk0.all.adiv <- alltime_data[alltime_data$visit=="Screening", c("USUBJID", "visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk0.adiv <- wk0.all.adiv
colnames(wk0.all.adiv)[3] <- "Week0adiv"
wk0.all.adiv <- subset(wk0.all.adiv, select=-c(visit))

wk4.all.adiv <- alltime_data[alltime_data$visit=="Week 4", c("USUBJID","visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk4.adiv <- wk4.all.adiv
colnames(wk4.all.adiv)[3] <- "Week4adiv"
wk4.all.adiv <- subset(wk4.all.adiv, select=-c(visit))

wk6.all.adiv <- alltime_data[alltime_data$visit=="Week 6", c("USUBJID","visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk6.adiv <- wk6.all.adiv
colnames(wk6.all.adiv)[3] <- "Week6adiv"
wk6.all.adiv <- subset(wk6.all.adiv, select=-c(visit))

wk22.all.adiv <- alltime_data[alltime_data$visit=="Week 22", c("USUBJID","visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk22.adiv <- wk22.all.adiv
colnames(wk22.all.adiv)[3] <- "Week22adiv"
wk22.all.adiv <- subset(wk22.all.adiv, select=-c(visit))

adiv.wk04 <- merge(wk0.all.adiv, wk4.all.adiv)
adiv.wk046 <- merge(adiv.wk04, wk6.all.adiv)
adiv.alltp <- merge(adiv.wk046, wk22.all.adiv)

#adiv.alltp <- adiv.alltp[,-1]
#mean.adiv.byvis <- apply(adiv.alltp[,-c(1:2)], 2, mean)
treatedresp <- wk0.all.adiv[wk0.all.adiv$TRTGRINDMAN=="Treated_Treated",]
#treatedrespY <- treatedrespY[treatedrespY$visit=="Screening",]
treatedrespY <- treatedresp[treatedresp$RelRSPwk22=="Yes",]
treatedrespN <- treatedresp[treatedresp$RelRSPwk22=="No",]
alltp.treatedrespY <- adiv.alltp[adiv.alltp$USUBJID %in% treatedrespY$USUBJID,]
n.alltp.treatedrespY <- nrow(alltp.treatedrespY)
rownames(alltp.treatedrespY) <- alltp.treatedrespY$USUBJID
alltp.treatedrespY <- as.matrix(subset(alltp.treatedrespY, select = -c(USUBJID, INDTRTGR, RelRSPwk22, REMISSwk22, TRTGRINDMAN)))
mean.adiv.byvis <- apply(alltp.treatedrespY, 2, mean)


alltp.treatedrespN <- adiv.alltp[adiv.alltp$USUBJID %in% treatedrespN$USUBJID,]
n.alltp.treatedrespN <- nrow(alltp.treatedrespN)
rownames(alltp.treatedrespN) <- alltp.treatedrespN$USUBJID
alltp.treatedrespN <- as.matrix(subset(alltp.treatedrespN,select = -c(USUBJID, INDTRTGR, RelRSPwk22, REMISSwk22, TRTGRINDMAN)))
mean.adiv.byvis <- apply(alltp.treatedrespN, 2, mean)


ggplot.wk04.adiv<-rbind(wk0.adiv, wk4.adiv)
ggplot.wk046.adiv<-rbind(ggplot.wk04.adiv, wk6.adiv)
ggplot.alltp.adiv<-rbind(ggplot.wk046.adiv, wk22.adiv)
treatedresp <- ggplot.alltp.adiv[ggplot.alltp.adiv$INDTRTGR=="Treated",]
treatedresp <- treatedresp[treatedresp$visit=="Screening",]
treatedrespY <- treatedresp[treatedresp$RelRSPwk22=="Yes",]
ggplot.alltp.treatedrespY <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% treatedrespY$USUBJID,]
ggplot.alltp.treatedrespY <- ggplot.alltp.treatedrespY[, -c(4:6)]


treatedrespN <- treatedresp[treatedresp$RelRSPwk22=="No",]
ggplot.alltp.treatedrespN <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% treatedrespN$USUBJID,]
ggplot.alltp.treatedrespN <- ggplot.alltp.treatedrespN[, -c(4:6)]



fried.trtdrespY <- friedman.test(alltp.treatedrespY)
fried.trtdrespY$p.value
friedmc.trtdrespY <- friedmanmc(alltp.treatedrespY)
friedmc.trtdrespY.pval <- friedmc.trtdrespY$dif.com
friedmc.trtdrespY.pval <- friedmc.trtdrespY.pval[friedmc.trtdrespY.pval$difference=="TRUE",]
pairwise.wilcox.test(ggplot.alltp.treatedrespY$invsimpson, ggplot.alltp.treatedrespY$visit, p.adj="BH", exact=F, paired=T)


friedman.test(alltp.treatedrespN)
friedmc <- friedmanmc(alltp.treatedrespN)
friedmc
pairwise.wilcox.test(ggplot.alltp.treatedrespN$invsimpson, ggplot.alltp.treatedrespN$visit, p.adj="BH", exact=F, paired=T)



#adiv.alltp <- adiv.alltp[,-1]
#mean.adiv.byvis <- apply(adiv.alltp[,-c(1:2)], 2, mean)
placeboresp <- wk0.all.adiv[wk0.all.adiv$INDTRTGR=="Placebo",]
#placeborespY <- placeborespY[placeborespY$visit=="Screening",]
placeborespY <- placeboresp[placeboresp$RelRSPwk22=="Yes",]
placeborespN <- placeboresp[placeboresp$RelRSPwk22=="No",]
alltp.placeborespY <- adiv.alltp[adiv.alltp$USUBJID %in% placeborespY$USUBJID,]
#alltp.placeborespY <- alltp.placeborespY[, -c(4:5)]
rownames(alltp.placeborespY) <- alltp.placeborespY$USUBJID
alltp.placeborespY <- as.matrix(alltp.placeborespY[,-c(1:5)])
mean.adiv.byvis <- apply(alltp.placeborespY, 2, mean)



alltp.placeborespN <- adiv.alltp[adiv.alltp$USUBJID %in% placeborespN$USUBJID,]
#alltp.placeborespN <- alltp.placeborespN[, -c(4:5)]
rownames(alltp.placeborespN) <- alltp.placeborespN$USUBJID
alltp.placeborespN <- as.matrix(alltp.placeborespN[,-c(1:5)])
mean.adiv.byvis <- apply(alltp.placeborespN, 2, mean)


ggplot.wk04.adiv<-rbind(wk0.adiv, wk4.adiv)
ggplot.wk046.adiv<-rbind(ggplot.wk04.adiv, wk6.adiv)
ggplot.alltp.adiv<-rbind(ggplot.wk046.adiv, wk22.adiv)
placeboresp <- ggplot.alltp.adiv[ggplot.alltp.adiv$INDTRTGR=="Placebo",]
placeboresp <- placeboresp[placeboresp$visit=="Screening",]
placeborespY <- placeboresp[placeboresp$RelRSPwk22=="Yes",]
ggplot.alltp.placeborespY <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% placeborespY$USUBJID,]
ggplot.alltp.placeborespY <- ggplot.alltp.placeborespY[, -c(4:5)]


placeborespN <- placeboresp[placeboresp$RelRSPwk22=="No",]
ggplot.alltp.placeborespN <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% placeborespN$USUBJID,]
ggplot.alltp.placeborespN <- ggplot.alltp.placeborespN[, -c(4:5)]


friedman.test(alltp.placeborespY)
friedmc <- friedmanmc(alltp.placeborespY)
friedmc
pairwise.wilcox.test(ggplot.alltp.placeborespY$invsimpson, ggplot.alltp.placeborespY$visit, p.adj="BH", exact=F, paired=T)


friedman.test(alltp.placeborespN)
friedmc <- friedmanmc(alltp.placeborespN)
friedmc
pairwise.wilcox.test(ggplot.alltp.placeborespN$invsimpson, ggplot.alltp.placeborespN$visit, p.adj="BH", exact=F, paired=T)

source('../code/alltp.adivXvisitXtrtgrXrelRSPwk22.R')

pdf("figures/Figure2_alltp.adivXvisitXindtrtXrelRSPwk22.pdf", height = 6, width = 10)
alltp.adivXvisXindtrtXrelRSPwk22.plot
dev.off()

alltp.treatedrespY <- data.frame(alltp.treatedrespY)
alltp.treatedrespN <- data.frame(alltp.treatedrespN)
alltp.placeborespY <- data.frame(alltp.placeborespY)
alltp.placeborespN <- data.frame(alltp.placeborespN)


#change in adiv over time?
wk0.all.adiv <- alltime_data[alltime_data$visit=="Screening", c("USUBJID", "visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk0.adiv <- wk0.all.adiv
colnames(wk0.all.adiv)[3] <- "Week0adiv"
wk0.all.adiv <- subset(wk0.all.adiv, select=-c(visit))

wk4.all.adiv <- alltime_data[alltime_data$visit=="Week 4", c("USUBJID","visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk4.adiv <- wk4.all.adiv
colnames(wk4.all.adiv)[3] <- "Week4adiv"
wk4.all.adiv <- subset(wk4.all.adiv, select=-c(visit))

wk6.all.adiv <- alltime_data[alltime_data$visit=="Week 6", c("USUBJID","visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk6.adiv <- wk6.all.adiv
colnames(wk6.all.adiv)[3] <- "Week6adiv"
wk6.all.adiv <- subset(wk6.all.adiv, select=-c(visit))

wk22.all.adiv <- alltime_data[alltime_data$visit=="Week 22", c("USUBJID","visit", "invsimpson", "INDTRTGR", "TRTGRINDMAN", "RelRSPwk22", "REMISSwk22")]
wk22.adiv <- wk22.all.adiv
colnames(wk22.all.adiv)[3] <- "Week22adiv"
wk22.all.adiv <- subset(wk22.all.adiv, select=-c(visit))

adiv.wk04 <- merge(wk0.all.adiv, wk4.all.adiv)
adiv.wk046 <- merge(adiv.wk04, wk6.all.adiv)
adiv.alltp <- merge(adiv.wk046, wk22.all.adiv)




#adiv.alltp <- adiv.alltp[,-1]
#mean.adiv.byvis <- apply(adiv.alltp[,-c(1:2)], 2, mean)
treatedrem <- wk0.all.adiv[wk0.all.adiv$TRTGRINDMAN=="Treated_Treated",]
#treatedremY <- treatedremY[treatedremY$visit=="Screening",]
treatedremY <- treatedrem[treatedrem$REMISSwk22=="Yes",]
treatedremN <- treatedrem[treatedrem$REMISSwk22=="No",]
alltp.treatedremY <- adiv.alltp[adiv.alltp$USUBJID %in% treatedremY$USUBJID,]
n.alltp.treatedremY <- nrow(alltp.treatedremY)
rownames(alltp.treatedremY) <- alltp.treatedremY$USUBJID
alltp.treatedremY <- as.matrix(subset(alltp.treatedremY,select = -c(USUBJID, INDTRTGR, RelRSPwk22, REMISSwk22, TRTGRINDMAN)))
mean.adiv.byvis <- apply(alltp.treatedremY, 2, mean)

alltp.treatedremN <- adiv.alltp[adiv.alltp$USUBJID %in% treatedremN$USUBJID,]
n.alltp.treatedremN <- nrow(alltp.treatedremN)
rownames(alltp.treatedremN) <- alltp.treatedremN$USUBJID
alltp.treatedremN <- as.matrix(subset(alltp.treatedremN,select = -c(USUBJID, INDTRTGR, RelRSPwk22, REMISSwk22, TRTGRINDMAN)))
mean.adiv.byvis <- apply(alltp.treatedremN, 2, mean)

ggplot.wk04.adiv<-rbind(wk0.adiv, wk4.adiv)
ggplot.wk046.adiv<-rbind(ggplot.wk04.adiv, wk6.adiv)
ggplot.alltp.adiv<-rbind(ggplot.wk046.adiv, wk22.adiv)
treatedrem <- ggplot.alltp.adiv[ggplot.alltp.adiv$INDTRTGR=="Treated",]
treatedrem <- treatedrem[treatedrem$visit=="Screening",]
treatedremY <- treatedrem[treatedrem$REMISSwk22=="Yes",]
ggplot.alltp.treatedremY <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% treatedremY$USUBJID,]
ggplot.alltp.treatedremY <- ggplot.alltp.treatedremY[, -c(4:6)]


treatedremN <- treatedrem[treatedrem$REMISSwk22=="No",]
ggplot.alltp.treatedremN <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% treatedremN$USUBJID,]
ggplot.alltp.treatedremN <- ggplot.alltp.treatedremN[, -c(4:6)]



fried.trtdremY <- friedman.test(alltp.treatedremY)
fried.trtdremY$p.value
friedmc.trtdremY <- friedmanmc(alltp.treatedremY)
friedmc.trtdremY.pval <- friedmc.trtdremY$dif.com
friedmc.trtdremY.pval <- friedmc.trtdremY.pval[friedmc.trtdremY.pval$difference=="TRUE",]
pairwise.wilcox.test(ggplot.alltp.treatedremY$invsimpson, ggplot.alltp.treatedremY$visit, p.adj="BH", exact=F, paired=T)


friedman.test(alltp.treatedremN)
friedmc <- friedmanmc(alltp.treatedremN)
friedmc
pairwise.wilcox.test(ggplot.alltp.treatedremN$invsimpson, ggplot.alltp.treatedremN$visit, p.adj="BH", exact=F, paired=T)



#adiv.alltp <- adiv.alltp[,-1]
#mean.adiv.byvis <- apply(adiv.alltp[,-c(1:2)], 2, mean)
placeborem <- wk0.all.adiv[wk0.all.adiv$INDTRTGR=="Placebo",]
#placeboremY <- placeboremY[placeboremY$visit=="Screening",]
placeboremY <- placeborem[placeborem$REMISSwk22=="Yes",]
placeboremN <- placeborem[placeborem$REMISSwk22=="No",]
alltp.placeboremY <- adiv.alltp[adiv.alltp$USUBJID %in% placeboremY$USUBJID,]
#alltp.placeboremY <- alltp.placeboremY[, -c(4:5)]
rownames(alltp.placeboremY) <- alltp.placeboremY$USUBJID
alltp.placeboremY <- as.matrix(alltp.placeboremY[,-c(1:5)])
mean.adiv.byvis <- apply(alltp.placeboremY, 2, mean)

alltp.placeboremN <- adiv.alltp[adiv.alltp$USUBJID %in% placeboremN$USUBJID,]
#alltp.placeboremN <- alltp.placeboremN[, -c(4:5)]
rownames(alltp.placeboremN) <- alltp.placeboremN$USUBJID
alltp.placeboremN <- as.matrix(alltp.placeboremN[,-c(1:5)])
mean.adiv.byvis <- apply(alltp.placeboremN, 2, mean)

ggplot.wk04.adiv<-rbind(wk0.adiv, wk4.adiv)
ggplot.wk046.adiv<-rbind(ggplot.wk04.adiv, wk6.adiv)
ggplot.alltp.adiv<-rbind(ggplot.wk046.adiv, wk22.adiv)
placeborem <- ggplot.alltp.adiv[ggplot.alltp.adiv$INDTRTGR=="Placebo",]
placeborem <- placeborem[placeborem$visit=="Screening",]
placeboremY <- placeborem[placeborem$REMISSwk22=="Yes",]
ggplot.alltp.placeboremY <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% placeboremY$USUBJID,]
ggplot.alltp.placeboremY <- ggplot.alltp.placeboremY[, -c(4:5)]


placeboremN <- placeborem[placeborem$REMISSwk22=="No",]
ggplot.alltp.placeboremN <- ggplot.alltp.adiv[ggplot.alltp.adiv$USUBJID %in% placeboremN$USUBJID,]
ggplot.alltp.placeboremN <- ggplot.alltp.placeboremN[, -c(4:5)]


friedman.test(alltp.placeboremY)
friedmc <- friedmanmc(alltp.placeboremY)
friedmc
pairwise.wilcox.test(ggplot.alltp.placeboremY$invsimpson, ggplot.alltp.placeboremY$visit, p.adj="BH", exact=F, paired=T)


friedman.test(alltp.placeboremN)
friedmc <- friedmanmc(alltp.placeboremN)
friedmc
pairwise.wilcox.test(ggplot.alltp.placeboremN$invsimpson, ggplot.alltp.placeboremN$visit, p.adj="BH", exact=F, paired=T)
```

```{r alltpByWk22RSP, include=F, eval=F}
summary(subset(alltime_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

set.seed(32016)


#ALL with strata
adonis(alltp_dist ~ visit*TRTGRINDMAN*RelRSPwk22, alltime_data, strata = alltime_data$USUBJID)
adonis(alltp_dist ~ visit*TRTGRINDMAN*REMISSwk22, alltime_data, strata = alltime_data$USUBJID)
```

```{r readdataAUCRF, include=FALSE}
#Read in all data
source('../code/APmd.setup.R')

APmd_adiv<-merge(APmd, adiv, by.x="group", by.y="group")#nonmatch removed
APmd_shared<-merge(APmd_adiv, ss3k_shared_U2s, by.x="group", by.y="Group")

APmd_shared$TRTGR_REMISSwk6 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$REMISSwk6))
APmd_shared$TRTGR_REMISSwk22 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$REMISSwk22))

APmd_shared$TRTGR_RelRSPwk6 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$RelRSPwk6))
APmd_shared$TRTGR_RelRSPwk22 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$RelRSPwk22))
APmd_RSPwk22<-APmd_shared

screen_RESPwk22<-APmd_RSPwk22[APmd_RSPwk22$visit=="Screening",] #removes any group with NA in RESPONSEwk22

screen_data<-screen_RESPwk22			

screen.trtd<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Placebo",]
screen.trtd<-screen.trtd[screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(screen.trtd[,1:6])
screen.plac<-screen_data[screen_data$TRTGRINDMAN!="Treated_Placebo",]
screen.plac<-screen.plac[screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(screen.plac[,1:6])

nao.screen_data<-na.omit(screen_data)

nao.screen.trtd<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Placebo_Placebo",]
nao.screen.trtd<-nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(nao.screen.trtd[,1:6])
nao.screen.plac<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Treated_Placebo",]
nao.screen.plac<-nao.screen.plac[nao.screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(nao.screen.plac[,1:6])

tax<- read.table('../data/Jan400.simple.taxonomy.txt', header=T, sep="\t")
nao.screen.all.distmat<-read.table("../data/Jan400.screening.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.all.dist<-as.dist(nao.screen.all.distmat)
nao.screen.trtd.distmat<-read.table("../data/Jan400.screening.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.trtd.dist<-as.dist(nao.screen.trtd.distmat)
nao.screen.plac.distmat<-read.table("../data/Jan400.screening.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.plac.dist<-as.dist(nao.screen.plac.distmat)

screen.all.distmat<-read.table("../data/Jan400.screen.all.thetayc.0.03.square.ave.dist")
screen.all.dist<-as.dist(screen.all.distmat)
screen.trtd.distmat<-read.table("../data/Jan400.screen.ust.thetayc.0.03.square.ave.dist")
screen.trtd.dist<-as.dist(screen.trtd.distmat)
screen.plac.distmat<-read.table("../data/Jan400.screen.plac.thetayc.0.03.square.ave.dist")
screen.plac.dist<-as.dist(screen.plac.distmat)
```	

```{r RelRSPwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("RelRSPwk6 by AUC-RF \\nRelative response defined as a decrease in CDAI from Screening greater than 29.5%.")
set.seed(32016)
levels(nao.screen.trtd$RelRSPwk6) <- c(0,1)
mdotu_relrespWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
mdotu_relrespWK6<-na.omit(mdotu_relrespWK6)
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
otu_relrespWK6 <- mdotu_relrespWK6[, c('RelRSPwk6', colnames(mdotu_relrespWK6)[grep('Otu[0123456789]', colnames(mdotu_relrespWK6))])]
#summary(otu_relrespWK6[,c("RelRSPwk6")])
#summary(otu_relrespWK6[,c("RelRSPwk6")])
otu_relrespWK6<-na.omit(otu_relrespWK6)
#summary(otu_relrespWK6[,c("RelRSPwk6")])
set.seed(32016)
otu_relrespWK6_rf <- AUCRF(RelRSPwk6~., data=otu_relrespWK6, ranking='MDA', ntree=500, pdel=0.2)
##otu_relrespWK6_rf # prints list of optimal OTUs
otu_relrespWK6_probs <- predict(otu_relrespWK6_rf$RFopt, type='prob')[,2]
otu_relrespWK6_roc <- roc(otu_relrespWK6$RelRSPwk6~otu_relrespWK6_probs)


md_relrespWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_relrespWK6[,c("RelRSPwk6")])
md_relrespWK6<-na.omit(md_relrespWK6)
#summary(md_relrespWK6[,c("RelRSPwk6")])
set.seed(32016)
md_relrespWK6_rf <- AUCRF(RelRSPwk6~., data=md_relrespWK6, ranking='MDA', ntree=500, pdel=0.3)
##md_relrespWK6_rf # prints list of optimal features
md_relrespWK6_probs <- predict(md_relrespWK6_rf$RFopt, type='prob')[,2]
md_relrespWK6_roc <- roc(md_relrespWK6$RelRSPwk6~md_relrespWK6_probs)


mdotu_relrespWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
mdotu_relrespWK6<-na.omit(mdotu_relrespWK6)
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
set.seed(32016)
mdotu_relrespWK6_rf <- AUCRF(RelRSPwk6~., data=mdotu_relrespWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_relrespWK6_rf # prints list of optimal features
mdotu_relrespWK6_probs <- predict(mdotu_relrespWK6_rf$RFopt, type='prob')[,2]
mdotu_relrespWK6_roc <- roc(mdotu_relrespWK6$RelRSPwk6~mdotu_relrespWK6_probs)
mdotu_relrespWK6_rf
otu_relrespWK6_rf
md_relrespWK6_rf

mdotuvmdWK6relresprocsP <- roc.test(mdotu_relrespWK6_roc, md_relrespWK6_roc)
otuvmdWK6relresprocsP <- roc.test(otu_relrespWK6_roc, md_relrespWK6_roc)
mdotuvmdWK6relresprocsPval <- signif(mdotuvmdWK6relresprocsP$p.value, digits=2)
otuvmdWK6relresprocsPval <- signif(otuvmdWK6relresprocsP$p.value, digits=2)
```

```{r RelRSPwk6AUCRFinline, include=F, fig.height=5, fig.width=10}
cutoff <- coords(roc=md_relrespWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff

### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_relrespWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_relrespWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_relrespWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Response")
legend('bottomright', c(sprintf('Week 0 OTUs : AUC=%.3f',otu_relrespWK6_roc$auc), sprintf('Week 0 Clinical Data: AUC=%.3f',md_relrespWK6_roc$auc), sprintf('Combined: AUC=%.3f',mdotu_relrespWK6_roc$auc)), lwd=2, col=c('royalblue', 'grey5', 'red'), bty='n', lty=1)

cutoff <- coords(roc=mdotu_relrespWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_relrespWK6_spec <- coords(mdotu_relrespWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_relrespWK6_sens <- coords(mdotu_relrespWK6_roc, x=mdotu_relrespWK6_spec, input='spec', ret='sens')
mdotu_relrespWK6_thr <- coords(mdotu_relrespWK6_roc, x=mdotu_relrespWK6_spec, input='spec', ret='thr')

points(coords(mdotu_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_relrespWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_relrespWK6_spec <- coords(otu_relrespWK6_roc, x=cutoff, input='thr', ret='spec')
otu_relrespWK6_sens <- coords(otu_relrespWK6_roc, x=otu_relrespWK6_spec, input='spec', ret='sens')
otu_relrespWK6_thr <- coords(otu_relrespWK6_roc, x=otu_relrespWK6_spec, input='spec', ret='thr')
points(coords(otu_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_relrespWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_relrespWK6_spec <- coords(md_relrespWK6_roc, x=cutoff, input='thr', ret='spec')
md_relrespWK6_sens <- coords(md_relrespWK6_roc, x=md_relrespWK6_spec, input='spec', ret='sens')
md_relrespWK6_thr <- coords(md_relrespWK6_roc, x=md_relrespWK6_spec, input='spec', ret='thr')

points(coords(md_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)

plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='Response Week 6', xaxt='n', cex.axis=1.2)
title('Response Status Compared to Combined Model')
stripchart(at=1, mdotu_relrespWK6_probs[mdotu_relrespWK6$'RelRSPwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_relrespWK6_probs[mdotu_relrespWK6$'RelRSPwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_relrespWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
abline(h=cutoff, lty=2)
legend("top", lty=2, sprintf('Model Threshold'))

#mdotu_relrespWK6_rf # prints list of optimal features
otu_relrespWK6_rf 
```

```{r REMISSwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("REMISSwk6 by AUC-RF")
set.seed(32016)
levels(nao.screen.trtd$REMISSwk6) <- c(0,1)
mdotu_remWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]

mdotu_remWK6<-na.omit(mdotu_remWK6)

otu_remWK6 <- mdotu_remWK6[, c('REMISSwk6', colnames(mdotu_remWK6)[grep('Otu[0123456789]', colnames(mdotu_remWK6))])]

otu_remWK6<-na.omit(otu_remWK6)

set.seed(32016)
otu_remWK6_rf <- AUCRF(REMISSwk6~., data=otu_remWK6, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK6_rf # prints list of optimal OTUs
otu_remWK6_probs <- predict(otu_remWK6_rf$RFopt, type='prob')[,2]
otu_remWK6_roc <- roc(otu_remWK6$REMISSwk6~otu_remWK6_probs)


md_remWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_remWK6[,c("REMISSwk6")])
md_remWK6<-na.omit(md_remWK6)
#summary(md_remWK6[,c("REMISSwk6")])
set.seed(32016)
md_remWK6_rf <- AUCRF(REMISSwk6~., data=md_remWK6, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK6_rf # prints list of optimal features
md_remWK6_probs <- predict(md_remWK6_rf$RFopt, type='prob')[,2]
md_remWK6_roc <- roc(md_remWK6$REMISSwk6~md_remWK6_probs)


mdotu_remWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('REMISSwk6', "AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", 'invsimpson', colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_remWK6[,c("REMISSwk6")])
mdotu_remWK6<-na.omit(mdotu_remWK6)
#summary(mdotu_remWK6[,c("REMISSwk6")])
set.seed(32016)
mdotu_remWK6_rf <- AUCRF(REMISSwk6~., data=mdotu_remWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK6_rf # prints list of optimal features
mdotu_remWK6_probs <- predict(mdotu_remWK6_rf$RFopt, type='prob')[,2]
mdotu_remWK6_roc <- roc(mdotu_remWK6$REMISSwk6~mdotu_remWK6_probs)




otu_relrespWK6xremWK6rocsP <- roc.test(otu_relrespWK6_roc, otu_remWK6_roc)
otu_relrespWK6xremWK6rocsPval <- signif(otu_relrespWK6xremWK6rocsP$p.value, digits=2)

mdotu_relrespWK6xremWK6rocsP <- roc.test(mdotu_relrespWK6_roc, mdotu_remWK6_roc)
mdotu_relrespWK6xremWK6rocsPval <- signif(mdotu_relrespWK6xremWK6rocsP$p.value, digits=2)

mdotuvmdWK6remrocsP <- roc.test(mdotu_remWK6_roc, md_remWK6_roc)
otuvmdWK6remrocsP <- roc.test(otu_remWK6_roc, md_remWK6_roc)
mdotuvmdWK6remrocsPval <- signif(mdotuvmdWK6remrocsP$p.value, digits=2)
otuvmdWK6remrocsPval <- signif(otuvmdWK6remrocsP$p.value, digits=2)
```

```{r REMISSwk6AUCRFinline, fig.height=5, fig.width=10, include=FALSE}
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_remWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Remission")
legend('bottomright', c(sprintf('Week 0 OTUs : AUC=%.3f', otu_remWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK6_roc$auc), sprintf('Week 0 Clinical Data: AUC=%.3f',md_remWK6_roc$auc)), lwd=2, col=c('royalblue', 'red', "grey5"), bty='n', lty=1)

cutoff <- coords(roc=mdotu_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_remWK6_spec <- coords(mdotu_remWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_remWK6_sens <- coords(mdotu_remWK6_roc, x=mdotu_remWK6_spec, input='spec', ret='sens')
mdotu_remWK6_thr <- coords(mdotu_remWK6_roc, x=mdotu_remWK6_spec, input='spec', ret='thr')

points(coords(mdotu_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_remWK6_spec <- coords(otu_remWK6_roc, x=cutoff, input='thr', ret='spec')
otu_remWK6_sens <- coords(otu_remWK6_roc, x=otu_remWK6_spec, input='spec', ret='sens')
otu_remWK6_thr <- coords(otu_remWK6_roc, x=otu_remWK6_spec, input='spec', ret='thr')
points(coords(otu_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_remWK6_spec <- coords(md_remWK6_roc, x=cutoff, input='thr', ret='spec')
md_remWK6_sens <- coords(md_remWK6_roc, x=md_remWK6_spec, input='spec', ret='sens')
md_remWK6_thr <- coords(md_remWK6_roc, x=md_remWK6_spec, input='spec', ret='thr')

points(coords(md_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)

plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='Remission Week 6', xaxt='n', cex.axis=1.2)
title('Remission Status Compared to Combined Model')
stripchart(at=1, mdotu_remWK6_probs[mdotu_remWK6$REMISSwk6=='0'], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK6_probs[mdotu_remWK6$REMISSwk6=='1'], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
abline(h=cutoff, lty=2)
legend("top", lty=2, sprintf('Model Threshold'))


#mdotu_remWK6_rf # prints list of optimal features
otu_remWK6_rf 


```

```{r RelRSPwk22-AUC-RF, include=F, fig.height=5, fig.width=10}
print("RelRSPwk22 by AUC-RF \\nRelative respWK22onse defined as a decrease in CDAI from Screening greater than 29.5%.")
set.seed(32016)

levels(nao.screen.trtd$RelRSPwk22) <- c(0,1)
nao.screen.trtd.trtd <- nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN == "Treated_Treated", ]
nrow(nao.screen.trtd.trtd)
summary(nao.screen.trtd.trtd$RelRSPwk22)
mdotu_relrespWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('RelRSPwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
mdotu_relrespWK22<-na.omit(mdotu_relrespWK22)
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
otu_relrespWK22 <- mdotu_relrespWK22[, c('RelRSPwk22', colnames(mdotu_relrespWK22)[grep('Otu[0123456789]', colnames(mdotu_relrespWK22))])]
#summary(otu_relrespWK22[,c("RelRSPwk22")])
#summary(otu_relrespWK22[,c("RelRSPwk22")])
otu_relrespWK22<-na.omit(otu_relrespWK22)
#summary(otu_relrespWK22[,c("RelRSPwk22")])
set.seed(32016)
otu_relrespWK22_rf <- AUCRF(RelRSPwk22~., data=otu_relrespWK22, ranking='MDA', ntree=500, pdel=0.2)
##otu_relrespWK22_rf # prints list of optimal OTUs
otu_relrespWK22_probs <- predict(otu_relrespWK22_rf$RFopt, type='prob')[,2]
otu_relrespWK22_roc <- roc(otu_relrespWK22$RelRSPwk22~otu_relrespWK22_probs)


md_relrespWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('RelRSPwk22','AGE', 'SEX', 'RACE', 'BAMINO', 'BANTIBIO', 'BCORT', 'BIMM', 'BL_BMI', 'DDUR', 'DDUR10', 'DDUR15', 'tissinvol', 'BL_FLACT', 'BL_FCALP', 'BL_CRP', 'BOWSTRICT')]
#summary(md_relrespWK22[,c("RelRSPwk22")])
md_relrespWK22<-na.omit(md_relrespWK22)
#summary(md_relrespWK22[,c("RelRSPwk22")])
set.seed(32016)
md_relrespWK22_rf <- AUCRF(RelRSPwk22~., data=md_relrespWK22, ranking='MDA', ntree=500, pdel=0.3)
##md_relrespWK22_rf # prints list of optimal features
md_relrespWK22_probs <- predict(md_relrespWK22_rf$RFopt, type='prob')[,2]
md_relrespWK22_roc <- roc(md_relrespWK22$RelRSPwk22~md_relrespWK22_probs)


mdotu_relrespWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('RelRSPwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
mdotu_relrespWK22<-na.omit(mdotu_relrespWK22)
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
set.seed(32016)
mdotu_relrespWK22_rf <- AUCRF(RelRSPwk22~., data=mdotu_relrespWK22, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_relrespWK22_rf # prints list of optimal features
mdotu_relrespWK22_probs <- predict(mdotu_relrespWK22_rf$RFopt, type='prob')[,2]
mdotu_relrespWK22_roc <- roc(mdotu_relrespWK22$RelRSPwk22~mdotu_relrespWK22_probs)


#pdf#pdf('data/figures/relRSPwk22aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_relrespWK22_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_relrespWK22_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_relrespWK22_roc, add=T, lwd=2, col='red', lty=1)
title("RelRSPwk22")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_relrespWK22_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_relrespWK22_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_relrespWK22_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk22abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_relrespWK22_probs[mdotu_relrespWK22$'RelRSPwk22'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_relrespWK22_probs[mdotu_relrespWK22$'RelRSPwk22'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_relrespWK22_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_relrespWK22_rf # prints list of optimal features
```

```{r REMISSwk22-AUC-RF, include=F, fig.height=5, fig.width=10}
print("REMISSwk22 by AUC-RF")
set.seed(32016)
summary(nao.screen.trtd.trtd$REMISSwk22)
levels(nao.screen.trtd.trtd$REMISSwk22) <- c(0,1)
mdotu_remWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_remWK22[,c("REMISSwk22")])
mdotu_remWK22<-na.omit(mdotu_remWK22)
#summary(mdotu_remWK22[,c("REMISSwk22")])
otu_remWK22 <- mdotu_remWK22[, c('REMISSwk22', colnames(mdotu_remWK22)[grep('Otu[0123456789]', colnames(mdotu_remWK22))])]
#summary(otu_remWK22[,c("REMISSwk22")])
#summary(otu_remWK22[,c("REMISSwk22")])
otu_remWK22<-na.omit(otu_remWK22)
#summary(otu_remWK22[,c("REMISSwk22")])
set.seed(32016)
otu_remWK22_rf <- AUCRF(REMISSwk22~., data=otu_remWK22, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK22_rf # prints list of optimal OTUs
otu_remWK22_probs <- predict(otu_remWK22_rf$RFopt, type='prob')[,2]
otu_remWK22_roc <- roc(otu_remWK22$REMISSwk22~otu_remWK22_probs)


md_remWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_remWK22[,c("REMISSwk22")])
md_remWK22<-na.omit(md_remWK22)
#summary(md_remWK22[,c("REMISSwk22")])
set.seed(32016)
md_remWK22_rf <- AUCRF(REMISSwk22~., data=md_remWK22, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK22_rf # prints list of optimal features
md_remWK22_probs <- predict(md_remWK22_rf$RFopt, type='prob')[,2]
md_remWK22_roc <- roc(md_remWK22$REMISSwk22~md_remWK22_probs)


mdotu_remWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22', 'AGE', 'SEX', 'RACE', 'BAMINO', 'BANTIBIO', 'BCORT', 'BIMM', 'BL_BMI', 'DDUR', 'DDUR10', 'DDUR15', 'tissinvol', 'BL_FLACT', 'BL_FCALP', 'BL_CRP', 'BOWSTRICT', 'invsimpson', colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_remWK22[,c("REMISSwk22")])
mdotu_remWK22<-na.omit(mdotu_remWK22)
#summary(mdotu_remWK22[,c("REMISSwk22")])
set.seed(32016)
mdotu_remWK22_rf <- AUCRF(REMISSwk22~., data=mdotu_remWK22, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK22_rf # prints list of optimal features
mdotu_remWK22_probs <- predict(mdotu_remWK22_rf$RFopt, type='prob')[,2]
mdotu_remWK22_roc <- roc(mdotu_remWK22$REMISSwk22~mdotu_remWK22_probs)



### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK22_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_remWK22_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK22_roc, add=T, lwd=2, col='red', lty=1)
title("REMISSwk22")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_remWK22_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_remWK22_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK22_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)

plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_remWK22_probs[mdotu_remWK22$'REMISSwk22'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK22_probs[mdotu_remWK22$'REMISSwk22'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK22_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff

mdotu_remWK22_rf # prints list of optimal features
```


```{r REMISSwk22-AUC-RF-alltrtd, include=F, fig.height=5, fig.width=10}
print("REMISSwk22 by AUC-RF")
set.seed(32016)
nao.screen_data.alltrtd <- nao.screen_data[nao.screen_data$TRTGRINDMAN!="Placebo_Placebo",]
summary(nao.screen_data.alltrtd$TRTGRINDMAN)
levels(nao.screen_data.alltrtd$REMISSwk22) <- c(0,1)
mdotu_remWK22.alltrtd <- nao.screen_data.alltrtd[nao.screen_data.alltrtd$visit=='Screening', c('REMISSwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen_data.alltrtd)[grep('Otu[0123456789]', colnames(nao.screen_data.alltrtd))])]
#summary(mdotu_remWK22.alltrtd[,c("REMISSwk22")])
mdotu_remWK22.alltrtd<-na.omit(mdotu_remWK22.alltrtd)
#summary(mdotu_remWK22.alltrtd[,c("REMISSwk22")])
otu_remWK22.alltrtd <- mdotu_remWK22.alltrtd[, c('REMISSwk22', colnames(mdotu_remWK22.alltrtd)[grep('Otu[0123456789]', colnames(mdotu_remWK22.alltrtd))])]
#summary(otu_remWK22.alltrtd[,c("REMISSwk22")])
#summary(otu_remWK22.alltrtd[,c("REMISSwk22")])
otu_remWK22.alltrtd<-na.omit(otu_remWK22.alltrtd)
#summary(otu_remWK22.alltrtd[,c("REMISSwk22")])
set.seed(32016)
otu_remWK22.alltrtd_rf <- AUCRF(REMISSwk22~., data=otu_remWK22.alltrtd, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK22.alltrtd_rf # prints list of optimal OTUs
otu_remWK22.alltrtd_probs <- predict(otu_remWK22.alltrtd_rf$RFopt, type='prob')[,2]
otu_remWK22.alltrtd_roc <- roc(otu_remWK22.alltrtd$REMISSwk22~otu_remWK22.alltrtd_probs)


md_remWK22.alltrtd <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_remWK22.alltrtd[,c("REMISSwk22")])
md_remWK22.alltrtd<-na.omit(md_remWK22.alltrtd)
#summary(md_remWK22.alltrtd[,c("REMISSwk22")])
set.seed(32016)
md_remWK22.alltrtd_rf <- AUCRF(REMISSwk22~., data=md_remWK22.alltrtd, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK22.alltrtd_rf # prints list of optimal features
md_remWK22.alltrtd_probs <- predict(md_remWK22.alltrtd_rf$RFopt, type='prob')[,2]
md_remWK22.alltrtd_roc <- roc(md_remWK22.alltrtd$REMISSwk22~md_remWK22.alltrtd_probs)


mdotu_remWK22.alltrtd <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22', 'AGE', 'SEX', 'RACE', 'BAMINO', 'BANTIBIO', 'BCORT', 'BIMM', 'BL_BMI', 'DDUR', 'DDUR10', 'DDUR15', 'tissinvol', 'BL_FLACT', 'BL_FCALP', 'BL_CRP', 'BOWSTRICT', 'invsimpson', colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_remWK22.alltrtd[,c("REMISSwk22")])
mdotu_remWK22.alltrtd<-na.omit(mdotu_remWK22.alltrtd)
#summary(mdotu_remWK22.alltrtd[,c("REMISSwk22")])
set.seed(32016)
mdotu_remWK22.alltrtd_rf <- AUCRF(REMISSwk22~., data=mdotu_remWK22.alltrtd, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK22.alltrtd_rf # prints list of optimal features
mdotu_remWK22.alltrtd_probs <- predict(mdotu_remWK22.alltrtd_rf$RFopt, type='prob')[,2]
mdotu_remWK22.alltrtd_roc <- roc(mdotu_remWK22.alltrtd$REMISSwk22~mdotu_remWK22.alltrtd_probs)



### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK22.alltrtd_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_remWK22.alltrtd_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK22.alltrtd_roc, add=T, lwd=2, col='red', lty=1)
title("REMISSwk22")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_remWK22.alltrtd_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_remWK22.alltrtd_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK22.alltrtd_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)

plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_remWK22.alltrtd_probs[mdotu_remWK22.alltrtd$'REMISSwk22'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK22.alltrtd_probs[mdotu_remWK22.alltrtd$'REMISSwk22'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK22.alltrtd_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff

mdotu_remWK22.alltrtd_rf # prints list of optimal features
```

```{r , include=F}

tax<-read.table("../data/Jan400.simple.taxonomy.txt", header=T, sep='\t')


#Abundance stripchart or most predictive otus
oturf_otus  <- as.character(otu_relrespWK22_rf$Xopt)
no_abunds <- otu_relrespWK22[otu_relrespWK22$RelRSPwk22=='0', oturf_otus]/10000 + 1e-5
no_abunds <-na.omit(no_abunds)
yes_abunds <- otu_relrespWK22[otu_relrespWK22$RelRSPwk22=='1', oturf_otus]/10000 + 1e-5
yes_abunds<- na.omit(yes_abunds)

layout(1)
par(mar=c(4, 9, 0, 1))
plot(1, type="n", ylim=c(0,length(oturf_otus )*2), xlim=c(1e-5,3), log="x", ylab="", xlab="Relative Abundance (%)", xaxt="n", yaxt="n")
index <- 1
for(i in oturf_otus ){
	stripchart(at=index-0.35, jitter(yes_abunds[,i], amount=1e-5), pch=21, bg="royalblue1", method="jitter", jitter=0.2, add=T, cex=1, lwd=0.5)
	stripchart(at=index+0.35, jitter(no_abunds[,i], amount=1e-5), pch=21, bg="orange", method="jitter", jitter=0.2, add=T, cex=1, lwd=0.5)
	segments(mean(yes_abunds[,i]),index-0.7,mean(yes_abunds[,i]),index, lwd=3)
	segments(mean(no_abunds[,i]),index+0.7,mean(no_abunds[,i]),index, lwd=3)
	index <- index + 2
}
oturf_otus<-as.factor(otu_relrespWK22_rf$Xopt)
axis(2, at=seq(1,index-2,2), labels=tax[tax$OTU %in% oturf_otus, 'Label'][as.factor(oturf_otus)], las=1, line=-0.5, tick=F, cex.axis=0.8)
axis(1, at=c(1e-5, 1e-4, 1e-3, 1e-2, 1e-1), label=c("0", "0.01", "0.1", "1", "10"))
legend('topright', legend=c("No", "Yes"), pch=c(21, 21), pt.bg=c("orange","royalblue1"), cex=0.7)
```

```{r week6readdata, include=F}

md<-read.table(file="../data/Jan400.mdshared.wmg.txt", sep="\t", header=T) #wmg=with merged groups
APmd.wk6<-subset(md, select=c(group, USUBJID, Site, visit, TRTGR, TRTGRINDMAN:COUNTRY, BAMINO, BANTIBIO, BCORT, BIMM, BL_BMI:tissinvol, BRSP_TNF, BL_FLACT:BL_FLACTLEV, BOWSTRICT, RESPONSEwk4:REMISSwk22, RSP_Rwk8n22:RSP_SRwk8N22, REMISSwk6, RESPONSEwk6, cdail_wk0, cdail_wk4, cdail_wk6, cdail_wk8, cdail_wk22, CDSTOOLM_Week.6, CDPAINM_Week.6, CDWELBEM_Week.6, CDARTRTM_Week.6, CDARALGM_Week.6, CDIRITM_Week.6, CDUVEITM_Week.6, CDERYTHM_Week.6, CDPYODEM_Week.6, CDSTOMAM_Week.6, CDFISURM_Week.6, CDFISTUM_Week.6, CDABSCEM_Week.6, CDOTFISM_Week.6, CDFEVERM_Week.6, CDDRGM_Week.6, CDAMASSM_Week.6, CDHCTM_Week.6, CDWEIGHM_Week.6))

levels(APmd.wk6$TRTGR)<-c("Placebo", "Treated")
levels(APmd.wk6$TRTGRINDMAN)<-c("Placebo_Not", "Placebo_Placebo", "Placebo_Treated", "Treated_Not", "Treated_Placebo", "Treated_Treated")

APmd.wk6$TRTGR_REMISSwk6 <- as.factor(paste(APmd.wk6$TRTGR, APmd.wk6$REMISSwk6))
APmd.wk6$TRTGR_REMISSwk22 <- as.factor(paste(APmd.wk6$TRTGR, APmd.wk6$REMISSwk22))

APmd.wk6$TRTGR_RelRSPwk6 <- as.factor(paste(APmd.wk6$TRTGR, APmd.wk6$RelRSPwk6))
APmd.wk6$TRTGR_RelRSPwk22 <- as.factor(paste(APmd.wk6$TRTGR, APmd.wk6$RelRSPwk22))

APmd.wk6$CDSTOOLM_Week.6[APmd.wk6$CDSTOOLM_Week.6=="."]<-"NA"
APmd.wk6$CDSTOOLM_Week.6 <-as.integer(APmd.wk6$CDSTOOLM_Week.6)
APmd.wk6$CDHCTM_Week.6[APmd.wk6$CDHCTM_Week.6=="."]<-"NA"
APmd.wk6$CDHCTM_Week.6 <-as.integer(APmd.wk6$CDHCTM_Week.6)
summary(subset(APmd.wk6, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

deltaCDAIwk0_22<-APmd.wk6$cdail_wk22-APmd.wk6$cdail_wk0
deltaCDAIwk0_8<-APmd.wk6$cdail_wk8-APmd.wk6$cdail_wk0
deltaCDAIwk0_4<-APmd.wk6$cdail_wk4-APmd.wk6$cdail_wk0
deltaCDAIwk0_6<-APmd.wk6$cdail_wk6-APmd.wk6$cdail_wk0
percentdelta0_4<-deltaCDAIwk0_4/APmd.wk6$cdail_wk0*100
percentdelta0_6<-deltaCDAIwk0_6/APmd.wk6$cdail_wk0*100
percentdelta0_8<-deltaCDAIwk0_8/APmd.wk6$cdail_wk0*100
percentdelta0_22<-deltaCDAIwk0_22/APmd.wk6$cdail_wk0*100
deltaCDAIwk8_22<-APmd.wk6$cdail_wk22-APmd.wk6$cdail_wk8
percentdelta8_22<-deltaCDAIwk8_22/APmd.wk6$cdail_wk8*100
APmd.wk6$percentdelta0_4<-percentdelta0_4
APmd.wk6$percentdelta0_6<-percentdelta0_6
APmd.wk6$percentdelta0_8<-percentdelta0_8
APmd.wk6$percentdelta0_22<-percentdelta0_22
APmd.wk6$percentdelta8_22<-percentdelta8_22
APmd.wk6$RelRSPwk4<-as.factor(ifelse(APmd.wk6$percentdelta0_4>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk6<-as.factor(ifelse(APmd.wk6$percentdelta0_6>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk8<-as.factor(ifelse(APmd.wk6$percentdelta0_8>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk22<-as.factor(ifelse(APmd.wk6$percentdelta0_22>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk8_22<-as.factor(ifelse(APmd.wk6$percentdelta8_22>=-29.5, print("No"), print("Yes")))
APmd.wk6$visit<-factor(APmd.wk6$visit, levels = c("Screening", "Week 4", "Week 6", "Week 22"))
adiv<-read.table(file="../data/Jan400.all.na.omit.cmd.groups.summary", sep="\t", header=T)
ss3k_shared<-read.table(file="../data/Jan400.all.na.omit.cmd.0.03.subsample.shared", sep="\t", header=T)#Jan400 subsampled to 3K visit=Week.6
APmd.wk6_adiv<-merge(APmd.wk6, adiv, by.x="group", by.y="group")#nonmatch removed
APmd.wk6_shared<-merge(APmd.wk6_adiv, ss3k_shared_U2s, by.x="group", by.y="Group")
summary(APmd.wk6_shared[,1:20])
APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$USUBJID!='C0743T26030100002',]
APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$group!='E4859941-5',]
levs<-APmd.wk6_shared %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs

APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$TRTGRINDMAN!="Placebo_Not",]
APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$TRTGRINDMAN!="Treated_Not",]

allwk6_data<-APmd.wk6_shared[APmd.wk6_shared$visit=="Week 6",]
summary(allwk6_data[,1:20])

trtd.wk6_data<-allwk6_data[allwk6_data$TRTGR=="Treated",]
summary(trtd.wk6_data[,1:20])
nrow(trtd.wk6_data[trtd.wk6_data$visit=="Week 6",])
plac.wk6_data<-allwk6_data[allwk6_data$TRTGR!="Treated",]
summary(plac.wk6_data[,1:20])

wk0_6_data<-APmd.wk6_shared[APmd.wk6_shared$visit!="Week 4",]
wk0_6_data<-wk0_6_data[wk0_6_data$visit!="Week 22",]

summary(subset(wk0_6_data, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson)))

levs<-wk0_6_data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
summary(levs)
subidsover2<-levs[levs$no_rows>=2,]
summary(subidsover2)
subidsover2
zero6subs<-as.character(subidsover2$USUBJID)
zero6subs_data <- wk0_6_data[wk0_6_data$USUBJID %in% zero6subs,]


zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Placebo_Not",]
zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Treated_Not",]
summary(subset(zero6subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

trtd.zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Placebo_Placebo",]
trtd.zero6subs_data<-trtd.zero6subs_data[trtd.zero6subs_data$TRTGRINDMAN!="Placebo_Treated",]
plac.zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Treated_Treated",]
plac.zero6subs_data<-plac.zero6subs_data[plac.zero6subs_data$TRTGRINDMAN!="Treated_Placebo",]
summary(subset(trtd.zero6subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
summary(subset(plac.zero6subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

week6.N.all <- nrow(zero6subs_data[zero6subs_data$visit=='Week 6',])
week6.trtd <- trtd.zero6subs_data[trtd.zero6subs_data$visit=='Week 6',]
week6.N.trtd <- nrow(trtd.zero6subs_data[trtd.zero6subs_data$visit=='Week 6',])
week6.N.trtd.respY <- nrow(week6.trtd[week6.trtd$RelRSPwk6=='Yes',])
week6.N.trtd.respN <- nrow(week6.trtd[week6.trtd$RelRSPwk6=='No',])
week6.plac <- plac.zero6subs_data[plac.zero6subs_data$visit=='Week 6',]
week6.N.plac <- nrow(plac.zero6subs_data[plac.zero6subs_data$visit=='Week 6',])
week6.N.plac.respY <- nrow(week6.plac[week6.plac$RelRSPwk6=='Yes',])
week6.N.plac.respN <- nrow(week6.plac[week6.plac$RelRSPwk6=='No',])
week6.N.trtd.REMY <- nrow(week6.trtd[week6.trtd$REMISSwk6=='Yes',])
week6.N.trtd.REMN <- nrow(week6.trtd[week6.trtd$REMISSwk6=='No',])
week6.N.plac.REMY <- nrow(week6.plac[week6.plac$REMISSwk6=='Yes',])
week6.N.plac.REMN <- nrow(week6.plac[week6.plac$REMISSwk6=='No',])

all.RelRSPwk6<-matrix(c('Week 6 Response (No, Yes)', trtd.RelRSPwk6_sum, plac.RelRSPwk6_sum, all.RelRSPwk6.kw.test.pval, all.RelRSPwk6_AMOVA), ncol = 5)

week6.N.resp.trtd_sum<-toString(c(week6.N.trtd.respN, week6.N.trtd.respY))
week6.N.resp.plac_sum<-toString(c(week6.N.plac.respN, week6.N.plac.respY))

week6.N.REM.trtd_sum<-toString(c(week6.N.trtd.REMN, week6.N.trtd.REMY))
week6.N.REM.plac_sum<-toString(c(week6.N.plac.REMN, week6.N.plac.REMY))

wk6.RelRSPwk6 <- matrix(c("Week 6", "Response Week 6 (No, Yes)", week6.N.resp.trtd_sum, week6.N.resp.plac_sum, week6.N.all), ncol=5)
colnames(wk6.RelRSPwk6) <- c("Stool Week", 'Status Week', 'Treated (n)', "Placebo (n)", "Total (n)")
wk6.REMISSwk6 <- matrix(c("Week 6", "Remission Week 6 (No, Yes)", week6.N.REM.trtd_sum, week6.N.REM.plac_sum, week6.N.all), ncol=5)
colnames(wk6.REMISSwk6) <- c("Stool Week", 'Status Week', 'Treated (n)', "Placebo (n)", "Total (n)")

wk6_wk6 <- rbind(wk6.RelRSPwk6, wk6.REMISSwk6)
wk6_wk6 <- wk6_wk6[,-5]

pdf("tables/wk6_wk6.sum.pdf", height=2, width=6)
grid.table(wk6_wk6)
dev.off()
```

```{r wk6Xwk6_phylum_otu, include=FALSE}
#screen_data <- allwk6_data
#source("../code/FigureDUNNO_wk6Xwk6phyla.R")

#pdf("figures/FigureDUNNO_wk6Xwk6phyla.pdf", height = 10, width = 12)
#plot_grid(phyRabRelRSPwk6_figure2, phyRabREMISSwk6_figure2, labels = c("A", "B"), align = "v", nrow=2)
#dev.off()

#source("../code/REMISSwk6Xwk6.sig.otu.figure.R")
```

```{r RelRSPwk6Xwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("RelRSPwk6 by AUC-RF \\nRelative response defined as a decrease in CDAI from Week.6 greater than 29.5%.")
set.seed(32016)
levels(trtd.wk6_data$RelRSPwk6) <- c(0,1)

mdotu_relrespWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", "invsimpson", colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
mdotu_relrespWK6xWK6<-na.omit(mdotu_relrespWK6xWK6)
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
otu_relrespWK6xWK6 <- mdotu_relrespWK6xWK6[, c('RelRSPwk6', colnames(mdotu_relrespWK6xWK6)[grep('Otu[0123456789]', colnames(mdotu_relrespWK6xWK6))])]
#summary(otu_relrespWK6xWK6[,c("RelRSPwk6")])
#summary(otu_relrespWK6xWK6[,c("RelRSPwk6")])
otu_relrespWK6xWK6<-na.omit(otu_relrespWK6xWK6)
summary(otu_relrespWK6xWK6[,c("RelRSPwk6")])
set.seed(32016)
otu_relrespWK6xWK6_rf <- AUCRF(RelRSPwk6~., data=otu_relrespWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
##otu_relrespWK6xWK6_rf # prints list of optimal OTUs
otu_relrespWK6xWK6_probs <- predict(otu_relrespWK6xWK6_rf$RFopt, type='prob')[,2]
otu_relrespWK6xWK6_roc <- roc(otu_relrespWK6xWK6$RelRSPwk6~otu_relrespWK6xWK6_probs)


md_relrespWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6")]
#summary(md_relrespWK6xWK6[,c("RelRSPwk6")])
md_relrespWK6xWK6<-na.omit(md_relrespWK6xWK6)
#summary(md_relrespWK6xWK6[,c("RelRSPwk6")])
set.seed(32016)
md_relrespWK6xWK6_rf <- AUCRF(RelRSPwk6~., data=md_relrespWK6xWK6, ranking='MDA', ntree=500, pdel=0.3)
##md_relrespWK6xWK6_rf # prints list of optimal features
md_relrespWK6xWK6_probs <- predict(md_relrespWK6xWK6_rf$RFopt, type='prob')[,2]
md_relrespWK6xWK6_roc <- roc(md_relrespWK6xWK6$RelRSPwk6~md_relrespWK6xWK6_probs)


mdotu_relrespWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", "invsimpson", colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
mdotu_relrespWK6xWK6<-na.omit(mdotu_relrespWK6xWK6)
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
set.seed(32016)
mdotu_relrespWK6xWK6_rf <- AUCRF(RelRSPwk6~., data=mdotu_relrespWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_relrespWK6xWK6_rf # prints list of optimal features
mdotu_relrespWK6xWK6_probs <- predict(mdotu_relrespWK6xWK6_rf$RFopt, type='prob')[,2]
mdotu_relrespWK6xWK6_roc <- roc(mdotu_relrespWK6xWK6$RelRSPwk6~mdotu_relrespWK6xWK6_probs)



#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_relrespWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_relrespWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(mdotu_relrespWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Response using Week 0 Clinical Data")
legend('bottomright', sprintf('Clinical Data: AUC=%.3f',md_relrespWK6xWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)
cutoff <- coords(roc=mdotu_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_relrespWK6xWK6_spec <- coords(mdotu_relrespWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_relrespWK6xWK6_sens <- coords(mdotu_relrespWK6xWK6_roc, x=mdotu_relrespWK6xWK6_spec, input='spec', ret='sens')
mdotu_relrespWK6xWK6_thr <- coords(mdotu_relrespWK6xWK6_roc, x=mdotu_relrespWK6xWK6_spec, input='spec', ret='thr')

points(coords(mdotu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_relrespWK6xWK6_spec <- coords(otu_relrespWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
otu_relrespWK6xWK6_sens <- coords(otu_relrespWK6xWK6_roc, x=otu_relrespWK6xWK6_spec, input='spec', ret='sens')
otu_relrespWK6xWK6_thr <- coords(otu_relrespWK6xWK6_roc, x=otu_relrespWK6xWK6_spec, input='spec', ret='thr')
points(coords(otu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_relrespWK6xWK6_spec <- coords(md_relrespWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
md_relrespWK6xWK6_sens <- coords(md_relrespWK6xWK6_roc, x=md_relrespWK6xWK6_spec, input='spec', ret='sens')
md_relrespWK6xWK6_thr <- coords(md_relrespWK6xWK6_roc, x=md_relrespWK6xWK6_spec, input='spec', ret='thr')

points(coords(md_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, md_relrespWK6xWK6_probs[md_relrespWK6xWK6$'RelRSPwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, md_relrespWK6xWK6_probs[md_relrespWK6xWK6$'RelRSPwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=md_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()


md_relrespWK6xWK6_rf # prints list of optimal features


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_relrespWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_relrespWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_relrespWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("RESPONSEwk6")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_relrespWK6xWK6_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_relrespWK6xWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_relrespWK6xWK6_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_relrespWK6xWK6_probs[mdotu_relrespWK6xWK6$'RelRSPwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_relrespWK6xWK6_probs[mdotu_relrespWK6xWK6$'RelRSPwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_relrespWK6xWK6_rf # prints list of optimal features
otu_relrespWK6xWK6_rf
md_relrespWK6xWK6_rf
```


```{r REMISSwk6Xwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("REMISSwk6 by AUC-RF")
set.seed(32016)
levels(trtd.wk6_data$REMISSwk6) <- c(0,1)
mdotu_remWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", "invsimpson", colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
mdotu_remWK6xWK6<-na.omit(mdotu_remWK6xWK6)
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
otu_remWK6xWK6 <- mdotu_remWK6xWK6[, c('REMISSwk6', colnames(mdotu_remWK6xWK6)[grep('Otu[0123456789]', colnames(mdotu_remWK6xWK6))])]
#summary(otu_remWK6xWK6[,c("REMISSwk6")])
#summary(otu_remWK6xWK6[,c("REMISSwk6")])
otu_remWK6xWK6<-na.omit(otu_remWK6xWK6)

#summary(otu_remWK6xWK6[,c("REMISSwk6")])
set.seed(32016)
otu_remWK6xWK6_rf <- AUCRF(REMISSwk6~., data=otu_remWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK6xWK6_rf # prints list of optimal OTUs
otu_remWK6xWK6_probs <- predict(otu_remWK6xWK6_rf$RFopt, type='prob')[,2]
otu_remWK6xWK6_roc <- roc(otu_remWK6xWK6$REMISSwk6~otu_remWK6xWK6_probs)


md_remWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6")]
#summary(md_remWK6xWK6[,c("REMISSwk6")])
md_remWK6xWK6<-na.omit(md_remWK6xWK6)
#summary(md_remWK6xWK6[,c("REMISSwk6")])
set.seed(32016)
md_remWK6xWK6_rf <- AUCRF(REMISSwk6~., data=md_remWK6xWK6, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK6xWK6_rf # prints list of optimal features
md_remWK6xWK6_probs <- predict(md_remWK6xWK6_rf$RFopt, type='prob')[,2]
md_remWK6xWK6_roc <- roc(md_remWK6xWK6$REMISSwk6~md_remWK6xWK6_probs)


mdotu_remWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('REMISSwk6', "AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", 'invsimpson', colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
mdotu_remWK6xWK6<-na.omit(mdotu_remWK6xWK6)
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
set.seed(32016)
mdotu_remWK6xWK6_rf <- AUCRF(REMISSwk6~., data=mdotu_remWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK6xWK6_rf # prints list of optimal features
mdotu_remWK6xWK6_probs <- predict(mdotu_remWK6xWK6_rf$RFopt, type='prob')[,2]
mdotu_remWK6xWK6_roc <- roc(mdotu_remWK6xWK6$REMISSwk6~mdotu_remWK6xWK6_probs)


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_remWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_remWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(mdotu_remWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Remission using Week 6 Clinical Data")
legend('bottomright', sprintf('Clinical Data: AUC=%.3f',md_remWK6xWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)

cutoff <- coords(roc=mdotu_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_remWK6xWK6_spec <- coords(mdotu_remWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_remWK6xWK6_sens <- coords(mdotu_remWK6xWK6_roc, x=mdotu_remWK6xWK6_spec, input='spec', ret='sens')
mdotu_remWK6xWK6_thr <- coords(mdotu_remWK6xWK6_roc, x=mdotu_remWK6xWK6_spec, input='spec', ret='thr')

points(coords(mdotu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_remWK6xWK6_spec <- coords(otu_remWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
otu_remWK6xWK6_sens <- coords(otu_remWK6xWK6_roc, x=otu_remWK6xWK6_spec, input='spec', ret='sens')
otu_remWK6xWK6_thr <- coords(otu_remWK6xWK6_roc, x=otu_remWK6xWK6_spec, input='spec', ret='thr')
points(coords(otu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_remWK6xWK6_spec <- coords(md_remWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
md_remWK6xWK6_sens <- coords(md_remWK6xWK6_roc, x=md_remWK6xWK6_spec, input='spec', ret='sens')
md_remWK6xWK6_thr <- coords(md_remWK6xWK6_roc, x=md_remWK6xWK6_spec, input='spec', ret='thr')

points(coords(md_remWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_remWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, md_remWK6xWK6_probs[md_remWK6xWK6$'REMISSwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, md_remWK6xWK6_probs[md_remWK6xWK6$'REMISSwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=md_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

md_remWK6xWK6_rf # prints list of optimal features


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_remWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Remission using Week 6 Clinical Data")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_remWK6xWK6_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_remWK6xWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK6xWK6_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_remWK6xWK6_probs[mdotu_remWK6xWK6$'REMISSwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK6xWK6_probs[mdotu_remWK6xWK6$'REMISSwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_remWK6xWK6_rf # prints list of optimal features
otu_remWK6xWK6_rf
md_remWK6xWK6_rf

WK6xWK6rocsP <- roc.test(otu_relrespWK6xWK6_roc, otu_remWK6xWK6_roc)
WK6xWK6rocsPval <- signif(WK6xWK6rocsP$p.value, digits=2)
```

```{r week22readdata, include=F}

md<-read.table(file="../data/Jan400.mdshared.wmg.txt", sep="\t", header=T) #wmg=with merged groups
APmd.wk22<-subset(md, select=c(group, USUBJID, Site, visit, TRTGR, TRTGRINDMAN:COUNTRY, BAMINO, BANTIBIO, BCORT, BIMM, BL_BMI:tissinvol, BRSP_TNF, BL_FLACT:BL_FLACTLEV, BOWSTRICT, RESPONSEwk4:REMISSwk22, RSP_Rwk8n22:RSP_SRwk8N22, REMISSwk22, RESPONSEwk22, cdail_wk0, cdail_wk4, cdail_wk22, cdail_wk8, cdail_wk22, CDSTOOLM_Week.22, CDPAINM_Week.22, CDWELBEM_Week.22, CDARTRTM_Week.22, CDARALGM_Week.22, CDIRITM_Week.22, CDUVEITM_Week.22, CDERYTHM_Week.22, CDPYODEM_Week.22, CDSTOMAM_Week.22, CDFISURM_Week.22, CDFISTUM_Week.22, CDABSCEM_Week.22, CDOTFISM_Week.22, CDFEVERM_Week.22, CDDRGM_Week.22, CDAMASSM_Week.22, CDHCTM_Week.22, CDWEIGHM_Week.22))

levels(APmd.wk22$TRTGR)<-c("Placebo", "Treated")
levels(APmd.wk22$TRTGRINDMAN)<-c("Placebo_Not", "Placebo_Placebo", "Placebo_Treated", "Treated_Not", "Treated_Placebo", "Treated_Treated")

APmd.wk22$TRTGR_REMISSwk22 <- as.factor(paste(APmd.wk22$TRTGR, APmd.wk22$REMISSwk22))
APmd.wk22$TRTGR_REMISSwk22 <- as.factor(paste(APmd.wk22$TRTGR, APmd.wk22$REMISSwk22))

APmd.wk22$TRTGR_RelRSPwk22 <- as.factor(paste(APmd.wk22$TRTGR, APmd.wk22$RelRSPwk22))
APmd.wk22$TRTGR_RelRSPwk22 <- as.factor(paste(APmd.wk22$TRTGR, APmd.wk22$RelRSPwk22))

APmd.wk22$CDSTOOLM_Week.22[APmd.wk22$CDSTOOLM_Week.22=="."]<-"NA"
APmd.wk22$CDSTOOLM_Week.22 <-as.integer(APmd.wk22$CDSTOOLM_Week.22)
APmd.wk22$CDHCTM_Week.22[APmd.wk22$CDHCTM_Week.22=="."]<-"NA"
APmd.wk22$CDHCTM_Week.22 <-as.integer(APmd.wk22$CDHCTM_Week.22)
summary(subset(APmd.wk22, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

deltaCDAIwk0_22<-APmd.wk22$cdail_wk22-APmd.wk22$cdail_wk0
deltaCDAIwk0_8<-APmd.wk22$cdail_wk8-APmd.wk22$cdail_wk0
deltaCDAIwk0_4<-APmd.wk22$cdail_wk4-APmd.wk22$cdail_wk0
deltaCDAIwk0_6<-APmd.wk22$cdail_wk22-APmd.wk22$cdail_wk0
percentdelta0_4<-deltaCDAIwk0_4/APmd.wk22$cdail_wk0*100
percentdelta0_6<-deltaCDAIwk0_6/APmd.wk22$cdail_wk0*100
percentdelta0_8<-deltaCDAIwk0_8/APmd.wk22$cdail_wk0*100
percentdelta0_22<-deltaCDAIwk0_22/APmd.wk22$cdail_wk0*100
deltaCDAIwk8_22<-APmd.wk22$cdail_wk22-APmd.wk22$cdail_wk8
percentdelta8_22<-deltaCDAIwk8_22/APmd.wk22$cdail_wk8*100
APmd.wk22$percentdelta0_4<-percentdelta0_4
APmd.wk22$percentdelta0_6<-percentdelta0_6
APmd.wk22$percentdelta0_8<-percentdelta0_8
APmd.wk22$percentdelta0_22<-percentdelta0_22
APmd.wk22$percentdelta8_22<-percentdelta8_22
APmd.wk22$RelRSPwk4<-as.factor(ifelse(APmd.wk22$percentdelta0_4>=-29.5, print("No"), print("Yes")))
APmd.wk22$RelRSPwk22<-as.factor(ifelse(APmd.wk22$percentdelta0_6>=-29.5, print("No"), print("Yes")))
APmd.wk22$RelRSPwk8<-as.factor(ifelse(APmd.wk22$percentdelta0_8>=-29.5, print("No"), print("Yes")))
APmd.wk22$RelRSPwk22<-as.factor(ifelse(APmd.wk22$percentdelta0_22>=-29.5, print("No"), print("Yes")))
APmd.wk22$RelRSPwk8_22<-as.factor(ifelse(APmd.wk22$percentdelta8_22>=-29.5, print("No"), print("Yes")))
APmd.wk22$visit<-factor(APmd.wk22$visit, levels = c("Screening", "Week 4", "Week 6", "Week 22"))
adiv<-read.table(file="../data/Jan400.all.na.omit.cmd.groups.summary", sep="\t", header=T)
ss3k_shared<-read.table(file="../data/Jan400.all.na.omit.cmd.0.03.subsample.shared", sep="\t", header=T)#Jan400 subsampled to 3K visit=Week.22
APmd.wk22_adiv<-merge(APmd.wk22, adiv, by.x="group", by.y="group")#nonmatch removed
APmd.wk22_shared<-merge(APmd.wk22_adiv, ss3k_shared_U2s, by.x="group", by.y="Group")
summary(APmd.wk22_shared[,1:20])
APmd.wk22_shared<-APmd.wk22_shared[APmd.wk22_shared$USUBJID!='C0743T26030100002',]
APmd.wk22_shared<-APmd.wk22_shared[APmd.wk22_shared$group!='E4859941-5',]
levs<-APmd.wk22_shared %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs

APmd.wk22_shared<-APmd.wk22_shared[APmd.wk22_shared$TRTGRINDMAN!="Placebo_Not",]
APmd.wk22_shared<-APmd.wk22_shared[APmd.wk22_shared$TRTGRINDMAN!="Treated_Not",]

allwk22_data<-APmd.wk22_shared[APmd.wk22_shared$visit=="Week 22",]
summary(allwk22_data[,1:20])

trtd.wk22_data<-allwk22_data[allwk22_data$TRTGR=="Treated",]
summary(trtd.wk22_data[,1:20])

plac.wk22_data<-allwk22_data[allwk22_data$TRTGR!="Treated",]
summary(plac.wk22_data[,1:20])

wk0_22_data<-APmd.wk22_shared[APmd.wk22_shared$visit!="Week 4",]
wk0_22_data<-wk0_22_data[wk0_22_data$visit!="Week 6",]

summary(subset(wk0_22_data, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson)))

levs<-wk0_22_data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
summary(levs)
subidsover2<-levs[levs$no_rows>=2,]
summary(subidsover2)
subidsover2
zero22subs<-as.character(subidsover2$USUBJID)
zero22subs_data <- wk0_22_data[wk0_22_data$USUBJID %in% zero22subs,]


zero22subs_data<-zero22subs_data[zero22subs_data$TRTGRINDMAN!="Placebo_Not",]
zero22subs_data<-zero22subs_data[zero22subs_data$TRTGRINDMAN!="Treated_Not",]
zero22subs_data<-zero22subs_data[zero22subs_data$TRTGRINDMAN!="Placebo_Treated",]
zero22subs_data<-zero22subs_data[zero22subs_data$TRTGRINDMAN!="Treated_Placebo",]
summary(subset(zero22subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

trtd.zero22subs_data<-zero22subs_data[zero22subs_data$TRTGRINDMAN=="Treated_Treated",]
plac.zero22subs_data<-zero22subs_data[zero22subs_data$TRTGRINDMAN=="Placebo_Placebo",]
summary(subset(trtd.zero22subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
summary(subset(plac.zero22subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

week22.N.all <- nrow(zero22subs_data[zero22subs_data$visit=='Week 22',])
week22.trtd <- trtd.zero22subs_data[trtd.zero22subs_data$visit=='Week 22',]
week22.N.trtd <- nrow(trtd.zero22subs_data[trtd.zero22subs_data$visit=='Week 22',])
week22.N.trtd.respY <- nrow(week22.trtd[week22.trtd$RelRSPwk22=='Yes',])
week22.N.trtd.respN <- nrow(week22.trtd[week22.trtd$RelRSPwk22=='No',])
week22.plac <- plac.zero22subs_data[plac.zero22subs_data$visit=='Week 22',]
week22.N.plac <- nrow(plac.zero22subs_data[plac.zero22subs_data$visit=='Week 22',])
week22.N.plac.respY <- nrow(week22.plac[week22.plac$RelRSPwk22=='Yes',])
week22.N.plac.respN <- nrow(week22.plac[week22.plac$RelRSPwk22=='No',])
week22.N.trtd.REMY <- nrow(week22.trtd[week22.trtd$REMISSwk22=='Yes',])
week22.N.trtd.REMN <- nrow(week22.trtd[week22.trtd$REMISSwk22=='No',])
week22.N.plac.REMY <- nrow(week22.plac[week22.plac$REMISSwk22=='Yes',])
week22.N.plac.REMN <- nrow(week22.plac[week22.plac$REMISSwk22=='No',])

all.RelRSPwk22<-matrix(c('Week 22 Response (No, Yes)', trtd.RelRSPwk22_sum, plac.RelRSPwk22_sum, all.RelRSPwk22.kw.test.pval, all.RelRSPwk22_AMOVA), ncol = 5)

week22.N.resp.trtd_sum<-toString(c(week22.N.trtd.respN, week22.N.trtd.respY))
week22.N.resp.plac_sum<-toString(c(week22.N.plac.respN, week22.N.plac.respY))

week22.N.REM.trtd_sum<-toString(c(week22.N.trtd.REMN, week22.N.trtd.REMY))
week22.N.REM.plac_sum<-toString(c(week22.N.plac.REMN, week22.N.plac.REMY))

wk22.RelRSPwk22 <- matrix(c("Week 22", "Response Week 22 (No, Yes)", week22.N.resp.trtd_sum, week22.N.resp.plac_sum, week22.N.all), ncol=5)
colnames(wk22.RelRSPwk22) <- c("Stool Week", 'Status Week', 'Treated (n)', "Placebo (n)", "Total (n)")
wk22.REMISSwk22 <- matrix(c("Week 22", "Remission Week 22 (No, Yes)", week22.N.REM.trtd_sum, week22.N.REM.plac_sum, week22.N.all), ncol=5)
colnames(wk22.REMISSwk22) <- c("Stool Week", 'Status Week', 'Treated (n)', "Placebo (n)", "Total (n)")

wk22_wk22 <- rbind(wk22.RelRSPwk22, wk22.REMISSwk22)
wk22_wk22 <- wk22_wk22[,-5]

pdf("tables/wk22_wk22.sum.pdf", height=2, width=6)
grid.table(wk22_wk22)
dev.off()
```

```{r freqOTUs list for table, eval=T, include=F}
#mdotu_respWK4_Xopt<-mdotu_respWK4_rf$Xopt
#mdotu_relrespWK4_Xopt<-mdotu_relrespWK4_rf$Xopt
#mdotu_remWK4_Xopt<-mdotu_remWK4_rf$Xopt
#otu_respWK4_Xopt<-otu_respWK4_rf$Xopt
#otu_relrespWK4_Xopt<-otu_relrespWK4_rf$Xopt
#otu_remWK4_Xopt<-otu_remWK4_rf$Xopt

#mdotu_respWK6_Xopt<-mdotu_respWK6_rf$Xopt
mdotu_relrespWK6_Xopt<-mdotu_relrespWK6_rf$Xopt
mdotu_remWK6_Xopt<-mdotu_remWK6_rf$Xopt
#otu_respWK6_Xopt<-otu_respWK6_rf$Xopt
otu_relrespWK6_Xopt<-otu_relrespWK6_rf$Xopt
otu_remWK6_Xopt<-otu_remWK6_rf$Xopt

#mdotu_respWK8_Xopt<-mdotu_respWK8_rf$Xopt
#mdotu_relrespWK8_Xopt<-mdotu_relrespWK8_rf$Xopt
#mdotu_remWK8_Xopt<-mdotu_remWK8_rf$Xopt
#otu_respWK8_Xopt<-otu_respWK8_rf$Xopt
#otu_relrespWK8_Xopt<-otu_relrespWK8_rf$Xopt
#otu_remWK8_Xopt<-otu_remWK8_rf$Xopt

#mdotu_respWK22_Xopt<-mdotu_respWK22_rf$Xopt
mdotu_relrespWK22_Xopt<-mdotu_relrespWK22_rf$Xopt
mdotu_remWK22_Xopt<-mdotu_remWK22_rf$Xopt
#otu_respWK22_Xopt<-otu_respWK22_rf$Xopt
otu_relrespWK22_Xopt<-otu_relrespWK22_rf$Xopt
otu_remWK22_Xopt<-otu_remWK22_rf$Xopt

rows<-c("AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen_data)[grep('Otu[0123456789]', colnames(nao.screen_data))])

#colnames<-c("mdotu_respWK4_Xopt", "mdotu_relrespWK4_Xopt", "mdotu_remWK4_Xopt", "otu_respWK4_Xopt", "otu_relrespWK4_Xopt", "otu_remWK4_Xopt", "mdotu_respWK6_Xopt", "mdotu_relrespWK6_Xopt", "mdotu_remWK6_Xopt", "otu_respWK6_Xopt", "otu_relrespWK6_Xopt", "otu_remWK6_Xopt", "mdotu_respWK8_Xopt", "mdotu_relrespWK8_Xopt", "mdotu_remWK8_Xopt", "otu_respWK8_Xopt", "otu_relrespWK8_Xopt", "otu_remWK8_Xopt", "mdotu_respWK22_Xopt", "mdotu_relrespWK22_Xopt", "mdotu_remWK22_Xopt", "otu_respWK22_Xopt", "otu_relrespWK22_Xopt", "otu_remWK22_Xopt")

colnames<-c("mdotu_relrespWK6_Xopt", "mdotu_remWK6_Xopt", "otu_relrespWK6_Xopt", "otu_remWK6_Xopt", "mdotu_relrespWK22_Xopt", "mdotu_remWK22_Xopt", "otu_relrespWK22_Xopt", "otu_remWK22_Xopt")

matches<-data.frame(row.names = rows)

for( i in colnames){
	matches[row.names(matches) %in% get(i), i]<- 1
}

matches$Sums<-apply(matches, 1, sum, na.rm=T)

freqVars<-matches[matches$Sums>=4,]
freqVars<-freqVars[order(-freqVars$Sums),]

freqOTUs<-freqVars[rownames(freqVars)[grep('Otu[0123456789]', rownames(freqVars))],]
#freqOTUs

freqOTUs_list<-as.factor(row.names(freqOTUs))
```


```{r, inline vars, include=F}

tbl1$Species.Richness..Alpha.diversity.[tbl1$Clinical.Variable=="CDAI"]

stool.adonis.inline.pval <- signif(as.numeric(na.omit(stool.adonis$aov.tab$`Pr(>F)`)), digits = 3)
BL_CRP.adonis.inline.pval <- signif(as.numeric(na.omit(BL_CRP.adonis$aov.tab$`Pr(>F)`)), digits = 3)

BL_FCALP.adonis.inline.pval <- signif(as.numeric(na.omit(BL_FCALP.adonis$aov.tab$`Pr(>F)`)), digits = 3)
BL_FLACT.adonis.inline.pval <-  signif(as.numeric(na.omit(BL_FLACT.adonis$aov.tab$`Pr(>F)`)), digits = 3)

AGE.adonis.inline.pval <-  signif(as.numeric(na.omit(AGE.adonis$aov.tab$`Pr(>F)`)), digits = 3)

tissinvol.adonis.inline.pval <-  signif(as.numeric(na.omit(tissinvol.adonis$aov.tab$`Pr(>F)`)), digits = 3)

BCORT.adonis.inline.pval <-  signif(as.numeric(na.omit(BCORT.adonis$aov.tab$`Pr(>F)`)), digits = 3)

DDUR.adonis.inline.pval <-  signif(as.numeric(na.omit(DDUR.adonis$aov.tab$`Pr(>F)`)), digits = 3)

wk6remadivpval <- table2_df[2,4]
wk6RSPbdivpval <- table2_df[1,5]
wk6REMbdivpval <- table2_df[2,5]
wk22RSPbdivpval <- table2_df[3,5]
wk22REMbdivpval <- table2_df[4,5]
```


```{r remwk6aucrfCV, cache=T, include=FALSE, eval=F}
set.seed(32016)
otu_remWK6_rf_cv <- AUCRFcv(otu_remWK6_rf)
set.seed(32016)
mdotu_remWK6_rf_cv <- AUCRFcv(mdotu_remWK6_rf)

print("OTU Week 6 rem  cross validation")
otu_remWK6_rf_cv

print("MDOTU Week 6 rem  cross validation")
mdotu_remWK6_rf_cv
```

```{r remWK22aucrfCV, cache=T, include=FALSE, eval=F}
set.seed(32016)
otu_remWK22_rf_cv <- AUCRFcv(otu_remWK22_rf)
set.seed(32016)
mdotu_remWK22_rf_cv <- AUCRFcv(mdotu_remWK22_rf)

print("OTU Week 22 rem cross validation")
otu_remWK22_rf_cv

print("MDOTU Week 22 rem  cross validation")
mdotu_remWK22_rf_cv
```

```{r wk6Xwk6aucrfCV, cache=T, include=FALSE, eval=F}
set.seed(32016)
otu_remWK6xWK6_rf_cv <- AUCRFcv(otu_remWK6xWK6_rf)

print("Week 6 by Week 6 cross validation")
otu_remWK6xWK6_rf_cv
```

```{r}
```

\vspace{35mm}
	
Running title: microbiota of Ustekinumab-treated Crohnâ€™s Disease patients.
	
\vspace{35mm}
Matthew K. Doherty${^2}$, Tao Ding${^2}$${^\alpha}$, Charlie Koumpouras${^2}$, Shannon Telesco${^1}$, Calixte Monast${^1}$, and Patrick D. Schloss${^2}$${^\dagger}$

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

1\. Janssen Pharmaceutical Companies of Johnson ${\&}$ Johnson, Spring House, PA, USA

2\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI, USA

${\alpha}$ Currently at Department of Biology, New York University, New York, NY, USA.





\newpage








## Abstract
The fecal microbiota is a rich source of biomarkers that have previously been shown to be predictive of numerous disease states. Less well studied is whether these biomarkers can be predictive of response to therapy. Here we sought to predict the therapuetic response of Crohnâ€™s disease (CD) patients enrolled in a double-blinded, placebo-controlled, Phase 2b clinical trial to test the efficacy of Ustekinumab (UST). CD is a global health concern characterized by patches of ulceration and inflammation along the gastrointestinal tract and alterations to the microbial community structure. Using stool samples collected over the course of 22 weeks, we characterized the composition of these patients' fecal bacterial communities by sequencing the V4 region of the 16S rRNA gene. We were able to differentiate patients in remission from those with active disease 6 post treatment induction by using RF models trained on the composition of their baseline microbiome and baseline clinical metadata (AUC = `r mdotu_remWK6_roc$auc`, specificity = `r mdotu_remWK6_spec`, sensitivity = `r mdotu_remWK6_sens`). Top predictive OTUs that were ubiquitous among patients included *Faecalibacterium* and *Escherichia/Shigella*. Among patients in remission 6 weeks post induction, the median baseline inverse Simpson index was `r signif(median.REMwk6.invsimp$Yes/median.REMwk6.invsimp$No, digits=2)` times higher than treated patients with active disease at week 6. Their baseline community structures were similarly different. Two OTUs, *Faecalibacterium* and *Bacteroides*, were significantly more abundant at baseline in patients who were in remission 6 weeks post induction. In treated patients we could follow to week twenty-two, the ${\alpha}$-diversity of UST treated clinical responders increased over time, in contrast to nonresponsive patients. The fecal microbiota at baseline was also associated with markers for disease severity, such as Crohnâ€™s Disease Activity Index (CDAI), stool frequency, CRP, fecal lactoferrin, and fecal calprotectin. The observed baseline differences in fecal microbiota and changes due to therapeutic response support using the microbiota as a biomarker for the establishment and maintenance CD remission.

_Importance:_ Finding biomarkers that give clinicians the ability to predict response to CD treatment at diagnosis will increase the likelihood of faster induction and maintenance of remission. The fecal microbiota could be a useful non-invasive biomarker for directing or monitoring the treatment of CD patients. OTUs associated with remission post induction induction, especially *Faecalibacterium*, could be biomarkers for successful UST treatment of TNF-${\alpha}$ refractory CD patients.

**Keywords: Crohn's Disease, IBD, fecal microbiome, microbiota, biologics, prediction, biomarkers, remission, Faecalibacterium, Ustekinumab, Stelara, machine learning, random forest**

\newpage

###Introduction
The microbiome has been correlated with a variety of diseases and has shown promise as a predictive tool for disease outcome for gingivitis [@Huang_gingivitis_2014], cardiovascular disease [@Wang_cvdrisk_2016], *Clostridium difficile* infection [@Schubert_cdiff_2015; @Seekatz_cdiff_2016], and colorectal cancer [@zackular_CRC_2014; @baxter_FIT_2016]. Additionally, the microbiome has been shown to affect the efficacy of various therapies, including vaginal microbicides [@Klatt_microbicide_2017], cardiac drugs [@Haiser_cardiac_2013], and cancer treatements [@Sivan_cancer_2015; @Vetizou_cancer_2015], and therefore could be used to predict therapeutic response. In relation to inflammatory bowel disease (IBD), previous studies have shown that the bacterial gut microbiota correlates with disease severity in new-onset, pediatric Crohnâ€™s disease (CD) patients [@gevers_pedsCD_2014; @wang_pedsCD_2016]. Additionally, recent studies have shown promise for the gut microbiota as it relates to IBD and therapeutic response [@Ananthakrishnan_IBD_2017; @Shaw_response_2016]. It remains to be determined, however, whether the composition of the fecal gut microbiota can predict and monitor response to CD therapy. Considering the involvement of the immune system and previous evidence for involvement of the microbiome, it is likely that response to CD therapy can be predicted using microbiome data.

CD is a global health concern causing large economic and healthcare utilization impacts on society [@ananthakrishnan_epidemiology_2015; @floyd_economicburden_2015]. CD is characterized by patches of ulceration and inflammation along the entire gastrointestinal tract, though mostly the ileum and colon. Currently, individuals with CD are treated based on disease location and risk of complications using escalating immunosuppressive treatment, and/or surgery, with the goal of achieving and sustaining remission [@randall_CDbiologics_2015; @wils_ust_2015]. Faster induction of remission following diagnosis reduces the risk of irreversible intestinal damage and disability [@wils_ust_2015; @colombel_deepremission_2015; @baert_mucosalhealing_2010]. Ideally, clinicians would be able to determine personalized treatment options for CD patients at diagnosis that would result in faster achievement of remission [@Lichtenstein_biomarkers_2010]. Therefore, recent research has been focused on identifying noninvasive, prognostic biomarkers to monitor CD severity and predict therapeutic response [@Chang_biomarkers_2015; @Boon_biomarkers_2015; @Falvey_biomarkers_2015]. 

The precise etiology of CD remains unknown, but host genetics, environmental exposure, and the gut microbiome appear to be involved [@ananthakrishnan_epidemiology_2015; @sartor_IBDpath_2006]. Individuals with CD have reduced microbial diversity in their guts, compared to healthy individuals, with a lower relative abundance of *Firmicutes* and an increased relative abundance of *Enterobacteraciae* and *Bacteroides*, at the phylum level [@wright_CDmicrobiome_2015; @manichanh_diversityCD_2006; @hansen_pedsIBD_2012; @haberman_pedsCD_2014; @gevers_pedsCD_2014]. Additionally, genome-wide association studies of individuals with CD identified several susceptibility loci, including loci involved in the IL-23 signaling pathway, which could impact the gut microbiota composition and function [@randall_CDbiologics_2015; @wright_CDmicrobiome_2015; @Riol-Blanco_IL23microbiome_2010; @Round_IL23microbiome_2009; @Eken_IL23CD_2014; @Shih_IL23Th17_2014]. If the fecal microbiota can be used to monitor disease severity and predict response to specific treatment modalities, then clinicians could use it as a noninvasive tool for prescribing therapies that result in faster remission [@tedjo_CDactivity_2016]. 

The FDA recently approved Ustekinumab (UST), a monoclonal antibody directed against the shared p40 subunit of IL-12 and IL-23, for the treatment of CD [Feagan_UST_2016; @wils_ust_2015; @sandborn_ust_2012; @sandborn_ust_2008; @kopylov_ust_2014, recentUSTpaper]. Given the potential impact of IL-23 on the microbiota [@Riol-Blanco_IL23microbiome_2010; @Round_IL23microbiome_2009; @Eken_IL23CD_2014; @Shih_IL23Th17_2014], we hypothesized that response to UST could be predicted or influenced by differences in patientsâ€™ gut microbiota and that UST treatment may alter the fecal microbiota. The effects of biologic treatment of IBD on the microbiota are not yet well described, but are hypothesized to be indirect, as these drugs act on host factors [@randall_CDbiologics_2015]. We analyzed the fecal microbiomes of patients who participated in a double-blinded, placebo-controlled Phase II clinical trial that demonstrated the safety and efficacy of UST for treating CD [@sandborn_ust_2012]. We tested whether clinical responders had a microbiota that was distinct from non-responders and if the fecal microbiota changed in patients treated with UST using 16S rRNA gene sequence data from these patients' stool samples. We also quantified the association between the fecal microbiota and disease severity. Our study demonstrates that these associations are useful in predicting and monitoring UST treatment outcome and suggest the fecal microbiota may be a broadly useful source of biomarkers for predicting response to treatment.

## Results

**Fecal microbiota based prediction of treatment response**

We characterized the fecal microbiota in a subset of anti-TNF-${\alpha}$ refractory CD patients, with moderate to severe CD, who took part in the double-blinded, CERTIFI clinical trial that demonstrated the efficacy of UST in treating CD [@sandborn_ust_2012].  Demographic and baseline disease characteristics of this subset are summarized in Table 1. Patients were randomly assigned to a treatment group in the induction phase of the study and at week 8 patients were re-randomized into maintenance therapy groups based on their induction response (Figure 1A). In our study response is defined as a decrease in a patient's initial Crohn's Disease Activity Index (CDAI) greater than 30%. The CDAI is the standard instrument for evaluating clinical symptoms and disease activity in CD [@PB_CDAI_2016; @Best_CDAI_1976]. The CDAI weights patient reported stool frequency, abdominal pain, and general well being over a week, in combination with weight change, hematocrit, opiate usage for diarrhea, and the presence of abdominal masses or other complications to determine the disease severity score [@Best_CDAI_1976; @PB_CDAI_2016].  The international definitions for the levels of CD activity include relrespission (CDAI < 150), mild to moderate (CDAI 150<220), moderate to severe (CDAI 220<450), and severe (CDAI < 450) [@PB_CDAI_2016]. Patients provided stool samples at baseline (screening) and at 4, 6, and 22 weeks post induction for analysis using 16S rRNA gene sequencing (Figure 1B).

We hypothesized that the baseline fecal microbiota could predict therapuetic response (CDAI deacrease >30%) 6 weeks post induction. To test this hypothesis, we constructed random-forest (RF) models to classify responders from non-responders 6 weeks post induction based on the relative abundance of fecal microbiota community members at baseline, clinical metadata at baseline, and the combination of microbiota and clinical data [@baxter_FIT_2016; @calle_aucrf_2011]. Clinical data included components of the CDAI, biomarkers for inflammation, and patient metadata described further in the methods section. We ran these models on `r nrow(nao.screen.trtd)` baseline stool samples from patients induced with UST. Clinical data alone resulted in an AUC of `r md_relrespWK6_roc$auc` (specificity = `r md_relrespWK6_spec`, sensitivity = `r md_relrespWK6_sens`) (Figure 2A). Using only microbiota data, the model predicted response with an AUC of `r otu_relrespWK6_roc$auc` (specificity = `r otu_relrespWK6_spec`, sensitivity = `r otu_relrespWK6_sens`). When combining clinical metadata with the microbiome, the model predicted response with an AUC of `r mdotu_relrespWK6_roc$auc` (specificity = `r mdotu_relrespWK6_spec`, sensitivity = `r mdotu_relrespWK6_sens`). These models were not significantly different in their ability to predict response. Optimal predictors were determined based on their mean decrease in accuracy (MDA) in the ability of the model to classify response (Figure 2B).

**Prediction of remission following treatment**

We also investigated whether the baseline fecal microbiota could predict therapuetic remission (CDAI < 150) 6 weeks post induction. To test this hypothesis, we again used RF models to classify patients in remission from those with active CD 6 weeks post induction. Clinical data alone resulted in an AUC of `r md_remWK6_roc$auc` (specificity = `r md_remWK6_spec`, sensitivity = `r md_remWK6_sens`) (Figure 2C). Using only fecal microbiota data the model had an AUC of `r otu_remWK6_roc$auc` (specificity = `r otu_remWK6_spec`, sensitivity = `r otu_remWK6_sens`). Finally, when combining clinical metadata with the microbiota we achieved an AUC of `r mdotu_remWK6_roc$auc` (specificity = `r mdotu_remWK6_spec`, sensitivity = `r mdotu_remWK6_sens`) for remission at week six. Prediction with clinical metadata alone did not perform as well as models using the baseline fecal microbiome (p = `r otuvmdWK6remrocsPval`) or the combined model (p = `r mdotuvmdWK6remrocsPval`). However, there was no significant difference between the baseline fecal microbiota model and the combined model **(p=)**. Also, our baseline fecal microbiota model was significatnly better able to classify remission compared to response (p = `r otu_relrespWK6xremWK6rocsPval`), whereas this was not true for the combined model (p = `r mdotu_relrespWK6xremWK6rocsPval`).

The majority of OTUs identified as optimal predictors in our model for remission were low in abundance across our cohort (Figure 2D). However, two OTUs appeared to be differentially abundant for patients in remission 6 weeks post induction. The relative abundance of *Escherichia/Shigella* (OTU00001) appeared lower in remitters (median = `r round(quantile(trtd.REMwk6.YES.relabund$Otu00001, type = 2, .50), 3)` IQR = `r  round(quantile(trtd.REMwk6.YES.relabund$Otu00001, type = 2, .25), 3)`-`r round(quantile(trtd.REMwk6.YES.relabund$Otu00001, type = 2, .75), 3)`) compared to patients with active CD (median = `r round(quantile(trtd.REMwk6.NO.relabund$Otu00001, type = 2, .50), 3)`, IQR = `r  round(quantile(trtd.REMwk6.NO.relabund$Otu00001, type = 2, .25), 3)`-`r round(quantile(trtd.REMwk6.NO.relabund$Otu00001, type = 2, .75), 3)`).  Also, the relative abundance of *Faecalibacterium* (OTU00007) was not only higher in remitters (median = `r round(quantile(trtd.REMwk6.YES.relabund$Otu00007, type = 2, .50), 3)`, IQR = `r  round(quantile(trtd.REMwk6.YES.relabund$Otu00007, type = 2, .25), 3)`-`r round(quantile(trtd.REMwk6.YES.relabund$Otu00007, type = 2, .75), 3)`) than patients with active CD (median = `r round(quantile(trtd.REMwk6.NO.relabund$Otu00007, type = 2, .50), 3)`, IQR = `r  round(quantile(trtd.REMwk6.NO.relabund$Otu00007, type = 2, .25), 3)`-`r round(quantile(trtd.REMwk6.NO.relabund$Otu00007, type = 2, .75), 3)`), it was present prior to the start of treatment in every patient who was in remission 6 weeks post induction. 

**Comparison of clinical responders and non-responders**

As our random forest models identified OTUs abundant across our cohort that were important in classifying response and remission, we further investigated differences in the baseline microbiota that could serve as potential biomarkers for successful UST treatment. We compared the baseline microbiomes of all `r nrow(APmd_shared[APmd_shared$visit=="Screening",])` patients who provided a baseline sample based on treatment group and treatment outcome 6 weeks post induction. Patients in remission 6 weeks post induction with UST had significantly higher diversity based on the inverse Simpson index than patients with active CD (respective median values = `r REMwk6.YES.invsimp` (IQR = `r  Yes.inv.25quantile`-`r Yes.inv.75quantile`), `r REMwk6.NO.invsimp` (IQR = `r  No.inv.25quantile`-`r No.inv.75quantile`), p = `r wk6remadivpval`). No other treatment or response groups were significantly different. ${\beta}$-diversity was significantly different for response and remission in UST treated patients 6 weeks post induction (response p = `r wk6RSPbdivpval`, remission p = `r all.REMISSwk6.adonis.pval`). No phyla were significantly different by treatment and response (Supplemental Figure 1) and no OTUs were significantly different among patients receiving placebo for induction, regardless of response and remission status. Two OTUs were significantly more abundant in patients in remission 6 weeks post induction compared to patients with active CD; *Bacteroides* (OTU19) (p = `r trtd.REMISSwk6.Otu19p`) and *Faecalibacterium* (OTU7) (p = `r trtd.REMISSwk6.Otu7p`) (Figure 3).  

**Variation in the baseline microbiota is associated with variation in clinical data**

Based on the associations we identified between baseline microbial diversity and response, we hypothesized that there were associations between the microbiota and clinical variables at baseline that could support the use of the microbiota as a non-invasive biomarker for disease activity [@tedjo_CDactivity_2016]. To test this hypothesis, we compared the baseline microbiota with clinical data at baseline for all `r nrow(APmd_shared[APmd_shared$visit=="Screening",])` samples provided at baseline (Supplemental Table 1). We observed small, but significant correlations for lower ${\alpha}$-diversity correlating with higher CDAI (${\rho}$ = `r signif(cor.cdai$estimate, digits = 3)`, p = `r tbl1$Species.Richness..Alpha.diversity.[tbl1$Clinical.Variable=="CDAI"]`), higher frequency of loose stools per week (${\rho}$ = `r signif(cor.stool$estimate, digits = 3)`, p = `r tbl1$Species.Richness..Alpha.diversity.[tbl1$Clinical.Variable=="Loose Stool Frequency (per week)"]`), and longer disease duration (${\rho}$ = `r signif(cor.DDUR$estimate, digits = 3)`, p = `r tbl1$Species.Richness..Alpha.diversity.[tbl1$Clinical.Variable=="Disease Duration (years)"]`). Corticosteroid use was associated with higher  ${\alpha}$-diversity (p = `r tbl1$Species.Richness..Alpha.diversity.[tbl1$Clinical.Variable=="Corticosteroid Use (Y/N)"]`). No significant association was observed between ${\alpha}$-diversity and CRP, fecal calprotectin, or fecal lactoferrin. However, the ${\beta}$-diversity was significantly different based on CRP (p = `r tbl1$Community.Structure..beta.diversity.[3]`), fecal calprotectin (p = `r tbl1$Community.Structure..beta.diversity.[4]`), and fecal lactoferrin (p = `r tbl1$Community.Structure..beta.diversity.[5]`). The ${\beta}$-diversity was also significantly different based on weekly loose stool frequency (p= `r tbl1$Community.Structure..beta.diversity.[2]`), age (p = `r tbl1$Community.Structure..beta.diversity.[8]`), the tissue affected (p = `r tbl1$Community.Structure..beta.diversity.[tbl1$Clinical.Variable=="Tissue Involvement"]`), corticosteroid use (p =`r tbl1$Community.Structure..beta.diversity.[tbl1$Clinical.Variable=="Corticosteroid Use (Y/N)"]`), and disease duration (p = `r tbl1$Community.Structure..beta.diversity.[tbl1$Clinical.Variable=="Disease Duration (years)"]`). No significant differences in the microbiota were observed for BMI, weight, or sex.

**The diversity of the microbiota changes in UST responders**

We tested whether treatment with UST alters the microbiota by performing a Friedman test comparing ${\alpha}$-diversity at each time point within each treatment group based on response 22 weeks post induction. We included `r (nrow(alltp.ust)/4)` patients induced and maintained with UST (`r (nrow(alltp.ust.RESPyes)/4)` responders, `r (nrow(alltp.ust.RESPno)/4)` non-responders) and `r (nrow(alltp.placplac)/4)` patients induced and maintained with placebo (`r (nrow(alltp.placplac.RESPyes)/4)` responders, `r (nrow(alltp.placplac.RESPno)/4)` non-responders), who provided samples at every time point (Figure 1). We saw no significant difference in the inverse Simpson index over time in patients who did not respond 22 weeks post induction, regardless of treatment, and in patients who received placebo (Figure 4). However, the median inverse Simpson index of responders 22 weeks post UST induction significantly changed over time (p = `r signif(fried.trtdrespY$p.value, digits = 1)`) having increased from baseline (median = `r median(alltp.treatedrespY$Week0adiv)`, IQR = `r quantile(alltp.treatedrespY$Week0adiv, 0.25)` - `r quantile(alltp.treatedrespY$Week0adiv, .75)`) to 4 weeks post UST induction(median = `r median(alltp.treatedrespY$Week4adiv)`, IQR = `r quantile(alltp.treatedrespY$Week4adiv, .25)` - `r quantile(alltp.treatedrespY$Week4adiv, .75)`), decreased from 4 to 6 weeks post induction (median = `r median(alltp.treatedrespY$Week6adiv)`, IQR = `r quantile(alltp.treatedrespY$Week6adiv, .25)` - `r quantile(alltp.treatedrespY$Week6adiv, .75)`), and was significantly higher than baseline (p < 0.05) at 22 weeks post induction (median = `r median(alltp.treatedrespY$Week22adiv)`, IQR = 
`r quantile(alltp.treatedrespY$Week22adiv, .25)` - `r quantile(alltp.treatedrespY$Week22adiv, .75)`).

**The microbiota post induction can distinguish between treatment outcomes**

Having demonstrated the microbiome changes in patients who responsd to UST treatment, we hypothesized that the microbiota could be used to monitor response to UST therapy by classifying patients based on disease activity [@tedjo_CDactivity_2016]. We again contrstructed a random forest classification model to distinguish between patients by UST treatment outcome based on their fecal microbiota [@baxter_FIT_2016; @calle_aucrf_2011]. The study design resulted in only `r week22.N.trtd` week twenty-two stool samples from patients induced and maintained with UST, so we focused our analysis on the `r nrow(trtd.wk6_data)` week 6 stool samples from patients induced with UST. We were again better able to distinguish patients in remission from patients with active CD compared to responders from non-responders (p = `r WK6xWK6rocsPval`; Figure 5A). Our model using week 6 stool samples for response 6 weeks post induction could classify patients who responded from non-responders with an AUC of `r otu_relrespWK6xWK6_roc$auc` (sensitivity = `r otu_relrespWK6xWK6_sens`, specificity = `r otu_relrespWK6xWK6_spec`). For remission 6 weeks post UST induction, the model had an AUC of `r otu_remWK6xWK6_roc$auc` (sensitivity = `r otu_remWK6xWK6_sens`, specificity = `r otu_remWK6xWK6_spec`) when classifying patients in remission from patients with active CD. Important classifiers again included *Fecalibacterium* (OTU7) (Figure 5B).

## Discussion

With this study we sought to determine whether the microbiota can be used to identify patients who will respond to UST therapy and to gain a more detailed understanding of if and how UST treatment affects the microbiota. We demonstrated that the microbiota could be useful in predicting remission due to UST therapy, compared to clinical metadata alone, in our unique patient cohort. We also showed found the fecal microbiota to be useful in uncovering associations between the microbiota and aspects of CD severity metrics and treatment outcomes. Finally, we found that the microbiota of treated responders changed over time. These results helped us to gain a better understanding of the interaction between the human gut microbiota and CD pathogenesis in adult patients refractory to anti-TNF-${\alpha}$ therapies with moderate to severe CD.

Porgnositc model generation paragraph

The presented prognostic model is useful for biomarker discovery and hypothesis generation about the biology of CD as it relates to the microbiome. Similar models could be further developed into a clinically useful prognostic tool. *Faecalibacterium* was the most frequently occurring OTU in our models. It is associated with health, comprising up to 5% of the relative abundance in healthy individuals, and has been shown to be rare in CD patients [@wright_CDmicrobiome_2015; @hansen_pedsIBD_2012; @naftali_tissinvol_2016; @sartor_microbesIBD_2016]. Each patient in remission six weeks post UST induction had measurable *Faecalibacterium* present at baseline. This supports the hypothesis that *Faecalibacterium* impacts CD pathogenesis. *Escherichia/Shigella* also occurred frequently in our models. This OTU is associated with inflammation and has been shown to be associated with CD pathogenesis [@sartor_microbesIBD_2016]. Many other taxa observed in our analysis had low abundance or were absent in the majority of patients. However, in many cases these taxa are related and may serve similar ecologic and metabolic roles in the gut environment. We hypothesize that these microbes may have genes that perform similar metabolic functions. These functions could be revealed by performing metagenomics on stool samples in future studies, especially in patients who achieve remission. 

We were better able to classify remission status compared to response status. We hypothesize that this is due to the relative nature of the response criteria compared to the threshold used to determine remission status. We defined response as a decrease in a patient's baseline CDAI of 30% or more, while remission was defined as a CDAI below 150. The original study used a decrease in CDAI of 100 points for their measure of response, but we felt using the relative percent change better represented a meaningful difference in disase activity and patient quality of life [@sandborn_ust_2012]. Additionally, the field appears to be moving away from CDAI and towards more objectively quantifiable biomarkers for inflammation as wells as endoscopic verification of mucosal healing. [@colombel_deepremission_2015]. 

We observed several associations between the microbiota and clinical variables that could impact how CD is monitored and treated in the future. Serum CRP, fecal calprotectin, and fecal lactoferrin are used as biomarkers to measure inflammation and CD severity. We found that the microbial community structure is different among patients based on these markers. This supports the hypothesis that the fecal microbiota could function as a biomarker for measuring disease activity in patients, especially in concert with established inflammatory biomarkers [@tedjo_CDactivity_2016; @boon_fmarkers_2015; @chang_monitoring_2015]. We also found that higher CDAI was associated with lower microbial diversity. This is consistent with other studies on the microbiota in individuals with CD compared to healthy individuals and studies looking at active disease compared to remission [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. However, these differences may have been driven by the CDAI subscore of weekly stool frequency (Supplementary Table 1), which is consistent with previous studies [@vandeputte_stoolcon_2016]. We also observed differences in the microbial community structure based on disease localization, which is consistent with a study by Naftali et al [@naftali_tissinvol_2016]. Our study also found that corticosteroid use impacts the composition of the human fecal microbiota, which is consistent with observations in mouse models [@huang_cort_2015]. As corticosteroid use appears to impact diversity, corticosteroid therapy may be useful when trying to positively impact microbial diversity during biologic therapy and thereby increase the possibility of response to CD therapies. We also observed that longer disease duration is associated with a reduction in fecal microbial diversity. This decreased diversity may be due to the long duration of inflammatory conditions in the gut. This observation and the increased likelihood of remission and mucosal healing in individuals treated with biologics earlier in the course of their disease is an argument for earlier biologic intervention [@monteleone_mongersen_2015; @monteleone_mongersen_2016; @ardizzone_mongersen_2016]. Hypothetically, earlier biologic intervention would occur before chronic inflammation resultled in reduced microbial diveristy. A more diverse microbiota may then promote remission and reduce the likelihood of relapse. However, the cost of biologics for patients is hindrance to early biologic intervention. Using aptamers in place of monoclonal antibodies may alleviate this expense [@orava_short_2013].

We observed that the ${\alpha}$-diversity of clinical responders increased over time, in contrast to nonresponsive patients. This observation could be due to lower inflammation and changes in disease activity corresponding to improved health in patients who responded to UST. We also addressed whether response to therapy can be predicted with the microbiota by developing a random-forest model that used relative microbial abundance data and/or clinical metadata for input. Again, we were better able to predict remission/non-remission than response/non-response. These findings are again consistent with other studies suggesting the microbiota could be useful as a biomarker in detecting remission versus active disease [@tedjo_CDactivity_2016].

The positive and negative associations between the microbiota and CD allow us to hypothesize on ways to alter the microbiota in order to increase the likelihood therapeutic response. Prior to the initiation of therapy, patients could have their fecal microbiome analyzed. The community data could then be used to direct the modification of a patient's microbiome prior to or during treatment with the goal of improving the outcome of UST treatment. Additionally, further research into the microbiota as a prognostic biomarker could eventually allow for the screening of patients with stool samples at diagnosis to better inform other treatment decisions. If the fecal microbiota can be used as a prognostic tool to non-invasively predict response to specific treatment modalities or inform treatment, then more personalized treatment could result in faster achievement of remission, thereby increasing patientsâ€™ quality of life and reducing economic and healthcare impacts. 



\newpage

###Methods

####Study Design and Sample Collection

Janssen Research and Development conducted a placebo-controlled, phase II clinical study of approximately 500 patients to assess the safety and efficacy of UST for treating anti-TNF-${\alpha}$ refractory, moderate to severe CD patients [@sandborn_ust_2012] (Figure 1). Institutional review board approval was acquired at each participating study center and patients provided written informed consten [@sandborn_ust_2012]. Patient data was deidentified for our study. Both patients and clinicians were blinded to their induction and maintenance treatment groups. Participants provided a stool sample prior to the initiation of the study and were then divided into treatment groups. Additional stool samples were provided 4 weeks post induction. At 6 weeks post induction an additional stool sample was collected, patients were scored for their response to UST based on CDAI, and then divided into groups receiving either subcutaneous injection of UST or placebo at weeks 8 and 16 as maintenance therapy. Response was defined as a decrease in a patient's initial CDAI of 30% or more. This value was determined by using the approximate percent change in CDAI from mild-moderate CD (220) to remission (150). Remission is defined as a CDAI below the threshold of 150. Finally, at 22 weeks patients provided an additional stool sample and were then scored using CDAI for their response to therapy. Frozen fecal samples were shipped to the University of Michigan and stored at -80Â°C prior to DNA extraction.

####DNA extraction and 16S rRNA gene sequencing 

Microbial genomic DNA was extracted using the PowerSoil-htp 96 Well Soil DNA Isolation Kit (MoBio Laboratories) and an EPMotion 5075 pipetting system [@baxter_FIT_2016; @zackular_CRC_2014]. The V4 region of the 16S rRNA gene from each sample was amplified and sequenced using the Illumina MiSeq$\text{\texttrademark}$ platform [@chang_monitoring_2015]. Sequences were curated as described previously using the mothur software package (v.1.34.4) [@schloss_mothur_2009; @Kozich_MiSeqSOP_2013]. Briefly, we curated the sequences to reduce sequencing and PCR errors [@schloss_PCRartifacts_2011], aligned the resulting sequences to the SILVA 16S rRNA sequence database [@Quast_silva_2013], and used UCHIME to remove any chimeric sequences [@edgar_uchime_2011]. Sequences were clustered into operational taxonomic units (OTU) at a 97% similarity cutoff using the average neighbor algorithm [@schloss_OTUanalysis_2011]. All sequences were classified using a naive Bayesian classifier trained against the RDP training set (version 14) and OTUs were assigned a classification based on which taxonomy had the majority consensus of sequences within a given OTU [@wang_taxonomy_2007].  

Following sequence curation using the mothur software package [@schloss_mothur_2009], we obtained a median of `r median(md$shared_count)` sequences per sample (IQR = `r quantile(md$shared_count, .25, type = 2)`-`r quantile(md$shared_count, .75, type = 2)`). Parallel sequencing of a mock community had an error rate of 0.017%. To limit effects of uneven sampling, we rarefied the dataset to 3,000 sequences per sample. Samples from patients that completed the clinical trial and had complete clinical metadata were included in our analysis. Of these samples, `r nrow(APmd_shared[APmd_shared$visit=="Screening",])` were provided prior to treatment as well as `r nrow(APmd_shared[APmd_shared$visit=="Week 4",])` provided at week four, `r nrow(APmd_shared[APmd_shared$visit=="Week 6",])` at week six, and `r nrow(APmd_shared[APmd_shared$visit=="Week 22",])` at week twenty-two post-treatment, for a total of `r nrow(APmd_shared)` samples. All fastq files and the MIMARKS spreadsheet with de-identified clinical metadata are available at **SRA**. 

####Gut microbiota biomarker discovery and statistical analysis 

R v.3.3.2 (2016-10-31) and mothur were used to analyze the data [@R]. To assess ${\alpha}$-diversity, the inverse Simpson index was calculated for each sample in the dataset. Spearman correlation tests were performed to compare the inverse Simpson index and continuous clinical data. Wilcoxon rank sum tests were performed for pairwise comparisons and Kruskal-Wallis rank sum tests for comparisons with more than two groups [@sokal_biometrystats_1995; @magurran_measuring_2004]. To measure ${\beta}$-diversity, the distance between samples was calculated using the thetaYC metric, which takes into account the types of bacteria and their abundance to calculate the differences between the communities [@yue_thetaYC_2005]. These distance matrices were assessed for overlap between sets of communities using the non-parametric analysis of molecular variance (AMOVA) and homogeneity of variance (HOMOVA) tests in mothur [@schloss_commstruct_2008], as well as the adonis function in the R package vegan (v.`r as.character(packageVersion("vegan"))`) [@oksanen_vegan_2016]. Change in ${\alpha}$-diversity over time based on week twenty-two response was assessed using a Friedman test on patients who provided a sample at each time point [@friedman_1937]. The Friedman test is a function in the R package stats (v.`r as.character(packageVersion('stats'))`). Multiple comparisons following a Friedman test were performed using the friedmanmc function in the package pgirmess (v.`r as.character(packageVersion("pgirmess"))`) [@pgirmess]. Change in beta-diversity over time by treatment group and response was assessed using the adonis function in vegan stratified by patient. We used the relative abundance of each OTU, inverse Simpson index, age, sex, current medications, BMI, disease duration, disease location, fecal calprotectin, fecal lactoferrin, C-reactive protein, bowel stricture, and CDAI subscores as input into our RF models constructed with the AUCRF R package (v.`r as.character(packageVersion("AUCRF"))`) [@AUCRF], in order to identify phylotypes or clinical variables that distinguish between various treatment and response groups, as well as to predict or determine response outcome [@breiman_rf_2001, @calle_aucrf_2011]. Optimal predictors were determined based on their mean decrease in accuracy (MDA) of the model to classify patients. Differentially abundant OTUs and phyla were selected through comparison of clinical groups using Kruskal-Wallis and Wilcox tests, where appropriate, to identify OTUs/phyla where there was a p-value less than 0.05 following a Benjamini-Hochberg correction for multiple comparisons [@Benjamini_Hochberg_1995].  Other R packages used in our analysis included ggplot2 v.`r as.character(packageVersion("ggplot2"))` [@ggplot2], dplyr v.`r as.character(packageVersion("dplyr"))` [@dplyr], pROC v.`r as.character(packageVersion("pROC"))` [@pROC], knitr v.`r as.character(packageVersion('knitr'))` [@knitr2014; @knitr2015; @knitr2017], gridExtra v.`r as.character(packageVersion("gridExtra"))` [@gridExtra], devtools v.`r as.character(packageVersion("devtools"))` [@devtools], knitcitations v.`r as.character(packageVersion("knitcitations"))` [@knitcitations], scales v.`r as.character(packageVersion("scales"))` [@scales], tidyr v.`r as.character(packageVersion("tidyr"))` [@tidyr], Hmisc v.`r as.character(packageVersion("Hmisc"))` [@Hmisc], and cowplot v.`r as.character(packageVersion("cowplot"))` [@cowplot]. A reproducible version of this analysis and manuscript are available at https://github.com/SchlossLab/Doherty_CDprediction_mBio_2017.

\newpage


## Tables
**Table 1: Summary of clinical metadata of chort at baseline**

```{r cohort.table, fig.height=5, fig.width=12, cache=F}

#source('../code/Table1_baseline_metadata.R')
```
\includegraphics{tables/Table1_baseline_metadata.pdf}

\newpage

**Supplemental Table 1: Diversity differences based on clinical metadata of chort at baseline**

```{r table1}
#source("../code/Supptable1.cohortdiv.R")

#xtable(tbl1)
```
\includegraphics{tables/Supp.table1_cohortdiversity.png}

\newpage

## Figures

**Figure 1: Experimental design as adapted from Sandborne et al 2012.** (A) Diagram of experimental design. Participants were divided into 4 groups of 125 individuals receiving placebo or 1, 3, or 6 mg/kg doses of UST by IV. At week 8, patients were divided into groups receiving either subcutaneous injection of UST or placebo at weeks 8 and 16 as maintenance therapy, based on response at week six. Finally, at 22 weeks patients were scored using CDAI for their response to therapy. (B) Stool sampling, treatment, and response evalution timeline.


```{r exp design}
```

\includegraphics{figures/Figure1_expdesign.pdf}

\newpage



**Figure 2: Prediction of week six disease status in patients treated with UST, using baseline samples** ROCs for (A) response and (C) remission using microbiota data, clinical metadata, and a combined model. Top predictive taxa for the microbiota model based on MDA for (B) response and (D) remission. 



```{r Figure4_wk0Xwk6pred}
#source('../code/Figure4_prediction.R')
```

\includegraphics{figures/Figure4_wk0Xwk6pred.pdf}

\newpage

**Supplemental Figure 1: Phyla from baseline stool samples in patients treated with UST by week six outcome** Compared the relative abundance of each phylum in UST teated patients based on (A) response and (B) remission status using a Wilcoxon rank sum test and to identify phyla where there was a p-value less than 0.05 following a Benjamini-Hochberg correction for multiple comparisons.

```{r SF1_wk6phyla}

```


\includegraphics{figures/SF1_wk6phyla.pdf}

\newpage

**Figure 3: Differential taxa in baseline stool samples from patients treated with UST, based on week six remission status** Compared the relative abundance of each OTU in UST teated patients based on (A) response and (B) remission status using a Wilcoxon rank sum test to identify OTUs where there was a p-value less than 0.05 following a Benjamini-Hochberg correction for multiple comparisons.

```{r Figure5_basesigOTUabund.REMISSwk6}
```

\includegraphics{figures/Figure5_basesigOTUabund.REMISSwk6.pdf}

\newpage

**Figure 4: Change in alpha diversity over time by induction treatment and week twenty-two response status.** The ${\alpha}$-diversity of `r (nrow(alltp.ust)/4)` patients induced and maintained with UST and `r (nrow(alltp.placplac)/4)` patients induced and maintained with placebo was assess at each time point. Friedman test were perfomed within each teatment and responder group. * indicates week twenty-two is significantly different from baseline (p <0.05).

```{r  Figure2_alltp.adivXvisitXindtrtXrelRSPwk22.pdf}
```

\includegraphics{figures/Figure2_alltp.adivXvisitXindtrtXrelRSPwk22.pdf}



\newpage

**Figure 5: Classification of week six response or remission status using week six stool samples from patients treated with UST** (A) ROCs for week six outcome based on the week six microbiome. (B) Top predictive taxa from week six stool for remission status at week six, based on MDA.

```{r Figure3_wk6Xwk6}
#source("../code/Figure3_responseREMwk6Xwk6.R")
```

\includegraphics{figures/Figure3_wk6Xwk6.pdf}

\newpage

# References
