---
title: The fecal microbiome as a tool for monitoring and predicting response outcomes
  in Ustekinumab-treated, anti-TNF-alpha refractory Crohn’s Disease patients.
csl: mbio.csl
fontsize: 11pt
output:
  md_document: default
  html_document: default
  pdf_document:
    keep_tex: yes
    latex_engine: xelatex
header-includes:
- \usepackage{setspace}
- \doublespacing
- \usepackage{lineno}
- \linenumbers
- \renewcommand{\familydefault}{\sfdefault}
- \usepackage{graphicx}
geometry: margin=1.0in
bibliography: My_Library_copy.bib
---

```{r knitr_settings, cache=FALSE, include=FALSE}
source('../code/knitr_settings.R')
knitr::opts_chunk$set("cache" = T)
source('../code/R_packages_setup.R')
```


```{r readdata, include=F}
#Read in all data
source('../code/APmd.setup.R')




screen_RESPwk22<-APmd_RSPwk22[APmd_RSPwk22$visit=="Screening",] #removes any group with NA in RESPONSEwk22

screen_data<-screen_RESPwk22				#na.omit(screen_RESPwk22)
summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

#screen_data<-screen_data[screen_data$USUBJID!='C0743T26030100002',]
#screen_data<-screen_data[screen_data$group!='E4859941-5',]

summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

#screen_data$CDSTOOLM_Screening[screen_data$CDSTOOLM_Screening=="."]<-"NA"
#screen_data$CDSTOOLM_Screening <-as.integer(screen_data$CDSTOOLM_Screening)
#screen_data$CDHCTM_Screening[screen_data$CDHCTM_Screening=="."]<-"NA"
#screen_data$CDHCTM_Screening <-as.integer(screen_data$CDHCTM_Screening)
summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

#levs<-screen_data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
#levs

#dup.group<-screen_data[screen_data$USUBJID=="C0743T26045400006",]
#subset(dup.group, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson))

#Remove "duplicate" sample with worse coverage and nseqs
#screen_data<-screen_data[screen_data$group!='E4859941-5',] #data = cleaned up duplicate removed
#summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

screen_data<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Not",]
screen_data<-screen_data[screen_data$TRTGRINDMAN!="Treated_Not",]
#screen_data<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Not",]

#pdf('data/figures/patientMD.#pdf', height=6, width=8)
summary(screen_data[,c("TRTGR", "TRTGRINDMAN", "visit", "RESPONSEwk6", "RelRSPwk6", "REMISSwk6")])
#dev.off()

screen.trtd<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Placebo",]
screen.trtd<-screen.trtd[screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(screen.trtd[,1:6])
screen.plac<-screen_data[screen_data$TRTGRINDMAN!="Treated_Placebo",]
screen.plac<-screen.plac[screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(screen.plac[,1:6])

nao.screen_data<-na.omit(screen_data)

nao.screen.trtd<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Placebo_Placebo",]
nao.screen.trtd<-nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(nao.screen.trtd[,1:6])
nao.screen.plac<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Treated_Placebo",]
nao.screen.plac<-nao.screen.plac[nao.screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(nao.screen.plac[,1:6])

#axes <- read.table('../data/Jan400.nao.screen_trtd.3107.subsample.thetayc.0.03.lt.ave.nmds.by3.axes', header=T)
tax<- read.table('../data/Jan400.simple.taxonomy.txt', header=T, sep="\t")
nao.screen.all.distmat<-read.table("../data/Jan400.screening.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.all.dist<-as.dist(nao.screen.all.distmat)
nao.screen.trtd.distmat<-read.table("../data/Jan400.screening.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.trtd.dist<-as.dist(nao.screen.trtd.distmat)
nao.screen.plac.distmat<-read.table("../data/Jan400.screening.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.plac.dist<-as.dist(nao.screen.plac.distmat)

screen.all.distmat<-read.table("../data/Jan400.screen.all.thetayc.0.03.square.ave.dist")
screen.all.dist<-as.dist(screen.all.distmat)
screen.trtd.distmat<-read.table("../data/Jan400.screen.ust.thetayc.0.03.square.ave.dist")
screen.trtd.dist<-as.dist(screen.trtd.distmat)
screen.plac.distmat<-read.table("../data/Jan400.screen.plac.thetayc.0.03.square.ave.dist")
screen.plac.dist<-as.dist(screen.plac.distmat)

trtd_trtd.data.distmat<-read.table("../data/Jan400.trtd_trtd.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
trtd_trtd.data.dist<-as.dist(trtd_trtd.data.distmat)
trtd_plac.data.distmat<-read.table("../data/Jan400.trtd_plac.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
trtd_plac.data.dist<-as.dist(trtd_plac.data.distmat)
plac_plac.data.distmat<-read.table("../data/Jan400.plac_plac.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
plac_plac.data.dist<-as.dist(plac_plac.data.distmat)
plac_trtd.data.distmat<-read.table("../data/Jan400.plac_trtd.accnos.na.omit.cmd.thetayc.0.03.square.ave.dist")
plac_trtd.data.dist<-as.dist(plac_trtd.data.distmat)
```	

```{r sum, include=F}
summary(nao.screen_data[,c("TRTGR")])
summary(screen_data[,c("TRTGR")])
```

```{r phylmum_otu}
source("../code/phylum.figures.R")
source("../code/REMISSwk6.sig.otu.figure.R")
```

```{r cdai.cor.test,  include=F}
set.seed(32016)
cdai.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, cdail_wk0)))
n.cdai.cortest<-nrow(cdai.cor.test.data)
n.cdai.cortest
cor.cdai<-cor.test(cdai.cor.test.data$cdail_wk0, cdai.cor.test.data$invsimpson, method = "spearman")
cor.cdai
cor.cdai.pval<-signif(cor.cdai$p.value, digits = 1)
cor.cdai.pval
cor.cdai.rho<-signif(cor.cdai$estimate, digits = 1)
cor.cdai.rho


```

```{r cdaiwk0betadiv, include=F}
set.seed(32016)
cdaiwk0.adonis<-adonis(nao.screen.all.dist ~ cdail_wk0, nao.screen_data)
cdaiwk0.adonis
cdaiwk0.adonis.pval<-signif(as.numeric(na.omit(cdaiwk0.adonis$aov.tab$`Pr(>F)`)), digits = 1)

CDAI_min<-toString(c('Min=', min(nao.screen_data$cdail_wk0)))
CDAI_min<-gsub(",", "", CDAI_min)
CDAI_min<-gsub(" ", "", CDAI_min)
CDAI_med<-toString(c("Median=", median(nao.screen_data$cdail_wk0)))
CDAI_med<-gsub(",", "", CDAI_med)
CDAI_med<-gsub(" ", "", CDAI_med)
CDAI_max<-toString(c("Max=", max(nao.screen_data$cdail_wk0)))
CDAI_max<-gsub(",", "", CDAI_max)
CDAI_max<-gsub(" ", "", CDAI_max)
CDAI_sum<-toString(c(CDAI_min, CDAI_med, CDAI_max))
CDAI_n<-n.cdai.cortest
CDAI_Spear_P<- toString(c('(', cor.cdai.pval, ')'))
CDAI_Spear_P<-gsub(",", "", CDAI_Spear_P)
CDAI_Spear_P<-gsub(" ", "", CDAI_Spear_P)
CDAI_Spear_rho<-toString(c('rho =', cor.cdai.rho))
CDAI_Spear_rho<-gsub(",", "", CDAI_Spear_rho)
#CDAI_Spear_rho<-gsub(" ", "", CDAI_Spear_rho)
CDAI_Spear_Prho<-c(CDAI_Spear_rho, CDAI_Spear_P)
CDAI_Spear<-toString(c(CDAI_Spear_Prho))
CDAI_Spear<-gsub(",", "", CDAI_Spear)
#CDAI_AMOVA_p<-toString(c(cdaiwk0.adonis.pval))
#CDAI_AMOVA_p<-gsub(",", "", CDAI_AMOVA_p)
#CDAI_AMOVA_p<-gsub(" ", "", CDAI_AMOVA_p)
#CDAI_AMOVA<-toString(c(CDAI_AMOVA_p))
#CDAI_AMOVA<-gsub(",", "", CDAI_AMOVA)
CDAI<- matrix(c('CDAI', CDAI_Spear_rho, cor.cdai$p.value, as.numeric(na.omit(cdaiwk0.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r stoolfreq adiv, include=F}
set.seed(32016)
stool.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, CDSTOOLM_Screening)))
n.stool.cortest<-nrow(stool.cor.test.data)
n.stool.cortest
cor.stool<-cor.test(stool.cor.test.data$CDSTOOLM_Screening, stool.cor.test.data$invsimpson, method = "spearman")
cor.stool
cor.stool.pval<-signif(cor.stool$p.value, digits = 1)
cor.stool.pval
cor.stool.rho<-signif(cor.stool$estimate, digits = 1)
cor.stool.rho

```

```{r stoolfreq bdiv, include=F}
set.seed(32016)
stool.adonis<-adonis(nao.screen.all.dist ~ CDSTOOLM_Screening, nao.screen_data)
stool.adonis
stool.adonis.pval<-signif(as.numeric(na.omit(stool.adonis$aov.tab$`Pr(>F)`)), digits = 1)

Stool_min<-toString(c('Min=', min(nao.screen_data$CDSTOOLM_Screening)))
Stool_min<-gsub(",", "", Stool_min)
Stool_min<-gsub(" ", "", Stool_min)
Stool_med<-toString(c("Median=", median(nao.screen_data$CDSTOOLM_Screening)))
Stool_med<-gsub(",", "", Stool_med)
Stool_med<-gsub(" ", "", Stool_med)
Stool_max<-toString(c("Max=", max(nao.screen_data$CDSTOOLM_Screening)))
Stool_max<-gsub(",", "", Stool_max)
Stool_max<-gsub(" ", "", Stool_max)
Stool_sum<-toString(c(Stool_min, Stool_med, Stool_max))
Stool_n<-n.stool.cortest
Stool_Spear_P<- toString(c("(", cor.stool.pval, ")"))
Stool_Spear_P<-gsub(",", "", Stool_Spear_P)
Stool_Spear_P<-gsub(" ", "", Stool_Spear_P)
Stool_Spear_rho<-toString(c('rho =', cor.stool.rho))
Stool_Spear_rho<-gsub(",", "", Stool_Spear_rho)
#Stool_Spear_rho<-gsub(" ", "", Stool_Spear_rho)
Stool_Spear_Prho<-c(Stool_Spear_rho, Stool_Spear_P)
Stool_Spear<-toString(c(Stool_Spear_Prho))
Stool_Spear<-gsub(",", "", Stool_Spear)

Stool<- matrix(c('Loose Stool Frequency (per week)', Stool_Spear_rho, cor.stool$p.value, as.numeric(na.omit(stool.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```


```{r CRP.cor.test, include=F}
set.seed(32016)
CRP.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BL_CRP)))
n.CRP.cortest<-nrow(CRP.cor.test.data)
n.CRP.cortest
cor.CRP<-cor.test(CRP.cor.test.data$BL_CRP, CRP.cor.test.data$invsimpson, method = "spearman")
cor.CRP
cor.CRP.pval<-signif(cor.CRP$p.value, digits = 1)
cor.CRP.pval
cor.CRP.rho<-signif(cor.CRP$estimate, digits = 1)
cor.CRP.rho

```

```{r BL_CRP.adonis,  include=F}
set.seed(32016)
BL_CRP.adonis<-adonis(nao.screen.all.dist ~ BL_CRP, nao.screen_data)
BL_CRP.adonis
BL_CRP.adonis.pval<-signif(as.numeric(na.omit(BL_CRP.adonis$aov.tab$`Pr(>F)`)), digits = 1)

CRP_min<-toString(c('Min=', min(nao.screen_data$BL_CRP)))
CRP_min<-gsub(",", "", CRP_min)
CRP_min<-gsub(" ", "", CRP_min)
CRP_med<-toString(c("Median=", median(nao.screen_data$BL_CRP)))
CRP_med<-gsub(",", "", CRP_med)
CRP_med<-gsub(" ", "", CRP_med)
CRP_max<-toString(c("Max=", max(nao.screen_data$BL_CRP)))
CRP_max<-gsub(",", "", CRP_max)
CRP_max<-gsub(" ", "", CRP_max)
CRP_sum<-toString(c(CRP_min, CRP_med, CRP_max))
CRP_n<-n.CRP.cortest
CRP_Spear_P<- toString(c('(', cor.CRP.pval, ")"))
CRP_Spear_P<-gsub(",", "", CRP_Spear_P)
CRP_Spear_P<-gsub(" ", "", CRP_Spear_P)
CRP_Spear_rho<-toString(c('rho =', cor.CRP.rho))
CRP_Spear_rho<-gsub(",", "", CRP_Spear_rho)
#CRP_Spear_rho<-gsub(" ", "", CRP_Spear_rho)
CRP_Spear_Prho<-c(CRP_Spear_rho, CRP_Spear_P)
CRP_Spear<-toString(c(CRP_Spear_Prho))
CRP_Spear<-gsub(",", "", CRP_Spear)

CRP<- matrix(c('C-Reactive Protein (mg/L serum)', CRP_Spear_rho, cor.CRP$p.value, as.numeric(na.omit(BL_CRP.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r FCALP.cor.test, include=F}
set.seed(32016)
FCALP.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BL_FCALP)))
n.FCALP.cortest<-nrow(FCALP.cor.test.data)
n.FCALP.cortest
cor.FCALP<-cor.test(FCALP.cor.test.data$BL_FCALP, FCALP.cor.test.data$invsimpson, method = "spearman")
cor.FCALP
cor.FCALP.pval<-signif(cor.FCALP$p.value, digits = 1)
cor.FCALP.pval
cor.FCALP.rho<-signif(cor.FCALP$estimate, digits = 1)
cor.FCALP.rho

```

```{r BL_FCALP.adonis,  include=F}
set.seed(32016)
BL_FCALP.adonis<-adonis(nao.screen.all.dist ~ BL_FCALP, nao.screen_data)
BL_FCALP.adonis
BL_FCALP.adonis.pval<-signif(as.numeric(na.omit(BL_FCALP.adonis$aov.tab$`Pr(>F)`)), digits = 1)

FCALP_min<-toString(c('Min=', min(nao.screen_data$BL_FCALP)))
FCALP_min<-gsub(",", "", FCALP_min)
FCALP_min<-gsub(" ", "", FCALP_min)
FCALP_med<-toString(c("Median=", median(nao.screen_data$BL_FCALP)))
FCALP_med<-gsub(",", "", FCALP_med)
FCALP_med<-gsub(" ", "", FCALP_med)
FCALP_max<-toString(c("Max=", max(nao.screen_data$BL_FCALP)))
FCALP_max<-gsub(",", "", FCALP_max)
FCALP_max<-gsub(" ", "", FCALP_max)
FCALP_sum<-toString(c(FCALP_min, FCALP_med, FCALP_max))
FCALP_n<-n.FCALP.cortest
FCALP_Spear_P<- toString(c('(', cor.FCALP.pval,")"))
FCALP_Spear_P<-gsub(",", "", FCALP_Spear_P)
FCALP_Spear_P<-gsub(" ", "", FCALP_Spear_P)
FCALP_Spear_rho<-toString(c('rho =', cor.FCALP.rho))
FCALP_Spear_rho<-gsub(",", "", FCALP_Spear_rho)
#FCALP_Spear_rho<-gsub(" ", "", FCALP_Spear_rho)
FCALP_Spear_Prho<-c(FCALP_Spear_rho, FCALP_Spear_P)
FCALP_Spear<-toString(c(FCALP_Spear_Prho))
FCALP_Spear<-gsub(",", "", FCALP_Spear)
FCALP_AMOVA_p<-toString(c(BL_FCALP.adonis.pval))
FCALP_AMOVA_p<-gsub(",", "", FCALP_AMOVA_p)
FCALP_AMOVA_p<-gsub(" ", "", FCALP_AMOVA_p)
FCALP_AMOVA<-toString(c(FCALP_AMOVA_p))
FCALP_AMOVA<-gsub(",", "", FCALP_AMOVA)
FCALP<- matrix(c('Fecal Calprotectin (µg/g)', FCALP_Spear_rho, cor.FCALP$p.value, as.numeric(na.omit(BL_FCALP.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r FLACT.cor.test, include=F}
set.seed(32016)
FLACT.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BL_FLACT)))
n.FLACT.cortest<-nrow(FLACT.cor.test.data)
n.FLACT.cortest
cor.FLACT<-cor.test(FLACT.cor.test.data$BL_FLACT, FLACT.cor.test.data$invsimpson, method = "spearman")
cor.FLACT
cor.FLACT.pval<-signif(cor.FLACT$p.value, digits = 1)
cor.FLACT.pval
cor.FLACT.rho<-signif(cor.FLACT$estimate, digits = 1)
cor.FLACT.rho

```

```{r BL_FLACT.adonis,  include=F}
set.seed(32016)
BL_FLACT.adonis<-adonis(nao.screen.all.dist ~ BL_FLACT, nao.screen_data)
BL_FLACT.adonis
BL_FLACT.adonis.pval<-signif(as.numeric(na.omit(BL_FLACT.adonis$aov.tab$`Pr(>F)`)), digits = 1)

FLACT_min<-toString(c('Min=', min(nao.screen_data$BL_FLACT)))
FLACT_min<-gsub(",", "", FLACT_min)
FLACT_min<-gsub(" ", "", FLACT_min)
FLACT_med<-toString(c("Median=", median(nao.screen_data$BL_FLACT)))
FLACT_med<-gsub(",", "", FLACT_med)
FLACT_med<-gsub(" ", "", FLACT_med)
FLACT_max<-toString(c("Max=", max(nao.screen_data$BL_FLACT)))
FLACT_max<-gsub(",", "", FLACT_max)
FLACT_max<-gsub(" ", "", FLACT_max)
FLACT_sum<-toString(c(FLACT_min, FLACT_med, FLACT_max))
FLACT_n<-n.FLACT.cortest
FLACT_Spear_P<- toString(c('(', cor.FLACT.pval,")"))
FLACT_Spear_P<-gsub(",", "", FLACT_Spear_P)
FLACT_Spear_P<-gsub(" ", "", FLACT_Spear_P)
FLACT_Spear_rho<-toString(c('rho =', cor.FLACT.rho))
FLACT_Spear_rho<-gsub(",", "", FLACT_Spear_rho)
#FLACT_Spear_rho<-gsub(" ", "", FLACT_Spear_rho)
FLACT_Spear_Prho<-c(FLACT_Spear_rho, FLACT_Spear_P)
FLACT_Spear<-toString(c(FLACT_Spear_Prho))
FLACT_Spear<-gsub(",", "", FLACT_Spear)
FLACT_AMOVA_p<-toString(c(BL_FLACT.adonis.pval))
FLACT_AMOVA_p<-gsub(",", "", FLACT_AMOVA_p)
FLACT_AMOVA_p<-gsub(" ", "", FLACT_AMOVA_p)
FLACT_AMOVA<-toString(c(FLACT_AMOVA_p))
FLACT_AMOVA<-gsub(",", "", FLACT_AMOVA)
FLACT<- matrix(c('Fecal Lactoferrin (µg/g)', FLACT_Spear_rho, cor.FLACT$p.value, as.numeric(na.omit(BL_FLACT.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r BMI.cortest, include=F}
set.seed(32016)
BMI.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, invsimpson, BL_BMI)))
n.BMI.cortest<-nrow(BMI.cor.test.data)
n.BMI.cortest
cor.BMI<-cor.test(BMI.cor.test.data$BL_BMI, BMI.cor.test.data$invsimpson, method = "spearman")
cor.BMI
cor.BMI.pval<-signif(cor.BMI$p.value, digits = 1)
cor.BMI.pval
cor.BMI.rho<-signif(cor.BMI$estimate, digits = 1)
cor.BMI.rho

```

```{r BL_BMI.adonis,  include=F}
set.seed(32016)
BL_BMI.adonis<-adonis(nao.screen.all.dist ~ BL_BMI, nao.screen_data)
BL_BMI.adonis
BL_BMI.adonis.pval<-signif(as.numeric(na.omit(BL_BMI.adonis$aov.tab$`Pr(>F)`)), digits = 1)

BMI_min<-toString(c('Min=', min(nao.screen_data$BL_BMI)))
BMI_min<-gsub(",", "", BMI_min)
BMI_min<-gsub(" ", "", BMI_min)
BMI_med<-toString(c("Median=", median(nao.screen_data$BL_BMI)))
BMI_med<-gsub(",", "", BMI_med)
BMI_med<-gsub(" ", "", BMI_med)
BMI_max<-toString(c("Max=", max(nao.screen_data$BL_BMI)))
BMI_max<-gsub(",", "", BMI_max)
BMI_max<-gsub(" ", "", BMI_max)
BMI_sum<-toString(c(BMI_min, BMI_med, BMI_max))
BMI_n<-n.BMI.cortest
BMI_Spear_P<- toString(c('(', cor.BMI.pval, ")"))
BMI_Spear_P<-gsub(",", "", BMI_Spear_P)
BMI_Spear_P<-gsub(" ", "", BMI_Spear_P)
BMI_Spear_rho<-toString(c('rho =', cor.BMI.rho))
BMI_Spear_rho<-gsub(",", "", BMI_Spear_rho)
#BMI_Spear_rho<-gsub(" ", "", BMI_Spear_rho)
BMI_Spear_Prho<-c(BMI_Spear_rho, BMI_Spear_P)
BMI_Spear<-toString(c(BMI_Spear_Prho))
BMI_Spear<-gsub(",", "", BMI_Spear)
BMI_AMOVA_p<-toString(c(BL_BMI.adonis.pval))
BMI_AMOVA_p<-gsub(",", "", BMI_AMOVA_p)
BMI_AMOVA_p<-gsub(" ", "", BMI_AMOVA_p)
BMI_AMOVA<-toString(c(BMI_AMOVA_p))
BMI_AMOVA<-gsub(",", "", BMI_AMOVA)
BMI<- matrix(c('BMI', BMI_Spear_rho, cor.BMI$p.value, as.numeric(na.omit(BL_BMI.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r weightcor, include=F}
set.seed(32016)
weight.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, CDWEIGHM_Screening)))
n.weight.cortest<-nrow(weight.cor.test.data)
n.weight.cortest
cor.weight<-cor.test(weight.cor.test.data$CDWEIGHM_Screening, weight.cor.test.data$invsimpson, method = "spearman")
cor.weight
cor.weight.pval<-signif(cor.weight$p.value, digits = 1)
cor.weight.pval
cor.weight.rho<-signif(cor.weight$estimate, digits = 1)
cor.weight.rho

```

```{r CDWEIGHM_Screening.adonis,  include=F}
set.seed(32016)
CDWEIGHM_Screening.adonis<-adonis(nao.screen.all.dist ~ CDWEIGHM_Screening, nao.screen_data)
CDWEIGHM_Screening.adonis
CDWEIGHM_Screening.adonis.pval<-signif(as.numeric(na.omit(CDWEIGHM_Screening.adonis$aov.tab$`Pr(>F)`)), digits = 1)

weight_min<-toString(c('Min=', min(nao.screen_data$CDWEIGHM_Screening)))
weight_min<-gsub(",", "", weight_min)
weight_min<-gsub(" ", "", weight_min)
weight_med<-toString(c("Median=", median(nao.screen_data$CDWEIGHM_Screening)))
weight_med<-gsub(",", "", weight_med)
weight_med<-gsub(" ", "", weight_med)
weight_max<-toString(c("Max=", max(nao.screen_data$CDWEIGHM_Screening)))
weight_max<-gsub(",", "", weight_max)
weight_max<-gsub(" ", "", weight_max)
weight_sum<-toString(c(weight_min, weight_med, weight_max))
weight_n<-n.weight.cortest
weight_Spear_P<- toString(c('(', cor.weight.pval, ")"))
weight_Spear_P<-gsub(",", "", weight_Spear_P)
weight_Spear_P<-gsub(" ", "", weight_Spear_P)
weight_Spear_rho<-toString(c('rho =', cor.weight.rho))
weight_Spear_rho<-gsub(",", "", weight_Spear_rho)
#weight_Spear_rho<-gsub(" ", "", weight_Spear_rho)
weight_Spear_Prho<-c(weight_Spear_rho, weight_Spear_P)
weight_Spear<-toString(c(weight_Spear_Prho))
weight_Spear<-gsub(",", "", weight_Spear)
weight_AMOVA_p<-toString(c(CDWEIGHM_Screening.adonis.pval))
weight_AMOVA_p<-gsub(",", "", weight_AMOVA_p)
weight_AMOVA_p<-gsub(" ", "", weight_AMOVA_p)
weight_AMOVA<-toString(c(weight_AMOVA_p))
weight_AMOVA<-gsub(",", "", weight_AMOVA)
weight<- matrix(c('Weight (kg)', weight_Spear_rho, cor.weight$p.value, as.numeric(na.omit(CDWEIGHM_Screening.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r AGE.cor.test, include=F}
set.seed(32016)
AGE.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, invsimpson, AGE)))
n.AGE.cortest<-nrow(AGE.cor.test.data)
n.AGE.cortest
cor.AGE<-cor.test(AGE.cor.test.data$AGE, AGE.cor.test.data$invsimpson, method = "spearman")
cor.AGE
cor.AGE.pval<-signif(cor.AGE$p.value, digits = 1)
cor.AGE.pval
cor.AGE.rho<-signif(cor.AGE$estimate, digits = 1)
cor.AGE.rho


AGE.total.mean<-mean(AGE.cor.test.data$AGE)
AGE.total.sd<-sd(AGE.cor.test.data$AGE)

AGE.trtd<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Treated",]
AGE.trtd.mean<-signif(mean(AGE.cor.test.data$AGE), digits = 2)
AGE.trtd.sd<-sd(AGE.cor.test.data$AGE)

AGE.plac<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Placebo",]
AGE.plac.mean<-signif(mean(AGE.cor.test.data$AGE), digits=2)
AGE.plac.sd<-sd(AGE.cor.test.data$AGE)



```

```{r AGE.adonis,  include=F}
set.seed(32016)
AGE.adonis<-adonis(nao.screen.all.dist ~ AGE, nao.screen_data)
AGE.adonis
AGE.adonis.pval<-signif(as.numeric(na.omit(AGE.adonis$aov.tab$`Pr(>F)`)), digits = 1)

AGE_min<-toString(c('Min=', min(nao.screen_data$AGE)))
AGE_min<-gsub(",", "", AGE_min)
AGE_min<-gsub(" ", "", AGE_min)
AGE_med<-toString(c("Median=", median(nao.screen_data$AGE)))
AGE_med<-gsub(",", "", AGE_med)
AGE_med<-gsub(" ", "", AGE_med)
AGE_max<-toString(c("Max=", max(nao.screen_data$AGE)))
AGE_max<-gsub(",", "", AGE_max)
AGE_max<-gsub(" ", "", AGE_max)
AGE_sum<-toString(c(AGE_min, AGE_med, AGE_max))
AGE_n<-n.AGE.cortest

AGE_Spear_P<- toString(c('(', cor.AGE.pval, ")"))
AGE_Spear_P<-gsub(",", "", AGE_Spear_P)
AGE_Spear_P<-gsub(" ", "", AGE_Spear_P)
AGE_Spear_rho<-toString(c('rho =', cor.AGE.rho))
AGE_Spear_rho<-gsub(",", "", AGE_Spear_rho)
#AGE_Spear_rho<-gsub(" ", "", AGE_Spear_rho)
AGE_Spear_Prho<-c(AGE_Spear_rho, AGE_Spear_P)
AGE_Spear<-toString(c(AGE_Spear_Prho))
AGE_Spear<-gsub(",", "", AGE_Spear)
AGE_AMOVA_p<-toString(c(AGE.adonis.pval))
AGE_AMOVA_p<-gsub(",", "", AGE_AMOVA_p)
AGE_AMOVA_p<-gsub(" ", "", AGE_AMOVA_p)
AGE_AMOVA<-toString(c(AGE_AMOVA_p))
AGE_AMOVA<-gsub(",", "", AGE_AMOVA)
AGE<- matrix(c('Age (years)', AGE_Spear_rho, cor.AGE$p.value, as.numeric(na.omit(AGE.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r SEX-wilcox, include=F}
set.seed(32016)
SEX.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, SEX)))
n.MSEX.wilcoxtest<-nrow(SEX.wilcox.test.data[SEX.wilcox.test.data$SEX=="M",])
n.MSEX.wilcoxtest
n.FSEX.wilcoxtest<-nrow(SEX.wilcox.test.data[SEX.wilcox.test.data$SEX=="F",])
n.FSEX.wilcoxtest
wilcox.SEX<-wilcox.test(invsimpson ~ SEX, SEX.wilcox.test.data)
summary(wilcox.SEX)
wilcox.SEX.pval<-signif(wilcox.SEX$p.value, digits = 1)
wilcox.SEX.pval
wilcox.SEX.method<-wilcox.SEX$method
subT<-wilcox.SEX.method

```

```{r SEX.adonis,  include=F}
set.seed(32016)
SEX.adonis<-adonis(nao.screen.all.dist ~ SEX, nao.screen_data)
SEX.adonis
SEX.adonis.pval<-signif(as.numeric(na.omit(SEX.adonis$aov.tab$`Pr(>F)`)), digits = 1)
MSEX<-toString(c(n.MSEX.wilcoxtest))
FSEX<-toString(c(n.FSEX.wilcoxtest))


SEX_sum<-toString(c(FSEX, "/", MSEX))
SEX_sum<-gsub(",", "", SEX_sum)
SEX_sum<-gsub(" ", "", SEX_sum)

SEX<- matrix(c('Sex (F/M)', "-", wilcox.SEX$p.value, as.numeric(na.omit(SEX.adonis$aov.tab$`Pr(>F)`))), ncol = 4)

```

```{r BCORT-wilcox, include=F}
set.seed(32016)
BCORT.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BCORT)))
YesBCORT.wilcoxtest<-BCORT.wilcox.test.data[BCORT.wilcox.test.data$BCORT=="Yes",]
n.YesBCORT.wilcoxtest<-nrow(YesBCORT.wilcoxtest)
NoBCORT.wilcoxtest<-BCORT.wilcox.test.data[BCORT.wilcox.test.data$BCORT=="No",]
n.NoBCORT.wilcoxtest<-nrow(NoBCORT.wilcoxtest)
wilcox.BCORT<-wilcox.test(invsimpson ~ BCORT, BCORT.wilcox.test.data)
summary(wilcox.BCORT)
wilcox.BCORT.pval<-signif(wilcox.BCORT$p.value, digits = 1)
wilcox.BCORT.pval
wilcox.BCORT.method<-wilcox.BCORT$method
subT<-wilcox.BCORT.method
nYesString<-toString(c("Yes=", n.YesBCORT.wilcoxtest))
nYesString<-gsub(",", "", nYesString)
nNoString<-toString(c("No=", n.NoBCORT.wilcoxtest))
nNoString<-gsub(",", "", nNoString)

```

```{r BCORT.adonis,  include=F}
set.seed(32016)
BCORT.adonis<-adonis(nao.screen.all.dist ~ BCORT, nao.screen_data)
BCORT.adonis
BCORT.adonis.pval<-signif(as.numeric(na.omit(BCORT.adonis$aov.tab$`Pr(>F)`)), digits = 1)

YesBCORT<-toString(c(n.YesBCORT.wilcoxtest))

NoBCORT<-toString(c(n.NoBCORT.wilcoxtest))


BCORT_sum<-toString(c(YesBCORT, "/", NoBCORT))
BCORT_sum<-gsub(",", "", BCORT_sum)
BCORT_sum<-gsub(" ", "", BCORT_sum)

BCORT<- matrix(c('Corticosteroid Use (Y/N)', "-", wilcox.BCORT$p.value, as.numeric(na.omit(BCORT.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r DDUR.cortest, include=F}
set.seed(32016)
cor.test(nao.screen_data$invsimpson, nao.screen_data$DDUR, method = "spearman")


DDUR.cor.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, DDUR)))
n.DDUR.cortest<-nrow(DDUR.cor.test.data)
n.DDUR.cortest
cor.DDUR<-cor.test(DDUR.cor.test.data$DDUR, DDUR.cor.test.data$invsimpson, method = "spearman")
cor.DDUR
cor.DDUR.pval<-signif(cor.DDUR$p.value, digits = 1)
cor.DDUR.pval
cor.DDUR.rho<-signif(cor.DDUR$estimate, digits = 1)
cor.DDUR.rho


```

```{r DDUR.adonis,  include=F}
set.seed(32016)
DDUR.adonis<-adonis(nao.screen.all.dist ~ DDUR, nao.screen_data)
DDUR.adonis
DDUR.adonis.pval<-signif(as.numeric(na.omit(DDUR.adonis$aov.tab$`Pr(>F)`)), digits = 1)

DDUR_min<-toString(c('Min=', min(nao.screen_data$DDUR)))
DDUR_min<-gsub(",", "", DDUR_min)
DDUR_min<-gsub(" ", "", DDUR_min)
DDUR_med<-toString(c("Median=", median(nao.screen_data$DDUR)))
DDUR_med<-gsub(",", "", DDUR_med)
DDUR_med<-gsub(" ", "", DDUR_med)
DDUR_max<-toString(c("Max=", max(nao.screen_data$DDUR)))
DDUR_max<-gsub(",", "", DDUR_max)
DDUR_max<-gsub(" ", "", DDUR_max)
DDUR_sum<-toString(c(DDUR_min, DDUR_med, DDUR_max))
DDUR_n<-n.DDUR.cortest
DDUR_Spear_P<- toString(c("(", cor.DDUR.pval, ")"))
DDUR_Spear_P<-gsub(",", "", DDUR_Spear_P)
DDUR_Spear_P<-gsub(" ", "", DDUR_Spear_P)
DDUR_Spear_rho<-toString(c('rho =', cor.DDUR.rho))
DDUR_Spear_rho<-gsub(",", "", DDUR_Spear_rho)
#DDUR_Spear_rho<-gsub(" ", "", DDUR_Spear_rho)
DDUR_Spear_Prho<-c(DDUR_Spear_rho, DDUR_Spear_P)
DDUR_Spear_Prho<-toString(c(DDUR_Spear_Prho))
DDUR_Spear_Prho<-gsub(",", "", DDUR_Spear_Prho)

DDUR<- matrix(c('Disease Duration (years)', DDUR_Spear_rho, cor.DDUR$p.value, as.numeric(na.omit(DDUR.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r kwtissinvol, include=F}
set.seed(32016)
summary(nao.screen_data$tissinvol)

tissinvol.kw.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, tissinvol)))
levels(tissinvol.kw.data$tissinvol)[2]<-"Colon-Ileum"
tissinvol.kw<-kruskal.test(invsimpson ~ tissinvol, tissinvol.kw.data)
tissinvol.kw
#plot(invsimpson ~ tissinvol, tissinvol.kw.data)
Colon.kw<-tissinvol.kw.data[tissinvol.kw.data$tissinvol=="Colon",]
n.Colon.kw<-nrow(Colon.kw)
Colon_Ileum.kw<-tissinvol.kw.data[tissinvol.kw.data$tissinvol=="Colon-Ileum",]
n.Colon_Ileum.kw<-nrow(Colon_Ileum.kw)
Ileum.kw<-tissinvol.kw.data[tissinvol.kw.data$tissinvol=="Ileum",]
n.Ileum.kw<-nrow(Ileum.kw)
kmc<-kruskalmc(invsimpson ~ tissinvol, tissinvol.kw.data)
kw.tissinvol<-kruskal.test(invsimpson ~ tissinvol, tissinvol.kw.data)
summary(kw.tissinvol)
kw.tissinvol.pval<-signif(kw.tissinvol$p.value, digits = 1)
kw.tissinvol.pval
kw.tissinvol.method<-kw.tissinvol$method
subT<-kw.tissinvol.method
nColonString<-toString(c("Colon =", n.Colon.kw))
nColonString<-gsub(",", "", nColonString)
nColon_IleumString<-toString(c("Colon-Ileum =", n.Colon_Ileum.kw))
nColon_IleumString<-gsub(",", "", nColon_IleumString)
nIleumString<-toString(c("Ileum =", n.Ileum.kw))
nIleumString<-gsub(",", "", nIleumString)

kruskalmc(invsimpson ~ tissinvol, tissinvol.kw.data)
```

```{r tissinvol.adonis,  include=F}
set.seed(32016)
tissinvol.adonis<-adonis(nao.screen.all.dist ~ tissinvol, nao.screen_data)
tissinvol.adonis
tissinvol.adonis.pval<-signif(as.numeric(na.omit(tissinvol.adonis$aov.tab$`Pr(>F)`)), digits = 1)

Colon<-toString(c('Colon=', n.Colon.kw))
Colon<-gsub(",", "", Colon)
Colon<-gsub(" ", "", Colon)
ColIle<-toString(c('Colon-Ileum=', n.Colon_Ileum.kw))
ColIle<-gsub(",", "", ColIle)
ColIle<-gsub(" ", "", ColIle)
Ileum<-toString(c('Ileum=', n.Ileum.kw))
Ileum<-gsub(",", "", Ileum)
Ileum<-gsub(" ", "", Ileum)

tissinvol_sum<-toString(c(Colon, ColIle, Ileum))

tissinvol<- matrix(c('Tissue Involvement', "-", kw.tissinvol$p.value, as.numeric(na.omit(tissinvol.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r BOWSTRICT-wilcox, include=F}
set.seed(32016)

BOWSTRICT.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, BOWSTRICT)))
n.YesBOWSTRICT.wilcoxtest<-nrow(BOWSTRICT.wilcox.test.data[BOWSTRICT.wilcox.test.data$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.wilcoxtest
n.NoBOWSTRICT.wilcoxtest<-nrow(BOWSTRICT.wilcox.test.data[BOWSTRICT.wilcox.test.data$BOWSTRICT=="Other",])
n.NoBOWSTRICT.wilcoxtest
wilcox.BOWSTRICT<-wilcox.test(invsimpson ~ BOWSTRICT, BOWSTRICT.wilcox.test.data)
summary(wilcox.BOWSTRICT)
wilcox.BOWSTRICT.pval<-signif(wilcox.BOWSTRICT$p.value, digits = 1)
wilcox.BOWSTRICT.pval

#plot(invsimpson~BOWSTRICT, nao.screen_data)
```

```{r BOWSTRICT.adonis,  include=F}
set.seed(32016)
BOWSTRICT.adonis<-adonis(nao.screen.all.dist ~ BOWSTRICT, nao.screen_data)
BOWSTRICT.adonis
BOWSTRICT.adonis.pval<-signif(as.numeric(na.omit(BOWSTRICT.adonis$aov.tab$`Pr(>F)`)), digits = 1)

YesBOWSTRICT<-toString(c('Yes=', n.YesBOWSTRICT.wilcoxtest))
YesBOWSTRICT<-gsub(",", "", YesBOWSTRICT)
YesBOWSTRICT<-gsub(" ", "", YesBOWSTRICT)
NoBOWSTRICT<-toString(c("No=", n.NoBOWSTRICT.wilcoxtest))
NoBOWSTRICT<-gsub(",", "", NoBOWSTRICT)
NoBOWSTRICT<-gsub(" ", "", NoBOWSTRICT)

BOWSTRICT_sum<-toString(c(NoBOWSTRICT, YesBOWSTRICT))

	
BOWSTRICT<-matrix(c('Bowel Stricture', "-", wilcox.BOWSTRICT$p.value, as.numeric(na.omit(BOWSTRICT.adonis$aov.tab$`Pr(>F)`))), ncol = 4)
```

```{r cohort description, include=F, cache=F}
N.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR)))
N_trtd <- nrow(N.cohort.data[N.cohort.data$TRTGR=="Treated",])
N_plac <- nrow(N.cohort.data[N.cohort.data$TRTGR=="Placebo",])
N_total <- nrow(N.cohort.data)
N.cohort<- matrix(c('', paste("n =", N_trtd), paste("n =", N_plac), paste("n =",N_total)), ncol = 4)

set.seed(32016)
AGE.total.mean<-signif(mean(AGE.cor.test.data$AGE), digits = 2)
AGE.total.sd<-signif(sd(AGE.cor.test.data$AGE), digits = 2)
AGE_total<-toString(c(AGE.total.mean, "±", AGE.total.sd))
AGE_total<-gsub(",", "", AGE_total)
#AGE_total<-gsub(" ", "", AGE_total)

AGE.trtd<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Treated",]
AGE.trtd.mean<-signif(mean(AGE.trtd$AGE), digits = 2)
AGE.trtd.sd<-signif(sd(AGE.trtd$AGE), digits = 2)
AGE_trtd<-toString(c(AGE.trtd.mean, " ± ", AGE.trtd.sd))
AGE_trtd<-gsub(",", "", AGE_trtd)
#AGE_trtd<-gsub(" ", "", AGE_trtd)

AGE.plac<-AGE.cor.test.data[AGE.cor.test.data$TRTGR=="Placebo",]
AGE.plac.mean<-signif(mean(AGE.plac$AGE), digits = 2)
AGE.plac.sd<-signif(sd(AGE.plac$AGE), digits = 2)
AGE_plac<-toString(c(AGE.plac.mean, " ± ", AGE.plac.sd))
AGE_plac<-gsub(",", "", AGE_plac)
#AGE_plac<-gsub(" ", "", AGE_plac)

AGE.cohort<- matrix(c('Age (years)', AGE_trtd, AGE_plac, AGE_total), ncol = 4)

SEX.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, SEX)))
n.MSEX.total<-nrow(SEX.cohort.data[SEX.cohort.data$SEX=="M",])
n.MSEX.total
n.FSEX.total<-nrow(SEX.cohort.data[SEX.cohort.data$SEX=="F",])
n.FSEX.total

SEX_total<-toString(c(signif(n.MSEX.total/(n.FSEX.total+n.MSEX.total)*100, 3)))
SEX_total<-gsub(",", "", SEX_total)
#SEX_total<-gsub(" ", "", SEX_total)

SEX.trtd<-SEX.cohort.data[SEX.cohort.data$TRTGR=="Treated",]
n.MSEX.trtd<-nrow(SEX.trtd[SEX.trtd$SEX=="M",])
n.MSEX.trtd
n.FSEX.trtd<-nrow(SEX.trtd[SEX.trtd$SEX=="F",])
n.FSEX.trtd
SEX_trtd<-toString(c(signif(n.MSEX.trtd/(n.FSEX.trtd + n.MSEX.trtd)*100, 3)))
SEX_trtd<-gsub(",", "", SEX_trtd)
#SEX_trtd<-gsub(" ", "", SEX_trtd)


SEX.plac<-SEX.cohort.data[SEX.cohort.data$TRTGR=="Placebo",]
n.MSEX.plac<-nrow(SEX.plac[SEX.plac$SEX=="M",])
n.MSEX.plac
n.FSEX.plac<-nrow(SEX.plac[SEX.plac$SEX=="F",])
n.FSEX.plac
SEX_plac<-toString(c(signif(n.MSEX.plac/(n.FSEX.plac +n.MSEX.plac)*100, 3)))
SEX_plac<-gsub(",", "", SEX_plac)
#SEX_plac<-gsub(" ", "", SEX_plac)

SEX.cohort<- matrix(c('Sex (% Male)', SEX_trtd, SEX_plac, SEX_total), ncol = 4)

RACE.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, RACE)))
n.CRACE.total<-nrow(RACE.cohort.data[RACE.cohort.data$RACE=="Caucasian",])
n.CRACE.total
n.ORACE.total<-nrow(RACE.cohort.data[RACE.cohort.data$RACE!="Caucasian",])
n.ORACE.total

RACE_total<-toString(c(signif(n.CRACE.total/(n.CRACE.total+n.ORACE.total)*100, 3)))
RACE_total<-gsub(",", "", RACE_total)
#RACE_total<-gsub(" ", "", RACE_total)

RACE.trtd<-RACE.cohort.data[RACE.cohort.data$TRTGR=="Treated",]
n.CRACE.trtd<-nrow(RACE.trtd[RACE.trtd$RACE=="Caucasian",])
n.CRACE.trtd
n.ORACE.trtd<-nrow(RACE.trtd[RACE.trtd$RACE!="Caucasian",])
n.ORACE.trtd
RACE_trtd<-toString(c(signif(n.CRACE.trtd/(n.CRACE.trtd+n.ORACE.trtd)*100, 3)))
RACE_trtd<-gsub(",", "", RACE_trtd)
#RACE_trtd<-gsub(" ", "", RACE_trtd)


RACE.plac<-RACE.cohort.data[RACE.cohort.data$TRTGR=="Placebo",]
n.CRACE.plac<-nrow(RACE.plac[RACE.plac$RACE=="Caucasian",])
n.CRACE.plac
n.ORACE.plac<-nrow(RACE.plac[RACE.plac$RACE!="Caucasian",])
n.ORACE.plac
RACE_plac<-toString(c(signif(n.CRACE.plac/(n.CRACE.plac+n.ORACE.plac)*100, 3)))
RACE_plac<-gsub(",", "", RACE_plac)
#RACE_plac<-gsub(" ", "", RACE_plac)

RACE.cohort<- matrix(c('Race (% Caucasian)', RACE_trtd, RACE_plac, RACE_total), ncol = 4)

BCORT.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, BCORT)))
n.YesBCORT.total<-nrow(BCORT.cohort.data[BCORT.cohort.data$BCORT=="Yes",])
n.YesBCORT.total
n.NoBCORT.total<-nrow(BCORT.cohort.data[BCORT.cohort.data$BCORT!="Yes",])
n.NoBCORT.total

BCORT_total<-toString(c(signif(n.YesBCORT.total/(n.YesBCORT.total+n.NoBCORT.total)*100, 3)))
BCORT_total<-gsub(",", "", BCORT_total)
BCORT_total<-gsub(" ", "", BCORT_total)

BCORT.trtd<-BCORT.cohort.data[BCORT.cohort.data$TRTGR=="Treated",]
n.YesBCORT.trtd<-nrow(BCORT.trtd[BCORT.trtd$BCORT=="Yes",])
n.YesBCORT.trtd
n.NoBCORT.trtd<-nrow(BCORT.trtd[BCORT.trtd$BCORT!="Yes",])
n.NoBCORT.trtd
BCORT_trtd<-toString(c(signif(n.YesBCORT.trtd/(n.YesBCORT.trtd+n.NoBCORT.trtd)*100, 3)))
BCORT_trtd<-gsub(",", "", BCORT_trtd)
BCORT_trtd<-gsub(" ", "", BCORT_trtd)


BCORT.plac<-BCORT.cohort.data[BCORT.cohort.data$TRTGR=="Placebo",]
n.YesBCORT.plac<-nrow(BCORT.plac[BCORT.plac$BCORT=="Yes",])
n.YesBCORT.plac
n.NoBCORT.plac<-nrow(BCORT.plac[BCORT.plac$BCORT!="Yes",])
n.NoBCORT.plac
BCORT_plac<-toString(c(signif(n.YesBCORT.plac/(n.YesBCORT.plac+n.NoBCORT.plac)*100, 3)))
BCORT_plac<-gsub(",", "", BCORT_plac)
BCORT_plac<-gsub(" ", "", BCORT_plac)

BCORT.cohort<- matrix(c('Corticosteroid Use (%)', BCORT_trtd, BCORT_plac, BCORT_total), ncol = 4)

BL_BMI.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, BL_BMI)))
BL_BMI.total.mean<-signif(mean(BL_BMI.cohort.data$BL_BMI), digits = 2)
BL_BMI.total.sd<-signif(sd(BL_BMI.cohort.data$BL_BMI), digits = 2)
BL_BMI_total<-toString(c(BL_BMI.total.mean, " ± ", BL_BMI.total.sd))
BL_BMI_total<-gsub(",", "", BL_BMI_total)
#BL_BMI_total<-gsub(" ", "", BL_BMI_total)

BL_BMI.trtd<-BL_BMI.cohort.data[BL_BMI.cohort.data$TRTGR=="Treated",]
BL_BMI.trtd.mean<-signif(mean(BL_BMI.trtd$BL_BMI), digits = 2)
BL_BMI.trtd.sd<-signif(sd(BL_BMI.trtd$BL_BMI), digits = 2)
BL_BMI_trtd<-toString(c(BL_BMI.trtd.mean, " ± ", BL_BMI.trtd.sd))
BL_BMI_trtd<-gsub(",", "", BL_BMI_trtd)
#BL_BMI_trtd<-gsub(" ", "", BL_BMI_trtd)

BL_BMI.plac<-BL_BMI.cohort.data[BL_BMI.cohort.data$TRTGR=="Placebo",]
BL_BMI.plac.mean<-signif(mean(BL_BMI.plac$BL_BMI), digits = 2)
BL_BMI.plac.sd<-signif(sd(BL_BMI.plac$BL_BMI), digits = 2)
BL_BMI_plac<-toString(c(BL_BMI.plac.mean, " ± ", BL_BMI.plac.sd))
BL_BMI_plac<-gsub(",", "", BL_BMI_plac)
#BL_BMI_plac<-gsub(" ", "", BL_BMI_plac)

BL_BMI.cohort<- matrix(c('BMI (kg/m^2)', BL_BMI_trtd, BL_BMI_plac, BL_BMI_total), ncol = 4)


DDUR.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, DDUR)))
DDUR.total.mean<-signif(mean(DDUR.cohort.data$DDUR), digits = 2)
DDUR.total.sd<-signif(sd(DDUR.cohort.data$DDUR), digits = 2)
DDUR_total<-toString(c(DDUR.total.mean, " ± ", DDUR.total.sd))
DDUR_total<-gsub(",", "", DDUR_total)
#DDUR_total<-gsub(" ", "", DDUR_total)

DDUR.trtd<-DDUR.cohort.data[DDUR.cohort.data$TRTGR=="Treated",]
DDUR.trtd.mean<-signif(mean(DDUR.trtd$DDUR), digits = 2)
DDUR.trtd.sd<-signif(sd(DDUR.trtd$DDUR), digits = 2)
DDUR_trtd<-toString(c(DDUR.trtd.mean, " ± ", DDUR.trtd.sd))
DDUR_trtd<-gsub(",", "", DDUR_trtd)
#DDUR_trtd<-gsub(" ", "", DDUR_trtd)

DDUR.plac<-DDUR.cohort.data[DDUR.cohort.data$TRTGR=="Placebo",]
DDUR.plac.mean<-signif(mean(DDUR.plac$DDUR), digits = 2)
DDUR.plac.sd<-signif(sd(DDUR.plac$DDUR), digits = 2)
DDUR_plac<-toString(c(DDUR.plac.mean, " ± ", DDUR.plac.sd))
DDUR_plac<-gsub(",", "", DDUR_plac)
#DDUR_plac<-gsub(" ", "", DDUR_plac)

DDUR.cohort<- matrix(c('Disease Duration (years)', DDUR_trtd, DDUR_plac, DDUR_total), ncol = 4)

cdail_wk0.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, cdail_wk0)))
cdail_wk0.total.mean<-signif(mean(cdail_wk0.cohort.data$cdail_wk0), digits = 2)
cdail_wk0.total.sd<-signif(sd(cdail_wk0.cohort.data$cdail_wk0), digits = 2)
cdail_wk0_total<-toString(c(cdail_wk0.total.mean, " ± ", cdail_wk0.total.sd))
cdail_wk0_total<-gsub(",", "", cdail_wk0_total)
#cdail_wk0_total<-gsub(" ", "", cdail_wk0_total)

cdail_wk0.trtd<-cdail_wk0.cohort.data[cdail_wk0.cohort.data$TRTGR=="Treated",]
cdail_wk0.trtd.mean<-signif(mean(cdail_wk0.trtd$cdail_wk0), digits = 2)
cdail_wk0.trtd.sd<-signif(sd(cdail_wk0.trtd$cdail_wk0), digits = 2)
cdail_wk0_trtd<-toString(c(cdail_wk0.trtd.mean, " ± ", cdail_wk0.trtd.sd))
cdail_wk0_trtd<-gsub(",", "", cdail_wk0_trtd)
#cdail_wk0_trtd<-gsub(" ", "", cdail_wk0_trtd)

cdail_wk0.plac<-cdail_wk0.cohort.data[cdail_wk0.cohort.data$TRTGR=="Placebo",]
cdail_wk0.plac.mean<-signif(mean(cdail_wk0.plac$cdail_wk0), digits = 2)
cdail_wk0.plac.sd<-signif(sd(cdail_wk0.plac$cdail_wk0), digits = 2)
cdail_wk0_plac<-toString(c(cdail_wk0.plac.mean, " ± ", cdail_wk0.plac.sd))
cdail_wk0_plac<-gsub(",", "", cdail_wk0_plac)
#cdail_wk0_plac<-gsub(" ", "", cdail_wk0_plac)

cdail_wk0.cohort<- matrix(c('CDAI', cdail_wk0_trtd, cdail_wk0_plac, cdail_wk0_total), ncol = 4)

BOWSTRICT.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, BOWSTRICT)))
n.YesBOWSTRICT.total<-nrow(BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.total
n.NoBOWSTRICT.total<-nrow(BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$BOWSTRICT=="Other",])
n.NoBOWSTRICT.total

BOWSTRICT_total<-toString(c(signif(n.YesBOWSTRICT.total/(n.YesBOWSTRICT.total+n.NoBOWSTRICT.total)*100, 3)))
BOWSTRICT_total<-gsub(",", "", BOWSTRICT_total)
BOWSTRICT_total<-gsub(" ", "", BOWSTRICT_total)

BOWSTRICT.trtd<-BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$TRTGR=="Treated",]
n.YesBOWSTRICT.trtd<-nrow(BOWSTRICT.trtd[BOWSTRICT.trtd$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.trtd
n.NoBOWSTRICT.trtd<-nrow(BOWSTRICT.trtd[BOWSTRICT.trtd$BOWSTRICT=="Other",])
n.NoBOWSTRICT.trtd
BOWSTRICT_trtd<-toString(c(signif(n.YesBOWSTRICT.trtd/(n.YesBOWSTRICT.trtd+n.NoBOWSTRICT.trtd)*100, 3)))
BOWSTRICT_trtd<-gsub(",", "", BOWSTRICT_trtd)
BOWSTRICT_trtd<-gsub(" ", "", BOWSTRICT_trtd)


BOWSTRICT.plac<-BOWSTRICT.cohort.data[BOWSTRICT.cohort.data$TRTGR=="Placebo",]
n.YesBOWSTRICT.plac<-nrow(BOWSTRICT.plac[BOWSTRICT.plac$BOWSTRICT=="Strict_stenosis",])
n.YesBOWSTRICT.plac
n.NoBOWSTRICT.plac<-nrow(BOWSTRICT.plac[BOWSTRICT.plac$BOWSTRICT=="Other",])
n.NoBOWSTRICT.plac
BOWSTRICT_plac<-toString(c(signif(n.YesBOWSTRICT.plac/(n.YesBOWSTRICT.plac+n.NoBOWSTRICT.plac)*100, 3)))
BOWSTRICT_plac<-gsub(",", "", BOWSTRICT_plac)
BOWSTRICT_plac<-gsub(" ", "", BOWSTRICT_plac)

BOWSTRICT.cohort<- matrix(c('Bowel Stricture (%)', BOWSTRICT_trtd, BOWSTRICT_plac, BOWSTRICT_total), ncol = 4)

tissinvol.cohort.data<-na.omit(subset(nao.screen_data, select= c(group, TRTGR, tissinvol)))
n.Colon.total<-nrow(tissinvol.cohort.data[tissinvol.cohort.data$tissinvol=="Colon",])
n.Colon.total <- signif(n.Colon.total/N_total*100, 3)
n.Colon_Ileum.total<-nrow(tissinvol.cohort.data[tissinvol.cohort.data$tissinvol=="Colon_Ileum",])
n.Colon_Ileum.total <- signif(n.Colon_Ileum.total/N_total*100, 3)
n.Ileum.total<-nrow(tissinvol.cohort.data[tissinvol.cohort.data$tissinvol=="Ileum",])
n.Ileum.total <- signif(n.Ileum.total/N_total*100,3)


tissinvol_total<-toString(c(n.Colon.total, "/", n.Colon_Ileum.total, "/", n.Ileum.total))
tissinvol_total<-gsub(",", "", tissinvol_total)
tissinvol_total<-gsub(" ", "", tissinvol_total)

tissinvol.trtd<-tissinvol.cohort.data[tissinvol.cohort.data$TRTGR=="Treated",]
n.Colon.trtd<-nrow(tissinvol.trtd[tissinvol.trtd$tissinvol=="Colon",])
n.Colon.trtd <- signif(n.Colon.trtd/N_trtd*100, 3)
n.Colon_Ileum.trtd<-nrow(tissinvol.trtd[tissinvol.trtd$tissinvol=="Colon_Ileum",])
n.Colon_Ileum.trtd <- signif(n.Colon_Ileum.trtd/N_trtd*100, 3)
n.Ileum.trtd<-nrow(tissinvol.trtd[tissinvol.trtd$tissinvol=="Ileum",])
n.Ileum.trtd <- signif(n.Ileum.trtd/N_trtd*100, 3)

tissinvol_trtd<-toString(c(n.Colon.trtd, "/", n.Colon_Ileum.trtd, "/", n.Ileum.trtd))
tissinvol_trtd<-gsub(",", "", tissinvol_trtd)
tissinvol_trtd<-gsub(" ", "", tissinvol_trtd)

tissinvol.plac<-tissinvol.cohort.data[tissinvol.cohort.data$TRTGR=="Placebo",]
n.Colon.plac<-nrow(tissinvol.plac[tissinvol.plac$tissinvol=="Colon",])
n.Colon.plac <- signif(n.Colon.plac/N_plac*100, 3)
n.Colon_Ileum.plac<-nrow(tissinvol.plac[tissinvol.plac$tissinvol=="Colon_Ileum",])
n.Colon_Ileum.plac <- signif(n.Colon_Ileum.plac/N_plac*100, 3)
n.Ileum.plac<-nrow(tissinvol.plac[tissinvol.plac$tissinvol=="Ileum",])
n.Ileum.plac <- signif(n.Ileum.plac/N_plac*100, 3)


tissinvol_plac<-toString(c(n.Colon.plac, "/", n.Colon_Ileum.plac, "/", n.Ileum.plac))
tissinvol_plac<-gsub(",", "", tissinvol_plac)
tissinvol_plac<-gsub(" ", "", tissinvol_plac)

tissinvol.cohort<- matrix(c('Tissue Involvement (%) \nColon/Ileocolic/Ileal', tissinvol_trtd, tissinvol_plac, tissinvol_total), ncol = 4)
```


```{r wk6-diversity, include=F}
set.seed(32016)
all.RelRSPwk6.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk6)))
all.n.YesRelRSPwk6.wilcoxtest<-nrow(all.RelRSPwk6.wilcox.test.data[all.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",])
all.n.YesRelRSPwk6.wilcoxtest
all.n.NoRelRSPwk6.wilcoxtest<-nrow(all.RelRSPwk6.wilcox.test.data[all.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",])
all.n.NoRelRSPwk6.wilcoxtest

all.RelRSPwk6.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk6, TRTGR)))
all.RelRSPwk6.kw.test.data$rRSPwk6TRTGR <- as.factor(paste(all.RelRSPwk6.kw.test.data$RelRSPwk6, all.RelRSPwk6.kw.test.data$TRTGR, sep = "_"))
all.RelRSPwk6.kw.test <- kruskal.test(all.RelRSPwk6.kw.test.data$invsimpson ~ all.RelRSPwk6.kw.test.data$rRSPwk6TRTGR, all.RelRSPwk6.kw.test.data)
all.RelRSPwk6.kw.test.pval <- all.RelRSPwk6.kw.test$p.value

set.seed(32016)
all.RelRSPwk6.adonis<-adonis(nao.screen.all.dist ~ RelRSPwk6, nao.screen_data, strata=nao.screen_data$TRTGR)
all.RelRSPwk6.adonis
all.RelRSPwk6.adonis.pval<-signif(as.numeric(na.omit(all.RelRSPwk6.adonis$aov.tab$`Pr(>F)`)), digits = 2)

all.YesRelRSPwk6<-toString(c(all.n.YesRelRSPwk6.wilcoxtest))
all.YesRelRSPwk6<-gsub(",", "", all.YesRelRSPwk6)
all.YesRelRSPwk6<-gsub(" ", "", all.YesRelRSPwk6)
all.NoRelRSPwk6<-toString(c(all.n.NoRelRSPwk6.wilcoxtest))
all.NoRelRSPwk6<-gsub(",", "", all.NoRelRSPwk6)
all.NoRelRSPwk6<-gsub(" ", "", all.NoRelRSPwk6)

all.RelRSPwk6_sum<-toString(c(all.NoRelRSPwk6, all.YesRelRSPwk6))

trtd.RelRSPwk6.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, RelRSPwk6)))
trtd.n.YesRelRSPwk6.wilcoxtest<-nrow(trtd.RelRSPwk6.wilcox.test.data[trtd.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",])
trtd.n.YesRelRSPwk6.wilcoxtest
trtd.n.NoRelRSPwk6.wilcoxtest<-nrow(trtd.RelRSPwk6.wilcox.test.data[trtd.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",])
trtd.n.NoRelRSPwk6.wilcoxtest

trtd.YesRelRSPwk6<-toString(c(trtd.n.YesRelRSPwk6.wilcoxtest))
trtd.YesRelRSPwk6<-gsub(",", "", trtd.YesRelRSPwk6)
trtd.YesRelRSPwk6<-gsub(" ", "", trtd.YesRelRSPwk6)
trtd.NoRelRSPwk6<-toString(c(trtd.n.NoRelRSPwk6.wilcoxtest))
trtd.NoRelRSPwk6<-gsub(",", "", trtd.NoRelRSPwk6)
trtd.NoRelRSPwk6<-gsub(" ", "", trtd.NoRelRSPwk6)

trtd.RelRSPwk6_sum<-toString(c(trtd.NoRelRSPwk6, trtd.YesRelRSPwk6))

plac.RelRSPwk6.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, RelRSPwk6)))
plac.n.YesRelRSPwk6.wilcoxtest<-nrow(plac.RelRSPwk6.wilcox.test.data[plac.RelRSPwk6.wilcox.test.data$RelRSPwk6=="Yes",])
plac.n.YesRelRSPwk6.wilcoxtest
plac.n.NoRelRSPwk6.wilcoxtest<-nrow(plac.RelRSPwk6.wilcox.test.data[plac.RelRSPwk6.wilcox.test.data$RelRSPwk6=="No",])
plac.n.NoRelRSPwk6.wilcoxtest

plac.YesRelRSPwk6<-toString(c(plac.n.YesRelRSPwk6.wilcoxtest))
plac.YesRelRSPwk6<-gsub(",", "", plac.YesRelRSPwk6)
plac.YesRelRSPwk6<-gsub(" ", "", plac.YesRelRSPwk6)
plac.NoRelRSPwk6<-toString(c(plac.n.NoRelRSPwk6.wilcoxtest))
plac.NoRelRSPwk6<-gsub(",", "", plac.NoRelRSPwk6)
plac.NoRelRSPwk6<-gsub(" ", "", plac.NoRelRSPwk6)

plac.RelRSPwk6_sum<-toString(c(plac.NoRelRSPwk6, plac.YesRelRSPwk6))

all.RelRSPwk6_AMOVA_p<-toString(c(all.RelRSPwk6.adonis.pval))
all.RelRSPwk6_AMOVA_p<-gsub(",", "", all.RelRSPwk6_AMOVA_p)
all.RelRSPwk6_AMOVA_p<-gsub(" ", "", all.RelRSPwk6_AMOVA_p)
all.RelRSPwk6_AMOVA<-toString(c(all.RelRSPwk6_AMOVA_p))
all.RelRSPwk6_AMOVA<-gsub(",", "", all.RelRSPwk6_AMOVA)
all.RelRSPwk6<-matrix(c('Week 6 Response (No, Yes)', trtd.RelRSPwk6_sum, plac.RelRSPwk6_sum, all.RelRSPwk6.kw.test.pval, all.RelRSPwk6_AMOVA), ncol = 5)

set.seed(32016)
all.REMISSwk6.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk6)))

all.REMISSwk6.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk6, TRTGR)))
all.REMISSwk6.kw.test.data$REMwk6TRTGR <- as.factor(paste(all.REMISSwk6.kw.test.data$REMISSwk6, all.REMISSwk6.kw.test.data$TRTGR, sep = "_"))
all.REMISSwk6.kw.test <- kruskal.test(all.REMISSwk6.kw.test.data$invsimpson ~ all.REMISSwk6.kw.test.data$REMwk6TRTGR, all.REMISSwk6.kw.test.data)
all.REMISSwk6.kw.test
all.REMISSwk6.kw.test.pval <- all.REMISSwk6.kw.test$p.value
all.REMISSwk6.kwmc.test <- kruskalmc(all.REMISSwk6.kw.test.data$invsimpson ~ all.REMISSwk6.kw.test.data$REMwk6TRTGR, all.REMISSwk6.kw.test.data)
all.REMISSwk6.kwmc.test

all.n.YesREMISSwk6.wilcoxtest<-nrow(all.REMISSwk6.wilcox.test.data[all.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",])
all.n.YesREMISSwk6.wilcoxtest
all.n.NoREMISSwk6.wilcoxtest<-nrow(all.REMISSwk6.wilcox.test.data[all.REMISSwk6.wilcox.test.data$REMISSwk6=="No",])
all.n.NoREMISSwk6.wilcoxtest
all.wilcox.REMISSwk6<-wilcox.test(invsimpson ~ REMISSwk6, all.REMISSwk6.wilcox.test.data)
all.wilcox.REMISSwk6
all.wilcox.REMISSwk6.pval<-signif(all.wilcox.REMISSwk6$p.value, digits = 2)
all.wilcox.REMISSwk6.pval

set.seed(32016)
all.REMISSwk6.adonis<-adonis(nao.screen.all.dist ~ REMISSwk6, nao.screen_data, strata=nao.screen_data$TRTGR)
all.REMISSwk6.adonis
all.REMISSwk6.adonis.pval<-signif(as.numeric(na.omit(all.REMISSwk6.adonis$aov.tab$`Pr(>F)`)), digits = 2)

all.YesREMISSwk6<-toString(c(all.n.YesREMISSwk6.wilcoxtest))
all.YesREMISSwk6<-gsub(",", "", all.YesREMISSwk6)
all.YesREMISSwk6<-gsub(" ", "", all.YesREMISSwk6)
all.NoREMISSwk6<-toString(c(all.n.NoREMISSwk6.wilcoxtest))
all.NoREMISSwk6<-gsub(",", "", all.NoREMISSwk6)
all.NoREMISSwk6<-gsub(" ", "", all.NoREMISSwk6)

all.REMISSwk6_sum<-toString(c(all.NoREMISSwk6, all.YesREMISSwk6))

trtd.REMISSwk6.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, REMISSwk6)))
trtd.n.YesREMISSwk6.wilcoxtest<-nrow(trtd.REMISSwk6.wilcox.test.data[trtd.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",])
trtd.n.YesREMISSwk6.wilcoxtest
trtd.n.NoREMISSwk6.wilcoxtest<-nrow(trtd.REMISSwk6.wilcox.test.data[trtd.REMISSwk6.wilcox.test.data$REMISSwk6=="No",])
trtd.n.NoREMISSwk6.wilcoxtest
trtd.YesREMISSwk6<-toString(c(trtd.n.YesREMISSwk6.wilcoxtest))
trtd.YesREMISSwk6<-gsub(",", "", trtd.YesREMISSwk6)
trtd.YesREMISSwk6<-gsub(" ", "", trtd.YesREMISSwk6)
trtd.NoREMISSwk6<-toString(c(trtd.n.NoREMISSwk6.wilcoxtest))
trtd.NoREMISSwk6<-gsub(",", "", trtd.NoREMISSwk6)
trtd.NoREMISSwk6<-gsub(" ", "", trtd.NoREMISSwk6)

trtd.REMISSwk6_sum<-toString(c(trtd.NoREMISSwk6, trtd.YesREMISSwk6))

plac.REMISSwk6.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, REMISSwk6)))
plac.n.YesREMISSwk6.wilcoxtest<-nrow(plac.REMISSwk6.wilcox.test.data[plac.REMISSwk6.wilcox.test.data$REMISSwk6=="Yes",])
plac.n.YesREMISSwk6.wilcoxtest
plac.n.NoREMISSwk6.wilcoxtest<-nrow(plac.REMISSwk6.wilcox.test.data[plac.REMISSwk6.wilcox.test.data$REMISSwk6=="No",])
plac.n.NoREMISSwk6.wilcoxtest
plac.YesREMISSwk6<-toString(c(plac.n.YesREMISSwk6.wilcoxtest))
plac.YesREMISSwk6<-gsub(",", "", plac.YesREMISSwk6)
plac.YesREMISSwk6<-gsub(" ", "", plac.YesREMISSwk6)
plac.NoREMISSwk6<-toString(c(plac.n.NoREMISSwk6.wilcoxtest))
plac.NoREMISSwk6<-gsub(",", "", plac.NoREMISSwk6)
plac.NoREMISSwk6<-gsub(" ", "", plac.NoREMISSwk6)

plac.REMISSwk6_sum<-toString(c(plac.NoREMISSwk6, plac.YesREMISSwk6))




all.REMISSwk6_AMOVA_p<-toString(c(all.REMISSwk6.adonis.pval))
all.REMISSwk6_AMOVA_p<-gsub(",", "", all.REMISSwk6_AMOVA_p)
all.REMISSwk6_AMOVA_p<-gsub(" ", "", all.REMISSwk6_AMOVA_p)
all.REMISSwk6_AMOVA<-toString(c(all.REMISSwk6_AMOVA_p))
all.REMISSwk6_AMOVA<-gsub(",", "", all.REMISSwk6_AMOVA)
all.REMISSwk6<-matrix(c('Week 6 Remission (No, Yes)', trtd.REMISSwk6_sum, plac.REMISSwk6_sum, all.REMISSwk6.kw.test.pval, all.REMISSwk6_AMOVA), ncol = 5)
```

```{r wk22-diversity, include=F}
set.seed(32016)
all.RelRSPwk22.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk22)))

all.RelRSPwk22.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, RelRSPwk22, TRTGRINDMAN)))
all.RelRSPwk22.kw.test.data$rRSPwk22TRTGRINDMAN <- as.factor(paste(all.RelRSPwk22.kw.test.data$RelRSPwk22, all.RelRSPwk22.kw.test.data$TRTGRINDMAN, sep = "_"))
all.RelRSPwk22.kw.test <- kruskal.test(all.RelRSPwk22.kw.test.data$invsimpson ~ all.RelRSPwk22.kw.test.data$rRSPwk22TRTGRINDMAN, all.RelRSPwk22.kw.test.data)
all.RelRSPwk22.kw.test.pval <- all.RelRSPwk22.kw.test$p.value


all.n.YesRelRSPwk22.wilcoxtest<-nrow(all.RelRSPwk22.wilcox.test.data[all.RelRSPwk22.wilcox.test.data$RelRSPwk22=="Yes",])
all.n.YesRelRSPwk22.wilcoxtest
all.n.NoRelRSPwk22.wilcoxtest<-nrow(all.RelRSPwk22.wilcox.test.data[all.RelRSPwk22.wilcox.test.data$RelRSPwk22=="No",])
all.n.NoRelRSPwk22.wilcoxtest
all.wilcox.RelRSPwk22<-wilcox.test(invsimpson ~ RelRSPwk22, all.RelRSPwk22.wilcox.test.data)
all.wilcox.RelRSPwk22
all.wilcox.RelRSPwk22.pval<-signif(all.wilcox.RelRSPwk22$p.value, digits = 2)
all.wilcox.RelRSPwk22.pval

set.seed(32016)
kruskalmc(nao.screen_data$invsimpson, nao.screen_data$RelRSPwk22, data=nao.screen_data)

set.seed(32016)
all.RelRSPwk22.adonis<-adonis(nao.screen.all.dist ~ RelRSPwk22, nao.screen_data, strata=nao.screen_data$TRTGRINDMAN)
all.RelRSPwk22.adonis
all.RelRSPwk22.adonis.pval<-signif(as.numeric(na.omit(all.RelRSPwk22.adonis$aov.tab$`Pr(>F)`)), digits = 2)

all.YesRelRSPwk22<-toString(c(all.n.YesRelRSPwk22.wilcoxtest))
all.YesRelRSPwk22<-gsub(",", "", all.YesRelRSPwk22)
all.YesRelRSPwk22<-gsub(" ", "", all.YesRelRSPwk22)
all.NoRelRSPwk22<-toString(c(all.n.NoRelRSPwk22.wilcoxtest))
all.NoRelRSPwk22<-gsub(",", "", all.NoRelRSPwk22)
all.NoRelRSPwk22<-gsub(" ", "", all.NoRelRSPwk22)

all.RelRSPwk22_sum<-toString(c(all.NoRelRSPwk22, all.YesRelRSPwk22))

trtd.RelRSPwk22.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, RelRSPwk22, TRTGRINDMAN)))
trtd.RelRSPwk22.wilcox.test.data<-trtd.RelRSPwk22.wilcox.test.data[trtd.RelRSPwk22.wilcox.test.data$TRTGRINDMAN=="Treated_Treated",]
trtd.n.YesRelRSPwk22.wilcoxtest<-nrow(trtd.RelRSPwk22.wilcox.test.data[trtd.RelRSPwk22.wilcox.test.data$RelRSPwk22=="Yes",])
trtd.n.YesRelRSPwk22.wilcoxtest
trtd.n.NoRelRSPwk22.wilcoxtest<-nrow(trtd.RelRSPwk22.wilcox.test.data[trtd.RelRSPwk22.wilcox.test.data$RelRSPwk22=="No",])
trtd.n.NoRelRSPwk22.wilcoxtest
trtd.YesRelRSPwk22<-toString(c(trtd.n.YesRelRSPwk22.wilcoxtest))
trtd.YesRelRSPwk22<-gsub(",", "", trtd.YesRelRSPwk22)
trtd.YesRelRSPwk22<-gsub(" ", "", trtd.YesRelRSPwk22)
trtd.NoRelRSPwk22<-toString(c(trtd.n.NoRelRSPwk22.wilcoxtest))
trtd.NoRelRSPwk22<-gsub(",", "", trtd.NoRelRSPwk22)
trtd.NoRelRSPwk22<-gsub(" ", "", trtd.NoRelRSPwk22)

trtd.RelRSPwk22_sum<-toString(c(trtd.NoRelRSPwk22, trtd.YesRelRSPwk22))

plac.RelRSPwk22.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, RelRSPwk22, TRTGRINDMAN)))
plac.RelRSPwk22.wilcox.test.data<-plac.RelRSPwk22.wilcox.test.data[plac.RelRSPwk22.wilcox.test.data$TRTGRINDMAN=="Placebo_Placebo",]
plac.n.YesRelRSPwk22.wilcoxtest<-nrow(plac.RelRSPwk22.wilcox.test.data[plac.RelRSPwk22.wilcox.test.data$RelRSPwk22=="Yes",])
plac.n.YesRelRSPwk22.wilcoxtest
plac.n.NoRelRSPwk22.wilcoxtest<-nrow(plac.RelRSPwk22.wilcox.test.data[plac.RelRSPwk22.wilcox.test.data$RelRSPwk22=="No",])
plac.n.NoRelRSPwk22.wilcoxtest
plac.YesRelRSPwk22<-toString(c(plac.n.YesRelRSPwk22.wilcoxtest))
plac.YesRelRSPwk22<-gsub(",", "", plac.YesRelRSPwk22)
plac.YesRelRSPwk22<-gsub(" ", "", plac.YesRelRSPwk22)
plac.NoRelRSPwk22<-toString(c(plac.n.NoRelRSPwk22.wilcoxtest))
plac.NoRelRSPwk22<-gsub(",", "", plac.NoRelRSPwk22)
plac.NoRelRSPwk22<-gsub(" ", "", plac.NoRelRSPwk22)

plac.RelRSPwk22_sum<-toString(c(plac.NoRelRSPwk22, plac.YesRelRSPwk22))


all.RelRSPwk22_AMOVA_p<-toString(c(all.RelRSPwk22.adonis.pval))
all.RelRSPwk22_AMOVA_p<-gsub(",", "", all.RelRSPwk22_AMOVA_p)
all.RelRSPwk22_AMOVA_p<-gsub(" ", "", all.RelRSPwk22_AMOVA_p)
all.RelRSPwk22_AMOVA<-toString(c(all.RelRSPwk22_AMOVA_p))
all.RelRSPwk22_AMOVA<-gsub(",", "", all.RelRSPwk22_AMOVA)
all.RelRSPwk22<-matrix(c('Week 22 Response (No, Yes)', trtd.RelRSPwk22_sum, plac.RelRSPwk22_sum, all.RelRSPwk22.kw.test.pval, all.RelRSPwk22_AMOVA), ncol = 5)

set.seed(32016)
all.REMISSwk22.wilcox.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk22)))
all.REMISSwk22.kw.test.data<-na.omit(subset(nao.screen_data, select= c(group, invsimpson, REMISSwk22, TRTGRINDMAN)))

all.REMISSwk22.kw.test.data$REMwk6TRTGRINDMAN <-
	as.factor(paste(all.REMISSwk22.kw.test.data$REMISSwk22, all.REMISSwk22.kw.test.data$TRTGRINDMAN, sep = "_"))
all.REMISSwk22.kw.test <- kruskal.test(all.REMISSwk22.kw.test.data$invsimpson ~ all.REMISSwk22.kw.test.data$REMwk6TRTGRINDMAN, all.REMISSwk22.kw.test.data)
all.REMISSwk22.kw.test.pval <- all.REMISSwk22.kw.test$p.value

all.n.YesREMISSwk22.wilcoxtest<-nrow(all.REMISSwk22.wilcox.test.data[all.REMISSwk22.wilcox.test.data$REMISSwk22=="Yes",])
all.n.YesREMISSwk22.wilcoxtest
all.n.NoREMISSwk22.wilcoxtest<-nrow(all.REMISSwk22.wilcox.test.data[all.REMISSwk22.wilcox.test.data$REMISSwk22=="No",])
all.n.NoREMISSwk22.wilcoxtest
all.wilcox.REMISSwk22<-wilcox.test(invsimpson ~ REMISSwk22, all.REMISSwk22.wilcox.test.data)
all.wilcox.REMISSwk22
all.wilcox.REMISSwk22.pval<-signif(all.wilcox.REMISSwk22$p.value, digits = 2)
all.wilcox.REMISSwk22.pval

set.seed(32016)
all.REMISSwk22.adonis<-adonis(nao.screen.all.dist ~ REMISSwk22, nao.screen_data, strata=nao.screen_data$TRTGRINDMAN)
all.REMISSwk22.adonis
all.REMISSwk22.adonis.pval<-signif(as.numeric(na.omit(all.REMISSwk22.adonis$aov.tab$`Pr(>F)`)), digits = 2)

kruskalmc(nao.screen_data$invsimpson, nao.screen_data$REMISSwk22, data=nao.screen.trtd)

all.YesREMISSwk22<-toString(c(all.n.YesREMISSwk22.wilcoxtest))
all.YesREMISSwk22<-gsub(",", "", all.YesREMISSwk22)
all.YesREMISSwk22<-gsub(" ", "", all.YesREMISSwk22)
all.NoREMISSwk22<-toString(c(all.n.NoREMISSwk22.wilcoxtest))
all.NoREMISSwk22<-gsub(",", "", all.NoREMISSwk22)
all.NoREMISSwk22<-gsub(" ", "", all.NoREMISSwk22)

all.REMISSwk22_sum<-toString(c(all.NoREMISSwk22, all.YesREMISSwk22))

trtd.REMISSwk22.wilcox.test.data<-na.omit(subset(nao.screen.trtd, select= c(group, invsimpson, REMISSwk22, TRTGRINDMAN)))
trtd.REMISSwk22.wilcox.test.data<-trtd.REMISSwk22.wilcox.test.data[trtd.REMISSwk22.wilcox.test.data$TRTGRINDMAN=="Treated_Treated",]
trtd.n.YesREMISSwk22.wilcoxtest<-nrow(trtd.REMISSwk22.wilcox.test.data[trtd.REMISSwk22.wilcox.test.data$REMISSwk22=="Yes",])
trtd.n.YesREMISSwk22.wilcoxtest
trtd.n.NoREMISSwk22.wilcoxtest<-nrow(trtd.REMISSwk22.wilcox.test.data[trtd.REMISSwk22.wilcox.test.data$REMISSwk22=="No",])
trtd.n.NoREMISSwk22.wilcoxtest
trtd.YesREMISSwk22<-toString(c(trtd.n.YesREMISSwk22.wilcoxtest))
trtd.YesREMISSwk22<-gsub(",", "", trtd.YesREMISSwk22)
trtd.YesREMISSwk22<-gsub(" ", "", trtd.YesREMISSwk22)
trtd.NoREMISSwk22<-toString(c(trtd.n.NoREMISSwk22.wilcoxtest))
trtd.NoREMISSwk22<-gsub(",", "", trtd.NoREMISSwk22)
trtd.NoREMISSwk22<-gsub(" ", "", trtd.NoREMISSwk22)

trtd.REMISSwk22_sum<-toString(c(trtd.NoREMISSwk22, trtd.YesREMISSwk22))

plac.REMISSwk22.wilcox.test.data<-na.omit(subset(nao.screen.plac, select= c(group, invsimpson, REMISSwk22, TRTGRINDMAN)))
plac.REMISSwk22.wilcox.test.data<-plac.REMISSwk22.wilcox.test.data[plac.REMISSwk22.wilcox.test.data$TRTGRINDMAN=="Placebo_Placebo",]
plac.n.YesREMISSwk22.wilcoxtest<-nrow(plac.REMISSwk22.wilcox.test.data[plac.REMISSwk22.wilcox.test.data$REMISSwk22=="Yes",])
plac.n.YesREMISSwk22.wilcoxtest
plac.n.NoREMISSwk22.wilcoxtest<-nrow(plac.REMISSwk22.wilcox.test.data[plac.REMISSwk22.wilcox.test.data$REMISSwk22=="No",])
plac.n.NoREMISSwk22.wilcoxtest
plac.YesREMISSwk22<-toString(c(plac.n.YesREMISSwk22.wilcoxtest))
plac.YesREMISSwk22<-gsub(",", "", plac.YesREMISSwk22)
plac.YesREMISSwk22<-gsub(" ", "", plac.YesREMISSwk22)
plac.NoREMISSwk22<-toString(c(plac.n.NoREMISSwk22.wilcoxtest))
plac.NoREMISSwk22<-gsub(",", "", plac.NoREMISSwk22)
plac.NoREMISSwk22<-gsub(" ", "", plac.NoREMISSwk22)

plac.REMISSwk22_sum<-toString(c(plac.NoREMISSwk22, plac.YesREMISSwk22))


all.REMISSwk22_AMOVA_p<-toString(c(all.REMISSwk22.adonis.pval))
all.REMISSwk22_AMOVA_p<-gsub(",", "", all.REMISSwk22_AMOVA_p)
all.REMISSwk22_AMOVA_p<-gsub(" ", "", all.REMISSwk22_AMOVA_p)
all.REMISSwk22_AMOVA<-toString(c(all.REMISSwk22_AMOVA_p))
all.REMISSwk22_AMOVA<-gsub(",", "", all.REMISSwk22_AMOVA)
all.REMISSwk22<-matrix(c('Week 22 Remission (No, Yes)', trtd.REMISSwk22_sum, plac.REMISSwk22_sum, all.REMISSwk22.kw.test.pval, all.REMISSwk22_AMOVA), ncol = 5)

```

```{r timereaddata, include=FALSE}
#Read in all data
#metadata
source('../code/APmd.setup.R')

screen_RESPwk22<-APmd_RSPwk22 #[APmd_RSPwk22$visit=="Screening",] #removes any group with NA in RESPONSEwk22

screen_data<-screen_RESPwk22				#na.omit(screen_RESPwk22)

#screen_data$CDSTOOLM_Screening[screen_data$CDSTOOLM_Screening=="."]<-"NA"
#screen_data$CDSTOOLM_Screening <-as.integer(screen_data$CDSTOOLM_Screening)
#screen_data$CDPAINM_Screening[screen_data$CDPAINM_Screening=="."]<-"NA"
#screen_data$CDPAINM_Screening <-as.integer(screen_data$CDPAINM_Screening)
#screen_data$CDWELBEM_Screening[screen_data$CDWELBEM_Screening=="."]<-"NA"
#screen_data$CDWELBEM_Screening <-as.integer(screen_data$CDWELBEM_Screening)
#screen_data$CDHCTM_Screening[screen_data$CDHCTM_Screening=="."]<-"NA"
#screen_data$CDHCTM_Screening <-as.integer(screen_data$CDHCTM_Screening)
#screen_data$CDAMASSM_Screening<-as.factor(screen_data$CDAMASSM_Screening)

#axes <- read.table('../data/Jan400.screen_ust.3107.subsample.thetayc.0.03.lt.ave.nmds.by3.axes', header=T)
tax<- read.table('../data/Jan400.simple.taxonomy.txt', header=T, sep="\t")
#summary(APmd_RSPwk22[,c("TRTGR", "visit")])
#summary(screen_RESPwk22[,1:49])
summary(subset(screen_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

levs<-screen_data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs
subidsover4<-levs[levs$no_rows>=4,]
summary(subidsover4)
fourtimesubs<-as.character(subidsover4$USUBJID)

fourtime_data <- screen_data[screen_data$USUBJID %in% fourtimesubs,]
#summary(subset(fourtime_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

#x5tp<-fourtime_data[fourtime_data$USUBJID=="C0743T26045400006",]
#subset(x5tp, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson))

#Remove "duplicate" sample with worse coverage and nseqs
alltime_data<-fourtime_data
alltime_data<-alltime_data[alltime_data$TRTGRINDMAN!="Placebo_Treated", ] #remove TRTGRINDMAN=Placebo_Treated for easier analysisof all time point data placebo vs treated

summary(subset(alltime_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
alltp.ust<-alltime_data[alltime_data$TRTGRINDMAN!="Placebo_Placebo",]
alltp.ust<-alltp.ust[alltp.ust$TRTGRINDMAN!="Placebo_Treated",]
summary(alltp.ust[,1:6])
alltp.plac<-alltime_data[alltime_data$TRTGRINDMAN!="Treated_Placebo",]
alltp.placplac<-alltp.plac[alltp.plac$TRTGRINDMAN!="Treated_Treated",]
summary(alltp.placplac[,1:6])
alltp.ust.RESPy<-alltp.ust[alltp.ust$RelRSPwk22=="Yes",]
summary(alltp.ust.RESPy[,1:6])
alltp.placplac.RESPy<-alltp.placplac[alltp.placplac$RelRSPwk22=="Yes",]
summary(alltp.placplac.RESPy[,1:6])
alltp.all.respY<-alltime_data[alltime_data$RelRSPwk22=="Yes",]
alltp.all.respN<-alltime_data[alltime_data$RelRSPwk22=="No",]
summary(alltp.all.respY[,1:6])

no22data<-screen_data[screen_data$visit!="Week 22",]
no22data<-no22data[no22data$TRTGRINDMAN!="Placebo_Not",]
no22data<-no22data[no22data$TRTGRINDMAN!="Treated_Not",]
summary(no22data$visit)
summary(subset(no22data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
levs<-no22data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs
subidsover3<-levs[levs$no_rows>=3,]
summary(subidsover3)
subidis4<-levs[levs$no_rows==4,]
no22subs3times<-as.character(subidsover3$USUBJID, permutations = 2000)
no22subs3times

no22_3x_data <- no22data[no22data$USUBJID %in% no22subs3times,]
summary(subset(no22_3x_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

x4tp<-no22_3x_data[no22_3x_data$USUBJID=="C0743T26045400006",]
subset(x4tp, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson))

#Remove "duplicate" sample with worse coverage and nseqs
wk046_data<-no22_3x_data[no22_3x_data$group!='E4859941-5',]

summary(subset(wk046_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

wk046_data<-wk046_data[wk046_data$USUBJID!='C0743T26030100002',]

summary(subset(wk046_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8, INDTRTGR)))

wk046wk22remitters<-wk046_data[wk046_data$REMISSwk22=="Yes",]
summary(subset(wk046wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

trtd.wk046wk22remitters<-wk046wk22remitters[wk046wk22remitters$TRTGRINDMAN!="Placebo_Placebo",]
trtd.wk046wk22remitters<-trtd.wk046wk22remitters[trtd.wk046wk22remitters$TRTGRINDMAN!="Placebo_Treated",]
summary(subset(trtd.wk046wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

plac.wk046wk22remitters<-wk046wk22remitters[wk046wk22remitters$TRTGRINDMAN!="Treated_Placebo",]
plac.wk046wk22remitters<-plac.wk046wk22remitters[plac.wk046wk22remitters$TRTGRINDMAN!="Treated_Treated",]
summary(subset(plac.wk046wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))


all.distmat<-read.table("../data/Jan400.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
all.dist<-as.dist(all.distmat)
ustdistmat<-read.table("../data/Jan400.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
ustdist<-as.dist(ustdistmat)
pladistmat<-read.table("../data/Jan400.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
pladist<-as.dist(pladistmat)
alltp_distmat<-read.table("../data/Jan400.alltp.noPU.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_dist<-as.dist(alltp_distmat)
alltp_ustdistmat<-read.table("../data/Jan400.all.na.omit.cmd.ust.alltp.thetayc.square.ave.dist")
alltp_ustdist<-as.dist(alltp_ustdistmat)
alltp_placplacdistmat<-read.table("../data/Jan400.alltp.plac.noPU.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_placplacdist<-as.dist(alltp_placplacdistmat)

wk046_distmat<-read.table("../data/Jan400.wk046.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk046_dist<-as.dist(wk046_distmat)
wk046_ustdistmat<-read.table("../data/Jan400.wk046.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk046_ustdist<-as.dist(wk046_ustdistmat)
wk046_placdistmat<-read.table("../data/Jan400.wk046.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk046_placdist<-as.dist(wk046_placdistmat)

wk0.ustdistmat<-read.table("../data/Jan400.alltp.ust.wk0.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk0.ustdist<-as.dist(wk0.ustdistmat)
wk4.ustdistmat<-read.table("../data/Jan400.alltp.ust.wk4.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk4.ustdist<-as.dist(wk4.ustdistmat)
wk6.ustdistmat<-read.table("../data/Jan400.alltp.ust.wk6.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk6.ustdist<-as.dist(wk6.ustdistmat)

wk0.placdistmat<-read.table("../data/Jan400.alltp.plac.wk0.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk0.placdist<-as.dist(wk0.placdistmat)
wk4.placdistmat<-read.table("../data/Jan400.alltp.plac.wk4.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk4.placdist<-as.dist(wk4.placdistmat)
wk6.placdistmat<-read.table("../data/Jan400.alltp.plac.wk6.na.omit.cmd.thetayc.0.03.square.ave.dist")
wk6.placdist<-as.dist(wk6.placdistmat)


alltp_ustdistmat.respY<-read.table("../data/Jan400.alltp.noPU.trtd.relRSP.Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_ustdist.respY<-as.dist(alltp_ustdistmat.respY)
alltp_placdistmat.respY<-read.table("../data/Jan400.alltp.noPU.plac.relRSP.Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_placdist.respY<-as.dist(alltp_placdistmat.respY)
alltp_distmat.respY<-read.table("../data/Jan400.alltp.noPU.relRSP.Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_dist.respY<-as.dist(alltp_distmat.respY)
alltp_distmat.respN<-read.table("../data/Jan400.alltp.noPU.relRSP.N.na.omit.cmd.thetayc.0.03.sq.ave.dist")
alltp_dist.respN<-as.dist(alltp_distmat.respN)

wk046.ust<-wk046_data[wk046_data$TRTGRINDMAN!="Placebo_Placebo",]
wk046.ust<-wk046.ust[wk046.ust$TRTGRINDMAN!="Placebo_Treated",]
summary(wk046.ust[,1:6])
wk046.plac<-wk046_data[wk046_data$TRTGRINDMAN!="Treated_Placebo",]
wk046.plac<-wk046.plac[wk046.plac$TRTGRINDMAN!="Treated_Treated",]
summary(wk046.plac[,1:6])

wk046.ust.relRSP22Y<-wk046.ust[wk046.ust$RelRSPwk22=="Yes",]
wk046.ust.relRSP22Y.distmat<-read.table("../data/Jan400.wk046.ust.relRSP22Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046.ust.relRSP22Y.dist<-as.dist(wk046.ust.relRSP22Y.distmat)

wk046.plac.relRSP22Y<-wk046.plac[wk046.plac$RelRSPwk22 =="Yes",]
wk046.plac.relRSP22Y.distmat<-read.table("../data/Jan400.wk046.plac.relRSP22Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046.plac.relRSP22Y.dist<-as.dist(wk046.plac.relRSP22Y.distmat)

wk046_data.relRSP22Y<-wk046_data[wk046_data$RelRSPwk22=="Yes",]
wk046_data.relRSP22Y.distmat<-read.table("../data/Jan400.wk046_data.relRSP22Y.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046_data.relRSP22Y.dist<-as.dist(wk046_data.relRSP22Y.distmat)

wk22remitters<-alltime_data[alltime_data$REMISSwk22=="Yes",]
summary(subset(wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

trtd.wk22remitters<-wk22remitters[wk22remitters$TRTGRINDMAN!="Placebo_Placebo",]
trtd.wk22remitters<-trtd.wk22remitters[trtd.wk22remitters$TRTGRINDMAN!="Placebo_Treated",]
summary(subset(trtd.wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

plac.wk22remitters<-wk22remitters[wk22remitters$TRTGRINDMAN!="Treated_Placebo",]
plac.wk22remitters<-plac.wk22remitters[plac.wk22remitters$TRTGRINDMAN!="Treated_Treated",]
summary(subset(plac.wk22remitters, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, REMISSwk22)))

wk22remitters.distmat<-read.table("../data/Jan400.alltp.wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk22remitters.dist<-as.dist(wk22remitters.distmat)
trtd.wk22remitters.distmat<-read.table("../data/Jan400.alltp.trtd.wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
trtd.wk22remitters.dist<-as.dist(trtd.wk22remitters.distmat)
plac.wk22remitters.distmat<-read.table("../data/Jan400.alltp.plac.wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
plac.wk22remitters.dist<-as.dist(plac.wk22remitters.distmat)

wk046wk22remitters.distmat<-read.table("../data/Jan400.wk046wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
wk046wk22remitters.dist<-as.dist(wk046wk22remitters.distmat)
trtd.wk046wk22remitters.distmat<-read.table("../data/Jan400.trtd.wk046wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
trtd.wk046wk22remitters.dist<-as.dist(trtd.wk046wk22remitters.distmat)
plac.wk046wk22remitters.distmat<-read.table("../data/Jan400.plac.wk046wk22remitters.na.omit.cmd.thetayc.0.03.sq.ave.dist")
plac.wk046wk22remitters.dist<-as.dist(plac.wk046wk22remitters.distmat)
```

```{r vars, include=F}
alltp.all.respY<-alltime_data[alltime_data$RelRSPwk22=="Yes",]
alltp.ust.RESPy<-alltp.ust[alltp.ust$RelRSPwk22=="Yes",]
alltp.placplac.RESPy<-alltp.placplac[alltp.placplac$RelRSPwk22=="Yes",]

#set.seed(32016)
#adonis(alltp_dist ~ TRTGR*RelRSPwk22, alltime_data, strata = alltime_data$visit)
```

```{r, eval=T, include=FALSE}
print("Summary of all subjects over weeks 0 4 and 6")
summary(subset(wk046_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

n.wk046 <- nrow(wk046_data[wk046_data$visit=="Week 6",])
n.wk046.trtd <- nrow(wk046.ust[wk046.ust$visit=="Week 6",])
n.wk046.plac <- nrow(wk046.plac[wk046.plac$visit=="Week 6",])
wk046.trtd.6 <- wk046.ust[wk046.ust$visit=="Week 6",]
wk046.plac.6 <- wk046.plac[wk046.plac$visit=="Week 6",]

n.wk046.trtd.respWK22Y <- nrow(wk046.trtd.6[wk046.trtd.6$RelRSPwk22 == "Yes",])
n.wk046.trtd.respWK22N <- nrow(wk046.trtd.6[wk046.trtd.6$RelRSPwk22 == "No",])
n.wk046.plac.respWK22Y <- nrow(wk046.plac.6[wk046.plac.6$RelRSPwk22 == "Yes",])
n.wk046.plac.respWK22N <- nrow(wk046.plac.6[wk046.plac.6$RelRSPwk22 == "No",])

all.wk046<-matrix(c('Week 22 Response (No, Yes)', trtd.RelRSPwk6_sum, plac.RelRSPwk6_sum, all.RelRSPwk6.kw.test.pval, all.RelRSPwk6_AMOVA), ncol = 5)

wk046.trtd_sum<-toString(c(n.wk046.trtd.respWK22N, n.wk046.trtd.respWK22Y))
wk046.plac_sum<-toString(c(n.wk046.plac.respWK22N, n.wk046.plac.respWK22Y))

wk046 <- matrix(c("Week 22 Response (No, Yes)", wk046.trtd_sum, wk046.plac_sum, n.wk046), ncol=4)
colnames(wk046) <- c("Samples at Week 0, 4, and 6", 'Treated (n)', "Placebo (n)", "Total (n)")


pdf("tables/wk046.sum.pdf", height=2, width=6)
grid.table(wk046)
dev.off()
```

```{r , include=F}
print("All subjects over weeks 0 4 and 6 by treatment and response at week 6")
set.seed(32016)
adonis(wk046_dist ~ visit*TRTGR*RelRSPwk6, wk046_data, strata = wk046_data$USUBJID, permutations = 2000)
```

```{r , include=F}
print("All subjects over weeks 0 4 and 6 by induction and relative response at week 22")
set.seed(32016)
wk046visXtrtgrXwk22.adonis <- adonis(wk046_dist ~ visit*TRTGR*RelRSPwk22, wk046_data, strata = wk046_data$USUBJID, permutations = 2000)

pdf("tables/SF2adonis.pdf", width = 8, height = 4)
grid.table(signif(wk046visXtrtgrXwk22.adonis$aov.tab, 2))
dev.off()
#print("All subjects over weeks 0 4 and 6 by induction and response at week 22")
#set.seed(32016)
#adonis(wk046_dist ~ visit*TRTGR*RESPONSEwk22, wk046_data, strata = wk046_data$USUBJID, permutations = 2000)
```

```{r, include=F}
print("Summary of all week 22 relative responders over weeks 0 4 and 6")
summary(subset(wk046_data.relRSP22Y, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))


print("Week 22 Relative Responders over weeks 0 4 and 6 by induction group and response at week 22")
set.seed(32016)
adonis(wk046_data.relRSP22Y.dist ~ visit*TRTGR, wk046_data.relRSP22Y, strata = wk046_data.relRSP22Y$USUBJID, permutations = 2000)
```

```{r , include=F}
print("Summary of all treated week 22 relative responders over weeks 0 4 and 6")
summary(subset(wk046.ust.relRSP22Y, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

set.seed(32016)
print("All treated week 22 responders over weeks 0 4 and 6")
adonis(wk046.ust.relRSP22Y.dist ~ visit, wk046.ust.relRSP22Y, strata = wk046.ust.relRSP22Y$USUBJID, permutations = 2000)
```

```{r , include=F}
print("Summary of all placebo week 22 relative responders over weeks 0 4 and 6")
summary(subset(wk046.plac.relRSP22Y, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

set.seed(32016)
print("All placebo week 22 responders over weeks 0 4 and 6")
adonis(wk046.plac.relRSP22Y.dist ~ visit, wk046.plac.relRSP22Y, strata = wk046.plac.relRSP22Y$USUBJID, permutations = 2000)
```

```{r wk046adivXvis}
source("../code/adivXvisit.facetTRTGR+RelRSPwk22.R")
```

```{r, include=F, }
summary(subset(alltime_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

set.seed(32016)


#ALL with strata
adonis(alltp_dist ~ visit*TRTGR*RelRSPwk22, alltime_data, strata = alltime_data$USUBJID)
```

```{r, include=F, }
print("all time points placebo subjects")
summary(subset(alltp.placplac, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

print("all time points placebo subjects")
#PLACEBO with strata
adonis(alltp_placplacdist ~ visit*RelRSPwk22, alltp.placplac, strata = alltp.placplac$USUBJID, permutations = 2000)
```

```{r, include=F, }
print("all time points treated subjects")
summary(subset(alltp.ust, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

print("all time points treated subjects")
#UST with strata
adonis(alltp_ustdist ~ visit*RelRSPwk22, alltp.ust, strata = alltp.ust$USUBJID, permutations = 2000)
```

```{r, include=F}
#alltp.all.respY<-alltp.all.respY[alltp.all.respY$TRTGRINDMAN!="Placebo_Treated",]
print("Summary of all week 22 responders")
summary(subset(alltp.all.respY, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

print("All week 22 responders by treatement group over all times")
set.seed(32016)
adonis(alltp_dist.respY ~ visit*TRTGR, alltp.all.respY, strata = alltp.all.respY$USUBJID, permutations = 2000)

print("All week 22 NONresponders by treatement group over all times")
set.seed(32016)
adonis(alltp_dist.respN ~ visit*TRTGR, alltp.all.respN, strata = alltp.all.respN$USUBJID, permutations = 2000)
```

```{r, include=F}
print("Summary of treated week 22 responders")
summary(subset(alltp.ust.RESPy, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

print("All treated week 22 responders over all times")
set.seed(32016)
adonis(alltp_ustdist.respY ~ visit, alltp.ust.RESPy, strata = alltp.ust.RESPy$USUBJID, permutations = 2000)
```

```{r visit all4tp, include=F}
set.seed(32016)
print("Summary of week 22 response")
summary(alltp.plac$RelRSPwk22)

print("Summary of placebo week 22 responders")
summary(subset(alltp.placplac.RESPy, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk6, RelRSPwk6, REMISSwk6, RESPONSEwk22, RelRSPwk22, REMISSwk22)))

print("All placebo week 22 responders alltimes")
set.seed(32016)
adonis(alltp_placdist.respY ~ visit, alltp.placplac.RESPy, strata = alltp.placplac.RESPy$USUBJID, permutations = 2000)
```

```{r readdataAUCRF, include=FALSE}
#Read in all data
source('../code/APmd.setup.R')

APmd_adiv<-merge(APmd, adiv, by.x="group", by.y="group")#nonmatch removed
APmd_shared<-merge(APmd_adiv, ss3k_shared_U2s, by.x="group", by.y="Group")

APmd_shared$TRTGR_REMISSwk6 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$REMISSwk6))
APmd_shared$TRTGR_REMISSwk22 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$REMISSwk22))

APmd_shared$TRTGR_RelRSPwk6 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$RelRSPwk6))
APmd_shared$TRTGR_RelRSPwk22 <- as.factor(paste(APmd_shared$TRTGR, APmd_shared$RelRSPwk22))
APmd_RSPwk22<-APmd_shared

screen_RESPwk22<-APmd_RSPwk22[APmd_RSPwk22$visit=="Screening",] #removes any group with NA in RESPONSEwk22

screen_data<-screen_RESPwk22				#na.omit(screen_RESPwk22)

screen.trtd<-screen_data[screen_data$TRTGRINDMAN!="Placebo_Placebo",]
screen.trtd<-screen.trtd[screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(screen.trtd[,1:6])
screen.plac<-screen_data[screen_data$TRTGRINDMAN!="Treated_Placebo",]
screen.plac<-screen.plac[screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(screen.plac[,1:6])

nao.screen_data<-na.omit(screen_data)

nao.screen.trtd<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Placebo_Placebo",]
nao.screen.trtd<-nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN!="Placebo_Treated",]
summary(nao.screen.trtd[,1:6])
nao.screen.plac<-nao.screen_data[nao.screen_data$TRTGRINDMAN!="Treated_Placebo",]
nao.screen.plac<-nao.screen.plac[nao.screen.plac$TRTGRINDMAN!="Treated_Treated",]
summary(nao.screen.plac[,1:6])

#axes <- read.table('../data/Jan400.nao.screen_trtd.3107.subsample.thetayc.0.03.lt.ave.nmds.by3.axes', header=T)
tax<- read.table('../data/Jan400.simple.taxonomy.txt', header=T, sep="\t")
nao.screen.all.distmat<-read.table("../data/Jan400.screening.all.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.all.dist<-as.dist(nao.screen.all.distmat)
nao.screen.trtd.distmat<-read.table("../data/Jan400.screening.ust.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.trtd.dist<-as.dist(nao.screen.trtd.distmat)
nao.screen.plac.distmat<-read.table("../data/Jan400.screening.plac.na.omit.cmd.thetayc.0.03.square.ave.dist")
nao.screen.plac.dist<-as.dist(nao.screen.plac.distmat)

screen.all.distmat<-read.table("../data/Jan400.screen.all.thetayc.0.03.square.ave.dist")
screen.all.dist<-as.dist(screen.all.distmat)
screen.trtd.distmat<-read.table("../data/Jan400.screen.ust.thetayc.0.03.square.ave.dist")
screen.trtd.dist<-as.dist(screen.trtd.distmat)
screen.plac.distmat<-read.table("../data/Jan400.screen.plac.thetayc.0.03.square.ave.dist")
screen.plac.dist<-as.dist(screen.plac.distmat)
```	

```{r RESPONSEwk6-AUC-RF, eval=FALSE, fig.height=5, fig.width=10, include=FALSE}
print("RESPONSEwk6 by AUC-RF")
set.seed(32016)
levels(nao.screen.trtd$RESPONSEwk6) <- c(0,1)
mdotu_respWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RESPONSEwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_respWK6[,c("RESPONSEwk6")])
mdotu_respWK6<-na.omit(mdotu_respWK6)
#summary(mdotu_respWK6[,c("RESPONSEwk6")])
otu_respWK6 <- mdotu_respWK6[, c('RESPONSEwk6', colnames(mdotu_respWK6)[grep('Otu[0123456789]', colnames(mdotu_respWK6))])]
#summary(otu_respWK6[,c("RESPONSEwk6")])
otu_respWK6<-na.omit(otu_respWK6)
#summary(otu_respWK6[,c("RESPONSEwk6")])
set.seed(32016)
otu_respWK6_rf <- AUCRF(RESPONSEwk6~., data=otu_respWK6, ranking='MDA', ntree=500, pdel=0.2)
#otu_respWK6_rf # prints list of optimal OTUs
otu_respWK6_probs <- predict(otu_respWK6_rf$RFopt, type='prob')[,2]
otu_respWK6_roc <- roc(otu_respWK6$RESPONSEwk6~otu_respWK6_probs)


md_respWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RESPONSEwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_respWK6[,c("RESPONSEwk6")])
md_respWK6<-na.omit(md_respWK6)
#summary(md_respWK6[,c("RESPONSEwk6")])
set.seed(32016)
md_respWK6_rf <- AUCRF(RESPONSEwk6~., data=md_respWK6, ranking='MDA', ntree=500, pdel=0.3)
#md_respWK6_rf # prints list of optimal features
md_respWK6_probs <- predict(md_respWK6_rf$RFopt, type='prob')[,2]
md_respWK6_roc <- roc(md_respWK6$RESPONSEwk6~md_respWK6_probs)


mdotu_respWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RESPONSEwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_respWK6[,c("RESPONSEwk6")])
mdotu_respWK6<-na.omit(mdotu_respWK6)
#summary(mdotu_respWK6[,c("RESPONSEwk6")])
set.seed(32016)
mdotu_respWK6_rf <- AUCRF(RESPONSEwk6~., data=mdotu_respWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_respWK6_rf # prints list of optimal features
mdotu_respWK6_probs <- predict(mdotu_respWK6_rf$RFopt, type='prob')[,2]
mdotu_respWK6_roc <- roc(mdotu_respWK6$RESPONSEwk6~mdotu_respWK6_probs)

#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_respWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_respWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(mdotu_respWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Response using Week 0 Clinical Data")
legend('bottomright', sprintf('Clinical Data: AUC=%.3f',md_respWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, md_respWK6_probs[md_respWK6$'RESPONSEwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, md_respWK6_probs[md_respWK6$'RESPONSEwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=md_respWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

md_respWK6_rf # prints list of optimal features


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_respWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_respWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_respWK6_roc, add=T, lwd=2, col='red', lty=1)
title("RESPONSEwk6")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_respWK6_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_respWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_respWK6_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_respWK6_probs[mdotu_respWK6$'RESPONSEwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_respWK6_probs[mdotu_respWK6$'RESPONSEwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_respWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_respWK6_rf # prints list of optimal features
otu_respWK6_rf
```

```{r RelRSPwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("RelRSPwk6 by AUC-RF \\nRelative response defined as a decrease in CDAI from Screening greater than 29.5%.")
set.seed(32016)
levels(nao.screen.trtd$RelRSPwk6) <- c(0,1)
mdotu_relrespWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
mdotu_relrespWK6<-na.omit(mdotu_relrespWK6)
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
otu_relrespWK6 <- mdotu_relrespWK6[, c('RelRSPwk6', colnames(mdotu_relrespWK6)[grep('Otu[0123456789]', colnames(mdotu_relrespWK6))])]
#summary(otu_relrespWK6[,c("RelRSPwk6")])
#summary(otu_relrespWK6[,c("RelRSPwk6")])
otu_relrespWK6<-na.omit(otu_relrespWK6)
#summary(otu_relrespWK6[,c("RelRSPwk6")])
set.seed(32016)
otu_relrespWK6_rf <- AUCRF(RelRSPwk6~., data=otu_relrespWK6, ranking='MDA', ntree=500, pdel=0.2)
##otu_relrespWK6_rf # prints list of optimal OTUs
otu_relrespWK6_probs <- predict(otu_relrespWK6_rf$RFopt, type='prob')[,2]
otu_relrespWK6_roc <- roc(otu_relrespWK6$RelRSPwk6~otu_relrespWK6_probs)


md_relrespWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_relrespWK6[,c("RelRSPwk6")])
md_relrespWK6<-na.omit(md_relrespWK6)
#summary(md_relrespWK6[,c("RelRSPwk6")])
set.seed(32016)
md_relrespWK6_rf <- AUCRF(RelRSPwk6~., data=md_relrespWK6, ranking='MDA', ntree=500, pdel=0.3)
##md_relrespWK6_rf # prints list of optimal features
md_relrespWK6_probs <- predict(md_relrespWK6_rf$RFopt, type='prob')[,2]
md_relrespWK6_roc <- roc(md_relrespWK6$RelRSPwk6~md_relrespWK6_probs)


mdotu_relrespWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
mdotu_relrespWK6<-na.omit(mdotu_relrespWK6)
#summary(mdotu_relrespWK6[,c("RelRSPwk6")])
set.seed(32016)
mdotu_relrespWK6_rf <- AUCRF(RelRSPwk6~., data=mdotu_relrespWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_relrespWK6_rf # prints list of optimal features
mdotu_relrespWK6_probs <- predict(mdotu_relrespWK6_rf$RFopt, type='prob')[,2]
mdotu_relrespWK6_roc <- roc(mdotu_relrespWK6$RelRSPwk6~mdotu_relrespWK6_probs)
mdotu_relrespWK6_rf
otu_relrespWK6_rf
md_relrespWK6_rf
```

```{r RelRSPwk6AUCRFinline, include=F, fig.height=5, fig.width=10}
#pdf('figures/mdrelRSPwk6aucrfcomb0.pdf', height=6, width=8)
### Plot ROC curves
#layout(matrix(c(1,2), nrow=1))
#par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
#plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_relrespWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_relrespWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(md_relrespWK6_roc, add=T, lwd=2, col='red', lty=1)
#title("Predicting Week 6 Response")
#legend('bottomright', sprintf('Week 0 Clinical Data: AUC=%.3f',md_relrespWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)
#points(coords(md_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='grey5', cex=1.5, pch=10)

#dev.off()
#dev.off()


#pdf('figures/RSPwk6abunds.pdf', height=6, width=8)
#plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='Response Week 6', xaxt='n', cex.axis=1.2)
#title('Response Status Compared to Clinical Model')
#stripchart(at=1, md_relrespWK6_probs[md_relrespWK6$'RelRSPwk6'==0], vertical=T, method='jitter', #jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
#stripchart(at=2, md_relrespWK6_probs[md_relrespWK6$'RelRSPwk6'==1], vertical=T, method='jitter', #jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
#mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
#axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=md_relrespWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#abline(h=cutoff, lty=2)
#legend("top", lty=2, sprintf('Model Threshold'))
#dev.off()

#md_relrespWK6_rf 

# prints list of optimal features
#pdf('figures/mdoturelRSPwk6aucrf.pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_relrespWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_relrespWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_relrespWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Response")
legend('bottomright', c(sprintf('Week 0 OTUs : AUC=%.3f',otu_relrespWK6_roc$auc), sprintf('Week 0 Clinical Data: AUC=%.3f',md_relrespWK6_roc$auc), sprintf('Combined: AUC=%.3f',mdotu_relrespWK6_roc$auc)), lwd=2, col=c('royalblue', 'grey5', 'red'), bty='n', lty=1)
#legend('bottomright', c(sprintf('Week 0 OTUs : AUC=%.3f',otu_relrespWK6_roc$auc), sprintf('Combined: AUC=%.3f',mdotu_relrespWK6_roc$auc)), lwd=2, col=c('royalblue', 'red'), bty='n', lty=1)

cutoff <- coords(roc=mdotu_relrespWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_relrespWK6_spec <- coords(mdotu_relrespWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_relrespWK6_sens <- coords(mdotu_relrespWK6_roc, x=mdotu_relrespWK6_spec, input='spec', ret='sens')
mdotu_relrespWK6_thr <- coords(mdotu_relrespWK6_roc, x=mdotu_relrespWK6_spec, input='spec', ret='thr')

points(coords(mdotu_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_relrespWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_relrespWK6_spec <- coords(otu_relrespWK6_roc, x=cutoff, input='thr', ret='spec')
otu_relrespWK6_sens <- coords(otu_relrespWK6_roc, x=otu_relrespWK6_spec, input='spec', ret='sens')
otu_relrespWK6_thr <- coords(otu_relrespWK6_roc, x=otu_relrespWK6_spec, input='spec', ret='thr')
points(coords(otu_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_relrespWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_relrespWK6_spec <- coords(md_relrespWK6_roc, x=cutoff, input='thr', ret='spec')
md_relrespWK6_sens <- coords(md_relrespWK6_roc, x=md_relrespWK6_spec, input='spec', ret='sens')
md_relrespWK6_thr <- coords(md_relrespWK6_roc, x=md_relrespWK6_spec, input='spec', ret='thr')

points(coords(md_relrespWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_relrespWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)
#dev.off()
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='Response Week 6', xaxt='n', cex.axis=1.2)
title('Response Status Compared to Combined Model')
stripchart(at=1, mdotu_relrespWK6_probs[mdotu_relrespWK6$'RelRSPwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_relrespWK6_probs[mdotu_relrespWK6$'RelRSPwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_relrespWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
abline(h=cutoff, lty=2)
legend("top", lty=2, sprintf('Model Threshold'))
#dev.off()

#mdotu_relrespWK6_rf # prints list of optimal features
otu_relrespWK6_rf 
```

```{r REMISSwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("REMISSwk6 by AUC-RF")
set.seed(32016)
levels(nao.screen.trtd$REMISSwk6) <- c(0,1)
mdotu_remWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_remWK6[,c("REMISSwk6")])
mdotu_remWK6<-na.omit(mdotu_remWK6)
#summary(mdotu_remWK6[,c("REMISSwk6")])
otu_remWK6 <- mdotu_remWK6[, c('REMISSwk6', colnames(mdotu_remWK6)[grep('Otu[0123456789]', colnames(mdotu_remWK6))])]
#summary(otu_remWK6[,c("REMISSwk6")])
#summary(otu_remWK6[,c("REMISSwk6")])
otu_remWK6<-na.omit(otu_remWK6)
#summary(otu_remWK6[,c("REMISSwk6")])
set.seed(32016)
otu_remWK6_rf <- AUCRF(REMISSwk6~., data=otu_remWK6, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK6_rf # prints list of optimal OTUs
otu_remWK6_probs <- predict(otu_remWK6_rf$RFopt, type='prob')[,2]
otu_remWK6_roc <- roc(otu_remWK6$REMISSwk6~otu_remWK6_probs)


md_remWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_remWK6[,c("REMISSwk6")])
md_remWK6<-na.omit(md_remWK6)
#summary(md_remWK6[,c("REMISSwk6")])
set.seed(32016)
md_remWK6_rf <- AUCRF(REMISSwk6~., data=md_remWK6, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK6_rf # prints list of optimal features
md_remWK6_probs <- predict(md_remWK6_rf$RFopt, type='prob')[,2]
md_remWK6_roc <- roc(md_remWK6$REMISSwk6~md_remWK6_probs)


mdotu_remWK6 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('REMISSwk6', "AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", 'invsimpson', colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_remWK6[,c("REMISSwk6")])
mdotu_remWK6<-na.omit(mdotu_remWK6)
#summary(mdotu_remWK6[,c("REMISSwk6")])
set.seed(32016)
mdotu_remWK6_rf <- AUCRF(REMISSwk6~., data=mdotu_remWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK6_rf # prints list of optimal features
mdotu_remWK6_probs <- predict(mdotu_remWK6_rf$RFopt, type='prob')[,2]
mdotu_remWK6_roc <- roc(mdotu_remWK6$REMISSwk6~mdotu_remWK6_probs)

set.seed(32016)
otu_remWK6_rf_cv <- AUCRFcv(otu_remWK6_rf)
set.seed(32016)
mdotu_remWK6_rf_cv <- AUCRFcv(mdotu_remWK6_rf)
```

```{r}
otu_remWK6_rf_cv
mdotu_remWK6_rf_cv
```

```{r REMISSwk6AUCRFinline, fig.height=5, fig.width=10, include=FALSE}
#pdf('figures/mdREMISSIwk6aucrfcomb0.pdf', height=6, width=8)
### Plot ROC curves
#layout(matrix(c(1,2), nrow=1))
#par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
#plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_remWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_remWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(md_remWK6_roc, add=T, lwd=2, col='red', lty=1)
#title("Predicting Week 6 Remission")
#legend('bottomright', sprintf('Week 0 Clinical Data: AUC=%.3f',md_remWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)
#cutoff <- coords(roc=md_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
#md_remWK6_spec <- coords(md_remWK6_roc, x=100, input='thr', ret='spec')
#md_remWK6_sens <- coords(md_remWK6_roc, x=md_remWK6_spec, input='spec', ret='sens')
#md_remWK6_thr <- coords(md_remWK6_roc, x=md_remWK6_spec, input='spec', ret='thr')
#points(coords(md_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='grey5', cex=1.5, pch=10)
#dev.off()

#pdf('figures/RSPwk6abunds.pdf', height=6, width=8)
#plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='Remission Week 6', xaxt='n', cex.axis=1.2)
#title('Remission Status Compared to Clinical Model')

#stripchart(at=1, md_remWK6_probs[md_remWK6$REMISSwk6=='0'], vertical=T, method='jitter', jitter=0.4, #pch=21, bg='orange', add=T, cex=1.5)
#stripchart(at=2, md_remWK6_probs[md_remWK6$REMISSwk6=='1'], vertical=T, method='jitter', jitter=0.4, #pch=21, bg='royalblue', add=T, cex=1.5)
#mtext(side=2, text='Probability of Remission', line=2.2, cex=0.8)
#axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
#cutoff <- coords(roc=md_remWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#abline(h=cutoff, lty=2)
#legend("top", lty=2, sprintf('Model Threshold'))
#dev.off()

#md_remWK6_rf 

# prints list of optimal features
#pdf('figures/mdotuREMISSIwk6aucrf.pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_remWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Remission")
legend('bottomright', c(sprintf('Week 0 OTUs : AUC=%.3f', otu_remWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK6_roc$auc), sprintf('Week 0 Clinical Data: AUC=%.3f',md_remWK6_roc$auc)), lwd=2, col=c('royalblue', 'red', "grey5"), bty='n', lty=1)

cutoff <- coords(roc=mdotu_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_remWK6_spec <- coords(mdotu_remWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_remWK6_sens <- coords(mdotu_remWK6_roc, x=mdotu_remWK6_spec, input='spec', ret='sens')
mdotu_remWK6_thr <- coords(mdotu_remWK6_roc, x=mdotu_remWK6_spec, input='spec', ret='thr')

points(coords(mdotu_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_remWK6_spec <- coords(otu_remWK6_roc, x=cutoff, input='thr', ret='spec')
otu_remWK6_sens <- coords(otu_remWK6_roc, x=otu_remWK6_spec, input='spec', ret='sens')
otu_remWK6_thr <- coords(otu_remWK6_roc, x=otu_remWK6_spec, input='spec', ret='thr')
points(coords(otu_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_remWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_remWK6_spec <- coords(md_remWK6_roc, x=cutoff, input='thr', ret='spec')
md_remWK6_sens <- coords(md_remWK6_roc, x=md_remWK6_spec, input='spec', ret='sens')
md_remWK6_thr <- coords(md_remWK6_roc, x=md_remWK6_spec, input='spec', ret='thr')

points(coords(md_remWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_remWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='Remission Week 6', xaxt='n', cex.axis=1.2)
title('Remission Status Compared to Combined Model')
stripchart(at=1, mdotu_remWK6_probs[mdotu_remWK6$REMISSwk6=='0'], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK6_probs[mdotu_remWK6$REMISSwk6=='1'], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
abline(h=cutoff, lty=2)
legend("top", lty=2, sprintf('Model Threshold'))
#dev.off()

#mdotu_remWK6_rf # prints list of optimal features
otu_remWK6_rf 
```


```{r RESPONSEwk22-AUC-RF, eval=FALSE, fig.height=5, fig.width=10, include=FALSE}
print("RESPONSEwk22 by AUC-RF")
set.seed(32016)
levels(nao.screen.trtd$RESPONSEwk22) <- c(0,1)
#nao.screen.trtd<-nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN=="Treated_Treated", ]

mdotu_respWK22 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RESPONSEwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_respWK22[,c("RESPONSEwk22")])
mdotu_respWK22<-na.omit(mdotu_respWK22)
#summary(mdotu_respWK22[,c("RESPONSEwk22")])
otu_respWK22 <- mdotu_respWK22[, c('RESPONSEwk22', colnames(mdotu_respWK22)[grep('Otu[0123456789]', colnames(mdotu_respWK22))])]
#summary(otu_respWK22[,c("RESPONSEwk22")])
otu_respWK22<-na.omit(otu_respWK22)
#summary(otu_respWK22[,c("RESPONSEwk22")])
set.seed(32016)
otu_respWK22_rf <- AUCRF(RESPONSEwk22~., data=otu_respWK22, ranking='MDA', ntree=500, pdel=0.2)
#otu_respWK22_rf # prints list of optimal OTUs
otu_respWK22_probs <- predict(otu_respWK22_rf$RFopt, type='prob')[,2]
otu_respWK22_roc <- roc(otu_respWK22$RESPONSEwk22~otu_respWK22_probs)


md_respWK22 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RESPONSEwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_respWK22[,c("RESPONSEwk22")])
md_respWK22<-na.omit(md_respWK22)
#summary(md_respWK22[,c("RESPONSEwk22")])
set.seed(32016)
md_respWK22_rf <- AUCRF(RESPONSEwk22~., data=md_respWK22, ranking='MDA', ntree=500, pdel=0.3)
#md_respWK22_rf # prints list of optimal features
md_respWK22_probs <- predict(md_respWK22_rf$RFopt, type='prob')[,2]
md_respWK22_roc <- roc(md_respWK22$RESPONSEwk22~md_respWK22_probs)


mdotu_respWK22 <- nao.screen.trtd[nao.screen.trtd$visit=='Screening', c('RESPONSEwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd))])]
#summary(mdotu_respWK22[,c("RESPONSEwk22")])
mdotu_respWK22<-na.omit(mdotu_respWK22)
#summary(mdotu_respWK22[,c("RESPONSEwk22")])
set.seed(32016)
mdotu_respWK22_rf <- AUCRF(RESPONSEwk22~., data=mdotu_respWK22, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_respWK22_rf # prints list of optimal features
mdotu_respWK22_probs <- predict(mdotu_respWK22_rf$RFopt, type='prob')[,2]
mdotu_respWK22_roc <- roc(mdotu_respWK22$RESPONSEwk22~mdotu_respWK22_probs)


#pdf('data/figures/RSPwk22aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_respWK22_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_respWK22_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_respWK22_roc, add=T, lwd=2, col='red', lty=1)
title("RESPONSEwk22")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_respWK22_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_respWK22_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_respWK22_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk22abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_respWK22_probs[mdotu_respWK22$'RESPONSEwk22'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_respWK22_probs[mdotu_respWK22$'RESPONSEwk22'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_respWK22_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_respWK22_rf # prints list of optimal features

```

```{r RelRSPwk22-AUC-RF, include=F, fig.height=5, fig.width=10}
print("RelRSPwk22 by AUC-RF \\nRelative respWK22onse defined as a decrease in CDAI from Screening greater than 29.5%.")
set.seed(32016)

levels(nao.screen.trtd$RelRSPwk22) <- c(0,1)
nao.screen.trtd.trtd <- nao.screen.trtd[nao.screen.trtd$TRTGRINDMAN == "Treated_Treated", ]
mdotu_relrespWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('RelRSPwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
mdotu_relrespWK22<-na.omit(mdotu_relrespWK22)
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
otu_relrespWK22 <- mdotu_relrespWK22[, c('RelRSPwk22', colnames(mdotu_relrespWK22)[grep('Otu[0123456789]', colnames(mdotu_relrespWK22))])]
#summary(otu_relrespWK22[,c("RelRSPwk22")])
#summary(otu_relrespWK22[,c("RelRSPwk22")])
otu_relrespWK22<-na.omit(otu_relrespWK22)
#summary(otu_relrespWK22[,c("RelRSPwk22")])
set.seed(32016)
otu_relrespWK22_rf <- AUCRF(RelRSPwk22~., data=otu_relrespWK22, ranking='MDA', ntree=500, pdel=0.2)
##otu_relrespWK22_rf # prints list of optimal OTUs
otu_relrespWK22_probs <- predict(otu_relrespWK22_rf$RFopt, type='prob')[,2]
otu_relrespWK22_roc <- roc(otu_relrespWK22$RelRSPwk22~otu_relrespWK22_probs)


md_relrespWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('RelRSPwk22','AGE', 'SEX', 'RACE', 'BAMINO', 'BANTIBIO', 'BCORT', 'BIMM', 'BL_BMI', 'DDUR', 'DDUR10', 'DDUR15', 'tissinvol', 'BL_FLACT', 'BL_FCALP', 'BL_CRP', 'BOWSTRICT')]
#summary(md_relrespWK22[,c("RelRSPwk22")])
md_relrespWK22<-na.omit(md_relrespWK22)
#summary(md_relrespWK22[,c("RelRSPwk22")])
set.seed(32016)
md_relrespWK22_rf <- AUCRF(RelRSPwk22~., data=md_relrespWK22, ranking='MDA', ntree=500, pdel=0.3)
##md_relrespWK22_rf # prints list of optimal features
md_relrespWK22_probs <- predict(md_relrespWK22_rf$RFopt, type='prob')[,2]
md_relrespWK22_roc <- roc(md_relrespWK22$RelRSPwk22~md_relrespWK22_probs)


mdotu_relrespWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('RelRSPwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
mdotu_relrespWK22<-na.omit(mdotu_relrespWK22)
#summary(mdotu_relrespWK22[,c("RelRSPwk22")])
set.seed(32016)
mdotu_relrespWK22_rf <- AUCRF(RelRSPwk22~., data=mdotu_relrespWK22, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_relrespWK22_rf # prints list of optimal features
mdotu_relrespWK22_probs <- predict(mdotu_relrespWK22_rf$RFopt, type='prob')[,2]
mdotu_relrespWK22_roc <- roc(mdotu_relrespWK22$RelRSPwk22~mdotu_relrespWK22_probs)


#pdf#pdf('data/figures/relRSPwk22aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_relrespWK22_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_relrespWK22_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_relrespWK22_roc, add=T, lwd=2, col='red', lty=1)
title("RelRSPwk22")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_relrespWK22_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_relrespWK22_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_relrespWK22_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk22abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_relrespWK22_probs[mdotu_relrespWK22$'RelRSPwk22'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_relrespWK22_probs[mdotu_relrespWK22$'RelRSPwk22'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_relrespWK22_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_relrespWK22_rf # prints list of optimal features
```

```{r REMISSwk22-AUC-RF, include=F, fig.height=5, fig.width=10}
print("REMISSwk22 by AUC-RF")
set.seed(32016)
levels(nao.screen.trtd.trtd$REMISSwk22) <- c(0,1)
mdotu_remWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_remWK22[,c("REMISSwk22")])
mdotu_remWK22<-na.omit(mdotu_remWK22)
#summary(mdotu_remWK22[,c("REMISSwk22")])
otu_remWK22 <- mdotu_remWK22[, c('REMISSwk22', colnames(mdotu_remWK22)[grep('Otu[0123456789]', colnames(mdotu_remWK22))])]
#summary(otu_remWK22[,c("REMISSwk22")])
#summary(otu_remWK22[,c("REMISSwk22")])
otu_remWK22<-na.omit(otu_remWK22)
#summary(otu_remWK22[,c("REMISSwk22")])
set.seed(32016)
otu_remWK22_rf <- AUCRF(REMISSwk22~., data=otu_remWK22, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK22_rf # prints list of optimal OTUs
otu_remWK22_probs <- predict(otu_remWK22_rf$RFopt, type='prob')[,2]
otu_remWK22_roc <- roc(otu_remWK22$REMISSwk22~otu_remWK22_probs)


md_remWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening")]
#summary(md_remWK22[,c("REMISSwk22")])
md_remWK22<-na.omit(md_remWK22)
#summary(md_remWK22[,c("REMISSwk22")])
set.seed(32016)
md_remWK22_rf <- AUCRF(REMISSwk22~., data=md_remWK22, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK22_rf # prints list of optimal features
md_remWK22_probs <- predict(md_remWK22_rf$RFopt, type='prob')[,2]
md_remWK22_roc <- roc(md_remWK22$REMISSwk22~md_remWK22_probs)


mdotu_remWK22 <- nao.screen.trtd.trtd[nao.screen.trtd.trtd$visit=='Screening', c('REMISSwk22', 'AGE', 'SEX', 'RACE', 'BAMINO', 'BANTIBIO', 'BCORT', 'BIMM', 'BL_BMI', 'DDUR', 'DDUR10', 'DDUR15', 'tissinvol', 'BL_FLACT', 'BL_FCALP', 'BL_CRP', 'BOWSTRICT', 'invsimpson', colnames(nao.screen.trtd.trtd)[grep('Otu[0123456789]', colnames(nao.screen.trtd.trtd))])]
#summary(mdotu_remWK22[,c("REMISSwk22")])
mdotu_remWK22<-na.omit(mdotu_remWK22)
#summary(mdotu_remWK22[,c("REMISSwk22")])
set.seed(32016)
mdotu_remWK22_rf <- AUCRF(REMISSwk22~., data=mdotu_remWK22, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK22_rf # prints list of optimal features
mdotu_remWK22_probs <- predict(mdotu_remWK22_rf$RFopt, type='prob')[,2]
mdotu_remWK22_roc <- roc(mdotu_remWK22$REMISSwk22~mdotu_remWK22_probs)



### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK22_roc, add=T, lwd=2, col='royalblue', lty=1)
#plot(md_remWK22_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK22_roc, add=T, lwd=2, col='red', lty=1)
title("REMISSwk22")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_remWK22_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_remWK22_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK22_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)

plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_remWK22_probs[mdotu_remWK22$'REMISSwk22'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK22_probs[mdotu_remWK22$'REMISSwk22'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK22_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff

mdotu_remWK22_rf # prints list of optimal features

set.seed(32016)
otu_remWK22_rf_cv <- AUCRFcv(otu_remWK22_rf)
set.seed(32016)
mdotu_remWK22_rf_cv <- AUCRFcv(mdotu_remWK22_rf)
```

```{r}
otu_remWK22_rf_cv
mdotu_remWK22_rf_cv
```

```{r , include=F}

tax<-read.table("../data/Jan400.simple.taxonomy.txt", header=T, sep='\t')


#Abundance stripchart or most predictive otus
oturf_otus  <- as.character(otu_relrespWK22_rf$Xopt)
no_abunds <- otu_relrespWK22[otu_relrespWK22$RelRSPwk22=='0', oturf_otus]/10000 + 1e-5
no_abunds <-na.omit(no_abunds)
yes_abunds <- otu_relrespWK22[otu_relrespWK22$RelRSPwk22=='1', oturf_otus]/10000 + 1e-5
yes_abunds<- na.omit(yes_abunds)

layout(1)
par(mar=c(4, 9, 0, 1))
plot(1, type="n", ylim=c(0,length(oturf_otus )*2), xlim=c(1e-5,3), log="x", ylab="", xlab="Relative Abundance (%)", xaxt="n", yaxt="n")
index <- 1
for(i in oturf_otus ){
	stripchart(at=index-0.35, jitter(yes_abunds[,i], amount=1e-5), pch=21, bg="royalblue1", method="jitter", jitter=0.2, add=T, cex=1, lwd=0.5)
	stripchart(at=index+0.35, jitter(no_abunds[,i], amount=1e-5), pch=21, bg="orange", method="jitter", jitter=0.2, add=T, cex=1, lwd=0.5)
	segments(mean(yes_abunds[,i]),index-0.7,mean(yes_abunds[,i]),index, lwd=3)
	segments(mean(no_abunds[,i]),index+0.7,mean(no_abunds[,i]),index, lwd=3)
	index <- index + 2
}
oturf_otus<-as.factor(otu_relrespWK22_rf$Xopt)
axis(2, at=seq(1,index-2,2), labels=tax[tax$OTU %in% oturf_otus, 'Label'][as.factor(oturf_otus)], las=1, line=-0.5, tick=F, cex.axis=0.8)
axis(1, at=c(1e-5, 1e-4, 1e-3, 1e-2, 1e-1), label=c("0", "0.01", "0.1", "1", "10"))
legend('topright', legend=c("No", "Yes"), pch=c(21, 21), pt.bg=c("orange","royalblue1"), cex=0.7)
```

```{r week6readdata, include=F}

md<-read.table(file="../data/Jan400.mdshared.wmg.txt", sep="\t", header=T) #wmg=with merged groups
APmd.wk6<-subset(md, select=c(group, USUBJID, Site, visit, TRTGR, TRTGRINDMAN:COUNTRY, BAMINO, BANTIBIO, BCORT, BIMM, BL_BMI:tissinvol, BRSP_TNF, BL_FLACT:BL_FLACTLEV, BOWSTRICT, RESPONSEwk4:REMISSwk22, RSP_Rwk8n22:RSP_SRwk8N22, REMISSwk6, RESPONSEwk6, cdail_wk0, cdail_wk4, cdail_wk6, cdail_wk8, cdail_wk22, CDSTOOLM_Week.6, CDPAINM_Week.6, CDWELBEM_Week.6, CDARTRTM_Week.6, CDARALGM_Week.6, CDIRITM_Week.6, CDUVEITM_Week.6, CDERYTHM_Week.6, CDPYODEM_Week.6, CDSTOMAM_Week.6, CDFISURM_Week.6, CDFISTUM_Week.6, CDABSCEM_Week.6, CDOTFISM_Week.6, CDFEVERM_Week.6, CDDRGM_Week.6, CDAMASSM_Week.6, CDHCTM_Week.6, CDWEIGHM_Week.6))

levels(APmd.wk6$TRTGR)<-c("Placebo", "Treated")
levels(APmd.wk6$TRTGRINDMAN)<-c("Placebo_Not", "Placebo_Placebo", "Placebo_Treated", "Treated_Not", "Treated_Placebo", "Treated_Treated")


APmd.wk6$CDSTOOLM_Week.6[APmd.wk6$CDSTOOLM_Week.6=="."]<-"NA"
APmd.wk6$CDSTOOLM_Week.6 <-as.integer(APmd.wk6$CDSTOOLM_Week.6)
APmd.wk6$CDHCTM_Week.6[APmd.wk6$CDHCTM_Week.6=="."]<-"NA"
APmd.wk6$CDHCTM_Week.6 <-as.integer(APmd.wk6$CDHCTM_Week.6)
summary(subset(APmd.wk6, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

deltaCDAIwk0_22<-APmd.wk6$cdail_wk22-APmd.wk6$cdail_wk0
deltaCDAIwk0_8<-APmd.wk6$cdail_wk8-APmd.wk6$cdail_wk0
deltaCDAIwk0_4<-APmd.wk6$cdail_wk4-APmd.wk6$cdail_wk0
deltaCDAIwk0_6<-APmd.wk6$cdail_wk6-APmd.wk6$cdail_wk0
percentdelta0_4<-deltaCDAIwk0_4/APmd.wk6$cdail_wk0*100
percentdelta0_6<-deltaCDAIwk0_6/APmd.wk6$cdail_wk0*100
percentdelta0_8<-deltaCDAIwk0_8/APmd.wk6$cdail_wk0*100
percentdelta0_22<-deltaCDAIwk0_22/APmd.wk6$cdail_wk0*100
deltaCDAIwk8_22<-APmd.wk6$cdail_wk22-APmd.wk6$cdail_wk8
percentdelta8_22<-deltaCDAIwk8_22/APmd.wk6$cdail_wk8*100
APmd.wk6$percentdelta0_4<-percentdelta0_4
APmd.wk6$percentdelta0_6<-percentdelta0_6
APmd.wk6$percentdelta0_8<-percentdelta0_8
APmd.wk6$percentdelta0_22<-percentdelta0_22
APmd.wk6$percentdelta8_22<-percentdelta8_22
APmd.wk6$RelRSPwk4<-as.factor(ifelse(APmd.wk6$percentdelta0_4>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk6<-as.factor(ifelse(APmd.wk6$percentdelta0_6>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk8<-as.factor(ifelse(APmd.wk6$percentdelta0_8>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk22<-as.factor(ifelse(APmd.wk6$percentdelta0_22>=-29.5, print("No"), print("Yes")))
APmd.wk6$RelRSPwk8_22<-as.factor(ifelse(APmd.wk6$percentdelta8_22>=-29.5, print("No"), print("Yes")))
APmd.wk6$visit<-factor(APmd.wk6$visit, levels = c("Screening", "Week 4", "Week 6", "Week 22"))
adiv<-read.table(file="../data/Jan400.all.na.omit.cmd.groups.summary", sep="\t", header=T)
ss3k_shared<-read.table(file="../data/Jan400.all.na.omit.cmd.0.03.subsample.shared", sep="\t", header=T)#Jan400 subsampled to 3K visit=Week.6
APmd.wk6_adiv<-merge(APmd.wk6, adiv, by.x="group", by.y="group")#nonmatch removed
APmd.wk6_shared<-merge(APmd.wk6_adiv, ss3k_shared_U2s, by.x="group", by.y="Group")
summary(APmd.wk6_shared[,1:20])
APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$USUBJID!='C0743T26030100002',]
APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$group!='E4859941-5',]
levs<-APmd.wk6_shared %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
levs

APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$TRTGRINDMAN!="Placebo_Not",]
APmd.wk6_shared<-APmd.wk6_shared[APmd.wk6_shared$TRTGRINDMAN!="Treated_Not",]

allwk6_data<-APmd.wk6_shared[APmd.wk6_shared$visit=="Week 6",]
summary(allwk6_data[,1:20])

trtd.wk6_data<-allwk6_data[allwk6_data$TRTGR=="Treated",]
summary(trtd.wk6_data[,1:20])

plac.wk6_data<-allwk6_data[allwk6_data$TRTGR!="Treated",]
summary(plac.wk6_data[,1:20])

wk0_6_data<-APmd.wk6_shared[APmd.wk6_shared$visit!="Week 4",]
wk0_6_data<-wk0_6_data[wk0_6_data$visit!="Week 22",]

summary(subset(wk0_6_data, select = c(group, USUBJID, visit, nseqs, coverage, sobs, invsimpson)))

levs<-wk0_6_data %>% group_by(USUBJID) %>% summarise(no_rows = length(USUBJID))
summary(levs)
subidsover2<-levs[levs$no_rows>=2,]
summary(subidsover2)
subidsover2
zero6subs<-as.character(subidsover2$USUBJID)
zero6subs_data <- wk0_6_data[wk0_6_data$USUBJID %in% zero6subs,]


zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Placebo_Not",]
zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Treated_Not",]
summary(subset(zero6subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

trtd.zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Placebo_Placebo",]
trtd.zero6subs_data<-trtd.zero6subs_data[trtd.zero6subs_data$TRTGRINDMAN!="Placebo_Treated",]
plac.zero6subs_data<-zero6subs_data[zero6subs_data$TRTGRINDMAN!="Treated_Treated",]
plac.zero6subs_data<-plac.zero6subs_data[plac.zero6subs_data$TRTGRINDMAN!="Treated_Placebo",]
summary(subset(trtd.zero6subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))
summary(subset(plac.zero6subs_data, select=c(group, USUBJID, visit, TRTGR, TRTGRINDMAN, RESPONSEwk8)))

week6.N.all <- nrow(zero6subs_data[zero6subs_data$visit=='Week 6',])
week6.trtd <- trtd.zero6subs_data[trtd.zero6subs_data$visit=='Week 6',]
week6.N.trtd <- nrow(trtd.zero6subs_data[trtd.zero6subs_data$visit=='Week 6',])
week6.N.trtd.respY <- nrow(week6.trtd[week6.trtd$RelRSPwk6=='Yes',])
week6.N.trtd.respN <- nrow(week6.trtd[week6.trtd$RelRSPwk6=='No',])
week6.plac <- plac.zero6subs_data[plac.zero6subs_data$visit=='Week 6',]
week6.N.plac <- nrow(plac.zero6subs_data[plac.zero6subs_data$visit=='Week 6',])
week6.N.plac.respY <- nrow(week6.plac[week6.plac$RelRSPwk6=='Yes',])
week6.N.plac.respN <- nrow(week6.plac[week6.plac$RelRSPwk6=='No',])
week6.N.trtd.REMY <- nrow(week6.trtd[week6.trtd$REMISSwk6=='Yes',])
week6.N.trtd.REMN <- nrow(week6.trtd[week6.trtd$REMISSwk6=='No',])
week6.N.plac.REMY <- nrow(week6.plac[week6.plac$REMISSwk6=='Yes',])
week6.N.plac.REMN <- nrow(week6.plac[week6.plac$REMISSwk6=='No',])

all.RelRSPwk6<-matrix(c('Week 6 Response (No, Yes)', trtd.RelRSPwk6_sum, plac.RelRSPwk6_sum, all.RelRSPwk6.kw.test.pval, all.RelRSPwk6_AMOVA), ncol = 5)

week6.N.resp.trtd_sum<-toString(c(week6.N.trtd.respN, week6.N.trtd.respY))
week6.N.resp.plac_sum<-toString(c(week6.N.plac.respN, week6.N.plac.respY))

week6.N.REM.trtd_sum<-toString(c(week6.N.trtd.REMN, week6.N.trtd.REMY))
week6.N.REM.plac_sum<-toString(c(week6.N.plac.REMN, week6.N.plac.REMY))

wk6.RelRSPwk6 <- matrix(c("Week 6", "Response Week 6 (No, Yes)", week6.N.resp.trtd_sum, week6.N.resp.plac_sum, week6.N.all), ncol=5)
colnames(wk6.RelRSPwk6) <- c("Stool Week", 'Status Week', 'Treated (n)', "Placebo (n)", "Total (n)")
wk6.REMISSwk6 <- matrix(c("Week 6", "Remission Week 6 (No, Yes)", week6.N.REM.trtd_sum, week6.N.REM.plac_sum, week6.N.all), ncol=5)
colnames(wk6.REMISSwk6) <- c("Stool Week", 'Status Week', 'Treated (n)', "Placebo (n)", "Total (n)")

wk6_wk6 <- rbind(wk6.RelRSPwk6, wk6.REMISSwk6)
wk6_wk6 <- wk6_wk6[,-5]

pdf("tables/wk6_wk6.sum.pdf", height=2, width=6)
grid.table(wk6_wk6)
dev.off()
```

```{r RelRSPwk6Xwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("RelRSPwk6 by AUC-RF \\nRelative response defined as a decrease in CDAI from Week.6 greater than 29.5%.")
set.seed(32016)
levels(trtd.wk6_data$RelRSPwk6) <- c(0,1)

mdotu_relrespWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", "invsimpson", colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
mdotu_relrespWK6xWK6<-na.omit(mdotu_relrespWK6xWK6)
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
otu_relrespWK6xWK6 <- mdotu_relrespWK6xWK6[, c('RelRSPwk6', colnames(mdotu_relrespWK6xWK6)[grep('Otu[0123456789]', colnames(mdotu_relrespWK6xWK6))])]
#summary(otu_relrespWK6xWK6[,c("RelRSPwk6")])
#summary(otu_relrespWK6xWK6[,c("RelRSPwk6")])
otu_relrespWK6xWK6<-na.omit(otu_relrespWK6xWK6)
summary(otu_relrespWK6xWK6[,c("RelRSPwk6")])
set.seed(32016)
otu_relrespWK6xWK6_rf <- AUCRF(RelRSPwk6~., data=otu_relrespWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
##otu_relrespWK6xWK6_rf # prints list of optimal OTUs
otu_relrespWK6xWK6_probs <- predict(otu_relrespWK6xWK6_rf$RFopt, type='prob')[,2]
otu_relrespWK6xWK6_roc <- roc(otu_relrespWK6xWK6$RelRSPwk6~otu_relrespWK6xWK6_probs)


md_relrespWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6")]
#summary(md_relrespWK6xWK6[,c("RelRSPwk6")])
md_relrespWK6xWK6<-na.omit(md_relrespWK6xWK6)
#summary(md_relrespWK6xWK6[,c("RelRSPwk6")])
set.seed(32016)
md_relrespWK6xWK6_rf <- AUCRF(RelRSPwk6~., data=md_relrespWK6xWK6, ranking='MDA', ntree=500, pdel=0.3)
##md_relrespWK6xWK6_rf # prints list of optimal features
md_relrespWK6xWK6_probs <- predict(md_relrespWK6xWK6_rf$RFopt, type='prob')[,2]
md_relrespWK6xWK6_roc <- roc(md_relrespWK6xWK6$RelRSPwk6~md_relrespWK6xWK6_probs)


mdotu_relrespWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('RelRSPwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", "invsimpson", colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
mdotu_relrespWK6xWK6<-na.omit(mdotu_relrespWK6xWK6)
#summary(mdotu_relrespWK6xWK6[,c("RelRSPwk6")])
set.seed(32016)
mdotu_relrespWK6xWK6_rf <- AUCRF(RelRSPwk6~., data=mdotu_relrespWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_relrespWK6xWK6_rf # prints list of optimal features
mdotu_relrespWK6xWK6_probs <- predict(mdotu_relrespWK6xWK6_rf$RFopt, type='prob')[,2]
mdotu_relrespWK6xWK6_roc <- roc(mdotu_relrespWK6xWK6$RelRSPwk6~mdotu_relrespWK6xWK6_probs)



#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_relrespWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_relrespWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(mdotu_relrespWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Response using Week 0 Clinical Data")
legend('bottomright', sprintf('Clinical Data: AUC=%.3f',md_relrespWK6xWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)
cutoff <- coords(roc=mdotu_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_relrespWK6xWK6_spec <- coords(mdotu_relrespWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_relrespWK6xWK6_sens <- coords(mdotu_relrespWK6xWK6_roc, x=mdotu_relrespWK6xWK6_spec, input='spec', ret='sens')
mdotu_relrespWK6xWK6_thr <- coords(mdotu_relrespWK6xWK6_roc, x=mdotu_relrespWK6xWK6_spec, input='spec', ret='thr')

points(coords(mdotu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_relrespWK6xWK6_spec <- coords(otu_relrespWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
otu_relrespWK6xWK6_sens <- coords(otu_relrespWK6xWK6_roc, x=otu_relrespWK6xWK6_spec, input='spec', ret='sens')
otu_relrespWK6xWK6_thr <- coords(otu_relrespWK6xWK6_roc, x=otu_relrespWK6xWK6_spec, input='spec', ret='thr')
points(coords(otu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_relrespWK6xWK6_spec <- coords(md_relrespWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
md_relrespWK6xWK6_sens <- coords(md_relrespWK6xWK6_roc, x=md_relrespWK6xWK6_spec, input='spec', ret='sens')
md_relrespWK6xWK6_thr <- coords(md_relrespWK6xWK6_roc, x=md_relrespWK6xWK6_spec, input='spec', ret='thr')

points(coords(md_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_relrespWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, md_relrespWK6xWK6_probs[md_relrespWK6xWK6$'RelRSPwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, md_relrespWK6xWK6_probs[md_relrespWK6xWK6$'RelRSPwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=md_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()


md_relrespWK6xWK6_rf # prints list of optimal features


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_relrespWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_relrespWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_relrespWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("RESPONSEwk6")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_relrespWK6xWK6_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_relrespWK6xWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_relrespWK6xWK6_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_relrespWK6xWK6_probs[mdotu_relrespWK6xWK6$'RelRSPwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_relrespWK6xWK6_probs[mdotu_relrespWK6xWK6$'RelRSPwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_relrespWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_relrespWK6xWK6_rf # prints list of optimal features
otu_relrespWK6xWK6_rf
md_relrespWK6xWK6_rf
```

```{r freqOTUs list for table, eval=T, include=F}
#mdotu_respWK4_Xopt<-mdotu_respWK4_rf$Xopt
#mdotu_relrespWK4_Xopt<-mdotu_relrespWK4_rf$Xopt
#mdotu_remWK4_Xopt<-mdotu_remWK4_rf$Xopt
#otu_respWK4_Xopt<-otu_respWK4_rf$Xopt
#otu_relrespWK4_Xopt<-otu_relrespWK4_rf$Xopt
#otu_remWK4_Xopt<-otu_remWK4_rf$Xopt

#mdotu_respWK6_Xopt<-mdotu_respWK6_rf$Xopt
mdotu_relrespWK6_Xopt<-mdotu_relrespWK6_rf$Xopt
mdotu_remWK6_Xopt<-mdotu_remWK6_rf$Xopt
#otu_respWK6_Xopt<-otu_respWK6_rf$Xopt
otu_relrespWK6_Xopt<-otu_relrespWK6_rf$Xopt
otu_remWK6_Xopt<-otu_remWK6_rf$Xopt

#mdotu_respWK8_Xopt<-mdotu_respWK8_rf$Xopt
#mdotu_relrespWK8_Xopt<-mdotu_relrespWK8_rf$Xopt
#mdotu_remWK8_Xopt<-mdotu_remWK8_rf$Xopt
#otu_respWK8_Xopt<-otu_respWK8_rf$Xopt
#otu_relrespWK8_Xopt<-otu_relrespWK8_rf$Xopt
#otu_remWK8_Xopt<-otu_remWK8_rf$Xopt

#mdotu_respWK22_Xopt<-mdotu_respWK22_rf$Xopt
mdotu_relrespWK22_Xopt<-mdotu_relrespWK22_rf$Xopt
mdotu_remWK22_Xopt<-mdotu_remWK22_rf$Xopt
#otu_respWK22_Xopt<-otu_respWK22_rf$Xopt
otu_relrespWK22_Xopt<-otu_relrespWK22_rf$Xopt
otu_remWK22_Xopt<-otu_remWK22_rf$Xopt

rows<-c("AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Screening", "CDPAINM_Screening", "CDWELBEM_Screening", "CDARTRTM_Screening", "CDARALGM_Screening", "CDIRITM_Screening", "CDUVEITM_Screening", "CDERYTHM_Screening", "CDPYODEM_Screening", "CDSTOMAM_Screening", "CDFISURM_Screening", "CDFISTUM_Screening", "CDABSCEM_Screening", "CDOTFISM_Screening", "CDFEVERM_Screening", "CDDRGM_Screening", "CDAMASSM_Screening", "CDHCTM_Screening", "CDWEIGHM_Screening", "invsimpson", colnames(nao.screen_data)[grep('Otu[0123456789]', colnames(nao.screen_data))])

#colnames<-c("mdotu_respWK4_Xopt", "mdotu_relrespWK4_Xopt", "mdotu_remWK4_Xopt", "otu_respWK4_Xopt", "otu_relrespWK4_Xopt", "otu_remWK4_Xopt", "mdotu_respWK6_Xopt", "mdotu_relrespWK6_Xopt", "mdotu_remWK6_Xopt", "otu_respWK6_Xopt", "otu_relrespWK6_Xopt", "otu_remWK6_Xopt", "mdotu_respWK8_Xopt", "mdotu_relrespWK8_Xopt", "mdotu_remWK8_Xopt", "otu_respWK8_Xopt", "otu_relrespWK8_Xopt", "otu_remWK8_Xopt", "mdotu_respWK22_Xopt", "mdotu_relrespWK22_Xopt", "mdotu_remWK22_Xopt", "otu_respWK22_Xopt", "otu_relrespWK22_Xopt", "otu_remWK22_Xopt")

colnames<-c("mdotu_relrespWK6_Xopt", "mdotu_remWK6_Xopt", "otu_relrespWK6_Xopt", "otu_remWK6_Xopt", "mdotu_relrespWK22_Xopt", "mdotu_remWK22_Xopt", "otu_relrespWK22_Xopt", "otu_remWK22_Xopt")

matches<-data.frame(row.names = rows)

for( i in colnames){
	matches[row.names(matches) %in% get(i), i]<- 1
}

matches$Sums<-apply(matches, 1, sum, na.rm=T)

freqVars<-matches[matches$Sums>=4,]
freqVars<-freqVars[order(-freqVars$Sums),]

freqOTUs<-freqVars[rownames(freqVars)[grep('Otu[0123456789]', rownames(freqVars))],]
#freqOTUs

freqOTUs_list<-as.factor(row.names(freqOTUs))
```

```{r REMISSwk6Xwk6-AUC-RF, include=F, fig.height=5, fig.width=10}
print("REMISSwk6 by AUC-RF")
set.seed(32016)
levels(trtd.wk6_data$REMISSwk6) <- c(0,1)
mdotu_remWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", "invsimpson", colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
mdotu_remWK6xWK6<-na.omit(mdotu_remWK6xWK6)
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
otu_remWK6xWK6 <- mdotu_remWK6xWK6[, c('REMISSwk6', colnames(mdotu_remWK6xWK6)[grep('Otu[0123456789]', colnames(mdotu_remWK6xWK6))])]
#summary(otu_remWK6xWK6[,c("REMISSwk6")])
#summary(otu_remWK6xWK6[,c("REMISSwk6")])
otu_remWK6xWK6<-na.omit(otu_remWK6xWK6)

#summary(otu_remWK6xWK6[,c("REMISSwk6")])
set.seed(32016)
otu_remWK6xWK6_rf <- AUCRF(REMISSwk6~., data=otu_remWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
#otu_remWK6xWK6_rf # prints list of optimal OTUs
otu_remWK6xWK6_probs <- predict(otu_remWK6xWK6_rf$RFopt, type='prob')[,2]
otu_remWK6xWK6_roc <- roc(otu_remWK6xWK6$REMISSwk6~otu_remWK6xWK6_probs)

set.seed(32016)
otu_remWK6xWK6_rf_cv <- AUCRFcv(otu_remWK6xWK6_rf)

md_remWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('REMISSwk6',"AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6")]
#summary(md_remWK6xWK6[,c("REMISSwk6")])
md_remWK6xWK6<-na.omit(md_remWK6xWK6)
#summary(md_remWK6xWK6[,c("REMISSwk6")])
set.seed(32016)
md_remWK6xWK6_rf <- AUCRF(REMISSwk6~., data=md_remWK6xWK6, ranking='MDA', ntree=500, pdel=0.3)
#md_remWK6xWK6_rf # prints list of optimal features
md_remWK6xWK6_probs <- predict(md_remWK6xWK6_rf$RFopt, type='prob')[,2]
md_remWK6xWK6_roc <- roc(md_remWK6xWK6$REMISSwk6~md_remWK6xWK6_probs)


mdotu_remWK6xWK6 <- trtd.wk6_data[trtd.wk6_data$visit=='Week 6', c('REMISSwk6', "AGE", "SEX", "BAMINO", "BANTIBIO", "BCORT", "BIMM", "BL_BMI", "DDUR", "DDUR10", "DDUR15", "tissinvol", "BRSP_TNF", "BL_FLACT", "BL_FCALP", "BL_CRP", "BL_CRPLEV", "BL_FCALPLEV", "BL_FLACTLEV", "BOWSTRICT", "CDSTOOLM_Week.6", "CDPAINM_Week.6", "CDWELBEM_Week.6", "CDARTRTM_Week.6", "CDARALGM_Week.6", "CDIRITM_Week.6", "CDUVEITM_Week.6", "CDERYTHM_Week.6", "CDPYODEM_Week.6", "CDSTOMAM_Week.6", "CDFISURM_Week.6", "CDFISTUM_Week.6", "CDABSCEM_Week.6", "CDOTFISM_Week.6", "CDFEVERM_Week.6", "CDDRGM_Week.6", "CDAMASSM_Week.6", "CDHCTM_Week.6", "CDWEIGHM_Week.6", 'invsimpson', colnames(trtd.wk6_data)[grep('Otu[0123456789]', colnames(trtd.wk6_data))])]
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
mdotu_remWK6xWK6<-na.omit(mdotu_remWK6xWK6)
#summary(mdotu_remWK6xWK6[,c("REMISSwk6")])
set.seed(32016)
mdotu_remWK6xWK6_rf <- AUCRF(REMISSwk6~., data=mdotu_remWK6xWK6, ranking='MDA', ntree=500, pdel=0.2)
#mdotu_remWK6xWK6_rf # prints list of optimal features
mdotu_remWK6xWK6_probs <- predict(mdotu_remWK6xWK6_rf$RFopt, type='prob')[,2]
mdotu_remWK6xWK6_roc <- roc(mdotu_remWK6xWK6$REMISSwk6~mdotu_remWK6xWK6_probs)


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
#plot(otu_remWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_remWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
#plot(mdotu_remWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Remission using Week 6 Clinical Data")
legend('bottomright', sprintf('Clinical Data: AUC=%.3f',md_remWK6xWK6_roc$auc), lwd=2, col='grey5', bty='n', lty=1)

cutoff <- coords(roc=mdotu_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
mdotu_remWK6xWK6_spec <- coords(mdotu_remWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
mdotu_remWK6xWK6_sens <- coords(mdotu_remWK6xWK6_roc, x=mdotu_remWK6xWK6_spec, input='spec', ret='sens')
mdotu_remWK6xWK6_thr <- coords(mdotu_remWK6xWK6_roc, x=mdotu_remWK6xWK6_spec, input='spec', ret='thr')

points(coords(mdotu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(mdotu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='red', cex=1.5, pch=10)

cutoff <- coords(roc=otu_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
otu_remWK6xWK6_spec <- coords(otu_remWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
otu_remWK6xWK6_sens <- coords(otu_remWK6xWK6_roc, x=otu_remWK6xWK6_spec, input='spec', ret='sens')
otu_remWK6xWK6_thr <- coords(otu_remWK6xWK6_roc, x=otu_remWK6xWK6_spec, input='spec', ret='thr')
points(coords(otu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(otu_remWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='royalblue', cex=1.5, pch=10)

cutoff <- coords(roc=md_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold'))
md_remWK6xWK6_spec <- coords(md_remWK6xWK6_roc, x=cutoff, input='thr', ret='spec')
md_remWK6xWK6_sens <- coords(md_remWK6xWK6_roc, x=md_remWK6xWK6_spec, input='spec', ret='sens')
md_remWK6xWK6_thr <- coords(md_remWK6xWK6_roc, x=md_remWK6xWK6_spec, input='spec', ret='thr')

points(coords(md_remWK6xWK6_roc, x=cutoff, input='threshold', ret='specificity'),coords(md_remWK6xWK6_roc, x=cutoff, input='threshold', ret='sensitivity'), col='black', cex=1.5, pch=10)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, md_remWK6xWK6_probs[md_remWK6xWK6$'REMISSwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, md_remWK6xWK6_probs[md_remWK6xWK6$'REMISSwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=md_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

md_remWK6xWK6_rf # prints list of optimal features


#pdf('data/figures/RSPwk6aucrf.#pdf', height=6, width=8)
### Plot ROC curves
layout(matrix(c(1,2), nrow=1))
par(mar=c(3,3,2,1), mgp=c(2,0.5,0))
plot(c(1,0),c(0,1), type='l', lty=2, xlim=c(1.01,0), ylim=c(-0.01,1.01), xaxs='i', yaxs='i', ylab='Sensitivity', xlab='Specificity', las=1)
plot(otu_remWK6xWK6_roc, add=T, lwd=2, col='royalblue', lty=1)
plot(md_remWK6xWK6_roc, add=T, lwd=2, col='grey5', lty=1)
plot(mdotu_remWK6xWK6_roc, add=T, lwd=2, col='red', lty=1)
title("Predicting Week 6 Remission using Week 6 Clinical Data")
legend('bottomright', c(sprintf('OTUs : AUC=%.3f',otu_remWK6xWK6_roc$auc),sprintf('Clinical Data: AUC=%.3f',md_remWK6xWK6_roc$auc),sprintf('Combined: AUC=%.3f',mdotu_remWK6xWK6_roc$auc)), lwd=2, col=c('royalblue','grey5', 'red'), bty='n', lty=1)
#dev.off()

#pdf('data/figures/RSPwk6abunds.#pdf', height=6, width=8)
plot(1, type='n', xlim=c(0.5,2.5), ylim=c(0,1), ylab='', xlab='', xaxt='n', cex.axis=1.2)
stripchart(at=1, mdotu_remWK6xWK6_probs[mdotu_remWK6xWK6$'REMISSwk6'==0], vertical=T, method='jitter', jitter=0.4, pch=21, bg='orange', add=T, cex=1.5)
stripchart(at=2, mdotu_remWK6xWK6_probs[mdotu_remWK6xWK6$'REMISSwk6'==1], vertical=T, method='jitter', jitter=0.4, pch=21, bg='royalblue', add=T, cex=1.5)
mtext(side=2, text='Probability of Response', line=2.2, cex=0.8)
axis(1, at=c(1,2), labels=c('No', 'Yes'), cex.axis=1.2)
cutoff <- coords(roc=mdotu_remWK6xWK6_roc, x='best', best.method='y', ret=c('threshold')) #gives best point on curve for cutoff
#dev.off()

mdotu_remWK6xWK6_rf # prints list of optimal features
otu_remWK6xWK6_rf
md_remWK6xWK6_rf
```

```{r}
otu_remWK6xWK6_rf_cv
```

```{r}
```


\vspace{35mm}
	
Running title: The fecal microbiome as a tool for monitoring and predicting response outcomes in Ustekinumab-treated, anti-TNF-alpha refractory Crohn’s Disease patients.
	
\vspace{35mm}
Matthew K. Doherty${^2}$, Tao Ding${^2}$${^\alpha}$, Charlie Koumpouras${^2}$, Shannon Telesco${^1}$, Calixte Monast${^1}$, and Patrick D. Schloss${^2}$${^\dagger}$

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

1\. Janssen Pharmaceutical Companies of Johnson ${\&}$ Johnson, Spring House, PA, USA

2\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI, USA

${\alpha}$ Currently at *...*





\newpage








## Abstract


_Abstract:_ The 16S rRNA gene from patient stool samples was sequenced using the Illumina MiSeq platform. The resulting sequences were curated and assigned to taxonomic groups using the mothur software package to determine the bacterial communities and relative abundance of bacterial species present in these patients. The relative abundance among the fecal microbiota, patient demographic data, and clinical metadata were used as input to a random forest machine-learning algorithm to predict disease severity and response to treatment with UST.

Fecal microbial diversity at baseline significantly correlates with markers for disease severity, such as Crohn’s Disease Activity Index (CDAI), stool frequency, and disease duration. Additionally, the overall community structure of the microbiome was significantly different based on stool frequency, CRP, fecal lactoferrin, fecal calprotectin, corticosteroid use, disease duration, and tissue involvement. Baseline fecal microbiome community structures and species diversity were significantly different among responders and non-responders to UST treatment. Faecalibacterium, among other taxa, was significantly more abundant in responders/remitters. Additionally, the microbiome of clinical responders changed over time, in contrast to nonresponsive subjects. Using AUC-RF, differences in the baseline microbiome and clinical metadata were able to predict response to UST, especially remission, with some AUCs approaching 0.85.

_Importance:_ Crohn’s disease (CD) is a global health issue characterized by patches of ulceration and inflammation along the gastrointestinal tract. Individuals with CD have reduced microbial diversity in their guts, compared to healthy individuals. It remains unclear if this reduced diversity is a result or cause of pathogenesis. We investigated the relationship between the fecal microbiome and clinical phenotypes in subjects with moderate to severe CD treated with Ustekinumab (UST) in a Phase 2b study to determine whether the fecal microbiome at baseline is predictive of disease severity and therapeutic response, as well as if the fecal microbiota changes due to therapy.

The ability to predict and monitor response to treatment using the microbiome will likely provide another clinical tool in treating CD patients. Additionally, the observed baseline differences in fecal microbiota and changes due to therapeutic response will allow further investigation into the microbes important in CD pathogenesis as well as establishing and maintaining CD remission. Finally, beneficial microbes associated with response to treatment could be developed as probiotics to increase the likelihood of response while undergoing treatment.

**Keywords: Crohn's Disease, fecal microbiome, biologics, prediction**

\newpage

###Introduction
Crohn’s disease (CD), an incurable inflammatory bowel disease (IBD), is a global health issue with increasing incidence. CD affects approximately 3 million people worldwide, causing large economic and healthcare utilization impacts on society [@ananthakrishnan_epidemiology_2015; @floyd_economicburden_2015; @molodecky_increasingIBD_2012]. CD is characterized by patches of ulceration and inflammation affecting the entire bowel wall along the gastrointestinal tract, most commonly in the ileum and colon. Individuals with CD experience frequent diarrhea, abdominal pain, fatigue, and weight loss resulting in significant health care costs, lower quality of life, and economic impacts due to loss of productivity [@floyd_economicburden_2015; @mantzaris_clinical_2015; @randall_CDbiologics_2015]. Current treatments for CD include antibiotics, anti-inflammatory drugs, immunomodulators, surgery, and biologic agents targeting tumor necrosis factor alpha (TNF-${\alpha}$), such as Infliximab (Remicade). Within 10 years of diagnosis, approximately half of individuals with CD will require surgery and the majority will experience escalating immunosuppressive treatment [@boyapati_pathCD_2015]. Currently, individuals with CD are treated based on disease location and risk of complications using escalating immunosuppressive treatment and/or surgery with the goal of achieving and sustaining remission [@randall_CDbiologics_2015; @wils_ust_2015]. Faster induction of remission following diagnosis reduces the risk of irreversible intestinal damage and disability [@wils_ust_2015; @colombel_deepremission_2015; @baert_mucosalhealing_2010] (7-9). Anti-TNF-${\alpha}$ therapy in combination with thiopurines has emerged as the preferred treatment for CD, but up to half of individuals with CD fail to respond or lose response to anti-TNF-${\alpha}$ therapy [@boyapati_pathCD_2015; @wils_ust_2015]. Ustekinumab (UST), a monoclonal antibody directed against the shared p40 subunit of IL-12 and IL-23, has been proposed as an alternative therapy for these patients [@sandborn_ust_2012]. While clinical trials have demonstrated that UST is a viable option for the treatment of CD [@wils_ust_2015; @sandborn_ust_2012; @sandborn_ust_2008; @kopylov_ust_2014] (7, 10-12), some patients within these trials were non-responsive to UST, which may be explained by differences in the patients’ gut microbiomes.

The precise etiology of CD remains unknown, but host genetics, environmental exposure, and the gut microbiome appear involved [@ananthakrishnan_epidemiology_2015; @sartor_IBDpath_2006].  Genome-wide association studies of individuals with CD identified several susceptibility genes including NOD2, a receptor involved in bacterial killing and innate immunity. Defects in NOD2 function affects microbial sensing, the regulation of IL-23 driven Th17 responses, and indirect modulation of the gut microbiome [@randall_CDbiologics_2015; @wright_CDmicrobiome_2015]. The gut microbiome has also been shown to play a key role in inflammation, immunity, and IBD [@haag_CDpath_2015]. Individuals with CD have reduced microbial diversity in their guts, compared to healthy individuals, with a lower relative abundance of Firmicutes and an increased relative abundance of Enterobacteraciae and Bacteroides, at the phylum level [@wright_CDmicrobiome_2015; @manichanh_diversityCD_2006; @hansen_pedsIBD_2012; @haberman_pedsCD_2014; @gevers_pedsCD_2014] . Additionally, previous studies have shown that the gut microbiome can be predictive of disease severity in new-onset, pediatric CD patients [@gevers_pedsCD_2014; @wang_pedsCD_2016]. It remains to be determined, however, whether the microbiome can predict response to therapy in CD [@wright_CDmicrobiome_2015]. Additionally, the effect of biologic treatment on the gut microbiome is not well understood. If the fecal microbiome can be used as a theraprognostic tool to non-invasively determine and monitor disease severity as well as predict response to specific treatment modalities, then more targeted treatment could result in reduced adverse effects of less effective therapies and faster achievement of remission. 

Our lab was approached to analyze the gut microbiomes of individuals who participated in a Phase II clinical trial to determine the efficacy of UST in treating CD [@sandborn_ust_2012]. Using stool samples taken prior to the start of the study, 16S rRNA gene sequence data from these patients will allow us to determine associations between clinical metadata, disease severity, and the fecal microbiome and whether clinical responders have a microbiome that is distinct from non-responders at baseline. Preliminary results generated with fecal samples from a subset of study participants and sequenced using the Roche 454 platform suggest that the fecal microbiota of moderate to severe CD patients refractory to anti-TNF-${\alpha}$ may differentiate individuals who will respond to treatment with UST; however, large interpersonal variation limited the power of our findings. This study attempts to overcome many of the limitations in our preliminary analysis by increasing our sample size to the full patient cohort and using the Illumina MiSeq platform to improve our sequencing depth. We demonstrate that the fecal microbiome is associated with baseline clinical metadata and that these associations and differences are useful in predicting disease severity and treatment outcome.



## Results

**Characteristics of Study Population**
We studied the fecal microbiota in a subset of TNF-${\alpha}$ refractory CD patients who took park in the CERTIFI clinical trial described in @sandborn_ust_2012. Briefly, patients with a history of moderate to severe CD were randomly assigned to a treatment group in the induction phase of the study. Subjects provided a stool sample at screening (Week 0), Week 4 and Week 6. At Week 8 patients were re-randomized into maintenance therapy groups. A final stool sample was provided at Week 22. Response to therapy was evaluated at week 4, 6, 8, and 22 based on change in CDAI. Samples from subjects that completed the clinical trial and had complete clinical metadata were included in our analysis. We used 16s rRNA gene sequencing to analyze the microbiome from 306 fecal samples provided prior to treatment as well as 258 Week 4, 289 Week 6, and 205 Week 22 post-treatment fecal samples, for a total of 1058 samples. Demographic and baseline disease characteristics are summarized in supplemental table 1.

**Comparison of microbiome at screening based on clinical variables**
To determine if there were any significant associations between microbial diversity and clinical variables of interest, we compared the microbiome with clinical data at Week 0. We determined species richness (${\alpha}$-diversity) using the inverse Simpson metric and assessed associations between species richness and clinical data using Spearman’s rank correlation, Wilcoxon rank-sum, or Kruskal-Wallis rank-sum tests. Associations between the overall community structure (ß-diversity) and clinical data were determined using the thetaYC distance metric as input to the adonis PERMANOVA function within the vegan R package [@oksanen_vegan_2016]. As seen in table 1, we observed a correlation between CDAI and species richness, with higher CDAI correlating to lower richness. The overall community structure was not different based on CDAI. When looking at CDAI subscores, we observed a significant association between species richness and the frequency of loose stools per week. The overall community structure was also significantly different based on weekly loose stool frequency. No significant association was observed between CRP and fecal calprotectin and species richness, while higher fecal lactoferrin weakly correlates with higher richness. The overall community structure was significantly different based on CRP, fecal calprotectin, and fecal lactoferrin. No significant differences in the microbiome were observed for BMI, weight, or sex. Overall community structure was different based on age. The overall community structure was also different based on the tissue affected. Species richness and the overall community structure were significantly different based on corticosteroid use. The community structure was significantly different based on disease duration and a significant correlation was seen between species richness and disease duration, with lower richness corresponding to longer disease.

*consider including LEfSe data for quick look at discriminate OTU based on sig dif clinical variables like in Gevers and Zackular papers?*

**Comparison of clinical responders and non-responders**
Next, We wanted to see if there were associations between the microbiome at baseline and response to treatment. For this study, response was defined as a 30% decrease from CDAI at baseline and remission defined as a CDAI below 150. Of the 306 screening samples analyzed, 232 were from subjects receiving UST and 74 from subjects receiving placebo. Baseline fecal microbiome community structures and species diversity were different among responders and non-responders to UST treatment. Based on response at the primary endpoint of the study, 6 weeks after IV induction, there was no difference in species richness between response groups, but there was a significant difference in the overall community structure of the entire cohort. This difference in community structure was not significant in treatment vs. placebo groups. Week 6 remitters were significantly different from non-remitters in both species richness (`r all.wilcox.REMISSwk6.pval`) and overall community structure (`r all.REMISSwk6.adonis.pval`). When looking at treated vs. untreated Week 6 remitters, the treated group had significant differences in both species richness and community structure while untreated remitters we not different from untreated non-remitters. At the secondary endpoint, 22 weeks after IV-induction and 14 weeks after maintenance dosing, there was no difference in species richness between response groups, but there was a significant difference in the overall community structure of the entire cohort. Week 22 remitters were significantly different from non-remitters in both species richness (`r all.wilcox.RelRSPwk22.pval`) and overall community structure (`r all.RelRSPwk22.adonis.pval`). However, these differences were not seen when the cohort was broken down by induction group. This could be due to changes in maintenance treatment.

**The microbiome by treatment and response over time**
One major question with regards to biologic treatment of IBD and the microbiome is whether treatment has an effect on the microbiome. We explored this question 2 different ways. We included subjects that had stool samples at all 4 time points and another analysis using subjects who provided samples at weeks 0, 4, and 6. We used PERMANOVA stratified on each subject, as a proxy for a repeated measures ANOVA, to determine if the microbiome changed over time. We found that taken together treatment does not affect the microbiome. No significant difference was seen based on visit when looking at all groups and response status at week 4, 6, or 8 over the first 3 time points, but there was a significant interaction between response at week 22 and visit (p=0.001) and between relative response, induction group, and visit (p=0.0445). 

This led us to examining just the week 22 responders vs. non-responders across visit. No significant difference over time was observed in non-responders. When we segregated week 22 responders, we saw a significant change in community structure over time. There was also a significant difference based on treatment group, but no significant interaction. When looking at treated vs. untreated responder groups, we observed a significant difference based on visit in the treated, Week 22 responder and in untreated responders across the first 3 visits prior to maintenance phase.

When looking at time in all subjects across all 4 time points we observed a significant interaction between visit and response, however no interaction between visit, treatment group, and response. In all subjects there was a significant difference in community structure based on response at Week 22. In treated subjects, we observed a significant interaction between response and visit, as well as a significant difference in community structure based on response at Week 22. No significant difference was observed in untreated responders across all 4 time points.

**Prediction of response based on the microbiome at screening**

Another major question in IBD and the microbiome is if response can be predicted using the microbiome. To address this we used AUCRF to develop a random forest classification model to differentiate responders from non-responders, as well as remitters from non-remitters, based on the relative abundance of fecal microbiome community members, clinical metadata, and combined microbiome and clinical data [@baxter_FIT_2016; @calle_aucrf_2011]. We ran these models for response and remission at Week 4, 6, 8, and 22 of the study. The optimal models for response and remission at the primary endpoint (Week 6) are shown in Figure 1. Using only clinical metadata to predict response, the model predicted response with an AUC of `r md_relrespWK6_roc$auc` with a specificity of `r md_relrespWK6_spec` and a sensitivity of `r md_relrespWK6_sens`. Using only microbiome data, the model predicted response with an AUC of `r otu_relrespWK6_roc$auc` with a specificity of `r otu_relrespWK6_spec` and a sensitivity of `r otu_relrespWK6_sens`. When combining clinical metadata with the microbiome, the model predicted response with an AUC of `r mdotu_relrespWK6_roc$auc` with a specificity of `r mdotu_relrespWK6_spec` and a sensitivity of `r mdotu_relrespWK6_sens`. With respect to Week 6 remission, using solely clinical metadata we achieved AUC of `r md_remWK6_roc$auc` with a specificity of `r md_remWK6_spec` and a sensitivity of `r md_remWK6_sens`. Using only fecal microbiome data we achieved an AUC of `r otu_remWK6_roc$auc` with a specificity of `r otu_remWK6_spec` and a sensitivity of `r otu_remWK6_sens`. When combining clinical metadata with the microbiome AUC of `r mdotu_remWK6_roc$auc` with a specificity of `r mdotu_remWK6_spec` and a sensitivity of `r mdotu_remWK6_sens`.

Across all weeks and responses, prediction with clinical metadata alone did not perform as well as models using the fecal microbiome at screening. Also, combining microbiome data with clinical metadata did not consistently improve prediction compared to microbiome data alone. Additionally we found several OTUs occurred frequently across models including Faecalibacterium, among other taxa that were significantly more abundant in responders/remitters. Their abundances can be seen in figure 4.

In addition to predicting future response, we wanted to determine if the microbiome could be used to monitor response to therapy. Again we used AUC-RF in order to determine if the fecal microbiome at Week 6 could be used to determine response or remission at Week 6. As seen in Supplemental Figure 1, using the microbiome alone we achieved an AUC of `r otu_relrespWK6xWK6_roc$auc` for response with a sensitivity of `r otu_relrespWK6xWK6_sens` and a specificity of `r otu_relrespWK6xWK6_spec`. For remission we had an AUC of `r otu_remWK6xWK6_roc$auc` with a sensitivity of `r otu_remWK6xWK6_sens` and specificity of `r otu_remWK6xWK6_spec`. Again we were better able to distinguish remitters from non-remitters than responders/non-responders. The clinical data were more reliable for determining disease activity at Week 6.


## Discussion

Our results examine the fecal microbiome of a subset of patients who participated in the CERTIFI trials to determine if the microbiome can predict response to therapy and if therapy has any effect on the microbiome. Several previous studies have looked at fecal and mucosal microbiomes in pediatric patients with new-onset and established disease and with established disease in adults [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. Unlike these studies, our patients were mostly Caucasian adults in their late thirties to early forties who failed to respond or lost response to anti-TNF-${\alpha}$ biologic treatment. We were able to find associations between the fecal microbiome of these patients and CDAI, stool frequency, fecal calprotectin, fecal lactoferrin, serum CRP, corticosteroid use, tissue involvement, and duration of disease. 

The association of the microbiome with clinically relevant biomarkers and disease activity metrics indicates that the microbiome may also function as a biomarker for CD activity. Given that serum CRP, calprotectin, and lactoferrin are used as biomarkers to measure intestinal inflammation and CD severity, it is interesting to see that the microbial community structure is different among patients based on these markers [@boon_fmarkers_2015; @chang_monitoring_2015]. This supports the idea that the microbiome could be useful as a biomarker for measuring disease activity in patients, especially when considered in relation to these biomarkers [@tedjo_CDactivity_2016]. Higher CDAI was associated with lower microbial diversity. This appears to be consistent with other studies on the microbiome in individuals with CD compared to healthy individuals and studies looking at active disease compared to remission [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. However, these differences may have been driven by weekly stool frequency, one component of the CDAI, where higher stool frequency is also negatively associated with microbial diversity. Given that higher stool frequency is associated with looser stool consistency, this finding appears consistent with the association between loose stools and lower diversity [@vandeputte_stoolcon_2016]. 

We also observed differences in the microbiome in relation to other clinical variables. The microbial community structure was different based on disease localization. These results are consistent with a study by Naftali et al finding distinct microbiotas for ileal versus colonic CD using mucosal tissue [@naftali_tissinvol_2016]. This study also found that corticosteroid use impacts the composition of the human fecal microbiome. This supports data seen in the mouse model where corticosteroid injections altered the fecal mouse microbiome [@huang_cort_2015]. As corticosteroid use appears to impact diversity, corticosteroids may be useful when trying to positively impact the microbiome during biologic therapy and increase the possibility of response to CD therapies. 

Unlike other studies, these patients had a CD diagnosis for an average of 12 years (Supplemental Table 1) [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. We observed that that longer disease duration is associated with a reduction in fecal microbial diversity. This decreased diversity may be due to the long duration of inflammatory conditions in the gut. One could hypothesize earlier biologic intervention may ‘preserve’ microbiome that promotes remission and reduces the likelihood of relapse. Publications have come out in support of earlier biologic intervention, as it appears to increase the likelihood of inducing remission and mucosal healing [@monteleone_mongersen_2015; @monteleone_mongersen_2016; @ardizzone_mongersen_2016]. However, the cost of biologics for patients is hindrance to early biologic intervention. Using aptamers in place of monoclonal antibodies may reduce this cost and make earlier intervention possible. Aptamers are short strands of DNA or RNA capable of specifically binding small molecules, proteins, and whole cells. Anti-TNF aptamers have been published that could potentially be used to test this in the mouse model [@orava_short_2013].

One important question for the microbiome and IBD is whether or not the microbiome is affected by treatment with biologics. This study attempted to answer that question by looking at the microbiome of our CD subjects across multiple time points during treatment. While we were unable to see direct effects of the drug on the fecal microbiome, we observed that the microbiome of clinical responders changed over time, in contrast to nonresponsive subjects. This was observed for responsive patients regardless of induction treatment, leading us to think we are seeing the effects of change in disease activity and health rather than any effects from treatment. This interpretation is consistent with studies using the microbiome to distinguish between remission and active CD [@tedjo_CDactivity_2016]. We did however observe a significant difference in community structure based on treatment and cannot eliminate the possibility of a direct effect on the microbiome in treated responders. 

Another important question in for the importance of the microbiome in IBD is whether response to therapy can be predicted with the microbiome. We attempted to address this by developing a random-forest model that used relative microbial abundance data and/or clinical metadata for input. We found we were better able to predict remission status compared to response status. Response may be less predictable due to the "floating target" nature of a relative decrease in CDAI compared to the hard threshold for remission (CDAI<150). We were also better able to distinguish remission/non-remission than response/non-response, 6 weeks after beginning treatment. This is consistent with other studies again suggesting the microbiome could be useful in detecting remission versus active disease [@tedjo_CDactivity_2016].

While using the presented model may not be useful clinically to predict response to therapy at this time, it is useful for hypothesis generation about the biology of CD as it relates to the microbiome. Some of the frequently occurring factors in our predictive models have already been linked to CD pathogenesis. As far as clinical biomarkers, fecal lactoferrin and fecal calprotectin occurred in the majority of models where clinical metadata was combined with the microbiome, supporting their importance as biomarkers for CD activity, especially in relation tot eh fecal microbiome [@boon_fmarkers_2015; @chang_monitoring_2015]. Faecalibacterium was the most frequently occurring OTU in our models. It is associated with health and has been shown to be low in CD patients [@wright_CDmicrobiome_2015; @hansen_pedsIBD_2012; @naftali_tissinvol_2016; @sartor_microbesIBD_2016]. Remission was much more likely in individuals who had measurable Faecalibacterium present at baseline. This supports the hypothesis that Faecalibacterium impacts CD. Escherichia/Shigella also occurred frequently in our models. This OTU is associated with inflammation and has been shown to negatively impact CD [@sartor_microbesIBD_2016]. Fusobacterium also appeared in our predictive models and is associated with CD and CRC, something CD patients are more likely to get [@sartor_microbesIBD_2016]. These observations and the positive/negative associations of these microbes and CD allow us to hypothesize on ways to alter the microbiome to increase the likelihood therapeutic response. Prior to the initiation of therapy, patients could get a fecal microbiome analysis. The community data could then be used to direct the patient to undergo a round of antibiotics to target and reduce the levels of Escherichia in the patient’s gut. Alternatively, the microbes found to be positively associated with response could be formulated into a daily probiotic patients could take while receiving therapy with the goal of increasing the likelihood of remission and mucosal healing.

With this study we sought to gain a more detailed understanding of if and how biologic treatment affects the microbiome, to determine whether the microbiome can be used to identify patients who will respond to therapy, and to gain a better understanding of the interaction between the human gut microbiome and CD pathogenesis in adult patients. We found the fecal microbiome to be useful in uncovering associations between the microbiome and aspects of CD severity metrics and treatment outcomes. We also demonstrated that the microbiome of treated responders changed over time, though it is not yet possible to determine any direct effect of treatment on the microbiome. Finally, we were able to show that the microbiome could be useful in predicting response to therapy, especially clinical remission, compared to clinical metadata alone in our unique patient cohort. While this prediction is not clinically useful as of yet, altering the weighting or binning of important factors in the model could make prediction of response or remission more reliable. This could eventually allow for pre-screening of patients with stool samples to predict successful treatment or better direct treatment. If the fecal microbiome can be used as a theraprognostic tool to non-invasively predict response to specific treatment modalities or inform treatment, then more personalized treatment could result in faster achievement of remission, thereby increasing patients’ quality of life and reducing economic and healthcare impacts. 

\newpage

###Methods

####Study Design and Sample Collection

Janssen Research and Development conducted a phase II clinical study of approximately 500 patients to assess the safety and efficacy of UST for treating anti-TNF-${\alpha}$ refractory CD patients [@sandborn_ust_2012]. Participants provided a stool sample prior to the initiation of the study and were then divided into 4 groups of 125 individuals receiving placebo or 1, 3, or 6 mg/kg doses of UST by IV. Additional stool samples were provided at week 4. At week 6 an additional stool sample was collected, patients were scored for their response to UST based on CD Activity Index (CDAI), and divided into groups receiving either subcutaneous injection of UST or placebo at weeks 8 and 16 as maintenance therapy. Finally, at 22 weeks patients provided an additional stool sample and were then scored using CDAI for their response to therapy. Frozen fecal samples were shipped to the University of Michigan and stored at -80°C prior to DNA extraction

####DNA extraction and 16S rRNA gene sequencing 

Microbial genomic DNA was extracted using the PowerSoil-htp 96 Well Soil DNA Isolation Kit (MoBio Laboratories) using an EPMotion 5075 pipetting system, as previously described [@baxter_FIT_2016; @zackular_CRC_2014]. The V4 region of the 16S rRNA gene from each sample was amplified and sequenced using the Illumina MiSeq Personal Sequencing platform as described elsewhere [@chang_monitoring_2015]. Sequences were curated as described previously using the mothur software package [@schloss_PCRartifacts_2011]. Briefly, we reduced sequencing and PCR errors, aligned the resulting sequences to the SILVA 16S rRNA sequence database, and removed any chimeric sequences flagged by UCHIME [@edgar_uchime_2011]. After curation, we obtained between 1 and 130,074 sequences per sample (median 13786), with a median length of 253 bp. To limit effects of uneven sampling, we rarefied the dataset to 3,000 sequences per sample. Parallel sequencing of a mock community revealed an error rate of 0.017 %. Sequences were clustered into operational taxonomic units (OTU), as previously described [@schloss_OTUanalysis_2011]. Briefly, OTUs were clustered at a 97% similarity cutoff and the relative abundance was calculated for OTUs in each sample. All sequences were classified using a naive Bayesian classifier trained against the RDP training set (version 11) and OTUs were assigned a classification based on which taxonomy had the majority consensus of sequences within a given OTU [@wang_taxonomy_2007].  All fastq files and the MIMARKS spreadsheet with de-identified clinical metadata are available at TBD. 

####Gut microbiome biomarker discovery analysis 

Mothur as well as the R software package were used for our data analysis. Alpha diversity metrics (e.g. Shannon, Inverse Simpson) were calculated for each sample in the dataset, and compared using non-parametric statistical tests (i.e Kruskal-Wallace and Wilcox Test) [@sokal_biometrystats_1995; @magurran_measuring_2004]. Beta diversity was calculated the distance between samples using the theta YC metric, which takes into account the types of bacteria and their abundance to calculate the differences between the communities [@yue_thetaYC_2005]. These distance matrices were visualized by generating non-metric dimensional scaling (NMDS) plots of the distances. Overlap between sets of communities was assessed using the non-parametric analysis of molecular variance (AMOVA) and homogeneity of variance (HOMOVA) tests [@schloss_commstruct_2008; @oksanen_vegan_2016]. Differentially abundant OTUs were selected using the biomarker discovery algorithm, LEfSe [linear discriminant analysis (LDA) effect size] for each pairwise comparison of clinical groups [@segata_biomarker_2011]. In short, This method uses the Wilcox non-parametric test to identify OTUs where there is a P-value less than 0.05 and then applies a LDA step to identify the effect sizes that are the most meaningful (i.e. greater than 2.0). We also used the relative abundance of each OTU across the samples and clinical metadata as input to the AUC-Random forest package available to identify phylotypes/clinical variables that would allow us to distinguish between various treatment and response groups [@breiman_rf_2001]. 





\newpage

## Tables
**Supplemental Table 1: Summary of clinical metadata of chort at baseline**

```{r cohort.table, fig.height=5, fig.width=12, cache=F}

source('../code/SupTable1_baseline_metadata.R')
```
includegraphics{tables/SupTable1_baseline_metadata.pdf

\newpage

**Table 1: Diversity differences based on clinical metadata of chort at baseline**

```{r table1}
source("../code/table1.cohortdiv.R")
```
includegraphics{tables/table1_cohortdiversity.pdf

\newpage

**Table 2: Diversity differenced bases on Response/Remission in treated subjects.**

```{r table2diversity, fig.height=5, fig.width=10}

source('../code/table2diversity.R')
```

includegraphics{tables/table2diversity.pdf

\newpage


**ST2: adonis summary table showing difference by visit/TRTGR/wk22resp**
includegraphics{tables/SF2adonis.pdf


\newpage

## Figures

**Figure 1: Phyla by response/remisison week 6 and OTU abundance week 6 remission**
![Fig1](figures/ggplot.phylum.figure.byRelRSPwk6.pdf)
![Fig1](figures/ggplot.phylum.figure.byREMISSwk6.pdf)

\newpage
**SF 1: Phyla by response/remission week 22**
![SF1](figures/ggplot.phylum.figure.byRelRSPwk22.pdf)
![SF1](figures/ggplot.phylum.figure.byREMISSwk22.pdf)
\newpage

**Figure 2: Change in alpha diversity over time**
![Figure 2](figures/wk046.adivXvisitXindtrtXrelRSPwk22.pdf)



\newpage
**Figure 3: Determine week 6 status by week 6 stool and impt OTUs and abunds**
![Figure 3](figures/Figure3.pdf)

\newpage
**Figure 4: Prediction of RESPONSE/REMISSION in treated subjects using all clinical metadata, baseline microbiome alone, and combined**
A. Response ROCs B. Response Model Performance vs. reality C. Top predictive taxa and abundance based on response	D. REMISSION ROCs E. REMISSION Model Performance vs. reality F. Top predictive taxa and abundance based on remission



```{r RelRSPwk6Fig}
source('../code/Figure1prediction.R')
```

![Prediction of RESPONSE/REMISSION in treated subjects using all clinical metadata, baseline microbiome alone, and combined. A. Response ROCs. B. Response Model Performance vs. reality. C. Top predictive taxa and abundance based on response. D. REMISSION ROCs. E. REMISSION Model Performance vs. reality. F. Top predictive taxa and abundance based on remission](figures/Figure4.pdf)

\newpage

**Supplemental Figure 3: Predicting Week 22 disease status using Week 0 samples**

```{r, include=T}

source('../code/predictionWeek22figures.R')
```

![SF3](figures/SF3.pdf)

\newpage
# References
