---
title: "Untitled"
author: "Matt Doherty"
date: "3/24/2017"
output: html_document
---


\vspace{35mm}
	
Running title: The fecal microbiome as a tool for monitoring and predicting response outcomes in Ustekinumab-treated, anti-TNF-alpha refractory Crohn’s Disease patients.
	
\vspace{35mm}
Matthew K. Doherty${^2}$, Tao Ding${^2}$${^\alpha}$, Charlie Koumpouras${^2}$, Shannon Telesco${^1}$, Calixte Monast${^1}$, and Patrick D. Schloss${^2}$${^\dagger}$

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

1\. Janssen Pharmaceutical Companies of Johnson ${\&}$ Johnson, Spring House, PA, USA

2\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI, USA

${\alpha}$ Currently at *...*





\newpage








## Abstract


_Abstract:_ The 16S rRNA gene from patient stool samples was sequenced using the Illumina MiSeq platform. The resulting sequences were curated and assigned to taxonomic groups using the mothur software package to determine the bacterial communities and relative abundance of bacterial species present in these patients. The relative abundance among the fecal microbiota, patient demographic data, and clinical metadata were used as input to a random forest machine-learning algorithm to predict disease severity and response to treatment with UST.

Fecal microbial diversity at baseline significantly correlates with markers for disease severity, such as Crohn’s Disease Activity Index (CDAI), stool frequency, and disease duration. Additionally, the overall community structure of the microbiome was significantly different based on stool frequency, CRP, fecal lactoferrin, fecal calprotectin, corticosteroid use, disease duration, and tissue involvement. Baseline fecal microbiome community structures and species diversity were significantly different among responders and non-responders to UST treatment. Faecalibacterium, among other taxa, was significantly more abundant in responders/remitters. Additionally, the microbiome of clinical responders changed over time, in contrast to nonresponsive subjects. Using AUC-RF, differences in the baseline microbiome and clinical metadata were able to predict response to UST, especially remission, with some AUCs approaching 0.85.

_Importance:_ Crohn’s disease (CD) is a global health issue characterized by patches of ulceration and inflammation along the gastrointestinal tract. Individuals with CD have reduced microbial diversity in their guts, compared to healthy individuals. It remains unclear if this reduced diversity is a result or cause of pathogenesis. We investigated the relationship between the fecal microbiome and clinical phenotypes in subjects with moderate to severe CD treated with Ustekinumab (UST) in a Phase 2b study to determine whether the fecal microbiome at baseline is predictive of disease severity and therapeutic response, as well as if the fecal microbiota changes due to therapy.

The ability to predict and monitor response to treatment using the microbiome will likely provide another clinical tool in treating CD patients. Additionally, the observed baseline differences in fecal microbiota and changes due to therapeutic response will allow further investigation into the microbes important in CD pathogenesis as well as establishing and maintaining CD remission. Finally, beneficial microbes associated with response to treatment could be developed as probiotics to increase the likelihood of response while undergoing treatment.

**Keywords: Crohn's Disease, fecal microbiome, biologics, prediction**

\newpage

###Introduction
Crohn’s disease (CD), an incurable inflammatory bowel disease (IBD), is a global health issue with increasing incidence. CD affects approximately 3 million people worldwide, causing large economic and healthcare utilization impacts on society [@ananthakrishnan_epidemiology_2015; @floyd_economicburden_2015; @molodecky_increasingIBD_2012]. CD is characterized by patches of ulceration and inflammation affecting the entire bowel wall along the gastrointestinal tract, most commonly in the ileum and colon. Individuals with CD experience frequent diarrhea, abdominal pain, fatigue, and weight loss resulting in significant health care costs, lower quality of life, and economic impacts due to loss of productivity [@floyd_economicburden_2015; @mantzaris_clinical_2015; @randall_CDbiologics_2015]. Current treatments for CD include antibiotics, anti-inflammatory drugs, immunomodulators, surgery, and biologic agents targeting tumor necrosis factor alpha (TNF-${\alpha}$), such as Infliximab (Remicade). Within 10 years of diagnosis, approximately half of individuals with CD will require surgery and the majority will experience escalating immunosuppressive treatment [@boyapati_pathCD_2015]. Currently, individuals with CD are treated based on disease location and risk of complications using escalating immunosuppressive treatment and/or surgery with the goal of achieving and sustaining remission [@randall_CDbiologics_2015; @wils_ust_2015]. Faster induction of remission following diagnosis reduces the risk of irreversible intestinal damage and disability [@wils_ust_2015; @colombel_deepremission_2015; @baert_mucosalhealing_2010] (7-9). Anti-TNF-${\alpha}$ therapy in combination with thiopurines has emerged as the preferred treatment for CD, but up to half of individuals with CD fail to respond or lose response to anti-TNF-${\alpha}$ therapy [@boyapati_pathCD_2015; @wils_ust_2015]. Ustekinumab (UST), a monoclonal antibody directed against the shared p40 subunit of IL-12 and IL-23, has been proposed as an alternative therapy for these patients [@sandborn_ust_2012]. While clinical trials have demonstrated that UST is a viable option for the treatment of CD [@wils_ust_2015; @sandborn_ust_2012; @sandborn_ust_2008; @kopylov_ust_2014] (7, 10-12), some patients within these trials were non-responsive to UST, which may be explained by differences in the patients’ gut microbiomes.

The precise etiology of CD remains unknown, but host genetics, environmental exposure, and the gut microbiome appear involved [@ananthakrishnan_epidemiology_2015; @sartor_IBDpath_2006].  Genome-wide association studies of individuals with CD identified several susceptibility genes including NOD2, a receptor involved in bacterial killing and innate immunity. Defects in NOD2 function affects microbial sensing, the regulation of IL-23 driven Th17 responses, and indirect modulation of the gut microbiome [@randall_CDbiologics_2015; @wright_CDmicrobiome_2015]. The gut microbiome has also been shown to play a key role in inflammation, immunity, and IBD [@haag_CDpath_2015]. Individuals with CD have reduced microbial diversity in their guts, compared to healthy individuals, with a lower relative abundance of Firmicutes and an increased relative abundance of Enterobacteraciae and Bacteroides, at the phylum level [@wright_CDmicrobiome_2015; @manichanh_diversityCD_2006; @hansen_pedsIBD_2012; @haberman_pedsCD_2014; @gevers_pedsCD_2014] . Additionally, previous studies have shown that the gut microbiome can be predictive of disease severity in new-onset, pediatric CD patients [@gevers_pedsCD_2014; @wang_pedsCD_2016]. It remains to be determined, however, whether the microbiome can predict response to therapy in CD [@wright_CDmicrobiome_2015]. Additionally, the effect of biologic treatment on the gut microbiome is not well understood. If the fecal microbiome can be used as a theraprognostic tool to non-invasively determine and monitor disease severity as well as predict response to specific treatment modalities, then more targeted treatment could result in reduced adverse effects of less effective therapies and faster achievement of remission. 

Our lab was approached to analyze the gut microbiomes of individuals who participated in a Phase II clinical trial to determine the efficacy of UST in treating CD [@sandborn_ust_2012]. Using stool samples taken prior to the start of the study, 16S rRNA gene sequence data from these patients will allow us to determine associations between clinical metadata, disease severity, and the fecal microbiome and whether clinical responders have a microbiome that is distinct from non-responders at baseline. We demonstrate that the fecal microbiome is associated with baseline clinical metadata and that these associations and differences are useful in predicting disease severity and treatment outcome.



## Results

**Characteristics of Study Population**
Using 16s rRNA gene sequencing, we studied the fecal microbiota in a subset of TNF-${\alpha}$ refractory CD patients who took park in the CERTIFI clinical trial [@sandborn_ust_2012]. Demographic and baseline disease characteristics of this subset are summarized in Table 1. Patients with a history of moderate to severe CD were randomly assigned to a treatment group in the induction phase of the study (Figure 1A). At Week 8 patients were re-randomized into maintenance therapy groups. Both patients and clinicians were blinded to their induction and maintenance treatment groups. Subjects provided a stool sample and were evaluated for response to therapy at screening (week 0), week 4, week 6, and week 22 post induction (Figure 1B). Response was evaluated based on the change in CDAI. For this study, "response" was defined as a decrease in CDAI of 30% from baseline and "remission" was defined as a CDAI below 150. 

**Comparison of microbiome at screening based on clinical variables**
_Question: disease severity and microboime at screening?_

Following curation using the mothur software package, we obtained between 1 and 130,074 sequences per sample (median 13786) [@schloss_mothur_2009]. To limit effects of uneven sampling, we rarefied the dataset to 3,000 sequences per sample. Parallel sequencing of a mock community revealed an error rate of 0.017 %. Samples from subjects that completed the clinical trial and had complete clinical metadata were included in our analysis. Of these samples, 306 were provided prior to treatment as well as 258 provided at week 4, 289 at week 6, and 205 at week 22 post-treatment, for a total of 1058 samples. 

To determine whether there are any significant associations between microbial diversity and clinical variables of interest, we compared the microbiome with clinical data at week 0 (Supplemental Table 1). We observed a weak, but significant correlation between CDAI and species diversity, with higher CDAI correlating to lower diversity (rho=`r signif(cor.cdai$estimate, digits = 3)`, p-value= `r signif(cor.cdai$p.value, digits = 3)`). The overall community structure was not different based on CDAI. When looking at CDAI subscores, we observed a weak, but significant association between species diversity and the frequency of loose stools per week (rho=`r signif(cor.stool$estimate, digits = 3)`, p-value= `r signif(cor.stool$p.value, digits = 3)`). The overall community structure was also significantly different based on weekly loose stool frequency `r signif(as.numeric(na.omit(stool.adonis$aov.tab$`Pr(>F)`)), digits = 3)`. No significant association was observed between CRP, fecal calprotectin, or fecal lactoferrin and species diversity, following multiple comparison correction. However, the overall community structure was significantly different based on CRP `r signif(as.numeric(na.omit(BL_CRP.adonis$aov.tab$`Pr(>F)`)), digits = 3)l`, fecal calprotectin `r signif(as.numeric(na.omit(BL_FCALP.adonis$aov.tab$`Pr(>F)`)), digits = 3)`, and fecal lactoferrin `r signif(as.numeric(na.omit(BL_FLACT.adonis$aov.tab$`Pr(>F)`)), digits = 3)`. No significant differences in the microbiome were observed for BMI, weight, or sex. Overall community structure was different based on age `r signif(as.numeric(na.omit(AGE.adonis$aov.tab$`Pr(>F)`)), digits = 3)`. The overall community structure was also different based on the tissue affected `r signif(as.numeric(na.omit(tissinvol.adonis$aov.tab$`Pr(>F)`)), digits = 3)`. Species diversity (p-value= `r signif(wilcox.BCORT$p.value, digits = 3)`)and the overall community structure (adonis p-value=`r signif(as.numeric(na.omit(BCORT.adonis$aov.tab$`Pr(>F)`)), digits = 3)`) were significantly different based on corticosteroid use. The community structure was significantly different (adonis p-value=`r signif(as.numeric(na.omit(DDUR.adonis$aov.tab$`Pr(>F)`)), digits = 3)`) based on disease duration and a weak, but significant correlation was seen between species diversity and disease duration (rho=`r signif(cor.DDUR$estimate, digits = 3)`, p-value= `r signif(cor.D$p.value, digits = 3)`), with lower diversity corresponding to longer disease.

**Comparison of clinical responders and non-responders**
_Q: Are responder different from non responders at baseline independent of treatment group_
CHECK THIS ANALYSIS AGAIN...

Next, we hypothesized that there are associations between the microbiome at baseline and response to treatment. to test this hypothesis we compared the week 0 microbiomes of subjects who repsonded or remitted to subjects who did not at week 6 and week 22.

Baseline fecal microbiome community structures and species diversity were different based on response/remission. Based on response at the primary endpoint of the study, 6 weeks after IV induction, there was no difference in species diversity between response groups, but there was a significant difference in the overall community structure of the entire cohort.

Week 6 remitters were significantly different from non-remitters in both species diversity (`r all.wilcox.REMISSwk6.pval`) and overall community structure (`r all.REMISSwk6.adonis.pval`). 


When looking at treated vs. untreated Week 6 remitters, the treated group had significant differences in both species diversity and community structure while untreated remitters we not different from untreated non-remitters. 

At the secondary endpoint, 22 weeks after IV-induction and 14 weeks after maintenance dosing, there was no difference in species diversity between response groups, but there was a significant difference in the overall community structure of the entire cohort. Week 22 remitters were significantly different from non-remitters in both species diversity (`r all.wilcox.RelRSPwk22.pval`) and overall community structure (`r all.RelRSPwk22.adonis.pval`). 



**The microbiome by treatment and response over time**
_Does microbiome change following treatemnt_
The effects of biologic treatment of IBD on the microbiome are not yet well described. We tested whether treatment with UST affects the microbiome using subjects who provided samples at weeks 0, 4, and 6. This allowed for us to analyze `r n.treated.wk046` treated subjects and `r n.plac.wk046` placebo subjects at each time point. Using the adonis function in the vegan R package [@oksanen_vegan_2016], we performed a PERMANOVA stratified on each subject, as a proxy for a repeated measures ANOVA, to determine if the community structure of microbiome changed over time. We included induction treatment group, response at each clinical endpoint, and sample date as parameters. 

We found that treatment alone does not affect the microbiome over time. No significant difference was seen based on sample date when looking at all treatment groups and week 6 response status, but there was a significant interaction between week 22 response and sample date `r (p=0.001)`. There was also a significant interaction and between week 22 response, induction group, and sample date `r (p=0.0445)` (Supplemental Table 2). 

This led us to further examining the microbial community structures in week 22 responders and nonresponders across sample date. No significant difference was observed in Week 22 non-responders over time. In week 22 responders, we saw a significant change in community structure over time. 

we observed a significant difference based on visit in the treated-week 22 responders and in untreated responders across the first 3 visits prior to maintenance phase.

we found that treated-week 22 responders had significantly differnt community structures over time.

WTF this mean?
There was also a significant difference based on treatment group, but no significant interaction. 

When looking at treated vs. untreated responder groups, 

Digging deeper,  

We also hypothesized that treatment may affect species diversity. We tested this by performing a freidman test comparing species diversity at each sample date within each induction treatment group based on their week 22 response status. As seen in Figure 4, we saw no significant difference in species diversity over time in subjects induced with placebo or treated-week 22 nonresponders. However, in treated-week 22 responders species diversity increased significantly from week 0 to week 4 (`r pvalue`) and remained higher than baseline at week 6. We hypothesize that this reflects decreased inflammation in the responding subjects.  


**The microbiome following treatment reflects disease status**
_Does microbiome relfect disease status at wk 6_

A paper recently published by @tedjo_CDactivity_2016 demonstrated a link between the microbiome and disease severity, where specific microbes were associated with being in remission capared to active CD. We hypothesized that the microbiome could be used to monitor response to therapy in a similar manner. We used AUC-RF in order to determine if the fecal microbiome at Week 6 could be used to determine if a study participant responded to therapy or was in remission at Week 6. As seen in Figure 4, using the microbiome alone we achieved an AUC of `r otu_relrespWK6xWK6_roc$auc` for response with a sensitivity of `r otu_relrespWK6xWK6_sens` and a specificity of `r otu_relrespWK6xWK6_spec`. For remission we had an AUC of `r otu_remWK6xWK6_roc$auc` with a sensitivity of `r otu_remWK6xWK6_sens` and specificity of `r otu_remWK6xWK6_spec`. We were better able to distinguish remitters from non-remitters than responders from non-responders.

The top microbes that were indicitive of disease status in this model included blanks at enriched in remitters and blanks enriched in nonremitters.



**Prediction of response based on the microbiome at screening**
_can differences at baseline predict later response to treatment_
Given the observed differencces in the fecal microbiome at baseline and week 6 in responders/remitters compared to nonresponders/nonremitters, we hypothesized that the fecal microbiome could predict response to therapy. To test this hypothesis, we used AUCRF to develop a random forest classification model to differentiate responders from non-responders, as well as remitters from non-remitters, based on the relative abundance of fecal microbiome community members, clinical metadata, and the combination of microbiome and clinical data [@baxter_FIT_2016; @calle_aucrf_2011]. We ran these models for response and remission at Week 6 and 22 of the study. The optimal models for response and remission at the primary endpoint (Week 6) are shown in Figure 1. Using only clinical metadata to predict response, the model predicted response with an AUC of `r md_relrespWK6_roc$auc` with a specificity of `r md_relrespWK6_spec` and a sensitivity of `r md_relrespWK6_sens`. Using only microbiome data, the model predicted response with an AUC of `r otu_relrespWK6_roc$auc` with a specificity of `r otu_relrespWK6_spec` and a sensitivity of `r otu_relrespWK6_sens`. When combining clinical metadata with the microbiome, the model predicted response with an AUC of `r mdotu_relrespWK6_roc$auc` with a specificity of `r mdotu_relrespWK6_spec` and a sensitivity of `r mdotu_relrespWK6_sens`. With respect to Week 6 remission, using solely clinical metadata we achieved AUC of `r md_remWK6_roc$auc` with a specificity of `r md_remWK6_spec` and a sensitivity of `r md_remWK6_sens`. Using only fecal microbiome data we achieved an AUC of `r otu_remWK6_roc$auc` with a specificity of `r otu_remWK6_spec` and a sensitivity of `r otu_remWK6_sens`. When combining clinical metadata with the microbiome AUC of `r mdotu_remWK6_roc$auc` with a specificity of `r mdotu_remWK6_spec` and a sensitivity of `r mdotu_remWK6_sens`.

Across all weeks and responses, prediction with clinical metadata alone did not perform as well as models using the fecal microbiome at screening. Also, combining microbiome data with clinical metadata did not consistently improve prediction compared to microbiome data alone. Additionally we found several OTUs occurred frequently across models including Faecalibacterium, among other taxa that were significantly more abundant in responders/remitters. Their abundances can be seen in figure 4.

## Discussion

Our results examine the fecal microbiome of a subset of patients who participated in the CERTIFI trials to determine if the microbiome can predict response to therapy and if therapy has any effect on the microbiome. Several previous studies have looked at fecal and mucosal microbiomes in pediatric patients with new-onset and established disease and with established disease in adults [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. Unlike these studies, our patients were mostly Caucasian adults in their late thirties to early forties who failed to respond or lost response to anti-TNF-${\alpha}$ biologic treatment. We were able to find associations between the fecal microbiome of these patients and CDAI, stool frequency, fecal calprotectin, fecal lactoferrin, serum CRP, corticosteroid use, tissue involvement, and duration of disease. 

The association of the microbiome with clinically relevant biomarkers and disease activity metrics indicates that the microbiome may also function as a biomarker for CD activity. Given that serum CRP, calprotectin, and lactoferrin are used as biomarkers to measure intestinal inflammation and CD severity, it is interesting to see that the microbial community structure is different among patients based on these markers [@boon_fmarkers_2015; @chang_monitoring_2015]. This supports the idea that the microbiome could be useful as a biomarker for measuring disease activity in patients, especially when considered in relation to these biomarkers [@tedjo_CDactivity_2016]. Higher CDAI was associated with lower microbial diversity. This appears to be consistent with other studies on the microbiome in individuals with CD compared to healthy individuals and studies looking at active disease compared to remission [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. However, these differences may have been driven by weekly stool frequency, one component of the CDAI, where higher stool frequency is also negatively associated with microbial diversity. Given that higher stool frequency is associated with looser stool consistency, this finding appears consistent with the association between loose stools and lower diversity [@vandeputte_stoolcon_2016]. 

We also observed differences in the microbiome in relation to other clinical variables. The microbial community structure was different based on disease localization. These results are consistent with a study by Naftali et al finding distinct microbiotas for ileal versus colonic CD using mucosal tissue [@naftali_tissinvol_2016]. This study also found that corticosteroid use impacts the composition of the human fecal microbiome. This supports data seen in the mouse model where corticosteroid injections altered the fecal mouse microbiome [@huang_cort_2015]. As corticosteroid use appears to impact diversity, corticosteroids may be useful when trying to positively impact the microbiome during biologic therapy and increase the possibility of response to CD therapies. 

Unlike other studies, these patients had a CD diagnosis for an average of 12 years (Supplemental Table 1) [@gevers_pedsCD_2014; @papa_pedsIBD_2012; @tedjo_CDactivity_2016]. We observed that that longer disease duration is associated with a reduction in fecal microbial diversity. This decreased diversity may be due to the long duration of inflammatory conditions in the gut. One could hypothesize earlier biologic intervention may ‘preserve’ microbiome that promotes remission and reduces the likelihood of relapse. Publications have come out in support of earlier biologic intervention, as it appears to increase the likelihood of inducing remission and mucosal healing [@monteleone_mongersen_2015; @monteleone_mongersen_2016; @ardizzone_mongersen_2016]. However, the cost of biologics for patients is hindrance to early biologic intervention. Using aptamers in place of monoclonal antibodies may reduce this cost and make earlier intervention possible. Aptamers are short strands of DNA or RNA capable of specifically binding small molecules, proteins, and whole cells. Anti-TNF aptamers have been published that could potentially be used to test this in the mouse model [@orava_short_2013].

One important question for the microbiome and IBD is whether or not the microbiome is affected by treatment with biologics. This study attempted to answer that question by looking at the microbiome of our CD subjects across multiple time points during treatment. While we were unable to see direct effects of the drug on the fecal microbiome, we observed that the microbiome of clinical responders changed over time, in contrast to nonresponsive subjects. This was observed for responsive patients regardless of induction treatment, leading us to think we are seeing the effects of change in disease activity and health rather than any effects from treatment. This interpretation is consistent with studies using the microbiome to distinguish between remission and active CD [@tedjo_CDactivity_2016]. We did however observe a significant difference in community structure based on treatment and cannot eliminate the possibility of a direct effect on the microbiome in treated responders. 

Another important question in for the importance of the microbiome in IBD is whether response to therapy can be predicted with the microbiome. We attempted to address this by developing a random-forest model that used relative microbial abundance data and/or clinical metadata for input. We found we were better able to predict remission status compared to response status. Response may be less predictable due to the "floating target" nature of a relative decrease in CDAI compared to the hard threshold for remission (CDAI<150). We were also better able to distinguish remission/non-remission than response/non-response, 6 weeks after beginning treatment. This is consistent with other studies again suggesting the microbiome could be useful in detecting remission versus active disease [@tedjo_CDactivity_2016].

While using the presented model may not be useful clinically to predict response to therapy at this time, it is useful for hypothesis generation about the biology of CD as it relates to the microbiome. Some of the frequently occurring factors in our predictive models have already been linked to CD pathogenesis. As far as clinical biomarkers, fecal lactoferrin and fecal calprotectin occurred in the majority of models where clinical metadata was combined with the microbiome, supporting their importance as biomarkers for CD activity, especially in relation tot eh fecal microbiome [@boon_fmarkers_2015; @chang_monitoring_2015]. Faecalibacterium was the most frequently occurring OTU in our models. It is associated with health and has been shown to be low in CD patients [@wright_CDmicrobiome_2015; @hansen_pedsIBD_2012; @naftali_tissinvol_2016; @sartor_microbesIBD_2016]. Remission was much more likely in individuals who had measurable Faecalibacterium present at baseline. This supports the hypothesis that Faecalibacterium impacts CD. Escherichia/Shigella also occurred frequently in our models. This OTU is associated with inflammation and has been shown to negatively impact CD [@sartor_microbesIBD_2016]. Fusobacterium also appeared in our predictive models and is associated with CD and CRC, something CD patients are more likely to get [@sartor_microbesIBD_2016]. These observations and the positive/negative associations of these microbes and CD allow us to hypothesize on ways to alter the microbiome to increase the likelihood therapeutic response. Prior to the initiation of therapy, patients could get a fecal microbiome analysis. The community data could then be used to direct the patient to undergo a round of antibiotics to target and reduce the levels of Escherichia in the patient’s gut. Alternatively, the microbes found to be positively associated with response could be formulated into a daily probiotic patients could take while receiving therapy with the goal of increasing the likelihood of remission and mucosal healing.

With this study we sought to gain a more detailed understanding of if and how biologic treatment affects the microbiome, to determine whether the microbiome can be used to identify patients who will respond to therapy, and to gain a better understanding of the interaction between the human gut microbiome and CD pathogenesis in adult patients. We found the fecal microbiome to be useful in uncovering associations between the microbiome and aspects of CD severity metrics and treatment outcomes. We also demonstrated that the microbiome of treated responders changed over time, though it is not yet possible to determine any direct effect of treatment on the microbiome. Finally, we were able to show that the microbiome could be useful in predicting response to therapy, especially clinical remission, compared to clinical metadata alone in our unique patient cohort. While this prediction is not clinically useful as of yet, altering the weighting or binning of important factors in the model could make prediction of response or remission more reliable. This could eventually allow for pre-screening of patients with stool samples to predict successful treatment or better direct treatment. If the fecal microbiome can be used as a theraprognostic tool to non-invasively predict response to specific treatment modalities or inform treatment, then more personalized treatment could result in faster achievement of remission, thereby increasing patients’ quality of life and reducing economic and healthcare impacts. 

\newpage

###Methods

####Study Design and Sample Collection

Janssen Research and Development conducted a phase II clinical study of approximately 500 patients to assess the safety and efficacy of UST for treating anti-TNF-${\alpha}$ refractory CD patients [@sandborn_ust_2012]. Participants provided a stool sample prior to the initiation of the study and were then divided into 4 groups of 125 individuals receiving placebo or 1, 3, or 6 mg/kg doses of UST by IV. Additional stool samples were provided at week 4. At week 6 an additional stool sample was collected, patients were scored for their response to UST based on CD Activity Index (CDAI), and divided into groups receiving either subcutaneous injection of UST or placebo at weeks 8 and 16 as maintenance therapy. Finally, at 22 weeks patients provided an additional stool sample and were then scored using CDAI for their response to therapy. Frozen fecal samples were shipped to the University of Michigan and stored at -80°C prior to DNA extraction

####DNA extraction and 16S rRNA gene sequencing 

Microbial genomic DNA was extracted using the PowerSoil-htp 96 Well Soil DNA Isolation Kit (MoBio Laboratories) using an EPMotion 5075 pipetting system, as previously described [@baxter_FIT_2016; @zackular_CRC_2014]. The V4 region of the 16S rRNA gene from each sample was amplified and sequenced using the Illumina MiSeq Personal Sequencing platform as described elsewhere [@chang_monitoring_2015]. Sequences were curated as described previously using the mothur software package [@schloss_PCRartifacts_2011]. Briefly, we reduced sequencing and PCR errors, aligned the resulting sequences to the SILVA 16S rRNA sequence database, and removed any chimeric sequences flagged by UCHIME [@edgar_uchime_2011]. Sequences were clustered into operational taxonomic units (OTU), as previously described [@schloss_OTUanalysis_2011]. Briefly, OTUs were clustered at a 97% similarity cutoff and the relative abundance was calculated for OTUs in each sample. All sequences were classified using a naive Bayesian classifier trained against the RDP training set (version 11) and OTUs were assigned a classification based on which taxonomy had the majority consensus of sequences within a given OTU [@wang_taxonomy_2007].  All fastq files and the MIMARKS spreadsheet with de-identified clinical metadata are available at TBD. 

####Gut microbiome biomarker discovery analysis 

Mothur as well as the R software package were used for our data analysis. Alpha diversity metrics (e.g. Shannon, Inverse Simpson) were calculated for each sample in the dataset, and compared using non-parametric statistical tests (i.e Kruskal-Wallace and Wilcox Test) [@sokal_biometrystats_1995; @magurran_measuring_2004]. Beta diversity was calculated the distance between samples using the theta YC metric, which takes into account the types of bacteria and their abundance to calculate the differences between the communities [@yue_thetaYC_2005]. These distance matrices were visualized by generating non-metric dimensional scaling (NMDS) plots of the distances. Overlap between sets of communities was assessed using the non-parametric analysis of molecular variance (AMOVA) and homogeneity of variance (HOMOVA) tests [@schloss_commstruct_2008; @oksanen_vegan_2016]. Differentially abundant OTUs were selected using the biomarker discovery algorithm, LEfSe [linear discriminant analysis (LDA) effect size] for each pairwise comparison of clinical groups [@segata_biomarker_2011]. In short, This method uses the Wilcox non-parametric test to identify OTUs where there is a P-value less than 0.05 and then applies a LDA step to identify the effect sizes that are the most meaningful (i.e. greater than 2.0). We also used the relative abundance of each OTU across the samples and clinical metadata as input to the AUC-Random forest package available to identify phylotypes/clinical variables that would allow us to distinguish between various treatment and response groups [@breiman_rf_2001]. 

####Statistical analysis


